window.ACTIVITY_DATA={template:"Genes",Genes:{title:"Genes Guide",search:"Search This Guide",home:"app/homebtn.svg",searchicon:"app/searchbtn.svg",help:"app/helpicon.svg",popupSearchImg:"app/popup-search.svg",download:"app/downloadbtn.svg",headerimg:"homepage.jpg",close:"app/closebtn.svg",collapse:"app/collapse-bt.svg",hhmi:"hhmi-logo.png",youtube:"youtube.png",facebook:"facebook.png",twitter:"twitter.png",helpPopupContent:"<div class='pop-items'> <div class='popup-box'> <div class='popup-box-heading'> <img class='home-icon' src='./assets/images/app/homebtn.svg' alt='home button' /> <span class='text'> Home   </div> <div class='popup-box-content'> <p class='popup-box-content-spacing'>On clicking on home button user will go to the landing page. User can access this button from every screen of this   guideThis butto is placed at top most left side of the header. On clicking on home button user will go to the landing page. User can access this butoon from every screen of this guideThis butto is placed at top most left side of the header. </p> </div> </div> <div class='popup-box'> <div class='popup-box-heading'> <img class='home-icon' src='./assets/images/app/collapse-bt.svg' alt='home button' /> <span class='text'> Collapse   </div> <div class='popup-box-content'> <p class='popup-box-content-spacing'>On clicking on collapse button our left side navigation will shift left and go out from the screen, and at the same time right side content will will shihft left to fill the white gap. </p> </div> </div> <div class='popup-box'> <div class='popup-box-heading'> <img class='home-icon' src='./assets/images/app/helpicon.svg' alt='Help' /> <span class='text'> Help   </div> <div class='popup-box-content'> <p class='popup-box-content-spacing'> On Clicking on the help button, a pop up nwindow will open and give you a brief about buttons and on screens. This button is placed at top right side of the header. On Clicking on the help button, a pop up window will open and give you a brief about buttons and their functions.  </p> </div> </div> <div class='popup-box'> <div class='popup-box-heading'> <img class='home-icon' src='./assets/images/app/downloadbtn.svg' alt='download' /> <span class='text'> Download   </div> <div class='popup-box-content'> <p class='popup-box-content-spacing'> On clicking on download button user will be abble to download the whole onlinne guide. This button is placed at top most right side of the header. </p> </div> </div> <div class='popup-box'> <div class='popup-box-heading'> <img class=' search-bg' src='./assets/images/app/popup-search.svg' alt='search' /> <span class='text'> Search   </div> <div class='popup-box-content'> <p class='popup-box-content-spacing'>  Clicking on search bar user can search any keyword and search any chapter or related content. This button is placed at centre the header. </p> </div> </div> </div></div>",footer:[{id:"1",contenttype:"icon",content:"facebook.png",link:"https://www.facebook.com/HowardHughesMed"},{id:"2",contenttype:"icon",content:"youtube.png",link:"https://www.youtube.com/channel/UCUWJVAgUcpvWwKJXba5XoxQ"},{id:"3",contenttype:"icon",content:"twitter.png",link:"https://twitter.com/hhminews"},{id:"4",contenttype:"popup",title:"Contact Us",content:"Contact Us",popupContent:"<div class='contact-content'> <div>Contact Information:</div> <div> Email: <a href=''>info@seaphages.org</a> </div> Website: <a href=''>seaphages.org</a> </div> "},{id:"5",contenttype:"popup",title:"Copyright",content:"Copyright",popupContent:"<div class = 'copyright-content'> <p>    Marianne Poxleitner1, Welkin Pope2, Deborah Jacobs-Sera2, Viknesh    Sivanathan3, Graham Hatfull2 Gonzaga University1, University of Pittsburgh2, Howard Hughes Medical Institute3 The Science Education Alliance-Phage Hunters Advancing Genomics and Evolutionary Science Supported by the Howard Hughes Medical Institute <br /> <br /> <br /> Contact Information: Email:<a tabIndex='1' href = 'mailto:info@seaphages.org'>info@seaphages.org</a> | Website:<a tabIndex='1' href = 'https://seaphages.org/' target  =  '_blank'>seaphages.org</a> <br /> <br /> <br /> With contributions from: <br /> <br /> Ilzat Ali, Aleem Mohamed, Emily Davis, and Ariel Egbunine, University of Maryland Baltimore County Priscilla Kobi<sup>3</sup>, William Biederman<sup>3</sup>, Danielle Heller<sup>3</sup> <br /> <br />Kevin Bradley, Infectious Disease Programs at the Association of Public Health Laboratories<br /><br />Steven Cresawn, James Madison University<br /><br />Dan Russell, University of Pittsburgh <br /><br />    Nicholas M. Jaramillo, Graphic Artist: Figures 3.0-2, 3.0-3,    3.0-4, 3.0-5,8.0-2 <br /><br /><br /><br />    <br />    Some text and figures adapted from the SEA-PHAGES Resource Guide ©    2012, Howard Hughes Medical Institute, and the NGRI In Silico    Phage Resource Guide,<i> Analyze © 2008</i>, Howard Hughes Medical    Institute, including Figures 13.0-4, 13.0-5, and 13.0-8 by Lucia    Barker  </p><p>© 2018, Howard Hughes Medical Institute. All rights reserved.</br> 4000 Jones Bridge Road </br> Chevy Chase, Maryland 20815</p> <p>Permission to use, copy, and distribute this manual or excerpts from this manual is granted provided that (1) the copyright notice above appears in all reproductions; (2) use is for noncommercial educational purposes only; (3) the manual or excerpts are not modified in any way. Requests beyond that scope should be directed to <a tabIndex='1' href = 'mailto:info@seaphages.org'>info@seaphages.org</a>.</p> <p>Use of the guide can be acknowledged as: <i>Phage Discovery Guide. Howard Hughes Medical Institute.</i></p> </div>"},{id:"6",contenttype:"image",content:"hhmi-logo.png",link:"https://www.hhmi.org/"}],navigation:[{id:"1",pageId:"1",templateId:"toc",name:"Chapter 1: Welcome",imgtext:"Welcome",img:"welcome.svg",child:!0,childElements:[{id:"1",pageId:"1-1",templateId:"chapterContent",name:"Chapter 1: SEA-GENES Overview"},{id:"2",pageId:"1-2",templateId:"chapterContent",name:"Chapter 1: Phage Infection Cycle"},{id:"3",pageId:"1-3",templateId:"chapterContent",name:"Chapter 1: Lab Safety"}]},{id:"2",pageId:"2",templateId:"toc",name:"Chapter 2: Molecular Cloning",img:"molecular-cloning.svg",imgtext:"Molecular Cloning",child:!0,childElements:[{id:"1",pageId:"2-1",templateId:"chapterContent",name:"Chapter 2: An Overview"},{id:"2",pageId:"2-2",templateId:"chapterContent",name:"Protocol 2.1a: Cloning into pExTra"},{id:"3",pageId:"2-3",templateId:"chapterContent",name:"Protocol 2.1b: Cloning into p2Halpha"},{id:"4",pageId:"2-4",templateId:"chapterContent",name:"Protocol 2.2: Polymerase Chain Reaction"},{id:"5",pageId:"2-5",templateId:"chapterContent",name:"Protocol 2.3: Gel Electrophoresis"},{id:"6",pageId:"2-6",templateId:"chapterContent",name:"Protocol 2.4: Column Purification of DNA"},{id:"7",pageId:"2-7",templateId:"chapterContent",name:"Protocol 2.5: Gel Extraction of DNA"},{id:"8",pageId:"2-8",templateId:"chapterContent",name:"Protocol 2.6: Restriction Enzyme Digests"},{id:"9",pageId:"2-9",templateId:"chapterContent",name:"Protocol 2.7: Isothermal Assembly"},{id:"10",pageId:"2-10",templateId:"chapterContent",name:"Protocol 2.8: DNA Ligation"},{id:"11",pageId:"2-11",templateId:"chapterContent",name:"Protocol 2.9: Chemical Transformation of Bacteria"},{id:"12",pageId:"2-12",templateId:"chapterContent",name:"Protocol 2.10: Clone Verification"},{id:"13",pageId:"2-13",templateId:"chapterContent",name:"Protocol 2.11: Inoculating a Culture"},{id:"14",pageId:"2-14",templateId:"chapterContent",name:"Protocol 2.12: Preparing Plasmid DNA"}]},{id:"3",pageId:"3",templateId:"toc",name:"Chapter 3: Phenotypic Assay",img:"phenotypic-assay.svg",imgtext:"Phenotypic Assay",child:!0,childElements:[{id:"1",pageId:"3-1",templateId:"chapterContent",name:"Chapter 3: An Overview"},{id:"2",pageId:"3-2",templateId:"chapterContent",name:"Protocol 3.1: Transformation of M. smegmatis by Electroporation"},{id:"3",pageId:"3-3",templateId:"chapterContent",name:"Protocol 3.2: Cytotoxicity Assay"},{id:"4",pageId:"3-4",templateId:"chapterContent",name:"Protocol 3.3: Defense Assay"}]},{id:"4",pageId:"4",templateId:"toc",name:"Chapter 4: Interaction Assay",img:"interaction-assay.svg",imgtext:"Interaction Assay",child:!0,childElements:[{id:"1",templateId:"chapterContent",pageId:"4-1",name:"Chapter  4: An Overview"},{id:"2",templateId:"chapterContent",pageId:"4-2",name:"Protocol 4.1: 2-Hybrid Selection Assay"},{id:"3",templateId:"chapterContent",pageId:"4-3",name:"Protocol 4.2: Selecting Candidate 2-Hybrid Hits"},{id:"4",templateId:"chapterContent",pageId:"4-4",name:"Protocol 4.3: Evaluating Candidate 2-Hybrid Hits"}]},{id:"14",templateId:"recipe-toc",pageId:"14",name:"Recipe Cards",img:"recipe-cards.svg",imgtext:"Recipe Cards",child:!1,appendix:!0,appendixElements:[{id:"1",templateId:"toc",pageId:"14-1",name:"Recipes for Solutions",img:"recipe-cards.svg",imgtext:"Recipe Cards",child:!0,childElements:[{id:"1",templateId:"chapterContent",pageId:"14-2",name:"AD Supplement"},{id:"2",templateId:"chapterContent",pageId:"14-3",name:"Calcium Chloride (1 M)"},{id:"3",templateId:"chapterContent",pageId:"14-4",name:"Calcium Nitrate (0.8 M )"},{id:"4",templateId:"chapterContent",pageId:"14-5",name:"Carbenicllin Stock Solution"},{id:"5",templateId:"chapterContent",pageId:"14-6",name:"Cycloheximide Stock Solution"},{id:"6",templateId:"chapterContent",pageId:"14-7",name:"Dextrose (40 %)"},{id:"7",templateId:"chapterContent",pageId:"14-8",name:"Enrichment Broth"},{id:"8",templateId:"chapterContent",pageId:"14-9",name:"EDTA"},{id:"9",templateId:"chapterContent",pageId:"14-10",name:"Glycerol (40 %)"},{id:"10",templateId:"chapterContent",pageId:"14-11",name:"Isopropanol (80 %)"},{id:"11",templateId:"chapterContent",pageId:"14-12",name:"Magnesium Chloride (1 M)"},{id:"12",templateId:"chapterContent",pageId:"14-13",name:"Magnesium Sulphate (1 M)"},{id:"13",templateId:"chapterContent",pageId:"14-14",name:"Nuclease Mix"},{id:"14",templateId:"chapterContent",pageId:"14-15",name:"Phage Buffer"},{id:"15",templateId:"chapterContent",pageId:"14-16",name:"Phage Precipitation Solution"},{id:"16",templateId:"chapterContent",pageId:"14-17",name:"SDS (10 %)"},{id:"17",templateId:"chapterContent",pageId:"14-18",name:"Tris Borate EDTA (TBE)"},{id:"18",templateId:"chapterContent",pageId:"14-19",name:"Tris, pH 7.5 (1 M)"},{id:"19",templateId:"chapterContent",pageId:"14-20",name:"Tween80 (20 %)"},{id:"20",templateId:"chapterContent",pageId:"14-21",name:"Uranyl Acetate Stain"},{id:"21",templateId:"chapterContent",pageId:"14-22",name:"Water"},{id:"22",templateId:"chapterContent",pageId:"14-23",name:"Zinc Chloride"}]},{id:"2",templateId:"toc",pageId:"14-24",name:"Recipes for Media",img:"recipe-cards.svg",imgtext:"Recipe Cards",child:!0,childElements:[{id:"1",templateId:"chapterContent",pageId:"14-25",name:"7H9 Liquid Medium(Neat)"},{id:"2",templateId:"chapterContent",pageId:"14-26",name:"7H9 Liquid Medium(Complete)"},{id:"3",templateId:"chapterContent",pageId:"14-27",name:"7H9 Liquid Medium(Complete with Tween80)"},{id:"4",templateId:"chapterContent",pageId:"14-28",name:"7H9 Top Agar (1X)"},{id:"5",templateId:"chapterContent",pageId:"14-29",name:"7H9 Top Agar (2X)"},{id:"6",templateId:"chapterContent",pageId:"14-30",name:"Luria Agar (L-Agar) Plates"},{id:"7",templateId:"chapterContent",pageId:"14-31",name:"Media Supplement for Streptomyces phage"},{id:"8",templateId:"chapterContent",pageId:"14-32",name:"Nutrient Broth"},{id:"9",templateId:"chapterContent",pageId:"14-33",name:"Nutrient Broth(with Supplements for S.griseus)"},{id:"10",templateId:"chapterContent",pageId:"14-34",name:"Nutrient Agar Plates(with Supplements for S.griseus)"},{id:"11",templateId:"chapterContent",pageId:"14-35",name:"PYCa Agar Plates"},{id:"12",templateId:"chapterContent",pageId:"14-36",name:"PYCa Liquid Medium"},{id:"13",templateId:"chapterContent",pageId:"14-37",name:"PYCa Top Agar"}]},{id:"3",templateId:"toc",pageId:"14-38",name:"Propagating Bacteria and Phage",img:"recipe-cards.svg",imgtext:"Recipe Cards",child:!0,childElements:[{id:"1",templateId:"chapterContent",pageId:"14-39",name:"Growing bacteria from a frozen stock"},{id:"2",templateId:"chapterContent",pageId:"14-40",name:"Growing liquid cultures of bacteria from a single colony"},{id:"3",templateId:"chapterContent",pageId:"14-41",name:"Preparing a culture without Tween80 from one with Tween80"},{id:"4",templateId:"chapterContent",pageId:"14-42",name:"Preparing frozen stocks of bacteria"},{id:"5",templateId:"chapterContent",pageId:"14-43",name:"Preparing Additional Phage Lysate & Long-term Storage"},{id:"6",templateId:"chapterContent",pageId:"14-44",name:"Received Biologicals from the Program? Follow these Next Steps."}]},{id:"4",templateId:"toc",pageId:"14-45",name:"Notes",img:"recipe-cards.svg",imgtext:"Recipe Cards",child:!0,childElements:[{id:"1",templateId:"chapterContent",pageId:"14-46",name:"Notes about Recipes"},{id:"2",templateId:"chapterContent",pageId:"14-47",name:"Autoclave Setting"}]}]}],chData:{1:{content:""},2:{content:""},3:{content:""},4:{content:""},14:{content:""},"1-1":{content:"<p > Welcome to the Science Education Alliance (SEA), a network of undergraduate researchers and instructors who collaborate on the PHAGES and GENES research projects. In the PHAGES project, researchers isolate novel bacteriophages from the environment by employing various techniques of microbiology, then fully annotate the bacteriophage genomes using an array of bioinformatic tools. Here, in the GENES project, researchers build on the discoveries of the PHAGES project, utilizing techniques of molecular biology and genetics to investigate the functions of genes encoded by these bacteriophages. </p> <h3 > Where are we headed? </h3> <p > The number of isolated phages with annotated genomes has increased by the thousands in recent years; this is made possible in part by undergraduate students who have contributed to the research enterprise through the SEA-PHAGES project. Their contributions have not only led to advances in our understanding of how genes are organized within phage genomes and distributed across phage populations, but has also revealed the incredible diversity of genes present within the phage population. When these genes are grouped into families based on the sequence similarities of the proteins they encode, to date, over twenty thousand unique families can be identified! Strikingly, the large majority of these families do not resemble any previously characterized proteins. Indeed, it is this remarkable and unexplored diversity that has earned bacteriophages the reputation as the “dark matter” of the biological world. These thousands of families of unknown functions thus represent a treasure trove from which to uncover novel biological functions. It is the aim of the SEA-GENES research project to experimentally explore this vast unknown. </p> <h3 > What might we find? </h3> <p > To consider this, it is important to revisit the details of a phage infection cycle, described in <a class='chap-linking'  tabindex='0' data-link-to='1-2' > Phage Infection Cycle Overview </a>, taking into account the myriad interactions that occur between the phage and its bacterial host. Phages have been co-evolving with the diversity of bacteria across the planet for eons, where each phage infection is a battle between the phage and its bacterial host for vital resources. Engaged in this high stakes arms race, both players have defense and counter-defense strategies that are continually evolving. The pursuit of gene-function elucidation therefore promises to uncover an arsenal of functions that reflect this ancient and on-going competition and shed new light on the intricacies of phage-host interactions. Such discoveries also hold the potential to greatly impact the fields of medicine and biotechnology. </p> <p > In medicine, particularly in an era of rampant antibiotic resistance, the phage gene pool is an invaluable resource for discovering novel strategies for controlling bacteria. Phage lysins, for example, are being developed as a strategy to treat bacterial infections, and recent studies of phage-host interactions are uncovering novel bacterial targets for the development of antimicrobials. In molecular biology, phage recombinases, as well as bacterial defense systems that target phage DNA, have enabled researchers to manipulate DNA with precision and make genetics modifications of cells in all domains of life. </p> <h3 > How do we elucidate gene-function? </h3> <p >‘Gene function’ refers to the specific role that a gene plays in an organism, which is determined by the structure of the final product it encodes, whether that be tRNA or, as in most cases, protein. This function can be complex, and is defined by all the work that a gene product performs. For example, a gene product may interact with an enzyme to catalyze a reaction that results in the production of a small molecule. A clear understanding of the role that a specific gene product performs cannot be gained from any one experiment, but instead requires the integration of multiple experimental approaches.</p> <p > There are several approaches that are routinely used to study the function of a gene product. These include bioinformatic inferences, phenotypic assays, and protein-protein interaction assays. Each of these approaches, described below, provides clues and preliminary insight that, when taken together, can be used to build a case for assigning function to a particular gene. Bioinformatic inferences have already been used, as part of the SEA-PHAGES project, to uncover phage gene function. Here, as part of the SEA-GENES project, phenotypic and protein-protein interaction assays will be used to gain experimental insights into phage gene function. </p> <ul role = 'list'> <li > <strong> Bioinformatic Inference.</strong> Gene-function studies often begin with bioinformatic inferences, where computer algorithms can be used to search for similarities between a gene of interest and other genes stored in databases. When strong similarities are identified to a gene with a known function, a gene of unknown function can be preliminarily assigned the same function. In the SEA-PHAGES project, phage genes were assigned functions based on similarity in sequence to genes and proteins of known function, using search tools such as BLAST, as well as on the order of genes within genomes, using tools such as Phamerator. The latter, called synteny, is based on a characteristic of phage and bacterial genomes in which genes that function in the same biological process often cluster together and maintain their relative positions on the genome. For a review of assigning functions using bioinformatic approaches, see the <a href = 'https://seaphagesbioinformatics.helpdocsonline.com/home' target = '_blank'> SEA-PHAGES Bioinformatics Guide </a>. </li> </ul> <ul role = 'list'> <li > <strong> Phenotypic Assays. </strong> The total and coordinated activity of all the genes expressed in an organism determines the observable traits and behavior, or phenotype<b>,</b> of that organism. As gene expression changes, so can its phenotype. The connection between gene expression and phenotypes is not only a fundamental tenet of biology; it is also central in the quest to study gene function. Phenotypic assays are a common strategy that involves altering gene expression in some way, either by turning it off completely, turning it on when it would not normally be expressed, or increasing or decreasing the amount of expression, and observing for a change in phenotype. An observable phenotypic change in response to the altered gene expression can provide important clues as to the role that gene plays. For example, expression of a phage gene in a bacterium may result in inhibition of bacterial growth, which suggests its involvement in a process essential for bacterial growth. This approach will be employed as part of the SEA-GENES project, and is described further in Chapter 3.</li> </ul> <ul role = 'list'> <li > <strong> Protein-Protein Interaction (PPI) Assays.</strong> PPI assays are employed to identify proteins that interact with one another. As proteins often do not function in isolation, but instead typically interact with other proteins as part of biological processes, PPI assays can be used to infer the function of one protein if the function of the interacting partner identified is known. For example, identifying a transcription factor as one interacting partner suggests that other protein may also be involved in transcription. This approach will be employed as part of the SEA-GENES project, and is described further in Chapter 4.</li> </ul> <h3 > SEA-GENES Research Workflow </h3><table> <thead><tr><th>Sequence</th><th colspan='2' rowspan='1'>Description of Research</tr></thead><tbody>  <tr> <td>1</td> <td><img src='./assets/images/welcome.svg' alt='First step of SEA-GENES Research Workflow' /></td> <td> Molecular cloning techniques will be used to engineer a system that enables<br> each gene of a mycobacteriophage to be expressed, individually, in <em>M.<br> smegmatis</em> </td> </tr> <tr> <td>2</td> <td><img src='./assets/images/molecular-cloning.svg' alt='Second step of SEA-GENES Research Workflow'/></td> <td> Each gene of a mycobacteriophage will be expressed, individually, in <em>M. <br>smegmatis</em> to identify those that are able to modify the behavior, or phenotype, of <br>the bacterium. </td> </tr> <tr> <td>3</td> <td><img src='./assets/images/phenotypic-assay.svg' alt='Third step of SEA-GENES Research Workflow'/></td> <td> Molecular cloning techniques will be used to engineer a second system that<br> enables specific mycobacteriophage genes identified by phenotypic assays to be <br> expressed, individually, in <em>E. coli.</em> </td> </tr> <tr> <td>4</td> <td><img src='./assets/images/interaction-assay.svg' alt='Fourth step of SEA-GENES Research Workflow' /></td> <td> Mycobacteriophage genes will be expressed in an <em>E. coli </em> protein-protein <br> interaction assay to identify potential interacting partners from the <em>M.<br> smegmatis </em>proteome. </td> </tr> </tbody> </table>"},"1-2":{content:"<p >A phage infection cycle can be broken down into 6 stages. A clear understanding of the strategies employed by phages to complete each infection stage and to counter bacterial defense systems is key to the study of phage-gene function. It provides a framework for anticipating functions that are likely to be encoded in the phage genome. For example, understanding that phages must degrade the bacterial cell wall at the end of the infection cycle then suggests that its genome is likely to encode for enzymes to accomplish that function. Described below are the stages of an infection cycle, many with examples of defense and counter defense strategies employed by bacteria and phages.</p> <h3>1. Adsorption</h3> <p >The first step of phage infection is attachment of a phage particle to the surface of its bacterial host, a process called adsorption. Adsorption is accomplished by phage-encoded <b>R</b>eceptor <b>B</b>inding <b>P</b>roteins (RBPs). On tailed phages, the RBPs are typically a component of the phage tail. These RBPs recognize specific structures of the host cell surface, enabling phages to not only attach to its hosts, but to also distinguish its hosts from other bacteria. </p> <p >However, accessing these host cell surface receptors is not always trivial. For instance, the bacterial cell surface may be decorated with polysaccharides that can occlude the cell surface receptor. In fact, this appears to be a strategy employed by bacteria to evade phage infection. In turn, phages can encode polysaccharide degrading enzymes, often also as part of the phage tail, to unmask its cell surface receptor.</p> <h3 >2. Genome Delivery</h3> <p >Successful adsorption then sets in motion the delivery of phage DNA from the capsid into the host cell. To achieve this, phages must clear a path and assemble a channel across the cell envelope. Phages encode enzymes such as peptidases and glycosidases to breakdown the rigid peptidoglycan cell wall, and specialized proteins to form a channel for DNA delivery. For myoviruses with contractile tails, this channel is formed by a preassembled tail-tube that emerges upon contraction of the tail. For podoviruses with short non-contractile tails, specialized tail proteins likely assemble <i>de novo </i>to form a channel across the cell envelope.</p>  <p >Once delivered into the host cell, phage DNA must evade degradation by host nucleases. For example, commonly encoded bacterial exonucleases target the double-stranded ends of linear phage genomes. To protect against these exonucleases, phages encode recombinases that join the two ends of its genome to form a circular DNA molecule. Phages that do not circularize their genomes encode proteins that either bind to and protect the ends of its linear DNA, or bind to the exonucleases and prevent their recognition of its DNA ends. Bacteria also encode at least two defense systems that utilize endonucleases to target phage DNA - the restriction-modification (RM) system and the CRISPR system. In both of these systems, bacterial endonucleases surveil the cell for specific “target” DNA sequences that, upon encountering, it cleaves to expose double-stranded DNA ends that can then be degraded by exonucleases. In the RM system, the endonucleases are called restriction enzymes. Because the host DNA may also contain these target sequences, the host encodes modification enzymes, such as methyl-transferases, to modify and thereby mask target sequences that may occur on its own DNA. Phages can similarly evade restriction enzymes by encoding their own modification enzymes, as well as proteins that stimulate the activity of host modification enzymes, to rapidly modify and thereby protect phage DNA. Phages can also encode proteins that, when co-delivered into the host cell with its DNA, either bind to and protect target sequences that occur on its genome, or act like decoys by binding to and sequestering the host restriction enzymes. In the CRISPR system, the endonucleases are called <b>C</b>RISPR-<b>as</b>sociated (Cas) proteins. Cas proteins target DNA sequences that are specified by a region of host DNA called CRISPR. Phages can inactivate CRISPR-Cas systems by encoding <b>a</b>nti-<b>CR</b>ISPR (<b>acr</b>) proteins that inhibit the ability of Cas proteins to target or cleave DNA, or their own CRISPR-Cas system that targets and cleaves host CRISPR DNA.</p> <h3>3. Macromolecular Synthesis</h3> <p >Immediately upon delivery of phage DNA into the cell, and while evading nucleases, phage gene expression and genome replication begins. </p> <p><strong>3a. Gene Expression.</strong>Transcription of phage genes is often initiated by host RNA polymerase and completed by either host or phage-encoded RNA polymerase, while translation is primarily executed by host translation machinery. For a productive infection, phage gene expression must be tightly regulated, and phages encode a myriad of proteins to regulate gene expression.</p> <p >At the level of transcription, phages encode transcription regulators that, by enabling, promoting, or repressing transcription from given promoters, can not only modulate the levels of expression, but can also exert temporal control over gene expression. These transcription regulators can also completely inhibit the activity of host RNA polymerase, a strategy employed by phages to prevent transcription of host genes, thereby maximizing host recourses for transcription of phage genes.  </p><p> Phage-encoded proteins are also able to modulate gene expression at the level of translation. For example, phages can encode mRNA-binding proteins to stabilize transcripts or occlude ribosome binding, or ribonucleases that target transcripts for degradation. Phages can also encode their own tRNAs, or proteins that modify tRNAs, which alter the cellular pool of these molecules and may facilitate translation of phage genes. </p>  <p  > In defense, host-encoded proteins can also inhibit phage gene expression, with phages often equipped to counter these bacterial defense strategies. For example, when host gene expression is inhibited by an infecting phage, the relative amounts of a host-encoded transcript-cleaving toxin, which is more stable than the co-expressed host-encoded anti-toxin, increases. These “free” toxins can then promote the degradation of transcripts to shutdown phage gene expression. To counter such <b>T</b>oxin-<b>A</b>nti-toxin (TA) systems, phages can encode their own anti-toxins, which can bind to and neutralize the host toxins. As another means of defense, bacterial proteins can cleave tRNAs to inhibit translation. In turn, phages can restore translation by encoding polynucleotide kinase, phosphatase, and RNA ligase to repair cleaved tRNA. </p>  <p  > <strong>3b. Genome Replication.</strong> Replication of phage genomes is often performed by both host and phage-encoded replication proteins such as DNA polymerase, helicase, and primase. As with transcription, phages can also encode proteins that specifically inhibit replication of the host genome as means to maximize reserves for replication of its own genome. Indeed, phages can also encode nucleases that degrade the host genome as well as enzymes that catalyze nucleotide formation to increase the pool of nucleotides available for phage genome replication. Phages that degrade host genomes also encode proteins that modify and protects its own DNA from degradation. </p> <p  > <strong>3c. Lysogeny.</strong><b> </b>At an early stage of macromolecular synthesis, temperate phages have the unique ability to pause their infection cycle and adopt a different pattern of gene expression and genome replication. This state, called lysogeny, is achieved when the phage-encoded immunity repressor is expressed and able to continually inhibit transcription from promoters that are critical for a productive infection. The decision to enter lysogeny, and indeed the duration of lysogeny, is determine by host and phage-encoded proteins that modulate the expression or activity of the immunity repressor in response to environmental stimuli. These include host-encoded proteins that sense DNA damage, and phage-encoded receptors that can detect the amount of host bacteria or active infections in the immediate environment. </p> <p  > During lysogeny, phages must also ensure that their genomes are faithfully replicated and stably segregated as the host cells continue to grow and divide. To achieve this, phages typically insert their genomes into the host chromosome. This integration is performed by phage-encoded recombinases, called integrases. When phages resume a productive infection, the same integrases acts in concert with phage-encoded excisionases to excise phage genomes from the host chromosome. Non-integrating phages, on the other hand, maintain their genomes extra-chromosomally, like a very large plasmid, during lysogeny. These phages encode partitioning proteins that help ensure their genomes, once replicated, are properly partitioned as the host cell divides. These partitioning proteins include centromere proteins and actin- or tubulin-like polymerizing proteins that bind to and actively segregate phage DNA, respectively. </p><h3> 4. Host Immunity  </h3> <p  > During the course on an infection, phages may promote survival of the infected host. For example, immunity repressor proteins expressed during lysogeny are able to recognize and similarly repress gene expression of identical or similar superinfecting phages. Indeed, phages can employ a variety of strategies to immunize its host from superinfection. For example, phage-encoded proteins can prevent superinfection by identical or similar phages by occluding cell surface receptors or by inhibiting cell wall degrading enzymes required for invasion. Phages can also provide immunity against more distantly related phages, for example, by encoding restriction enzymes that target successfully delivered phage DNA for degradation. </p> <h3 > 5. Assembly  </h3> <p  > Towards the end of a productive infection cycle, the components of new phage particles begin to accumulate in the host. In stages, these components begin assembling into new phage particles; scaffolding proteins and chaperones guide assembly of capsid proteins into capsids, and the numerous tail proteins into tail tubes and tail fibers, while packaging proteins such as terminase and portal proteins package phage genomes into capsids. Finally, connector and adapter proteins bridge these preassembled parts to form mature phage particles. </p>  <h3> 6. Release  </h3> <p  > As new phage particles accumulate in the host, so do phage-encoded proteins that are important for the final act of an infection cycle – phage release from the host. Accumulating holin proteins associate with the host cell membrane to form holes, through which lysins, with various hydrolase activities, can access and degrade the cell wall. This combined activity results in cell lysis and the release of phage particles that can begin new infection cycles.  </p>"},"1-3":{content:"<p>This document offers an introduction to lab safety, and is a good starting point for a conversation about working responsibly in the lab.While every laboratory has its own specific rules and regulations, there are some universal guidelines that you should always follow. They are designed to keep you and your lab mates safe and to comply with state and federal regulations. Please read these rules and guidelines carefully and talk to your instructor about any additional rules that must be adhered to in your workspace.</p> <h3>General Lab Safety Practices</h3> <ul role = 'list'> <li>No eating or drinking.</li> <li>No open-toe shoes.</li> <li>Never work alone in a lab.</li> <li>Tie back long hair.</li> <li>Avoid wearing baggy clothing.</li> <li>Never leave a lit Bunsen burner unattended.</li> <li>Keep workspace clear of clutter.</li> <li>Minimize loud talking and distractions.</li> <li>Know the location of fire extinguishers, emergency eye wash stations, and emergency showers.</li> <li>Properly dispose of waste generated from an experiment. Not everything can go down the drain or in a garbage can.</li> <li>No mouth pipetting!</li> <li>Wear safety glasses at all times.</li> <li>Wear gloves when working with bacteria.</li> <li>Wash your hands before leaving lab.</li> <li>If you are not sure, ask first!</li> </ul> <h3>Guidelines for Working with Bacteria</h3> <p>The following are recommendations that should be used to supplement general lab practices and established safety rules at your institution when working with bacteria.</p> <p><ol role = 'list'> <h4>Before working with bacteria:</h4></p> <li>Ensure that all participants are aware of lab practices and safety rules, which should include the recommendations listed in this document.</li> <li>Make sure that all participants are aware that: <ul role = 'list' style='list-style-type:disc'> <li>The bacteria recommended for use in this course are safe for those with a normal immune system. If an individual has any health concerns related to exposure to bacteria, including being immuno-compromised or pregnant, the individual should consult a physician to determine the appropriate level of participation in lab activities.</li> <li>The primary hazards of working with the bacteria used in this course are from exposure via mucosal membrane routes (eyes, mouth, and nose), parenteral inoculation (broken skin), or ingestion.</li> </ul> </li> <p><h4>While working with bacteria:</h4></p> <li>Gloves and lab coats should be used. Change gloves when contaminated or when integrity is compromised. Do not touch face, apply cosmetics, adjust contact lenses, bite nails, etc.</li> <li>Care should be taken to avoid generating aerosols of cultures or suspensions of bacteria; do not sonicate or vortex bacterial cultures or suspensions; transfer bacterial cultures and suspensions with care and deliberation. Safety glasses and surgical masks, or a biosafety cabinet, are recommended if there is risk of generating aerosols.</li> <li>Label cultures, tubes and plates containing bacteria appropriately.</li> <li>Lab work surfaces should be wiped-down with an appropriate phenolic disinfectant before and after use. An example of an appropriate disinfectant for Mycobacteria is CiDecon Detergent Disinfectant Concentrate, available from multiple vendors including Fisher Scientific and VWR. Disinfectants should be used according to the manufacturer instructions. Once cleaned, the work surface should then be wiped-down with 70 % ethanol to remove residual disinfectant from the surface.<br> Note: As a disinfectant, 70 % EtOH is sufficient to kill most bacterial contaminants on surfaces. However, it is not effective on Mycobacteria because of the mycolic acid wall. Use of a disinfectant like CiDecon will remove mycobacterial contaminants.</li> <li>Materials that have come in contact with bacteria and that are intended for reuse (e.g. glass flasks) should be treated with CiDecon before washing. Spent CiDecon can be disposed of in sinks with tap-water running.</li> <li>Materials for disposal that have come in contact with bacteria (e.g. microcentrifuge tubes or agar plates) should be disposed of in biohazard bins.</li> <li>Sticks or loops used for streaking or inoculating bacteria should not be flamed after use. Instead, sticks should be disposed of in a biohazard bin, while metal loops should be treated with CiDecon followed by 70 % ethanol.</li> <p><h4>After working with bacteria:</h4></p> <li>Wash hands thoroughly.</li> <li>Dispose of all unused bacterial cultures appropriately by mixing the contents with CiDecon, bleach, or autoclaving.</li> <li>Materials for disposal that have come in contact with bacteria (e.g. microcentrifuge tubes or agar plates) should be disposed of in biohazard bins.</li> <li>Waste collected in biohazard bins should be inactivated using an autoclave, or removed by a licensed waste removal service provider.</li> <li>Materials that have come in contact with bacteria and that are intended for reuse (e.g. glass flasks) should be treated with bleach or CiDecon before washing. Spent bleach and CiDecon can be disposed of in sinks with tap water running.</li> </ol> <h3>Guidelines for Avoiding Contamination</h3> <p>Aseptic technique is used to prevent contamination in microbiology experiments. The growth of contaminating bacteria, fungi, and viruses can be avoided by disinfecting your bench, creating an updraft with a Bunsen burner, and moving with care and deliberation. When working with DNA and molecular biology techniques, it is essential to disinfect your bench and to ALWAYS wear gloves and a lab coat in order to reduce the chances of introducing contaminating DNA or degrading nucleases into your sample.</p> <p>The information provided in this document is adapted from the SEA-PHAGES Phage Discovery Guide and Phage Discovery Instructors Guide © 2018, Howard Hughes Medical Institute.</p>"},"2-1":{content:"<h3>Plasmids and Gene Expression in Bacteria</h3> <p>A typical bacterial genome is 5 million base pairs (5 Mbp) in length, though genomes as small as 0.1 Mbp and as large as 14 Mbp have been reported. In most bacteria, these millions of DNA base pairs are arranged as a single double-stranded, circular chromosome that is organized into genes encoding for several thousand different proteins. The <i>Mycobacterium smegmatis</i> genome contains ~ 7 Mbp, organized into 6,717 genes. As bacteria grow and divide, the chromosome is copied by the replication machinery and partitioned into the two resulting daughter cells, ensuring stable maintenance of all chromosomal genes.</p> <p>Not all of the genes in a bacterial cell are necessarily found on this chromosome; bacteria can carry additional genes on DNA molecules known as plasmids, which<b> </b>like the chromosome, are also replicated and partitioned as bacteria grow and divide. Plasmids are small, circular molecules of dsDNA that are typically less than 10 kbp and contain just a handful of genes. Owing to its small size, plasmids are easily engineered and manipulated in the lab and can be made to contain almost any genetic sequence imaginable. The process of building a plasmid is known as molecular cloning and is described in the next section. For SEA-GENES, two plasmids have been engineered to facilitate the study of gene function, pExTra and p2H&alpha;, which you will manipulate to contain your phage gene of interest (Figure 2.01).</p> <p> All plasmids, including those that exist in nature or that have been created in a lab, contain two key features. The first, the origin of replication, contains all the necessary elements that enable replication of the plasmid, including the DNA sequence where replication is initiated by the bacterial machinery. The origin of replication for a plasmid thus determines the bacterial hosts in which the plasmid can be replicated and maintained. The pExTra plasmid has two origins of replication, enabling it to be replicated in <i>Escherichia coli </i>and <i>M. smegmatis, </i>whereas the p2H&alpha; plasmid can only be replicated in <i>E. coli.</i> The second feature common to all plasmids is a gene that confers some type of advantage to the plasmid-containing cell, which in turn helps to ensure that the plasmid is maintained within a bacterial population. Oftentimes this advantageous gene encodes for antibiotic resistance, allowing the plasmid-containing cell to survive in the presence of a given antibiotic. In the lab, this feature can be used to identify and isolate, or select, bacteria that contain the specific plasmid from cells that do not. The pExTra and p2H&alpha; plasmids confer resistance to the antibiotics Kanamycin and Spectinomycin, respectively.</p> <div class='img-wrap'> <div class='img-container'> <img src='./assets/images/c2-plasmid-maps.png' alt='This image contains two colored circles; one labeled p-extra and one labeled p-two-h-alpha. The p-extra circle has 6 colored and labeled regions in the following order. p-tet, m-cherry reporter, kanamycin resistance gene, e-coli origin of replication, tet repressor, and m-smegmatis origin of replication. Phage gene insertion site is marked with an arrow pointing to a spot on the circle between p-tet and m-cherry reporter. The p-two-h-alpha circle has 4 colored and labeled regions in the following order. p-lac-asterix, alpha domain, e-coli origin of replication, and spectinomycin resistance gene. Phage gene insertion site is marked with an arrow pointing to a spot on the circle immediately after alpha domain.' /> </div>  <div class='img-hr'></div><div class='title-container'>   Figure 2.01. pExTra and p2H&alpha; plasmid maps </div> </div> <p> In addition to these two features, the pExTra and p2H&alpha; plasmids have been engineered to each contain a promoter, which is a DNA sequence that is recognized and bound by RNA polymerase. Once bound at the promoter, RNA polymerase can initiate transcription of the gene(s) immediately downstream of the promoter. In SEA-GENES research, and as part of Chapter 2, you will manipulate pExTra and p2H&alpha; plasmids to contain your phage gene of interest immediately downstream of the promoter. By doing so, you will create a means to express your phage gene of interest in bacteria to begin exploring phage gene function as part of the research outlined in chapters 3 and 4. </p> <h4>Molecular Cloning Techniques for the Construction of Plasmids</h4> <p> The pExTra and p2H&alpha; plasmids will be modified using a process known as molecular cloning, which is a set of experimental methods for the assembly, isolation, and propagation of recombinant DNA plasmids. 'Recombinant' simply means that these DNA molecules are composed of a mixture of sequences from different sources and would not naturally occur. For example, the recombinant pExTra plasmids we will build will contain sequences originally from mycobacteriophage, bacteria, naturally occurring plasmids, and sea anemones! Since the advent of molecular cloning in the 1970s, researchers have developed a number of cloning technologies and workflows that enable efficient engineering of DNA molecules such as plasmids. The workflows that we will use to clone phage genes into pExTra and p2H&alpha; plasmids are described in detail in Protocol 2.1a and Protocol 2.1b, and are composed of the following 4 core steps. </p> <p> <strong>Step 1.</strong> Copying and isolating a particular DNA fragment referred to as an 'insert'. </p> <p> <strong>Step 2.</strong> Assembling the insert with a relevant plasmid, which in the context of cloning is referred to as a 'plasmid backbone.' </p> <p> <strong>Step 3.</strong> Introducing the new recombinant plasmid into bacteria by a process known as 'transformation' and selecting bacteria carrying the plasmid. </p> <p> <strong>Step 4.</strong> Purifying the recombinant plasmid and verifying that it contains the correct insert. </p> <p> <strong> Step 1. Copying and isolating a particular DNA fragment referred to as an 'insert'. </strong> <br /> Polymerase Chain Reaction, or PCR, is a powerful technique that allows a specific fragment of DNA to be copied exponentially, <i>in vitro</i>, generating large quantities of DNA from very little starting DNA (Figure 2.02). </p> <div class='img-wrap'> <div class='img-container'> <img src='./assets/images/figure-2-02-a-phage-gene-of-interest.png' alt='The image is of a multi-faceted geometric shape with a black line emerging from one point. The black line has a sequence of 5 thick colored arrows pointing in the same direction. In the middle of the thick blue arrow is a small black arrow pointing down to a grouping of 18 similar thick, blue arrows.' /> </div>  <div class='img-hr'></div><div class='title-container'>   Figure 2.02. A phage gene of interest is copied, or 'amplified', by PCR. </div> </div> <p>PCR requires several key components:</p> <table> <thead><tr><th>Component</th><th>Description of Component</th></tr></thead><tbody> <tr> <td>Template DNA</td> <td> DNA sample that contains the sequence to be copied. This can be a purified DNA <br> sample such as extracted genomic DNA or plasmid DNA, or it can be a more complex <br> DNA-containing sample such as a phage lysate or bacterial colony. </td> </tr> <tr> <td>DNA Polymerase</td> <td> An enzyme that synthesizes new DNA molecules. Polymerases from thermophilic <br> organisms are used due to their high heat tolerance. </td> </tr> <tr> <td>Nucleotides</td> <td> The building blocks used by DNA polymerase to make new DNA molecules. </td> </tr> <tr> <td>Primers</td> <td> Short pieces of single stranded DNA (15-60 nucleotides) that are complementary to a <br> portion of the template DNA sequence, marking the beginning and end of the sequence <br> to be copied. PCR primers are custom-built in a laboratory and can be engineered to <br> add specific sequences to the ends of a DNA fragment. </td> </tr> </tbody> </table> <br> <p> The reaction consists of three steps (denaturation, annealing, and extension), each occurring at a specific temperature. Reaction components are mixed together and placed in a thermocycler machine, which cycles through these three temperatures, usually 20-40 times. During each cycle of denaturation, annealing, and extension, the specific segment of DNA is amplified exponentially. Exact temperatures and times for these steps are influenced by primer sequence, DNA length, and the polymerase used.<br /> </p> <table> <thead><tr><th>PCR Step</th><th>Temp.</th><th>Duration</th><th>Description of step</th></tr></thead><tbody> <tr> <td>Denaturation</td> <td>94 - 98 °C</td> <td>5 -10 sec</td> <td> High temperature denatures the two strands of the <br> template DNA. </td> </tr> <tr> <td>Annealing</td> <td>50 -72 °C</td> <td>5 sec</td> <td> The temperature is lowered, allowing the primers to bind <br> to the complementary region of the template DNA strand. </td> </tr> <tr> <td>Extension</td> <td>68 - 72 °C</td> <td>10 - 30 sec per kb of amplified segment</td> <td> DNA polymerase adds nucleotides to the end of each <br> annealed primer (in the 5' to 3' direction) to build a new <br> double stranded copy of the segment. </td> </tr> </tbody> </table> <br> <p> The result of a PCR reaction can be assessed by gel electrophoresis, a technique in which DNA is passed through an agarose gel by an electric current. Negatively charged DNA molecules migrate through the sieve-like matrix according to their size, allowing for separation of DNA molecules based on their size. When visualized using a DNA dye such as ethidium bromide, DNA molecules will appear as bands whose size can be determined by comparison to a standard of known sizes, or 'ladder'. Usually, the desired result of a PCR reaction is a single band of the correct size; however, in some cases a PCR may yield multiple bands or no bands, requiring further optimization. It is necessary to purify the desired PCR product away from the other components of the reaction as they may interfere with downstream assembly steps. This can be done by washing the entire reaction over a commercially available silica column or first extracting a specific band of DNA from an agarose gel and washing that over a column. </p> <p> Once an insert has been generated by PCR and purified, it can be assembled with a plasmid backbone using a variety of methods, each of which requires modification of the ends of the inserts. In SEA-GENES we will utilize two assembly strategies, which are described below. </p> <p> <h4> Step 2. Assembling the insert with a relevant plasmid, which in the context of cloning is referred to as a 'plasmid backbone.' </h4> There are two strategies that will be employed for the assembly reaction, either restriction enzyme digest and ligation, or isothermal assembly. </p> <p> <strong>Restriction enzyme digestion (RED) and ligation.</strong> Assembling an insert and plasmid backbone by RED and ligation can essentially be thought of as a 'cut and paste' reaction that takes two linear DNA molecules and joins them to form a single circular molecule. The 'cut' is performed by restriction endonuclease enzymes, which recognize and cleave double-stranded DNA at specific 4–6 bp palindromic sequences called restriction sites. Some restriction enzymes cleave both strands at the same site, generating 'blunt ends', whereas other enzymes cleave the strands at different locations a few bp away from each other, generating single-stranded 'sticky ends.' These sticky ends allow for efficient binding of two pieces of DNA with complementary overhangs. To create compatible ends, the insert and plasmid should be digested with the same restriction enzyme and thus, must have the same restriction sites. If not already present, primers can be designed to introduce desired restriction sites to the ends of an insert sequence (flanking the gene) by PCR. Similarly, plasmids can be modified to contain necessary restriction sites, and often have several that can be used for cloning in a region known as the 'multiple cloning site.' Cleavage of a circular plasmid molecule by one or more restriction enzymes creates a linear plasmid backbone with appropriate ends for assembly. Digestion of the insert and plasmid with two different restriction enzymes (known as a double digest) ensures proper directional assembly. Some restriction sites occur naturally within gene sequences or plasmid sequences; when designing a cloning strategy, it is essential that restriction sites be unique within the insert and plasmid such that the right number of fragments of the correct size are generated upon digestion. </p> <p> The 'paste' step is achieved by the enzyme DNA ligase, which covalently connects the sugar phosphate backbones of two DNA molecules with complementary sticky ends generated by restriction enzyme digest. Interestingly, the ligase enzyme used for this reaction is from the phage T4! Together, these two enzymatic steps produce a recombinant plasmid that is ready for transformation into bacteria. </p> <div class='img-wrap'> <div class='img-container'> <img src='./assets/images/figure-2-03-Insert-dna-plasmid.png' alt='This image includes the grouping of 18 thick, blue arrows followed by an addition symbol followed by a color-coded circle with an antibiotic resistance gene, an origin of replication and a promter with a gap at the top of the circle. An arrow points from this circle to another with the same labels and a thick blue arrow that was part of the grouping at the beginning filling the gap in the circle. ' /> </div>  <div class='img-hr'></div><div class='title-container'>   Figure 2.03. 'Insert' DNA and prepared plasmid 'backbone' DNA are assembled into a new plasmid. </div> </div> <p> <strong>Isothermal Assembly.</strong> Similar to RED and ligation, isothermal assembly 'cuts and pastes' together multiple linear DNA fragments. This method is an attractive alternative to RED and ligation cloning as it involves fewer individual steps and does not require DNA segments to have specific, unique restriction sites. However, it does require that the DNA segments have identical, overlapping sequences known as 'homology regions' at both ends; these homology regions should be at least 20 bp in length. As in the above strategy, primers can be engineered to introduce these homology regions into the insert or plasmid backbone by PCR, providing the necessary sequence match. Circular plasmids must be linearized for use in isothermal assembly; this is usually achieved by amplifying the plasmid by PCR, creating a linear plasmid backbone with the homology regions situated at the 5&prime; and 3&prime; ends. </p> <p> In an isothermal assembly reaction, three enzymatic steps occur in a single tube during a single incubation step at 50 ºC: </p> <ol role = 'list'> <li> An exonuclease chews back double stranded DNA at each 5' end, leaving 3' overhangs or 'sticky ends'. The exonuclease is quickly inactivated at 50 ºC. This ensures that only the DNA ends are digested, not the entire strand. Since the two DNA segments have overlapping sequences at their ends, the sticky ends that are generated can base pair to each other, allowing the two different DNA segments to anneal to each other. This base pairing is very efficient due to the extended length of the sticky ends—much more efficient than the base pairing of sticky ends generated by restriction enzyme digest, which are typically only a few bp long. </li> <li>DNA Polymerase fills in the gaps on the annealed segments, and</li> <li> DNA ligase covalently links the sugar phosphate backbones, creating a complete DNA molecule ready for bacterial transformation. </li> </ol> <h4> Step 3. Introducing the new recombinant plasmid into bacteria by a process known as 'transformation' and selecting bacteria carrying the plasmid. </h4> <p> Transformation is the process of introducing a plasmid into bacterial cells. Cells that are capable of taking up a piece of foreign DNA are said to be 'competent.' Transformation is an essential step in molecular cloning of plasmids, as the bacterial cells are used to recover the cloned plasmid of interest from a ligation or isothermal assembly reaction. Transformed cells are also used to store and replicate the plasmid. Although many species of bacteria are or can be engineered to be competent, <i>E. coli</i> is the true workhorse of molecular cloning. <i>E. coli</i> grows quickly (optimally, doubling every~ 20 min) and is relatively easy to work with. As such, molecular biologists have modified the <i>E. coli </i>genome to facilitate its ability to take up and stably store recombinant plasmid DNA. For instance, many cloning strains have a mutation in the <i>endA</i> gene, disrupting the function of the encoded endonuclease and protecting plasmid molecules from degradation. </p> <p>To transform bacteria, plasmid DNA is incubated with an ice-cold mixture of competent cells and glycerol. The plasmid molecules can be introduced into the cells by one of two methods: heat shock or electroporation. In the heat shock method, competent cells are treated with divalent cations (such as CaCl<sub>2</sub>) and briefly incubated at a high temperature (42 ºC); this creates pores in the membrane that the plasmid can travel through. Similarly, in the electroporation method, membrane pores are created by pulsing cells with an electric current. Although transformation is an efficient method, not all cells will take up the DNA. In both methods, transformed bacteria that have taken up the plasmid of interest must be 'selected' for based on the antibiotic resistance conferred by the plasmid. This is achieved by plating the transformation mixture on solid medium containing a selective concentration of antibiotic; only those cells that have successfully taken up and replicated the recombinant plasmid are able to grow.</p><div class='img-wrap'> <div class='img-container'> <img src='./assets/images/figure-2-04-during-a-transformation-reaction.png' alt='A color-coded circle with labels antibiotic resistance gene, origin of replication, promter and a thick, blue arrow is followed by an addition symbol followed by a grouping of 8 oblong shapes followed by an arrow pointing to another grouping of 8 oblong shapes, three of which contain the color-coded circles at the beginning.' /> </div>  <div class='img-hr'></div><div class='title-container'>   Figure 2.04. During a transformation reaction, a fraction of bacteria will take up the DNA. These bacteria are said to be 'transformed'. </div> </div> <h4> Step 4. Purifying the recombinant plasmid and verifying that it contains the correct insert. </h4> <p> Once the transformed <i>E. coli</i> cells harboring the plasmid clone have formed colonies on an antibiotic selection plate, it is necessary to purify the recombinant plasmid and verify that it was constructed correctly before it is used in any downstream experiments. To purify the plasmid, cells are grown to high density in selective liquid medium then lysed by the addition of SDS and NaOH, which denatures all DNA (chromosomal and plasmid) in the sample. This process, known as alkaline lysis, allows for separation of plasmid DNA from chromosomal DNA; upon neutralization of the alkaline solution, plasmid molecules are re-natured, whereas strands of chromosomal DNA are unable to re-anneal and become an insoluble aggregate that precipitates out of solution. Plasmid DNA can then be bound to a silica column, washed to remove salts, proteins and other contaminants, and eluted with sterile, nuclease-free water (ddH<sub>2</sub>O). </p> <div class='img-wrap'> <div class='img-container'> <img src='./assets/images/figure-2-05-bacteria-containing.png' alt='An arrow is pointing from eleven oblong shapes containing circles with green and blue markers to a four by four grid of the circles with the blue and green markers.' /> </div>  <div class='img-hr'></div><div class='title-container'>   Figure 2.05. Bacteria containing a plasmid can be grown to high denstities, from which large quantities of the plasmid can be extracted for use in downsteam applications. </div> </div> <p> There are multiple methods that can be used to confirm that a recombinant plasmid contains the expected insert. The exact sequence of nucleotides in the insert-containing region of the plasmid can be precisely determined by a DNA sequencing technique known as Sanger sequencing. Alternatively, PCR and gel electrophoresis can be used diagnostically to determine whether a plasmid contains an insert of the expected size. One benefit of the latter method is that the plasmid does not need to be purified from cells before running the diagnostic; a transformed colony can be used directly as PCR template. In a typical cloning workflow multiple candidate clones are tested for a correct insert, and verifying a plasmid clone prior to going through the purification process can conserve time and reagents. </p> <p> Once a plasmid has been purified and verified, it is ready for use in the experimental procedures outlined in Chapters 3 or 4! </p>"},"2-2":{content:"  <h3>Objective:</h3> <p>To construct a DNA molecule that will enable expression of a phage gene in <i>M. smegmatis.</i> </p> <h4>Rationale:</h4> <p> To express a phage gene in <i>M. smegmatis</i>, the gene must first be introduced into the bacterium, where it must then be stably maintained. These requirements can be achieved with the use of an expression plasmid, such as pExTra. Because pExTra contains a promoter (P<sub>tet</sub>) that is activated in the presence of an 'inducer' molecule, expression of the phage gene can be turned on as needed. Here is presented a multi-step workflow for the cloning of a phage gene into pExTra. Briefly, a specific phage gene sequence will be copied from a phage lysate by PCR using a gene-specific primer that is complementary to the first ~20 bp of the gene (the forward primer) and one that is complementary to the last ~20 bp of the gene (the reverse primer). These primer sequences also introduce essential homology regions at the ends of the amplified gene sequence; these regions are a perfect sequence match to plasmid sequences upstream and downstream of the phage gene insertion site, enabling assembly of this insert and plasmid backbone by the isothermal assembly method. After isothermal assembly, recombinant plasmids are transformed into <i>E. coli</i> and selected clones are verified by diagnostic colony PCR. Correct plasmids will be propagated and purified from <i>E. coli</i> cells. This recombinant pExTra plasmid will then be ready for use in the host <i>M. smegmatis</i>. </p> <p> <h4>Procedure :</h4> <p > Cloning a phage gene into a the pExTra plasmid is a multi-step process, and requires knowledge about key features of the plasmid. The multi-step process is provided in the table below and includes: </p> <ul role = 'list'> <li > the sequence of steps, </li> <li > a description of each step, </li> <li > a general protocol for executing the step, and </li> <li > specific information for cloning into pExTra that must be accounted for when utilising the general protocol. </li> </ul> <p > Key features of the pExTra plasmid are presented with the plasmid diagram. Refer back to information provided within this protocol throughout the process of cloning genes into pExTra.</p> <table style='margin:auto;'><thead><tr><th>Step</th><th>Step Description & Protocols</th><th>Protocol-Specific Information</th></tr></thead> <tbody>  <tr> <td> <p>1</p> </td> <td> <p>Amplify Phage Gene(s) <br> <a class='chap-linking'  tabindex='0' data-link-to='2-4'>by Polymerase Chain Reaction - Protocol 2.2</a></p> </td> <td> <p> <strong>Template DNA:</strong> Phage lysate or phage DNA <br> <strong>2X Polymerase MasterMix:</strong> Q5 <br> <strong>Primers:</strong> Gene-specific </p> </td> </tr> <tr> <td> <p>2</p> </td> <td> <p>Verify Size(s) of PCR Amplified DNA <br> <a class='chap-linking'  tabindex='0' data-link-to='2-5'>by Gel Electrophoresis - Protocol 2.3</a></p> <p> If Expected Fragment is Amplified, <br> Proceed to Step 3 Protocol 2.4</p> <p> If Multiple Fragments are Amplified, <br> Proceed to Step 3 Protocol 2.5 or <br> Return to Step 1</p> <p> If No Fragment is Amplified, <br> Return to Step 1</p> </td> <td aria-label='No value'> </td> </tr> <tr> <td> <p>3</p> </td> <td> <p> Purify Amplified DNA <br><a class='chap-linking'  tabindex='0' data-link-to='2-6'> by Column Purification of DNA - Protocol 2.4</a> <br> <a class='chap-linking'  tabindex='0' data-link-to='2-7'>by Gel Extraction of DNA - Protocol 2.5</a> </p> </td> <td aria-label='No value'> </td> </tr> <tr> <td> 4 </td> <td> <p>Assemble Amplified DNA into pExTra <br> <a class='chap-linking'  tabindex='0' data-link-to='2-9'>by Isothermal Assembly - Protocol 2.7</a></p> </td> <td> <p> <strong>Plasmid Size:</strong> 6 kbp</p> </td> </tr> <tr> <td> 5 </td> <td> <p>Transform E. coli with Assembled Plasmid DNA <br> <a class='chap-linking'  tabindex='0' data-link-to='2-11'>by Chemical Transformation of Bacteria - Protocol 2.9 </a></p> <p> If Colonies are Obtained, <br> Proceed to Step 6</p> <p>If No Colonies are Obtained, <br> Return to Step 4</p></p> </td> <td> <p> <strong>Medium: </strong> LB agar + Kanamycin (50μg/ml)</p> <strong>Incubation Temperature: </strong> 37 °C</p> </td> </tr> <tr> <td> <p>6</p> </td> <td> <p>Confirm Insert Size of Candidate Cloned <br> Plasmid(s) <br><a class='chap-linking'  tabindex='0' data-link-to='2-12'> by Clone Verification - Protocol 2.10</a></p> </td> <td> <p> <strong> Template DNA:</strong> Colony of transformed cells <br> <strong> 2X Polymerase MasterMix:</strong> Taq <br> <strong> Primers: </strong> pExTra_universal F and pExTra_universal R</p> </td> </tr> <tr> <td> <p>7</p> </td> <td> <p>Propogate E. coli with Candidate Cloned <br> Plasmid(s) <br><a class='chap-linking'  tabindex='0' data-link-to='2-13'> by Inoculating a Culture - Protocol 2.11</a></p> </td> <td> <p> <strong>Medium:</strong> LB + Kanamycin (50μg/ml) <br> <strong> Incubation Temperature:</strong> 37 °C</p> </td> </tr> <tr> <td> <p>8</p> </td> <td> <p>Extract Cloned Plasmid DNA <br><a class='chap-linking'  tabindex='0' data-link-to='2-14'> by Preparing Plasmid DNA - Protocol 2.12</a></p> </td> <td aria-label='No value'> </td> </tr> </tbody> </table> <div class='img-wrap'> <div class='img-container'> <img src='./assets/images/pextra-plasmid.png' alt='A color-coded circle for p-extra, with 6 colored and labeled regions in the following order. p-tet, m-cherry reporter, kanamycin resistance gene, e-coli origin of replication, tet repressor, and m-smegmatis origin of replication. Phage gene insertion site is marked with an arrow pointing to a spot on the circle between p-tet and m-cherry reporter.'> </div> <div class='img-hr'></div><div class='title-container'>  pExTra plasmid map </div></div> <p><strong>Helpful tips:</strong></p> "},"2-3":{content:"  <h3>Objective:</h3> <p> To construct a DNA molecule that will enable expression of a phage gene in an <i>E. coli</i> protein-protien interaction (PPI) assay. </p> <h4>Rationale:</h4> <p> To express a phage gene in&nbsp;<i>E. coli</i> in order to look for interactions between the phage gene product and host protein fragments, the gene must first be introduced into an appropriate expression plasmid, p2H&alpha;. The p2H&alpha; plasmid contains all the elements necessary for stable propagation in <i>E. coli</i> cells, and expression of inserted phage genes is controlled by the P<sub>lac*</sub> promoter, which is turned on in the presence of an inducer molecule. Here,&nbsp;a multi-step workflow for the cloning of a phage gene into p2H&alpha; is presented. Briefly, a phage gene sequence will be amplified, not from a lysate, but using the pExTra recombinant plasmid as template DNA. Primers that anneal to the pExTra plasmid upstream and downstream of the inserted phage gene will be used, and the same primer pair can be used universally for all recombinant pExTra plasmids. Amplification of phage genes using these universal primers ensures that the appropriate restriction sites are found at the ends of the insert; these same sites are found on the p2H&alpha; plasmid. Both plasmid and insert will be digested with appropriated restriction enzymes and ligated together, and recombinant plasmids are transformed into <i>E. coli</i>. Selected clones will be verified by diagnostic colony PCR, and correct plasmids will be propagated and purified from <i>E. coli</i> cells. This recombinant p2H&alpha; plasmid will then be ready for use in the <i>E. coli</i> B2H selection assay presented in Chapter 4. </p> <h4>Procedure :</h4> <p > Cloning a phage gene into a the p2H&alpha; plasmid is a multi-step process, and requires knowledge about key features of the plasmid. The multi-step process is provided in the table below and includes: </p> <ul role = 'list'> <li > the sequence of steps, </li> <li > a description of each step, </li> <li > a general protocol for executing the step, and </li> <li > specific information for cloning into p2H&alpha; that must be accounted for when utilizing the general protocol. </li> </ul> <p >Key features of the p2H&alpha; plasmid are presented with the plasmid diagram. Refer back to information provided within this protocol throughout the process of cloning genes into p2H&alpha;. </p> <table style='margin:auto'><thead><tr><th>Step</th><th>Step Description & Protocols</th><th>Protocol-Specific Information</th></tr></thead> <tbody> <tr> <td> <p>1</p> </td> <td> <p>Amplify Phage Gene(s) <br><a class='chap-linking'  tabindex='0' data-link-to='2-4'> by Polymerase Chain Reaction - Protocol 2.2</a></p> </td> <td> <p> <strong>Template DNA:</strong> Recombinant pExTra with appropriate gene <br> insert  <strong>2X Polymerase MasterMix:</strong> Q5 <br> <strong>Primers: </strong>pExTra_universal_F and pExTra_universal_R </p> </td> </tr> <tr> <td> <p>2</p> </td> <td> <p>Verify Size(s) of PCR Amplified DNA <br><a class='chap-linking'  tabindex='0' data-link-to='2-5'> by Gel Electrophoresis - Protocol 2.3</a></p> <p> If Expected Fragment is Amplified, <br> Proceed to Step 3 Protocol 2.4</p> <p> If Multiple Fragments are Amplified, <br> Proceed to Step 3 Protocol 2.5 or <br> Return to Step 1</p> <p> If No Fragment is Amplified, <br> Return to Step 1</p> </td> <td aria-label='No value'> </td> </tr> <tr> <td> <p>3</p> </td> <td> <p> Purify Amplified DNA <br> <a class='chap-linking'  tabindex='0' data-link-to='2-6'>by Column Purification of DNA - Protocol 2.4 </a><br><a class='chap-linking'  tabindex='0' data-link-to='2-7'> by Gel Extraction of DNA - Protocol 2.5 </a></p> </td> <td aria-label='No value'> </td> </tr> <tr> <td> 4 </td> <td> <p>Restriction Enzyme Double Digest of Insert <br> <a class='chap-linking'  tabindex='0' data-link-to='2-8'>by Restriction Enzyme Digest - Protocol 2.6</a></p> </td> <td> <p> <strong>Enzymes: </strong> NdeI and BamHI OR NdeI and SbfI</p> </td> </tr> <tr> <td> 5 </td> <td> <p>Purify Digested Insert<br><a class='chap-linking'  tabindex='0' data-link-to='2-6'> by Column Purification of DNA - Protocol 2.4 </a></p> </td> <td aria-label='No value'> </td> </tr> <tr> <td> <p>6</p> </td> <td> <p>Ligation of Insert and p2H&alpha; backbone <br><a class='chap-linking'  tabindex='0' data-link-to='2-10'> by DNA Ligation - Protocol 2.8</a></p> </td> <td> <p> <strong> Plasmid size:</strong> 5 kb </p> </td> </tr> <tr> <td> <p>7</p> </td> <td> <p>Transform E. coli with Assembled Plasmid DNA <br> <a class='chap-linking'  tabindex='0' data-link-to='2-11'>by Chemical Transformation of Bacteria - Protocol 2.9</a> <br></p> <p>If Colonies are Obtained, <br> Proceed to Step 8</p> <p> If No Colonies are Obtained, <br> Return to Step 4</p> </td> <td> <p> <strong> Medium:</strong> LB agar + Spectinomycin (50μg/ml) <br> <strong> Incubation Temperature: </strong> 30 °C or 37 °C </p> </td> </tr> <tr> <td> <p>8</p> </td> <td> <p>Confirm Insert Size of Candidate Cloned <br> Plasmid(s) <br><a class='chap-linking'  tabindex='0' data-link-to='2-12'> by Clone Verification - Protocol 2.10</a></p> </td> <td> <p> <strong> Template DNA:</strong> Colony of transformed cells <br> <strong> 2X Polymerase MasterMix:</strong> Taq <br> <strong> Primers: </strong> o-p2H&alpha;F & o-p2H&alpha;R</p> </td> </tr> <tr> <td> <p> 9</p> </td> <td> <p> Propogate E. coli with Candidate Cloned <br> Plasmid(s) <br><a class='chap-linking'  tabindex='0' data-link-to='2-13'> by Inoculating a Culture - Protocol 2.11</a> </p> </td> <td> <p> <strong> Medium:</strong> LB + Spectinomycin (50μg/ml) <br> <strong> Incubation Temperature: </strong> 30 °C or 37 °C </p> </td> </tr> <tr> <td> <p> 10 </p> </td> <td> <p>Extract Cloned Plasmid DNA <br> <a class='chap-linking'  tabindex='0' data-link-to='2-14'>by Preparing Plasmid DNA - Protocol 2.12</a></p> </td> <td aria-label='No value'> </td> </tr> </tbody> </table> <div class='img-wrap'> <div class='img-container'> <img src='./assets/images/p2ha-plasmid-map.png' alt='A color-coded circle for p-two-h-alpha, with 4 colored and labeled regions in the following order. p-lac-asterix, alpha domain, e-coli origin of replication, and spectinomycin resistance gene. A list of 4 enzymes, not-one, N-d-e one, Bam-H-one, and S-b-f one  is marked with an arrow pointing to a spot on the circle immediately after alpha domain.'> </div> <div class='img-hr'></div><div class='title-container'>  p2H&alpha; plasmid map</div></div>  "},"2-4":{content:"  <h3>Objective:</h3> <p>To generate copies of a specific segment of DNA </p> <h4>Rationale:</h4> <p> Polymerase Chain Reaction (PCR) is a powerful technique that allows researchers to make large quantities of a specific segment of DNA. PCR is utilized in many scientific disciplines, including medicine, forensics, and genomics. In molecular biology, PCR is an essential step in many cloning strategies, including those we will employ for cloning phage genes into pExTra and p2H&alpha; plasmids.</p> <h4>Supplies :</h4> <ul role = 'list'> <li> Template DNA* </li> <li> DNA Primers, forward and reverse (10 μM )* </li> <li> 2x Polymerase MasterMix* </li> <li> Sterile, nuclease-free water </li> <li> Ice </li> <li> Thermocycler </li> <li> PCR tubes </li> <li> Microcentrifuge tubes  </li> </ul> <p>* Each PCR will require specific template DNA and DNA primers, as well as the 2X DNA polymerase MasterMix. Consult the protocol specific information provided in <a class='chap-linking'  tabindex='0' data-link-to='2-2'>Protocol 2.1a: Cloning into pExTra</a> or<a class='chap-linking'  tabindex='0' data-link-to='2-3'> Protocol 2.1b: Cloning into p2H&alpha;</a>.</p> <h4>Procedure :</h4> <p ><strong><b style='color:#b22222'> Important : </b> </strong>Gloves should always be worn when doing molecular cloning.</p> <ol role = 'list' style='list-style-type:upper-alpha;'> <li> Set up the PCR: <ol role = 'list'> <li><p> Obtain and label a PCR tube for each PCR to be performed. </li> <li> For each PCR, add all the following components to the tube. <br> <strong><b style='color:#b22222'> Important : </b></strong>Follow the order of addition as indicated in the table below, and use a new pipette tip for each addition.</p><table > <thead><tr><th>Order of Addition</th><th>PCR Component</th><th>Volume</th></tr></thead><tbody> <tr> <td >1</td> <td>sterile water</td> <td>9.0 &micro;l</td> </tr> <tr> <td >2</td> <td>forward primer</td> <td>1.25 &micro;l</td> </tr> <tr> <td >3</td> <td>reverse primer</td> <td>1.25 &micro;l</td> </tr> <tr> <td >4</td> <td>template DNA</td> <td>1.0 &micro;l</td> </tr> <tr> <td >5</td> <td>2X polymerase MasterMix</td> <td>12.5 &micro;l</td> </tr> <tr> <td aria-label='No value'> </td> <td><strong>Total volume</strong></td> <td><strong>25 &micro;l</strong></td> </tr> </tbody> </table> </li> </ol> </li> </ol> <ol role = 'list' start='3' style='list-style-type: decimal;'> <li> Close the PCR tube cap(s) tightly and do a quick spin or gently tap on bench to settle all liquid to the bottom of the tube. </li> <li> Place tube(s) in ice.  </li> </ol>  B. Set-up the thermocycle PCR program.<ol role = 'list'> <li> Set the heated lid temperature to 95 <b>°</b>C. </li> <li> Set the thermocycler parameters based on the DNA polymerase being used. <br> <strong>Note:</strong> Annealing temperatures may be adjusted based on the specific primer pairs. Extension times are based on the length of the DNA segment being amplified. <br> <table > <thead><tr><th aria-label='No value'></th><th colspan='2' rowspan='1'>Q5 DNA Polymerase</th><th colspan='2' rowspan='1'>Taq DNA Polymerase</th></tr></thead><tbody>  <tr> <td> <strong> PCR Step </strong> </td> <td > <strong> Temperature </strong></td> <td > <strong> Duration </strong></td> <td > <strong> Temperature </strong></td> <td > <strong> Duration </strong></td> </tr> <tr> <td>a. Initial Denaturation</td> <td > 95 <b>°</b>C </td> <td >5 min</td> <td > 94 <b>°</b>C </td> <td >5 min</td> </tr> <tr> <td>b. Denaturation</td> <td > 95 <b>°</b>C </td> <td >10 sec</td> <td >94 <b> °</b> C </td> <td >15 sec</td> </tr> <tr> <td>c. Annealing</td> <td > 60 <b>°</b>C </td> <td >10 sec</td> <td >60 <b>°</b>C </td> <td >15 sec</td> </tr> <tr> <td>d. Extension</td> <td > 72 <b>°</b>C </td> <td >30 sec/kb</td> <td >68 <b>°</b>C </td> <td >1 min/kb</td> </tr> <tr> <td colspan='5' >Repeat PCR Steps b -d for 29 cycles</td> </tr> <tr> <td>e. Final Extension</td> <td > 72 <b>°</b>C </td> <td >5 min</td> <td >68 <b>°</b>C </td> <td >5 min</td> </tr> <tr> <td>f. Hold</td> <td > 4 <b>°</b>C </td> <td >infinite</td> <td >4 <b>°</b>C </td> <td >infinite</td> </tr> </tbody> </table> </li> <li> Place the PCR tube(s) in the thermocycler. </li> <li> Run the PCR program. Once complete, store your reaction tubes at 4 °C until you are ready to proceed to <a class='chap-linking'  tabindex='0' data-link-to='2-5'>Protocol 2.3: Gel Electrophoresis. </a>  </li> </ol>  "},"2-5":{content:"<h3>Objective:</h3> <p>To visualize and determine the size of DNA segments</p> <h4>Rationale: </h4> <p>Gel electrophoresis is a technique that separates DNA molecules on the basis of size. DNA molecules are loaded onto an agarose gel and exposed to an electric current. The DNA molecules migrate through the sieve-like agarose gel due to their inherent negative charge; smaller DNA molecules move more easily, and therefore more quickly, than larger DNA molecules through the gel. The agarose gel contains a dye that binds to DNA and allows the DNA molecules that have now been separated by size to be visualized under ultraviolet light. The size of the DNA molecules can then be calculated by comparing to a ladder of DNA molecules of known sizes. </p> <p> <h4>Supplies:</h4></p> <ul role = 'list'> <li>Pre-poured agarose gel (0.8%)</li> <li>DNA loading dye (6X)</li> <li>Electrophoresis apparatus and power supply</li> <li>1X TBE or TAE running buffer</li> <li>DNA samples</li> <li>DNA ladder</li> <li>Gel imaging equipment</li> </ul> <h4>Procedure :</h4><p> <strong> <b> <b style='color:#b22222'> Important : </b> </b> </strong> Gloves should always be worn when doing molecular cloning.</p> <ol role = 'list' style='list-style-type:upper-alpha;'> <li>Prepare your DNA samples for gel electrophoresis. <ol role = 'list'> <li>Determine the volume of DNA sample to be loaded of the gel. <b> <b style='color:#b22222'> Important : </b> </b> For verification of PCR products, 5 - 10 &micro;l of sample is sufficient. For gel purification of PCR products, the entire PCR reaction volume (25 &micro;l) should be used.</li> <li>For each DNA sample to be loaded onto the gel, add the approproate volume of DNA sample to a clean microcentrifuge tube. Then add 1 &micro;l of concentrated 6X DNA loading dye for every 5 &micro;l of DNA sample. Pipet up and down gently to mix. </li> </ol> </li> <li>Load the DNA sample and loading dye mixture onto the agarose gel according to your instructor's directions and following the instructions below. <ol role = 'list'> <li>In the first well of the gel, load an appropriate volume of DNA ladder. Each manufacturer is different, so follow the manufacturer’s guidelines.<br> <b> <b style='color:#b22222'> Important : </b> </b> If gel electrophoresis is being performed as part of Protocol 2.5: Gel Extraction of DNA, skip a lane in the gel between each sample as well as the DNA ladder to avoid cross-contamination of samples.</li> <li>Using a fresh tip on your micropipettor for each sample, pipette each DNA sample and loading dye mixture into a new well in the gel. <ol role = 'list' start = '1' style='list-style-type:lower-alpha;'> <li>Holding the pipette in both hands, place your elbows on either side of the gel apparatus.</li> <li>Situate your eyes directly above the wells to make the wells easier to see.</li> <li>Place the pipette tip directly above the well, just below the surface of the buffer. Do not try to get the pipette tip into the well, or you might puncture the bottom of the gel.</li> <li>Slowly depress the pipette plunger, allowing the solution to slowly sink into the well.</li> <li>Remove the pipette from the gel before releasing the plunger.</li> </ol> </li> <li>Draw a picture of the gel in your lab notebook, making note of where your samples are relative to your classmates’ samples. (Once the gels are loaded, everyone’s samples look alike!)</li> <li>Plug the electrodes into the appropriate locations on the power supply. Turn on the power supply and set the voltage to 100 V.<br> <b> <b style='color:#b22222'> Important : </b> </b> Remember, DNA runs toward the RED electrode!</li> <li>Run the gel until the visible loading dye front has migrated two thirds the length of the gel. This will take between 30 - 60 minutes depending on the size of gel being used.<br> <b> <b style='color:#b22222'> Important : </b> </b> It is important to monitor the migration of the dye front in your gel, which migrates at the same speed as DNA molecules of approximately 300 bp. Running the gel for too short a duration will not allow for good separation of different sized DNA moecules. Running the gel for too long may result in small DNA molecules to run off the gel and into the buffer.</li> <li>Turn off the power supply. </li> </ol> </li> <li>Gel extraction <ol role = 'list'> <li>If extracting DNA of a particular size from your gel, proceed directly to Step B of Protocol 2.5: Gel Extraction of DNA. Otherwise, continue to Step D, below. </li> </ol> </li> <li>Photograph the gel. <ol role = 'list'> <li>Using gloves, carefully remove the gel from the electrophoresis chamber.</li> <li>Photograph your gel, following your instructor’s directions.</li> <li>Include a copy of the gel photograph in your laboratory notebook.</li> </ol> </li> <li>Clean up your work area. <ol role = 'list'> <li>Dispose of your gel as directed by your instructor.</li> <li>If another gel isn’t going to be run, empty the buffer into the sink, rinse out the gel apparatus, and set it aside to dry. The buffer can be reused, so check to see if anyone else will need to run a gel in the next few days.</li> </ol> </li> </ol> <p><strong>Helpful tips:</strong></p> "},"2-6":{content:"<h3>Objective:</h3> <p>To clean DNA samples after enzymatic reactions</p> <h4>Rationale: </h4> <p>Molecular cloning involves multiple reaction steps in which DNA molecules are mixed with different enzymes and buffers. Often, the components from one reaction are incompatible with the conditions for a second or downstream reaction. As such, the DNA must be purified away from the other components of a reaction before it can be used in a downstream reaction. A convenient way for purifying DNA is to use commercially-available column kits. Using these column kits, DNA from various samples are mixed with salts that denature residual proteins and allow DNA molecules larger than 50 bp to bind silica resin within the column, while other reaction are washed away. The DNA can then be eluted from the column using water. A separate strategy for purifying DNA is described in<a class='chap-linking'  tabindex='0' data-link-to='2-7'> Protocol 2.5: Gel Extraction of DNA</a>.</p> <p> <h4>Supplies:</h4></p> <ul role = 'list'> <li> DNA sample </li> <li> DNA Clean and Concentrator Kit (Zymo) </li> <li> sterile, nucelase free water </li> <li> microcentrifuge </li> <li> micrcentrifuge tubes </li> </ul> <h4>Procedure :</h4> <strong> <b style='color:#b22222'> Important : </b> </strong> Gloves should always be worn when doing molecular cloning.</p> <ol role = 'list' style='list-style-type:upper-alpha;'> <li> Bind DNA to the column. <ol role = 'list'> <li> For each DNA sample being purified, transfer the entire volume of the DNA sample to a new and clean microcentrifuge tube. <br> <b> <b style='color:#b22222'> Important : </b> </b> Make note of the DNA sample volume. </li> <li>Based on the volume of DNA sample transfered, add 5 times that volume of DNA Binding Buffer to the DNA sample. Pipette up and down gently to mix. <p > <b>Example: </b> <em>To 25 &micro;l of DNA sample, add 125 &micro;l of DNA Binding Buffer.</em> </p> </li> <li> <p > Label a spin column according to your DNA sample, place the spin column in a collection tube, and transfer the mixture of DNA sample and DNA Binding Buffer to the labeled spin column. </p> </li> <li> <p > Centrifuge at the maximum speed for 30 sec. </p> </li> <li> <p > Remove the spin column and discard the flow-through that accumulated in the collection tube into an appropriate liquid waste container. Then place the spin column back onto the same collection tube, and proceed to the next step.</p> </li> </ol> </li> <li> Wash the DNA now bound within the column. <ol role = 'list'> <li> Add 200 &micro;l of DNA Wash Buffer to the spin column. </li> <li> Centrifuge at the maximum speed for 30 sec. </li> <li> Remove the spin column and discard the flow-through from the collection tube. Then place the spin column back onto the same collection tube. </li> <li> Repeat Steps B1-3 </li> </ol> </li> <li>Elute the DNA <ol role = 'list'> <li> Transfer the spin column to a new and labelled microcentrifuge tube. </li> <li> Add 25 &micro;l of sterile water to the spin column. <br> <b> <b style='color:#b22222'> Important : </b> </b> To make sure that the water soaks the column matrix, pipette the water directly onto the 'white' matix in the center of the column. </li> <li> Incubate at room temperature for 1 min. </li> <li> Centrifuge at the maximum speed for 30 sec. </li> <li> The flow-through in the microcentrifuge tube is your eluted and purified DNA. </li> </ol> </li>  <li> Measure the concentration of your DNA. <ol role = 'list'> <li> Follow your instructor's directions to measure the concentration of your purified DNA. </li> <li>Record the concentration of DNA, in ng/&micro;l, on the side of the microcentrifuge tube and in your notebook.</li> <li>Take note of the A260/A280 (DNA/protein) ratio and A260/A230 (DNA/organic compounds) ratio for your purified DNA. A 'pure' DNA sample should have a A260/280 ratio of ~1.8 and an A260/A230 ratio of 2.0 - 2.2. </li> </ol> </li> </ol> <p><strong>Helpful tips:</strong></p> "},"2-7":{content:"<h3>Objective:</h3> <p>To extract DNA segments of a certain size from an agarose gel.</p> <h4>Rationale: </h4> <p>DNA molecules can easily be purified away from other reaction components, such as enzymes and buffers, using <a class='chap-linking'  tabindex='0' data-link-to='2-6'>Protocol 2.4: Column Purification of DNA</a>. However, column purification indiscriminately purifies all DNA molecules longer than 50 bp. To purify DNA molecules of a particular sizes from a mixture that contains DNA molecules of multiple sizes, DNA molecules can first be separated by size using gel electrophoresis, then physically removing parts of the agarose gel that contain DNA fragments of the desired size, and finally, purifying that DNA away from the components of the gel using a column. </p> <p> <h4>Supplies:</h4></p> <ul role = 'list'> <li> DNA sample </li> <li >Pre-poured agarose gel</li> <li >DNA loading dye (6X)</li> <li >Electrophoresis apparatus and power supply</li> <li >DNA ladder</li> <li> Razor blade, spatula, or gel extraction tool </li> <li> Gel DNA Recovery Kit (Zymo) </li> <li> Sterile, nucelase-free water </li> <li>Microcentrifuge</li> <li>Microcentrifuge tubes</li> <li>Thermocycler, heat block, or incubator set to a temperature within 37 - 55 <b>°</b>C.</li> </ul> <h4>Procedure :</h4> <strong> <b style='color:#b22222'> Important : </b> </strong> Gloves should always be worn when doing molecular cloning.</p> <ol role = 'list' style='list-style-type:upper-alpha;'> <li> Perform gel electrophoresis <ol role = 'list'> <li>For each DNA sample from which you wish to purify DNA of a particular size, perform gel electrophoresis using <a class='chap-linking'  tabindex='0' data-link-to='2-5'> Protocol 2.3: Gel Electrophoresis</a>.<br> <strong> <b style='color:#b22222'> Important : </b> </strong> Skip a lane in the gel between each sample as well as the DNA ladder to avoid cross-contamination of samples. </li> </ol> </li> <li> Excise the band of interest from the agarose gel. <ol role = 'list'> <li>For each band of agarose to be extracted from the gel, obtain and label a new microcentrifuge tube.</li> <li>Weight the microcentrifuge tube and write this initial mass on the side of the tube.</li> <li>Carefully place the agarose gel on the gel imaging apparatus or UV box.</li> <li>Wearing appropriate personal protective equipment to protect from UV light (e.g. a face shield) and following your instructor's directions, turn on the UV light source to visualize the DNA bands within the gel.</li> <li>Using a razor blade, spatula, or gel extraction tool, carefully cut out the DNA band of interest from the gel and place the extracted gel piece in the labeled microcentrifuge tube.<br> <strong> <b style='color:#b22222'> Important : </b> </strong> Take care to avoid other DNA bands in the same or neighbouring lanes. <br> <strong> Note: </strong> The gel slice can be stored at -20 <b>°</b>C for multiple days. </li> <li> Weight the microcentrifuge tube containing the gel piece. Write the final mass on the side of the tube. </li> <li> Calculate the mass of the gel slice by subtracting the initial mass of the tube from the final mass of the tube. </li> </ol> </li> <li> Extract the DNA from the agarose gel. <ol role = 'list'> <li> To the microcentrifuge containing the gel slice, add 300 &micro;l of Agarose Dissolving Buffer (ADB) for every 100 mg of gel slice. <br> <strong> Example: </strong> <em>For a gel slice with a mass of 150 mg, add 450 &micro;l of ADB</em> </li> <li> Incubate the tube at 37 - 55 <b>°</b>C for 5 - 10 minutes, until the agarose is completely dissolved. <br> <strong> <b style='color:#b22222'> Important : </b> </strong> To check if the agarose has completely dissolved, invert the tube 2 - 3 times to mix the contents of the tube. </li> <li > Label a spin column, place the spin column in a collection tube, and transfer the entire DNA/ADB mixture to the labeled spin column. </li> <li > Centrifuge at the maximum speed for 30 sec. </li> <li > Remove the spin column and discard the flow-through that accumulated in the collection tube into an appropriate liquid waste container. Then place the spin column back onto the same collection tube, and proceed to the next step. </li> </ol> </li> <li > Wash the DNA now bound within the column. <ol role = 'list'> <li > Add 200 &micro;l of DNA Wash Buffer to the spin column. </li> <li > Centrifuge at the maximum speed for 30 sec. </li> <li > Remove the spin column and discard the flow-through from the collection tube. Then place the spin column back onto the same collection tube. </li> <li > Repeats Steps D1-3.  </li> </ol> </li> <li >Elute the DNA <ol role = 'list'> <li > Transfer the spin column to a new and labelled microcentrifuge tube. </li> <li > Add 25 &micro;l of sterile water to the spin column. <br> <strong> <b style='color:#b22222'> Important : </b> </strong> To make sure that the water soaks the column matrix, pipette the water directly onto the 'white' matrix in the center of the column. </li> <li > Incubate at room temperature for 1 min. </li> <li > Centrifuge at the maximum speed for 30 sec. </li> <li > The flow-through in the microcentrifuge tube is your eluted and purified DNA.  </li> </ol> </li> <li >Measure the concentration of your DNA <ol role = 'list'> <li > Follow your instructor's directions to measure the concentration of your purified DNA. </li> <li >Record the concentration of DNA, in ng/&micro;l, on the side of the microcentrifuge tube and in your notebook.</li> <li >Take note of the A260/A280 (DNA/protein) ratio and A260/A230 (DNA/organic compounds) ratio for your purified DNA. A 'pure' DNA sample should have a A260/280 ratio of ~1.8 and an A260/A230 ratio of 2.0 - 2.2.</li> </ol> </li> </ol> <p><strong>Helpful tips:</strong></p> <p>1. UV light can be mutagenic to DNA . To limit the UV exposure of your DNA samples, use long-wavelength UV for as short as time as possible when extracting bands from a gel. </p> "},"2-8":{content:" <h3>Objective:</h3> <p> To prepare the ends of DNA fragments for molecular cloning. </p> <h4>Rationale: </h4><p>Restriction endonuclease enzymes recognize and cleave double-stranded DNA at specific 4-6 bp palindromic sequences called restriction sites. Some restriction enzymes cleave both DNA strands at the same position, generating 'blunt ends', whereas other restriction enzymes cleave each DNA strand at slightly different positions, generating single-stranded overhangs called 'sticky-ends'. These sticky-ends are very useful for molecular cloning, as they allow for two pieces of DNA, for example a gene insert and a plasmid, that have similar overhangs to pair efficiently and specifically. Once paired, the enzyme DNA ligase can be used to covalently link the two pieces of the together. </p> <h4>Supplies:</h4> <ul role = 'list'> <li>Purified DNA sample</li> <li>Restriction enzymes * (NEB)</li> <li>10X restriction enzyme buffer (NEB)</li> <li>Sterile, nuclease-free water</li> <li>Thermocycler, heat block, or incubator set to 37 <b>°</b>C </li> </ul> </div> <p >*For cloning genes into the p2H&alpha; plasmid, gene inserts and p2H&alpha; are both cleaved with NdeI and BamHI restriction enzymes. If the phage gene insert has a BamHI site within it, NdeI and SbfI enzymes should be used. If the phage gene insert has an NdeI site within it, do not proceed with this protocol. </p> <h4>Procedure:</h4> <p > <strong> <b style='color:#b22222'> Important : </b></strong> Gloves should always ben worn when doing molecular cloning. </p> <ol role = 'list' style='list-style-type:upper-alpha;'> <li> Determine the volume of DNA sample to be used. <ol role = 'list'> <li> For each DNA sample to be cleaved, use the concentration of the DNA sample to calculate the volume needed to obtain 0.5-1 &micro;g of DNA. <p> <strong><b style='color:#b22222'> Important : </b></strong> If you do not have enough sample to obtain 0.5 &micro;g of DNA, check with your instructor if it is suitable to use the maximum volume of DNA sample allowed in the reaction.</p></li> </ol> </li> <li> Set up the restriction enzyme digest reaction. <ol role = 'list'> <li> For each DNA sample to be cleaved, obtain and label a new microcentrifuge tube. </li> <li> Add the following components to the microcentrifuge tube for each restriction enzyme digest reaction. <p><strong> <b style='color:#b22222'> Important : </b></strong> Follow the order of addition as indicated in the table below, and use a new pipette tip for each addition. </p> <table > <thead><tr><th>Reaction Component</th><th>Volume</th></tr></thead><tbody>  <tr> <td>Sterile water</td> <td>up to 50 &micro;l</td> </tr> <tr> <td>10X reaction buffer</td> <td>5 &micro;l</td> </tr> <tr> <td>Restriction enzyme (each)</td> <td>1 &micro;l</td> </tr> <tr> <td>DNA sample</td> <td>predetermined volume</td> </tr> <tr> <td><strong>Total Volume</strong></td> <td><strong>50 &micro;l</strong></td> </tr> </tbody> </table> </li> <li>  Mix the contents of each tube gently and quick spin the tube in a microcentrifuge tube for less than 1 min to move all the liquid to the bottom of the tube.  </li> <li>Incubate the tube(s) at 37 <b>°</b>C for 1 hour, or follow your instructor's directions.</li> <li>After the incubation period, quick spin the tube in a microcentrifuge tube for less than 1 min to move all the liquid to the bottom of the tube. Store at -20 <b>°</b>C until ready to use for downstream applications. </li> </ol> </li> <li> Purify DNA from the reaction. <ol role = 'list'> <li> To purify DNA from the digest reaction for downstream applications, use <a class='chap-linking'  tabindex='0' data-link-to='2-6'>Protocol 2.4: Column Purification of DNA </a>.</li> </ol> </li> </ol> <p> <strong>Helpful tips:</strong> </p>"},"2-9":{content:"<h3>Objective:</h3> <p> To construct a plasmid from a gene insert and plasmid backbone DNA</p> <h4>Rationale: </h4><p>Isothermal assembly is a molecular cloning strategy that allows for the rapid assembly of multiple DNA fragments into a single DNA molecule. This strategy is quick, with multiple enzymatic reactions happening simultaneously and at one incubation temperature, but requires that the DNA segments to be assembled be linear and have overlapping sequences that are at least 20 bp in length. Isothermal assembly will be used to assemble PCR-amplified gene inserts with the pExTra backbone DNA (i.e. plasmid that has been made linear).</p> <h4>Supplies:</h4> <ul role = 'list'> <li>Purified gene-insert DNA</li> <li>Plasmid backbone DNA*</li> <li>2X NEBuilder HiFi DNA Assembly MasterMix (NEB)</li> <li>Ice</li> <li>Sterile, nuclease-free water</li> <li>Thermocycler, heatblock, or incubator set to 50 <b>°</b>C</li> <li>Microcentrifuge or PCR tubes.</li> </ul> <p> * Plasmid backbone DNA will be prepared and provided by your instructor. </p> <h4>Procedure:</h4> <ol role = 'list' style='list-style-type:upper-alpha;' > <li ><strong><b style='color:#b22222'> Important : </b></strong> Gloves should always be worn when doing molecular cloning. </strong> </li> <li>This isothermal assembly protocol is optimized for 50 ng of plasmid backbone DNA, and 2 molar excess of insert DNA. To setup the assembly reaction, the volume of plasmid backbone and insert DNA to be used must first be determined. <ol role = 'list'> <li>Determine the volume of plasmid backbone DNA to be used. <ol role = 'list' start='1' style='list-style-type: lower-alpha;' > <li><p>Based on the concetration of the plasmid DNA prepared by yor instructor, use the formula below to calculate the volume of sample needed to obtain 50 ng of plasmid backbone DNA.</p> <p> Formula: volume needed, &micro;l = (amount needed, ng) / (concentration of stock, ng/&micro;l) </p> <p> <em><strong>Example:</strong> F or plasmid backbone DNA at 100 ng/&micro;l, </em> </p> <p> <em>volume needed, &micro;l = (50 ng) / (100 ng/&micro;l) </em><em>= 0.5 &micro;l</em></p></li> <li>Record this volume in the table below. See helpful tip below. </li> </ol> </li> <li>Determine the volume of insert DNA to be used. <ol role = 'list' start='1' style='list-style-type: lower-alpha;'> <li><p>Based on the length and the concentration of the insert DNA, use the formula below to calculate the volume of sample needed to obtain a 2 molar excess of insert DNA to 50 ng of plasmid backbone DNA. </p><p>Formula: volume needed, &micro;l = 2 x 50 ng x (length of insert, bp / length of plasmid backbone, bp) / <em> </em>(concentration of insert DNA, ng/&micro;l) </p> <p> <em><strong> Example:</strong> For insert DNA that is 450 bp in length and prepared at 25 ng/&micro;l, and will be cloned into the 6 kbp pExTra plasmid backbone, </p> <p> volume needed, &micro;l = 2 x 50 ng x (450 bp / 6000 bp) / (25 ng/&micro;l) = 0.3 &micro;l</em></p></li> <li>Record this volume in the table below. See helpful tip below. </li> </ol> </li> </ol> </li> <li>Determine the volume of water to be used. <ol role = 'list' start='1' style='list-style-type: lower-alpha;' > <li>Sum the volume of plasmid backbone and insert DNA in the table below. <p> <strong> <b style='color:#b22222'> Important : </b> </strong>The sum volume should not exceed 5 &micro;l.</p></li> <li>Subtract this summed value from 5 &micro;l. This subtratced value is the volume of water to be used.</li> <li>Record this calculated volume for water in the table below. </li> </ol> </li> <li> Set up the isothermal assembly reaction. <p><strong> <b style='color:#b22222'> Important : </b> </strong>All components of the assembly reaction should be kept cold, on ice, during the setup process.</p></li> <ol role = 'list'> <li>Obtain an aliquot of 2X HiFi DNA Assembly MasterMix and place it on ice. Flick the tube with your finger a few times to mix prior to adding to the reaction.</li> <li>Obtain and label a PCR tube for each assembly reaction to be setup.</li> <li>Using the information recorded in the table below, setup the assembly reaction. </br><span style='color:#b22222'> Important : </span> Follow the order of addition as indicated in the table below., and use a new pipette tip for each addition. </br><table>  <thead><tr><th>Reaction Component</th><th>Volume, &micro;l</th></tr></thead><tbody> <tr> <td> Reaction Component </td> <td > Volume, &micro;l </td> </tr> <tr> <td>Plasmid backbone DNA (50 ng)</td> <td aria-label='No value'> </td> </tr> <tr> <td>DNA insert</td> <td aria-label='No value'> </td> </tr> <tr> <td>Sterile, nuclease-free water</td> <td aria-label='No value'> </td> </tr> <tr> <td>2X HiFi DNA Assembly MasterMix</td> <td >5 &micro;l</td> </tr> <tr> <td>Total Volume</td> <td >10 &micro;l</td> </tr> </tbody> </table> </li> <li>Once all the components have been added, gently pipette up and down 4 - 5 times to mix, and keep on ice.</li> <li> When ready to begin the assembly reaction, place the tubes at 50 <b>°</b>C for 30 min.</li> <li>Once the reaction is complete, place the reaction on ice. Then proceed to<a class='chap-linking'  tabindex='0' data-link-to='2-11'> Protocol 2.9 Transformation of Bacteria</a>, or store the reaction at -20 <b>°</b>C</li> </ol> </li> </ol> <p> <strong>Helpful tips:</strong> </p> <p>- It is not recommended to pipette volumes less than 1&micro;l. If the concentration of your insert or plasmid backbone sample requires less than 1 &micro;l be added, make an appropriate dilution of the sample. </p> <p><strong><em> Example 1 :</strong> F or plasmid backbone DNA at 100 ng/&micro;l, </em> </p> <p> <em>volume needed, &micro;l = (50 ng) / (100 ng/&micro;l) </em><em>= 0.5 &micro;l</em></p> <p><i>A 1:2 dilution can be made, mixing equal volumes of the plasmid backbone and sterile nuclease-free water, for a final DNA concentration of 50 ng/</i><em>&micro;l. </em></p> <p><em>volume needed, &micro;l = (50 ng) / (50 ng/&micro;l) </em><em>= 1 &micro;l</em></p> <p><strong> <em>Example 2: </em></strong> <em>For i</em><em>nsert DNA that is 450 bp in length and prepared at 25 ng/&micro;l, and will be cloned into the 6 kbp pExTra plasmid backbone, </p> <p>volume needed, &micro;l = 2 x 50 ng x (450 bp / 6000 bp) / (25 ng/&micro;l) = 0.3 &micro;l</em></p> <p><i>A 1:5 dilution can be made, mixing four volumes of sterile nuclease-free water for every volume of insert, for a final DNA concentration of 5 ng/</i><em>&micro;l. </em></p> <p><em>volume needed, &micro;l = 2 x 50 ng x (450 bp / 6000 bp) / (5 ng/&micro;l) = 1.5 &micro;l</em></p> "},"2-10":{content:" <h3>Objective:</h3> <p>To construct a plasmid from a gene insert and plasmid backbone DNA</p> <h4>Rationale:</h4><p> DNA ligation is the process by which the ends of two DNA molecules can be covalently linked together. This linkage is formed by the enzyme DNA ligase. When constructing a plasmid, DNA ligase can be used to covalently link together the ends of a gene insert with that of plasmid backbone DNA, thereby regenerating a circular plasmid, now with the gene insert. DNA ligation will be used to assemble the p2H&alpha; plasmid containing a phage gene of interest.</p> <h4>Supplies:</h4> <ul role = 'list'> <li >Purified gene-insert DNA</li> <li >plasmid backbone DNA*</li> <li >2X Instant Sticky-end Ligase MasterMix (NEB)</li> <li >Ice</li> <li >Sterile, nuclease-free water</li> <li >Thermocycler, heatblock, or incubator set to 65 <b>°</b>C</li> <li >Microcentrifuge or PCR tubes. </li> </ul> <p >* Linear plasmid backbone DNA will be prepared and provided by your instructor.</p> <p><strong><b style='color:#b22222'> Important : </b></strong> Gloves should always ben worn when doing molecular cloning. </p> <ol role = 'list' style='list-style-type:upper-alpha;' > <li >This DNA ligation protocol is optimized for a 3:1 molar ratio of insert DNA to plasmid backbone DNA. To setup the ligation reaction, the volume of insert and plasmid backbone DNA to be used must first be determined. <ol role = 'list'> <li>Determine the volume of plasmid backbone DNA to be used. <ol role = 'list' style='list-style-type: lower-alpha;' > <li><p>Based on the concetration of the plasmid DNA prepared by your instructor, use the formula below to calculate the volume of sample needed to obtain 50 ng of plasmid backbone DNA.</p> <p>Formula: volume needed, &micro;l = (amount needed, ng) / (concentration of stock, ng/&micro;l) </p> <em><p><strong>Example:</strong> F or plasmid backbone DNA at 100 ng/&micro;l, </em> </p> <p><em>volume needed, &micro;l = (50 ng) / (100 ng/&micro;l) </em><em>= 0.5 &micro;l</em></p></li> <li>Record this volume in the table below. </li> </ol> </li> <li>Determine the volume of gene insert DNA to be used. <ol role = 'list' style='list-style-type: lower-alpha;' > <li><p>Based on the length and the concentration of the insert DNA, use the formula below to calculate the volume of sample needed to obtain a 3 molar excess of insert DNA to 50 ng of plasmid backbone DNA.</p> <p> Formula: volume needed, &micro;l = 3 x 50 ng x (length of insert, bp / length of plasmid backbone, bp) / <em> </em>(concentration of insert DNA, ng/&micro;l) </p> <p> <em><strong>Example:</strong> For insert DNA that is 750 bp in length and prepared at 25 ng/&micro;l, and will be assembled with the 5 kbp p2H&alpha; plasmid backbone, </p> <p>volume needed, &micro;l = 3 x 50 ng x (750 bp / 5000 bp) / (25 ng/&micro;l) = 0.9 &micro;l</em></p></li> <li>Record this volume in the table below. </li> </ol> </li> <li >Determine the volume of water to be used. <ol role = 'list' style='list-style-type: lower-alpha;'> <li >Sum the volume of plasmid backbone and insert DNA in the table below.<br> <strong> <b style='color:#b22222'> Important : </b> </strong>The sum volume should not exceed 5 &micro;l.</li> <li >Subtract this summed value from 5 &micro;l. This subtrated value is the volume of water to be used.</li> <li >Record this calculated volume for water in the table below. </li> </ol> </li> </ol> </li> <li > Set up the DNA ligation reaction. <ol role = 'list'> <li >Obtain and label a microcentrifuge o.conr PCR tube for each DNA ligation reaction to be setup.</li> <li >Using the information recorded in the table below, setup the reaction. <br><strong> <b style='color:#b22222'> Important : </b></strong> <p>Follow the order of addition as indicated in the table below, and use a new pipette tip for each addition.</p><table ><thead><tr><th>Reaction Component</th><th>Volume, &micro;l</th></tr></thead> <tbody> <tr> <td>Plasmid backbone DNA (50 ng)</td> <td aria-label='No value'> </td> </tr> <tr> <td>DNA insert</td> <td aria-label='No value'> </td> </tr> <tr> <td>Sterile, nuclease-free water</td> <td aria-label='No value'> </td> </tr> <tr> <td>2X Instant Sticky-end Ligase MasterMix </td> <td >5 &micro;l</td> </tr> <tr> <td>Total Volume</td> <td >10 &micro;l</td> </tr> </tbody> </table></p> </li> <li >Once all the components have been added, gently pipette up and down 4 - 5 times to mix.</li> <li > Incubate the reactions on the bench, at room temperature, for 10 min.</li> <li >Once the reaction is complete, place the reaction on ice. Then proceed to <a class='chap-linking'  tabindex='0' data-link-to='2-11'>Protocol 2.9 Transformation of Bacteria </a>, or store the reaction at -20 <b>°</b>C </li> </ol> </li> </ol> <p> <strong>Helpful tips:</strong> </p> <p>- It is not recommended to pipette volumes less than 1&micro;l. If the concentration of your insert or plasmid backbone sample requires less than 1 &micro;l be added, make an appropriate dilution of the sample. </p> <p><em><strong>Example:</strong> For plasmid backbone DNA at 100 ng/&micro;l,</em> </p> <p><em>volume needed, &micro;l (50 ng) / (100 ng/&micro;l) </em><em>= 0.5 &micro;l</em></p> <p><i>A 1:2 dilution can be made, mixing equal volumes of the plasmid backbone and sterile nuclease-free water, for a final DNA concentration of 50 ng/</i><em>&micro;l. </em></p> <p><em>volume needed, &micro;l = (50 ng) / (50 ng/&micro;l) </em><em>= 1 &micro;l</em></p>"},"2-11":{content:" <h3>Objective:</h3> <p> To introduce plasmid DNA into bacterial cells.</p> <h4>Rationale:</h4> <p>Transformation is the process of introducing a piece of DNA, typically a plasmid, into bacterial cells. Once transformed, bacterial cells store and replicate the plasmid as it does its own genetic material. In molecular cloning, plasmids that are assembled <em>in vitro</em>, for example by isothermal assembly or DNA ligation, are transformed into bacteria so that the plasmid can be replicated. Transformed bacteria, when grown to high densities, can therefore be used to generate large quantities of plasmid DNA, which can subsequently be extracted. Transformations are also an important technique for introducing specific genes into bacteria in order to study gene function.</p> <p >In chemical transformation, plasmid DNA is mixed with bacterial cells, and this mixture is briefly incubated at a high temperature. This 'heat shock' allows plasmid DNA to enter the cells. During a transformation reaction, not all bacterial cells will be transformed with DNA. However, those bacteria that have been transformed with plasmid DNA can readily be selected based on antibiotic resistance conferred by the plasmid. To select transformed bacteria, bacterial cells from the transformation reaction are allowed to grow for an hour such that transformed cells begin expressing the antibiotic resistance gene, and all cells are then plated on agar supplemented with the appropriate antibiotic. Only cells that are transformed will survive, forming colonies on the agar plate.</p> <h4>Supplies:</h4> <ul role = 'list'> <li>Plasmid DNA</li> <li>Ice</li> <li>Microcentrifuge tubes</li> <li>Microcentrifuge</li> <li>Recovery broth*</li> <li>LB agar plates supplemented with the appropriate antibiotics</li> <li>Rattler beads or spreader, with 100 % EtOH</li> <li>Shaking incubator set to 37 <b>°</b>C</li> </ul> <ul role = 'list'> <li>Chemical competent cells</li> <li>Heat block set to 42 <b>°</b>C.</li> </ul> <p>*LB or SOC for <em>E. coli </em>cells</p> </div> <h4>Procedure:</h4> <strong><b style='color:#b22222'> Important : </b></strong> Competent cells should be kept on ice throughout Steps A - C. </p> <ol role = 'list' style='list-style-type:upper-alpha;' > <li>Retrieve chemical competent cells from -80 <b>°</b>C and place on ice, immediately, to thaw. <ol role = 'list'> <li>For each transformation reaction, 25 &micro;l of competent cells will be needed. </li> </ol> </li> <li>Determine the volume of DNA to use for each reaction. <ol role = 'list'> <li>If transforming bacteria with purified plasmid, use 1 &micro;l of plasmid DNA.</li> <li>If transforming bacteria with the mix from an isothermal assembly or DNA ligation reaction, use 2 &micro;l of the reaction mix.<br> <strong> <b style='color:#b22222'> Important : </b></strong> The volume of DNA used in a chemical transformation should not exceed 10 % of the total transformation reaction volume, as it will decrease the efficiency of the transformation reaction. </li> </ol> </li> <li>Set up the transformation reaction. <ol role = 'list'> <li>For each tarsnformation reaction, obtain and label a clean microcentrifuge tube.</li> <li>Add the volume of DNA, as determine in Step B, to the microcentrifuge tube.</li> <li>Place the microcentrifuge tube with DNA on ice.</li> <li>Add 25 &micro;l of chemical competent cells from Step A to the microcentrifuge tube.</li> <li>Incubate the mix on ice for 20 min. </li> </ol> </li> <li>Heat shock cells to allow for DNA uptake. <ol role = 'list'> <li>Transfer the microcentrifuge tube from ice directly to the 42 <b>°</b>C heat block, and incubate at 42 <b>°</b>C for 30 sec.</li> <li>Transfer the microcentrifuge tube directly back onto ice, and incubate on ice for 3 min. </li> </ol> </li> <li>Allow cells to recover. <ol role = 'list'> <li>Using aseptic technique, add 1 ml of LB recovery broth to the microcentrifuge tube.</li> <li>Incubate the miccrocentrifuge tube at 37 <b>°</b>C, with shaking, for 1 hr. </li> </ol> </li> <li>Prepare the appropriate selection plate(s) <ol role = 'list'> <li>Consult the information provided for the appropriate plasmid to determine the antibiotic resistance conferred by the plasmid. Obtain two plates for each transformation reaction.</li> <li>Label the plates with your name, date, incubation temperature, and experimental information according to your instructor's direction. </li> </ol> </li> <li>Spread the transformation reaction onto selection plate(s). <ol role = 'list'> <li>For each transformation reaction: <ol role = 'list' style='list-style-type: lower-alpha;'> <li>Transfer 100 &micro;l of the recovery mix to one of the two selection plates.</li> <li>Spin the 900 &micro;l of recovery mix remaining in the microcentrifuge tubes in a microcentrifuge at 2,000 x g for 5 min.</li> <li>Remove 800 &micro;l of the supernatant fraction, being careful to not disturb the pellet. Discard the supernatant in an appropriate liquid waste container.</li> <li>Resuspend the pellet in the remaining liquid, approximately 100 &micro;l, by pipetting up and down to mix.</li> <li>Transfer the entire resuspension onto the second of the two selection plates.</li> </ol> </li> <li>Using Rattler beads or a glass spreader that has been decontaminated using EtOH, spread the mixture to even distribute the cells on the agar surface of each selection plate.</li> <li>With the lids closed, allow the plates to sit on the bench until all the liquid has been fully absorbed into the agar.</li> <li>Invert the plates and incubate the plates at the temperature suitable for propagation of the plasmid.</li> </ol> </li> </ol> <p> <strong>Helpful tips:</strong> </p>"},"2-12":{content:"<h3>Objective:</h3> <p>To determine if an assembled plasmid contains insert DNA of the correct size. </p> <h4>Rationale:</h4><p> When plasmids are assembled, either by isothermal assembly or DNA ligation, and transformed into bacteria, it is important to verify that the assembly reaction occured as intended and that the assembled plasmid contains the correct insert. A strategy that is routinely employed for doing so involves verifying that the DNA insert is of the correct size. This can be achieved by amplifying the region of the plasmid that contains the insert by PCR, and analyzing the size of that amplified DNA by gel electrophoresis. To save on time and reagents, the plasmid within bacterial cells do not need to be purified for use as template DNA in a PCR reaction. Instead, whole bacterial cells containing the plasmid can be used directly in the PCR reaction. This is known as 'colony PCR'. </p> <h4>Supplies:</h4> <p>For colony PCR:</p> <ul role = 'list'> <li><em>E. coli </em>colonies containing the plasmid to be analysed</li> <li>PCR tubes</li> <li>2X One Taq Polymerase Master Mix</li> <li>DNA Primers (10 μM) </li> <li> Ice </li> <li> Sterile, nuclease-free water </li> <li> Thermocycler </li> </ul> <p> For gel electrophoresis </p> <ul role = 'list'> <li>Pre-poured agarose gel</li> <li>DNA loading dye (6X)</li> <li>Electrophoresis apparatus and power supply</li> <li>DNA samples</li> <li>DNA ladder</li> <li>Gel imaging equipment</li> </ul> </div> <h4>Procedure:</h4> <p > <strong>Note:</strong> This is not a new protocol, but instead a combination of two protocols: <a class='chap-linking'  tabindex='0' data-link-to='2-4'>Protocol 2.2: Polymerase Chain Reaction</a>  and <a class='chap-linking'  tabindex='0' data-link-to='2-5'> Protocol 2.3: Gel Eelctrophoresis </a>. </p> <ol role = 'list' style='list-style-type: upper-alpha;' > <li> Observe your transformation plate and determine which colonies to test. <ol role = 'list'> <li> On the bottom of the plate, circle and label colonies to be tested. Typically, 3 - 4 colonies from a given transformation reaction are tested. <br> <strong><b style='color:#b22222'> Important : </b></strong> Select well-isolated colonies to minimize cross-contamination. </li> </ol> </li> <li>Set up the colony PCR. <ol role = 'list'> <li>Prepare template DNA. <ol role = 'list' style='list-style-type: lower-alpha;'> <li>Obtain and label a new microcentrifuge tube for each colony to be tested.</li> <li>Add 100 &micro;l of water into each microcentrifuge tube.</li> <li>Gently touch a sterile pipette tip to a colony to be tested, removing only as much as a quarter of the colony. <br> <strong><b style='color:#b22222'> Important : </b> </strong>Do not transfer the entire colony.</li> <li>Swirl the pipette tip in the water within the microcentrifuge tube to transfer the bacterials cells. This is your template DNA.</li> </ol> </li> <li>Perform the PCR. <ol role = 'list'> <li>Consult the information for the appropriate plasmid to determine which primers and DNA polymerase to use for clone verification.</li> <li>Follow instructions in <a class='chap-linking'  tabindex='0' data-link-to='2-4'> Protocol 2.2: Polymerase Chain Reaction. </a> </li> </ol> </li> </ol> </li> <li>Determine the size of PCR amplified DNA. <ol role = 'list'> <li>Follow instructions in <a class='chap-linking'  tabindex='0' data-link-to='2-5'>Protocol 2.3: Gel Electrophoresis  </a>to determine the size of products for each colony PCR reaction.</li> <li>Determine which colonies contain plasmid DNA with an insert of the correct size.</li> <li>Record this information in your notebook and in a Clone Verification Data Card.</li> <li>To prepare large quantities of plasmid with an insert of the correct size, proceed to <a class='chap-linking'  tabindex='0' data-link-to='2-13'>Protocol 2.11: Inoculating a Culture. </a></li> </ol> </li> </ol> <p> <strong>Helpful tips:</strong> </p> "},"2-13":{content:" <h3>Objective:</h3><p> To inoculate a liquid culture from a single bacterial colony. </p> <h4>Rationale:</h4><p> Culturing bacteria is a routine and important experimental step during molecular cloning. As a means to obtain large quantities of a given plasmid DNA, bacteria transformed with that plasmid DNA are cultured to high density, before being harvested and the plasmids extracted. To ensure that all bacteria in the culture maintain and propogate copies of the plasmid DNA, the antibiotics for which the plasmid confers resistance to is added the the culture. Culturing bacteria is also an important step for many microbiological experiments. </p> <h4>Supplies:</h4> <ul role = 'list'> <li> Sterile test tubes with lids </li> <li>Liquid media supplemented with the appropriate antibiotic*</li> <li>Sterile innoculation or wooden sticks</li> <li>Temperature-controlled shaking incubator or rotor</li> <li>5 ml serological pipettes</li> <li>Pipette aid</li> </ul> <p >* <em>E. coli</em> should be cultured in LB liquid medium. Consult your instructor and the Plasmid Information Sheet to determine which antibiotic to use. <h4>Procedure:</h4> <ol role = 'list' style='list-style-type:upper-alpha;' > <li> For each culture to be inoculated, obtain and label a test tube with a lid. </li> <li> Using good aseptic technique and working close to a flame, with a serological pipette, transfer 5 ml of the appropriate liquid medium that has been supplemented with antibiotic to each test tube. </li> <li> For each culture to be inoculated, identify a well-isolated bacterial colony. On the bottom of the plate, circle and label the colony. </li> <li> Inoculate a culture. <br> <strong> <b style='color:#b22222'> Important : </b> </strong>Use aseptic technique, and work close to a flame, throughout Step D. <ol role = 'list'> <li> Using a sterile inoculation or wooden stick, gently touch the selected colony. </li> <li> Remove the lid of the test tube, and swirl the tip of the wooden stick that touched the bacterial colony in the liquid medium. </li> <li> Replace the lid of the test tube. </li> </ol> </li> <li>Incubate the test tube, with shaking or rotation, at the appropriate temperature for 16 - 24 hr. <ol role = 'list'> <li>Consult the information for the appropriate plasmid to determine the growth temperature.</li> </ol> </li> </ol> <p><strong>Helpful tips:</strong> </p>"},"2-14":{content:"<h3>Objective:</h3> <p> To prepare a stock of plasmid DNA for use in downstream experiments. </p> <h4>Rationale:</h4> <p>As a means to obtain large quantities of a given plasmid, bacteria transformed with that plasmid are cultured to high density, from which the plasmid can be extracted and purified. A routine strategy for doing so is called alkaline lysis. Here, NaOH serves multiple purposes, including lysis of bacteria and denaturation of double-stranded DNA. When the alkaline solution is neutralized, small plasmid DNA readily renatures into double-stranded DNA, whereas large DNA molecules such as the bacterial chromosomal DNA becomes a tangled and insoluble aggregate. The renatured plasmid DNA can then be selectively bound to a silica column and purified away from contaminating salts and proteins. Once 'washed', the plasmid DNA can be eluted from the silica column with water. </p> </div> <h4>Supplies:</h4> <ul role = 'list'> <li> High-density <em>E. coli</em> liquid culture </li> <li> Microcentrifuge tubes </li> <li> Microcentrifuge </li> <li> Plasmid mini-prep Kit (Zymo) </li> <li> Sterile, nuclease-free water </li> </ul> <h4>Procedure:</h4> <p><strong><b style='color:#b22222'> Important : </b></strong>Gloves should always be worn when doing molecular cloning.</p> <ol role = 'list' style='list-style-type:upper-alpha;'> <li> For each plasmid to be purified, obtain and label a microcentrifuge tube. </li> <li> Transfer 1 ml of the bacterial culture containing the plasmid to be purified into the microcentrifuge tube. </li> <li> Lyse the bacterial culture. <ol role = 'list'> <li> Centrifuge the tubes in a microcentrifuge at maximum speed for 5 min. <br><span style='color:#b22222'> Important : </span>Ensure that the centrifuge is properly balanced. </li> <li> Remove and discard the supernatant, being careful not to disturb the pellet. </li> <li> Recommended : For a higher yield, transfer an additional 1 ml of culture to the same tube and repeat Steps C1 - 2. </li> <li> Resuspend the cell pellet in 600 &micro;l of water. </li> <li> Add 100 &micro;l of 7X Lysis Buffer and carefully close the tube. The 7X Lysis Buffer is a viscous solution and contains a blue pH indicator. </li> <li> Invert the tube 3 - 4 times or until the solution is uniformly blue.</br> <span style='color:#b22222'> Important : </span> Do not shake the tube too vigorously as this can shear the chromosomal DNA into smaller fragments that cannot be easily separated from plasmid DNA. </li> </ol> </li> <li>Neutralize the lysed culture. <ol role = 'list'> <li>Add 350 &micro;l of chilled Neutralization Buffer. </li> <li> Invert the tube 3 - 4 times to mix or until the solution turns uniformly yellow, indicating complete neutralization. A precipitate will form.</br> <span style='color:#b22222'> Important : </span> Do not shake the tube too vigorously as this can shear the chromosomal DNA into smaller fragments that cannot be easily separated from plasmid DNA. </li> <li> Centrifuge at maximum speed for 4 min.</br> <span style='color:#b22222'> Important : </span> Ensure that the centrifuge is properly balanced. </li> </ol> </li> <li>Bind plasmid DNA to the column. <ol role = 'list'> <li> Label a spin column according to your DNA sample, and place the spin column in a collection tube. </li> <li> Being careful not to disturb the large pellet, transfer the supernatant into the spin column. There should be ~ 900 &micro;l. </li> <li <p > Centrifuge at the maximum speed for 30 sec. </p> </li> <li <p > Remove the spin column and discard the flow-through that accumulated in the collection tube into an appropriate liquid waste container. Then place the spin column back onto the same collection tube, and proceed to the next step. </p> </li> </ol> </li> <li <p > Wash the DNA now bound within the column. </p> <ol role = 'list'> <li > Add 200 &micro;l of Endo-Wash Buffer to the column </li> <li > Centrifuge at the maximum speed for 30 sec. </li> <li > Remove the spin column and discard the flow-through from the collection tube. Then place the spin column back onto the same collection tube. </li> <li > Add 400 &micro;l of Zyppy-Wash Buffer to the column. </li> <li > Centrifuge at the maximum speed for 30 sec. </li> <li > Remove the spin column and discard the flow-through from the collection tube. Then place the spin column back onto the same collection tube. </li> </ol> </li> <li > Elute the DNA <ol role = 'list'> <li > Transfer the spin column to a new and labelled microcentrifuge tube. </li> <li > Add 30 &micro;l of sterile water to the spin column.</br> <span style='color:#b22222'> Important : </span> To make sure that the water soaks the column matrix, pipette the water directly onto the 'white' matrix in the center of the column. </li> <li > Incubate at room temperature for 1 min. </li> <li > Centrifuge at the maximum speed for 30 sec. </li> <li > The flow-through in the microcentrifuge tube is your eluted and purified DNA. This is your plasmid DNA stock. </li> </ol> </li> <li > Measure the concentration of your DNA. <ol role = 'list'> <li > Follow your instructor's directions to measure the concentration of your purified DNA. </li> <li >Record the concentration of DNA, in ng/&micro;l, on the side of the microcentrifuge tube and in your notebook.</li> <li > Take note of the A260/A280 (DNA/protein) ratio and A260/A230 (DNA/organic compounds) ratio for your purified DNA. A 'pure' DNA sample should have a A260/280 ratio of ~1.8 and an A260/A230 ratio of 2.0 - 2.2. </li> </ol> </li> <li> Plasmid DNA should be stored at -20 <b>°</b>C</li> </ol> </div> <p> <strong>Helpful tips:</strong> </p> "},"3-1":{content:"<p>Chapter 2 guided you through the process of cloning a phage gene of interest into the plasmid, pExTra. In this chapter, the recombinant plasmid you built and purified will serve as an important tool in your research to investigate the function of this gene. Through the process of transformation by electroporation (Protocol 3.1), this plasmid will be introduced into <i>M. smegmatis</i>, the bacterial host of the mycobacteriophages encoding our genes of interest. pExTra was specifically engineered to enable the controlled expression of our phage genes in <i>M. smegmatis</i>, allowing us to ask the question: what happens when a particular phage gene is expressed in the host cell? The combined efforts of you and your fellow researchers will enable investigation of this research question on a genome-wide scale.</p> <h3> From genes to phenotype </h3> <p>As in all organisms, the expression of genes in bacteria, whether they are found on the bacterial chromosome, on a plasmid, or on an infecting phage genome, follows the central dogma: RNA polymerase transcribes the DNA gene sequence into an RNA molecule which is then translated into a peptide chain of amino acids by the ribosome. This amino acid sequence dictates the 3D structure that a peptide chain adopts; this in turn dictates that protein’s function, or the work that it performs in a cell. Together, the coordinated activities of the various proteins that are produced by an organism, determine the observable traits and behavior, or phenotype of that organism. In short, an organism’s phenotype reflects the set of genes being expressed. As the set of genes being expressed in an organism changes, so can the phenotype. For example, during phage infection of a bacterial cell, the expression of phage genes often changes the phenotype of the infected bacterial cell.</p> <h3> Manipulating gene expression to study phage gene function </h3> <p>The connection between gene expression and observable traits is not only a fundamental tenet of biology; it is also a valuable research tool in the quest to understand the function of a gene. By manipulating gene expression in some way and looking for an observable change in the organism, we can begin to associate the gene with processes it is involved in and thereby gather clues as to its function. The expression of a gene can be altered in various ways, either by turning it off completely, turning it on when it would not normally be expressed, or increasing or decreasing the amount of expression. In our investigation, we will express a given mycobacteriophage gene from the pExTra plasmid in the host cell and observe if and how it alters the host phenotype. To some extent, this approach<i> </i>mimics a typical phage infection of a mycobacterial cell, though with some key differences; for example, other phage gene products that may regulate or contribute to the function of our gene of interest will not be present in the cell. We know that many phage gene products manipulate host processes in some way in order to promote phage survival and propagation; consequently, we expect to observe some of the phenotypes that are typically observed during a phage infection, including growth inhibition of the host cell. Expression of the phage gene by itself allows us to ascribe any phenotypic change to the specific phage gene being studied</p> <p>For controlled expression of the phage gene, and for careful assessment of gene expression, the pExTra plasmid was engineered with two features:<br> 1. a promoter that allows gene expression to be turned on and off, and<br> 2. a reporter gene that indicates when expression is occurring.</p> <h3> 1. The pExTra inducible promoter </h3> <p>A promoter is a DNA sequence that controls transcription of the gene(s) immediately downstream. Bacterial promoters are typically composed of two 6 bp long sequences, referred to as the -10 and -35 sites, to which RNA polymerase binds to initiate gene transcription. If RNA polymerase can access a given promoter at all times, and thereby initiate transcription continuously, the promoter is said to be constitutively ‘on’. However, many promoters are regulated by additional factors and are only ‘on’ under specific conditions. Such promoters are known as inducible promoters and can be either negatively or positively regulated.</p> <p>In a negatively-regulated promoter, a repressor protein binds to a DNA sequence near or within the promoter DNA, known as an operator site. The bound repressor prevents access to the promoter by RNA polymerase, thereby maintaining the promoter in the ‘off’ state. pExTra plasmids contain a Tetracycline-inducible or ‘Tet’ promoter upstream of the phage gene sequence. This promoter is maintained in the “off” state by the tetracycline-repressor protein, TetR, and can be turned “on” by the addition of the molecule tetracycline. Tetracycline binds TetR, causing a conformational change that releases the repressor from the DNA, thereby enabling access to the promoter by RNA polymerase. Because tetracycline turns “on” the promoter and thereby “induces” gene expression, tetracycline is known as an inducer. In our experiments, we will use a chemically modified form of tetracycline as the inducer, known as anhydrotetracycline, aTc. As illustrated in the plasmid map in Protocol 2.1a, the gene encoding TetR is also found on pExTra, and the repressor is constitutively produced, ensuring that phage gene expression is tightly repressed until the aTc inducer is introduced into our experimental system.</p> <p>As a side note, the plasmid that we will employ in Chapter 4 contains a different negatively-regulated promoter, the ‘Lac*’ promoter, which is repressed by the transcriptional regulator LacI and induced by the chemical Isopropyl b-D-1-thiogalactopyranoside (IPTG). We will also utilize a positively regulated promoter in Chapter 4, in which ‘activator’ proteins recruit RNA polymerase to a promoter to initiate transcription. This positively-regulated promoter will be described in Chapter 4.</p> <h3> 2. The pExTra reporter gene </h3> <p>When we induce expression of a gene in order to investigate its effects, it is important to confirm that the gene is indeed being expressed. This is particularly important when the expression of a gene does not result in any observable change in phenotype. Typically, gene expression is confirmed by monitoring the levels of protein that accumulate in the cell by a technique known as Western blotting. This approach, however, is cumbersome and not suitable for the scope of our research, in which very many different genes will be expressed. Instead, a convenient strategy to monitor expression is by using a second gene, under the control of the same promoter, that produces an easily observable phenotype when expressed. Such genes are known as reporter genes, as they act to “report” on promoter activity. Commonly used reporter genes include genes encoding for fluorescent proteins that can change the color of the cell, or antibiotic resistance genes that allow cells to grow in the presence of antibiotics.</p> <p>To monitor the expression of a phage gene, the pExTra plasmid has been engineered to contain a reporter gene encoding for the red fluorescent protein mCherry downstream of the phage gene. If transcription from the Tet promoter occurs, both genes will be co-transcribed as a single mRNA but will be translated as two separate proteins—the phage gene product and mCherry. Accumulation of the mCherry protein turns cells pink, providing a clear readout of whether or not the transcript is being produced and thus providing some indication as to whether or not the phage gene product of interest is being produced.</p> <h3> Assessing the effects of phage gene expression on host phenotype </h3> <p>The pExTra expression system allows us to both control and monitor expression of a particular phage gene in its native host cell environment, but how can we detect and measure a gene-mediated change in host phenotype? This requires the use of a phenotypic assay, or an experiment designed to evaluate a specific host characteristic. In SEA-GENES research, we will be performing two phenotypic assays on <i>M. smegmatis</i> transformed with a recombinant pExTra plasmid:<br> 1. a Cytotoxicity Assay, and <br> 2. a Superinfection Immunity Assay.</p> <h3> 1. Cytotoxicity Assay </h3> <p>The first phenotypic assay is a cytotoxicity assay, which measures the effect of phage gene expression on the ability of <i>M. smegmatis</i> to grow; a gene product that slows or prevents bacterial growth is said to be ‘toxic’ and likely interferes with some essential cellular process. Identifying toxic phage gene products is not only important for our understanding of how phages interact with their bacterial hosts but is also a key step in the scientific endeavor to develop new antibacterial therapeutics. We already know of some phage gene products that are toxic to bacteria, for example, the holin and lysin proteins that work in concert to lyse the cell; however, within the vast unexplored phage sequence space, there are certain to be many novel mechanisms by which phage gene products disrupt cellular function.</p> <h3> 2. Defense Assay </h3> <p>The second phenotypic assay is a defense test which determines whether expression of a phage gene in <i>M. smegmatis</i> can protect the cell from infection by the phage that encodes the gene. An example of defense, called superinfection immunity, typically occurs through the same mechanism that enables a phage to form a lysogen. During lysogeny, the phage immunity repressor gene is expressed, and the immunity repressor proteins bind to and represses expression of phage genes required for the lytic cycle. When a second or similar phage infects the lysogen, the repressor proteins in the cell can similarly repress expression from the second phage, thereby providing “immunity” to the second infection. Many immunity repressors can confer this immunity when expressed by themselves without other phage-encoded functions. Genes encoding for immunity repressor proteins have not yet been identified in many temperate mycobacteriophage genomes, and even those that have been bioinformatically predicted are poorly characterized. Performing a defense assay provides important experimental validation of this prediction and can be used to inform and improve genome annotation methods. Furthermore, not only immunity repressors are capable of protecting bacterial hosts from infection; other phage gene products have been shown to prevent infection by closely related phages by a variety of mechanisms, including blocking the phage receptor or inhibiting DNA entry. Thus, this phenotypic assay may reveal novel genes involved in such phage-phage interactions. </p> <p>The recombinant pExTra plasmid you have constructed could be used in any number of phenotypic assays to observe gene-mediated effects on different host processes. Key to the interpretation of results from any of these assays, though, is the inclusion of appropriate experimental controls; it is essential to compare the behavior of <i>M. smegmatis</i> cells transformed with the pExTra plasmid of interest in these assays in the presence or absence of inducer, in order to draw any conclusions as to whether phage gene expression is responsible for an observed effect.</p> <h3> Genome-wide analysis of gene-mediated phenotypic effects </h3> <p>Combined, the recombinant pExTra plasmids built by you and your fellow researchers form a gene library, or a collection of individual genes cloned into an expression plasmid, that represents most or all of the genetic material in a particular phage genome. Together, you will perform phenotypic assays on the genes in this library, thereby screening the phage genome to identify the small subset of genes that have particular effects on the host phenotype. This collective analysis provides a genome-wide view of phage gene function, and as more and more of these phage gene libraries are built and screened, we can also begin to make comparisons between related genes within the broader phage population. Doing research at scale increases the chances of discovering promising leads to pursue in future research studies while using only a few experimental methods. </p> <p>Gaining a thorough understanding of how a gene functions in the context of a phage infection, is a large undertaking, requiring the integration of ample data collected by many different methods. The data generated by you and your fellow researchers from the phenotypic assays outlined in this chapter provide important experimental observations that can lead to new hypotheses and get us a few steps closer to better understanding how these genes function in nature!</p> "},"3-2":{content:"<h3>Objective:</h3> <p>To introduce plasmid DNA into bacterial cells.</p> <h4>Rationale: </h4> <p>Transformation is the process of introducing a piece of DNA, typically a plasmid, into bacterial cells. Once transformed, bacterial cells store and replicate the plasmid as it does its own genetic material. In molecular cloning, plasmids that are assembled in vitro, for example by isothermal assembly or DNA ligation, are transformed into bacteria so that the plasmid can be replicated. Transformed bacteria, when grown to high densities, can therefore be used to generate large quantities of plasmid DNA, which can subsequently be extracted. Transformations are also an important technique for introducing specific genes into bacteria in order to study gene function. </p> <p> In electroporation, plasmid DNA is mixed with bacterial cells. This mixture is then placed in a special container, called an electroporation cuvette, before being pulsed with an electric current. This electric shock allows plasmid DNA to enter cells. When compared to chemical transformations, transformation by electroporation occurs at a much higher efficiency. </p> <p> During a transformation reaction, not all bacterial cells will be transformed with DNA. However, those bacteria that have been transformed with plasmid DNA can readily be selected based on antibiotic resistance conferred by the plasmid. To select transformed bacteria, bacterial cells from the transformation reaction are allowed to grow for an hour such that transformed cells begin expressing the antibiotic resistance gene, and all cells are then plated on agar supplemented with the appropriate antibiotic. Only cells that are transformed will survive, forming colonies on the agar plate. </p> <p> <h4>Supplies:</h4></p> <ul role = 'list'> <li>Plasmid DNA</li> <li>Ice</li> <li>Microcentrifuge tubes</li> <li>Microcentrifuge</li> <li>Recovery broth (7H9 neat)</li> <li>7H10 agar plates supplemented with 20 μg/ml Kanamycin </li> <li>Rattler beads or spreader, with 100 % EtOH</li> <li>Shaking incubator set to 37 <b>°</b>C.</li> <li>Electrocompetent cells</li> <li>Electroporation cuvettes (gap: 0.1 cm)</li> <li>Electroporator (voltage: 1.8 kV)</li> <li>Paper towel or KimWipe</li> </ul> <h4>Procedure :</h4> <strong> <b style='color:#b22222'> Important : </b> </strong> Competent cells should be kept on ice throughout Steps A - B.</p> <ol role = 'list' style='list-style-type:upper-alpha;'> <li>Retrieve electrocompetent cells from -80 <b>°</b>C and place on ice, immediately, to thaw. <ol role = 'list'> <li>For each transformation reaction, 25 - 50 &micro;l of competent cells will be needed. Consult your instructor for the appropriate volume of cells to use. </li> </ol> </li> <li>Set up the transformation reaction. <ol role = 'list'> <li>For each transformation reaction, obtain and label a clean microcentrifuge tube and a electroporation cuvette. Place both vessels on ice. <ol role = 'list' start='1' style='list-style-type: lower-alpha;'> <li>Add 1 &micro;l of purified plasmid DNA to the microcentrifuge tube.<br> <strong> <b style='color:#b22222'> Important : </b> </strong> Do not used unpurified DNA. The presence of salts from reactions such as isothermal assembly or DNA ligation can result in overheating and electric arcing when a current is applied, which will kill the bacterial cells.</li> </ol> </li> <li>Add the appropriate volume of electrocompetent cells from Step A to the microcentrifuge tube.</li> <li>Pipette the mixture up and down gently, one time.<br> <strong> <b style='color:#b22222'> Important : </b> </strong> Act quickly to ensure that the mixture remains cold.</li> <li>Transfer the mixture into the pre-chilled electroporation cuvette.<br> <strong> <b style='color:#b22222'> Important : </b> </strong> Pipette the mixture into the thin gap between two metal electrodes at the bottom of the cuvette. Gently tap the cuvette on the bench to evenly settle the mixture at the bottom of the cuvette.<br> <strong> <b style='color:#b22222'> Important : </b> </strong> Be careful not to introduce any air bubbles in the mixture.</li> </ol> </li> <li>Electroporate the cells to allow for DNA uptake. <ol role = 'list'> <li>Adjust the settings on the electroporator according to your instructor's directions.</li> <li>Remove the cuvette from the ice, and using a paper towel, wipe off any moisture on the outside of the cuvette. Proceed quickly and carefully through the remaining steps.</li> <li>Place the cuvette into the electroporator holder.</li> <li>Pulse the cells, taking note of the pulse length value. It should be between 4 - 6 msec. <ol role = 'list' start='1' style='list-style-type: lower-alpha;'> <li>If a flash of light is seen or a loud popping sound is heard during the pulse, an electric arc has occured due to contaminating salts in the mixture. Repeat Procedure 2 from the begining, this time using DNA that has been diluted in water to reduce the salt concentration in the final mixture.</li> </ol> </li> </ol> </li> <li>Allow cells to recover. <ol role = 'list'> <li>Using aseptic technique, add 1 ml of the 7H9 recovery broth to the electroporation cuvette, then transfer the entire volume from the elctroporation cuvette into a clean and labelled microcentrifuge tube.</li> <li>Incubate the miccrocentrifuge tube at 37 <b>°</b>C, with shaking for 2 hr.</li> </ol> </li> <li>Prepare the appropriate selection plate(s) <ol role = 'list'> <li> Obtain two 7H10 agar plates supplemented with 20 μg/ml Kanamycin for each transformation reaction.</li> <li>Label the plates with your name, date, incubation temperature, and experimental information according to your instructor's direction.  </li> </ol> </li> <li>Spread the transformation reaction onto selection plate(s). <ol role = 'list'> <li>For each transformation reaction: <ol role = 'list' start='1' style='list-style-type: lower-alpha;'> <li >Transfer 100 &micro;l of the recovery mix to one of the two selection plates.</li> <li >Spin the 900 &micro;l of recovery mix remaining in the microcentrifuge tubes in a microcentrifuge at 2,000 x g for 5 min.</li> <li >Remove 800 &micro;l of the supernatant fraction, being careful to not disturb the pellet. Discard the supernatant in an appropriate liquid waste container.</li> <li >Resuspend the pellet in the remaining liquid, approximately 100 &micro;l, by pipetting up and down to mix.</li> <li >Transfer the entire resuspension onto the second of the two selection plates.</li> </ol> </li> <li >Using Rattler beads or a glass spreader that has been decontaminated using EtOH, spread the mixture to even distribute the cells on the agar surface of each selection plate.</li> <li >With the lids closed, allow the plates to sit on the bench until all the liquid has been fully absorbed into the agar.</li> <li >Invert the plates and incubate the plates at the temperature suitable for propagation of the plasmid.</li> </ol> </li> </ol> <p><strong>Helpful tips:</strong></p> "},"3-3":{content:"<h3>Objective:</h3> <p>To determine whether expression of a given mycobacteriophage gene is toxic to the bacterial host<em>, M. smegmatis.</em></p> <h4>Rationale: </h4><p >To assess if expression of a particular phage gene has an effect on the growth of <em>M. smegmatis</em>, cells transformed with the appropriate pExTra recombinant plasmid are serially diluted and spotted onto solid growth medium containing antibiotic, to maintain selection of the plasmid, and the inducer molecule anhydro-tetracycline (aTc), to induce expression of the gene. The same dilutions are also spotted on plates lacking aTc, an essential control that allows changes in growth phenotype to be identified. Expression of non-toxic genes will cause no change in the amount of bacterial growth, whereas expression of toxic genes will result in less or no growth as compared to the control plate, or may result in smaller colonies. Plates should be observed over the course of several days and colony color should be monitored. Pink colony color is an indication that the phage gene is being transcribed. </p> <h4>Supplies:</h4> <ul role = 'list'> <li><em>M. smegmatis</em> transformed with recombinant pExTra plasmid(s) encoding gene(s) of interest</li> <li><em>M. smegmatis</em> transformed with pExTra02 plasmid encoding positive control gene</li> <li><em>M. smegmatis</em> transformed with pExTra03 plasmid encoding negative control gene</li> <li>Microcentrifuge tubes</li> <li>7H9 liquid medium</li> <li>7H10 Agar plates supplemented with Kanamycin (20 μg/ml) and aTc (10 ng/ml and 100 ng/ml)</li> <li>Sterile wooden sticks</li> <li>Incubator set to 37 <b>°</b>C</li> <li>Vortex</li> <li>Image documentation equipment</li> <li>A printed plate template, available for download <a href = 'https://dzf8vqv24eqhg.cloudfront.net/userfiles/11001/22461/ckfinder/files/Cytotoxicity%20Plate_Template.pdf' target = '_blank' >here </a>.</li> </ul> </div> <h4>Procedure:</h4> <ol role = 'list' style='list-style-type:upper-alpha;'> <li>Identify colonies for testing. <ol role = 'list'> <li>Observe your transformation plate(s) containing <em>M. smegmatis</em> transformed with your gene(s) of interest after incubation for 4-5 days at 37 ºC . </li> <li>For each gene being tested, identify 3 well-isolated colonies.</li> <li>On the bottom of the plate, circle and label the 3 colonies to be tested. <br> <b> Note:</b> You will also need one positive control (pExTra02) and one negative control (pExTra03) colony, which should be obtained from your instructor. </li> </ol> </li> <li>Prepare colonies for testing. <ol role = 'list'> <li>Obtain and label a clean microcentrifuge tube for each of the 3 colonies to be tested, as well as for the positive and negative controls.</li> <li>Fill each labelled microcentrifuge tube with 500 &micro;l 7H9 liquid medium.</li> <li>Using a sterile wooden stick, transfer each selected colony into the corresponding microcentrifuge tube. Shake the stick so transfer as much of the colony on the stick into the liquid medium.</li> <li>Cap the microcentrifuge tube and vortex to disperse the colony. This is your undiluted cell suspension. <br> <b> Note: </b> <em>M. smegmatis </em>cells should be properly suspended in the liquid medium before proceeding to the next step. Suspensions may be clumpy, even after vortexing. <br> <b><b style='color:#b22222'> Important : </b> </b>Cells can settle in the tube when left on the bench, even after just a few minutes. Throughout the process of diluting and spotting your samples, it is critical to regularly mix the cell suspensions by pipetting up and down or vortexing for 1-2 seconds. </li> </ol> </li> <li>Set up 10-fold serial dilutions (Figure 3.2a). <ol role = 'list'> <li>For each of the cell suspensions, obtain 5 additional microcentriuge tubes, arrange these microcentrifuge tubes in a rack, and label them 10<sup>-1</sup>, 10<sup>-2</sup>, 10<sup>-3</sup>, 10<sup>-4</sup>, and 10<sup>-5</sup>.</li> <li >Add 900 &micro;l of 7H9 liquid medium to each of the tubes</li> <li>Perform Steps C 4 - 7 for each of your undiluted cell suspensions.</li> <li>Transfer 100 &micro;l of your undiluted cell suspension to the “10<sup>-1</sup>” tube. <br> <b><b style='color:#b22222'> Important : </b></b> Make sure that cell suspension is well mixed before transferring. Try to avoid transferring very large clumps of cells that were not properly suspended.</li> <li>Cap the tube and vortex briefly to mix. <ol role = 'list' > <li><b>Note:</b> This solution in this “10<sup>-1</sup>” tube contains 1/10<sup>th</sup> the number of cells as your undiluted sample. It is also referred to as a 1:10 dilution.</li> </ol> </li> <li>Using a new pipette tip, transfer 100 &micro;l of the “10 <sup>-1</sup>” sample to the “10<sup>-2</sup>” tube and vortex well. <ol role = 'list' > <li>This solution contains 1/100<sup>th</sup> as many cells as your undiluted sample. It can also be referred to as your 1:100 dilution.</li> </ol> </li> <li>Continue each successive dilution until you get to your last tube, using a fresh tip for each dilution.</li> </ol> </li> <div class= 'img-wrap '> <div class= 'img-container '> <img src= './assets/images/figure-3-2-1.jpg ' alt= 'An illustration of 6 partially-filled tubes, each containing 900 microliters, with a progressively increasing lighter color in each from left to right. The first tube on the left is labeled undiluted cell suspension and the rest are labeled 10 to the negative one through 10 to the negative 5. An arrow labeled 100  microliters is drawn between each tube.'> </div> <div class='img-hr'></div><div class= 'title-container '>  Figure 3.2a. Serial dilutions. A volume of the undiluted cell suspension (100 &micro;l) is transferred into 900 &micro;l of 7H9 liquid media in the 10-1 tube. After the tube is mixed, 100 &micro;l of the 10-1 sample is transferred into the 10-2 tube, and the tube is mixed. This serial dilution continues until the desired dilutions are obtained. The last tube will contain a final volume of 1000 &micro;l. </div></div> <li>Spot dilutions and controls on inducer plates. <ol role = 'list'> <li>Obtain the following 3 plates plate for each gene being tested: <ol role = 'list' style='list-style-type:lower-alpha;'> <li>one 'no inducer' plate,</li> <li>one 'aTc<sup>10</sup> inducer plate', and</li> <li>one 'aTc<sup>100</sup> incuder plate'.</li> </ol> </li> <li >Label the bottom edge of each agar plate with your name, the date, inducer (aTc) concentration, and the gene being tested.</li> <li >Place the agar plate on a printed copy of the plate template. <br> <b> Note: </b>The printable plate template is a mirror image of Figure 3.2b. This is to ensure that, when viewed from the bottom of the plate, the negative control is placed in the leftmost column, as in Figure 3.2b.</li> <li > One at a time, aseptically transfer 5 &micro;l of each dilution onto the appropriate position on each of the three agar plates, using a new pipette tip for each spot.<br><b> <b style='color:#b22222'> Important : </b> </b> To ensure that samples are well mixed, pipette up and down a few times before transferring each sample to the plate.<br> <b> Note: </b> The pattern of spotting is identical on each of the 3 plates.</p> </li> </ol> </ol> <ol role = 'list'> <li >Allow the plates to sit on the bench for approximately 30 min for the liquid to be absorbed into the agar.</li> <li >Once the liquid has been absorbed, invert the plates and incubate at 37 <b>°</b>C for a total of 5 days. </li> <div class= 'img-wrap '> <div class= 'img-container '> <img src= './assets/images/figure-3-2-2.jpg ' alt= ' A circle circumscribing a 5 by 5 grid. The rows from top to bottom are labeled: undiluted cell suspension, 10 to the negative 1 through 10 to the negative 5. The columns are labeled negative control, positive control, replicate number 1, replicate number 2, and replicate number 3'> </div><div class='img-hr'></div><div class= 'title-container '>  Figure 3.2b. The printable 'plate template' serves as a guide for spotting the samples on the agar plate, such that when plates are viewed from the bottom, the samples are distributed as above. </div></div> <li>Observe for cytotoxicity. <ol role = 'list'> <li>Plates should be observed for phenotypes on days 2 or 3 and on days 4 or 5 of incubation. <br> <b> Note: </b> All colonies should grow on the 'no inducer' plate, and may or may not grow in the presence of aTc inducer.</li> <li>Photograph your plates, following your instructor’s directions. </li> </ol> </li> </ol> <p><strong>Helpful tips:</strong> </p> <div >When observing plates for cytotoxicity phenotypes, in addition to comparing the growth of the experimental samples on the three concentrations of aTc, it is also helpful to compare their growth to that of the pExTra03 control. If the pExTra03 control exhibits signs of cytotoxicity on aTc, it suggests there may be a technical issue and that the experiment should be repeated!</div> "},"3-4":{content:"<h3>Objective:</h3> <p> To determine whether expression of a phage gene protects the bacterial host, <em>M. smegmatis</em> from infection by phages. </p> <h4>Rationale:</h4> <p> During an infection, the expression of certain phage genes can protect the infected host cell from infection by a second phage. For example, during lysogeny, temperate phages produce an immunity repressor to prevent the expression of its own genes that are involved in lytic replication; expression of this immunity repressor gene also protects the lysogen from infection by a second closely related phage. Other phage gene products have also been found to protect host cells from infection by closely or more distantly related phages. In order to test the ability of individual phage genes to confer protection, or defense, to the host cell, the efficiency of plaque formation by a given phage on bacterial lawns expressing each phage gene will be determined. </p> <h4>Supplies:</h4> <ul role = 'list'> <li> <em>M. smegmatis</em> transformed with recombinant pExTra plasmid(s) encoding gene(s) of interest </li> <li><em>M. smegmatis</em> transformed with pExTra01 control plasmid</li> <li>7H9 liquid medium (neat)</li> <li> 7H9 liquid medium (Complete, with Tween80), supplemented with Kanamycin (10 μg/ml)** </li> <li>1X 7H9 top agar supplemented with Kanamycin (20 μg/ml)</li> <li> 1X 7H9 top agar supplemented with Kanamycin (20 μg/ml) and aTc (100 ng/ml) </li> <li>7H10 agar plate(s) supplemented with Kanamycin (20 μg/ml)</li> <li> 7H10 agar plate(s) supplemented with Kanamycin (20 μg/ml) and aTc (100 ng/ml) </li> <li>Sterile test tubes with lids</li> <li>Microcentrifuge tubes</li> <li>Tabletop centrifuge</li> <li>Sterile innoculation or wooden sticks</li> <li>Temperature-controlled shaking incubator or rotor set to 37 ºC</li> <li>5 ml serological pipettes</li> <li>Pipette aid</li> </ul> <p> ** It is recommended to grow cultures in 10 μg/ml Kanamycin as this yields more consistent growth. </p> <h4>Procedure:</h4> <ol role = 'list' style='list-style-type:upper-alpha;'> <li> Generate two identical liquid cultures, or replicates, of <em>M. smegmatis</em> transformed with a recombinant pExTra plasmid encoding a gene of interest. <ol role = 'list'> <li> Observe your transformation plate(s) containing <em>M. smegmatis</em> transformed with your gene(s) of interest after incubation for 4 - 5 days at 37 ºC . </li> <li> For each gene to be tested, identify and circle two well-isolated colonies. </li> <li> For each of the two colonies selected, follow <a class='chap-linking'  tabindex='0' data-link-to='2-13'> Protocol 2.11 </a> to inoculate a liquid culture in 7H9 liquid medium (Complete, with Tween80) supplemented with Kanamycin (20 μg/ml). <br /> Note: There should be two cultures inoculated for a given gene of interest. To differentiate between these two replicate cultures, add ''rep 1'' to the label for one culture, and ''rep 2'' to the other. </li> <li> Incubate the replicates cultures, with shaking or rotation, at 37 ºC for 24 - 48 hours, until the cultures have saturated or have an optical density (O.D.) of >3. Avoid using liquid cultures that have been inoculated for more than 48 hours. </li> <li> <strong><b style='color:#b22222'> Important : </b></strong> Control cultures (two replicates per class) should be inoculated with colonies of <em>M. smegmatis </em> transformed with the pExTra01 plasmid, following steps A1-4. </li> </ol> </li> <li> Wash cells to remove Tween80. <ol role = 'list'> <li> Aseptically transfer 1 ml of each saturated liquid culture from Step A4 to a clean and labeled microcentrifuge tube. </li> <li> Spin the microcentrifuge tubes in a tabletop centrifuge at 2,000 x g for 10 min to pellet cells. </li> <li>Discard the supernatant.</li> <li> Aseptically transfer 1 ml of 7H9 liquid medium (neat) to each microcentrifuge tube and gently pipette up and down to resuspend each pellet. </li> <li> For each resuspended cell pellet, aseptically transfer 250 &micro;l of the cell resuspension to two sterile and labeled test tubes.<br /> Note: There should be four test tubes (two for each replicate) for each gene being tested. Per class, there should also be four pExTra01 control test tubes. </li> </ol> </li> <li> Prepare bacterial lawns on plates<strong> </strong><em>without</em> inducer (aTc). <ol role = 'list'> <li> For each gene being tested, obtain two 7H10 agar plates supplemented with Kanamycin (20 μg/ml). <ol role = 'list' start='1' style='list-style-type: lower-alpha;'> <li> Label the bottom of the plates with the following information (see Figure 3.3a): <ol role = 'list' start ='1' style='list-style-type: lower-roman;' > <li>your initials, date, and incubation temperature,</li> <li>no aTc</li> <li>the phage gene being tested,</li> <li>''rep 1'' on one plate and ''rep 2'' on the other,</li> <li> information about the phage lysate(s) that will be used to infect cells of your bacterial lawn.<br /> Note: You and your instructor should decide on the phage lysate(s) that will be used. </li> <li> Divide the plate into rows for each phage lysate to be tested, with a maximum of 4 rows per plate. </li> <li> Divide each row into sections for each lysate dilution to be spotted, with a maximum of 6 sections per row. </li> </ol> </li> </ol> </li> <li> Obtain the ''rep 1'' and ''rep 2'' test tubes with host bacteria from step B5. </li> <li>Obtain 1X 7H9 top agar supplemented with Kanamycin (20 μg/ml).</li> <li> Using a sterile 5 ml pipette, transfer 3.5 ml of molten top agar to the ''rep 1'' test tube containing host bacteria and then immediately draw the solution back into the same pipette.<br /> <span style='color:#b22222'> Important : </span> Try to avoid making or withdrawing bubbles, as they can interfere with interpretation of results. </li> <li> Dispense the top agar-bacteria mixture onto the labeled agar plate. <ol role = 'list' start='1' style='list-style-type: lower-alpha;'> <li> The top agar should not sit in the pipette for more than a few seconds because the agar will begin to solidify. </li> <li> Gently, but quickly, tilt the plate in multiple directions until the top agar mixture evenly coats agar plate </li> </ol> </li> <li> Allow the plate and to sit <em>undisturbed </em>for 20 minutes or until the top agar solidifies completely. </li> <li> Repeat Steps C4 - C6 for the ''rep 2'' test tube containing host bacteria prepared in step B5. </li> <li> <span style='color:#b22222'> Important : </span> Bacterial lawns should also be prepared for both replicates of the pExTra01 control, generating two class control plates without aTc. </li> </ol> </li> <li> Prepare bacterial lawns on plates <em>with</em> inducer (aTc). <ol role = 'list'> <li> For each gene being tested, obtain two 7H10 agar plates supplemented with Kanamycin (20 μg/ml) and aTc (100 ng/ml). <ol role = 'list' start='1' style='list-style-type: lower-alpha;'> <li> Label the bottom of the plates with the following information (see Figure 3.3a): <ol role = 'list' start ='1' style='list-style-type: lower-roman;'> <li>your initials, date, and incubation temperature,</li> <li>aTc (100 ng/ml)</li> <li>the phage gene being tested,</li> <li>''rep 1'' on one plate and ''rep 2'' on the other,</li> <li> information about the phage lysate(s) that will be used to infect cells of your bacterial lawn.<br /> Note: You and your instructor should decide on the phage lysate(s) that will be used. </li> <li> Divide the plate into rows for each phage lysate to be tested, with a maximum of 4 rows per plate. </li> <li> Divide each row into sections for each lysate dilution to be spotted, with a maximum of 6 sections per row. </li> </ol> </li> </ol> </li> <li> Obtain the ''rep 1'' and ''rep 2'' test tubes with host bacteria from step B5. </li> <li> Obtain 1X 7H9 top agar supplemented with Kanamycin (20 μg/ml) and aTc (100 ng/ml). </li> <li> Using a sterile 5 ml pipette, transfer 3.5 ml of molten top agar to the ''rep 1'' test tube containing host bacteria and then immediately draw the solution back into the same pipette.<br /> <span style='color:#b22222'> Important : </span> Try to avoid making or withdrawing bubbles, as they can interfere with interpretation of results. </li> <li> Dispense the top agar-bacteria mixture onto the labeled agar plate. <ol role = 'list' start='1' style='list-style-type: lower-alpha;'> <li> The top agar should not sit in the pipette for more than a few seconds because the agar will begin to solidify. </li> <li> Gently, but quickly, tilt the plate in multiple directions until the top agar mixture evenly coats agar plate </li> </ol> </li> <li> Allow the plate and to sit <em>undisturbed </em>for 20 minutes or until the top agar solidifies completely. </li> <li> Repeat Steps D4 - D6 for the ''rep 2'' test tube containing host bacteria prepared in step B5. </li> <li> <span style='color:#b22222'> Important : </span> Bacterial lawns should also be prepared for both replicates of the pExTra01 control, generating two class control plates with aTc. </li> </ol> </li> <div class='img-wrap'> <div class='img-container'> <img src='./assets/images/figure-3-3-1.jpg ' alt='Four circles, each circumscribing 3 columns six squares long labeled x, y and z. The cells in each column are labeled dilution 1 through dilution 6. Around the edge of the first circle is written: initials, date, temperature, no a-T-c, gene number and replicate 1. Around the second circle is written: initials, date, temperature, no a-T-c, gene number and replicate 2. Around the third circle is written: initials, date, temperature, a-T-c 100 nanograms per mililiter, gene number and replicate 1.  Around the fourth circle is written: initials, date, temperature, a-T-c 100 nanograms per mililiter, gene number and replicate 2. ' /> </div>  <div class='img-hr'></div><div class='title-container'>   Figure 3.3a. Plates and labels for a Defense Assay. In the example provided, X, Y, and Z refer to 3 different phages that will be tested for their ability to infect M. smegmatis plated without inducer or expressing a particular phage gene (Gene #) and mcherry or mcherry by itself in the presence of aTc inducer. The assay is performed in replicate, labeled as ''rep 1'' and ''rep 2''. </div> </div> <li> Spot a dilution series of each phage lysate to be tested on the solidified bacterial lawns prepared in Steps C and D . <ol role = 'list'> <li> Your instructor will provide appropriately diluted phage lysate(s) or will provide instructions for you to prepare your own lysate dilutions. </li> <li> One at a time, aseptically transfer 3 &micro;l of all dilutions onto the proper location on each of the four bacterial lawns. </li> <li> Spot 3 &micro;l of sterile 7H9 liquid medium (neat) onto a free section of each lawn. This is your negative control. </li> <li> Allow the liquid from the spots to absorb into the agar for 30 minutes or longer. </li> <li> <span style='color:#b22222'> Important : </span> The same diluted samples should be spotted on the pExTra01 class control lawns. </li> <li> Incubate plates (right-side up, not inverted) at 37 ºC for 24– 48 hours.</li> </ol> </li> <li> Observe for defense. <ol role = 'list'> <li> Plates for each gene of interest and class control plates should be observed after 24 - 48 hours of incubation. For each plate, <ol role = 'list'  style='list-style-type: lower-alpha;'> <li>The bacterial lawn should be visible and confluent.</li> <li> Phage clearing or plaque formation within spots should be visible. </li> <li> Make note of the color of the lawn. On aTc, lawns may or may not appear pink in color. </li> <li> Note: If expression of a gene is toxic to the host, lawn growth on aTc may be sparse or no growth may be visible even after 48 hours at 37 ºC. </li> </ol> </li> <li> Photograph your plates, following your instructor’s directions. </li> </ol> </li> <li> Interpret your defense assay results <ol role = 'list'> <li> First, calculate the titer of each phage lysate tested based on the number of plaques that formed on the ''rep 1'' and ''rep 2'' bacterial lawns, in both the presence and absence of aTc inducer. Calculate the titers on ''rep 1'' and ''rep 2'' separately. </li> <li> Next, calculate the titer of the same phage lysate based on the number of plaques that formed on the control bacterial lawns in the presence and absence of inducer (aTc).<br /> Note: The control bacterial lawn contains host bacteria transformed with the pExTra01 plasmid encoding the <em>mcherry </em>reporter gene only. </li> <li> For each gene, compare the titers of each phage lysate in the presence or absence of aTc. <ol role = 'list' style='list-style-type: lower-alpha;' > <li> For the control, the titers should be similar on lawns with or without aTc. </li> <li> For a gene of interest, the titers may or may not be similar on lawns with or without aTc. </li> </ol> </li> <li> Then, determine the efficiency of plating (e.o.p.), using the following formula for each phage and replicate. Titers should be comparable on both control plates, thus, titers from just one representative replicate can be used for this calculation. <br /> e.o.p. = titer of phage on bacterial lawn expressing your gene of interest with aTc/ titer of phage on the control bacterial lawn with aTc <ol role = 'list' style='list-style-type: lower-alpha;'> <li> If the e.o.p. is 1, the phage is equally able to infect the bacterial host regardless of whether or not the phage gene of interest is expressed. An e.o.p. of 1 therefore indicates that the phage gene of interests does not provide the bacterial host with protection to the phage tested. </li> <li> If the e.o.p. is smaller than 0.1, the phage has a clear reduced ablity to infect the bacterial host when the phage gene of interest is expressed. An e.o.p. below 1 therefore indicates that the phage gene of interest provides the bacterial host some protection to the phage tested. The smaller the e.o.p., the more effective the protection. </li> <li> If the e.o.p. is between 0.1 - 1, any protection provided by the phage gene of interest is minimal, and the experiment should be repeated to confirm the result. </li> </ol> </li> <li> In some cases, leaky gene expression from the repressed pExTra promoter may be sufficient to confer protection to the host. If expression of a gene of interest is toxic, inhibiting growth of the lawn on aTc, the e.o.p. can be calculated by comparing the titers on plates without aTc inducer, using the following formula for each phage and replicate:<br /> e.o.p. = titer of phage on bacterial lawn encoding your gene of interest without aTc / titer of phage on control bacterial lawn without aTc </li> </ol> </li> </ol><p> <strong>Helpful tips:</strong></p> <p> For a review on spot titers, see Protocol 6.4 in the Phage Discovery Guide. </p> "},"4-1":{content:"<p> Gene products rarely function in isolation, and the vast majority of biological processes depend upon physical contact between protein molecules. Cellular proteins can work together in pairs or may act as part of larger complexes involving many different proteins. Uncovering the protein partners a given protein is able to bind to, or ‘interact’ with, can provide valuable insight into the biological function of that protein, linking that protein to known cellular processes and proteins with well-defined functions. In this chapter, you will further explore the functions of a subset of phage genes, investigating the ability of their products to interact with proteins found in the M. smegmatis proteome. Since many phage proteins are expected to interact with host proteins as part of the phage infection cycle, the identification of an interaction between a phage protein and a host protein may provide further insight into the role of that gene in the phage life cycle and generate new hypotheses for future study. </p> <p> There are several experimental approaches available to study the protein-protein interactions (PPIs), either in vivo (inside the cell) or in vitro (outside the cell, often using purified proteins). Each approach has its own benefits and pitfalls, but all of these techniques require an observable readout that allows one to detect that an interaction has occurred. A widely used experimental system to study PPIs in vivo is the 2-hybrid assay, in which a physical interaction between two specific proteins produced in a cell activates a specific reporter gene. This reporter gene confers a specific characteristic, or phenotype, to the cell and thus allows the PPI to be monitored. Many different 2-hybrid systems are available to study PPIs, most of which use fast-growing microorganisms such as yeast or E. coli and a variety of reporter gene readouts. The version of the 2-hybrid assay that will be utilized as part of SEA-GENES research is a bacterial 2-hybrid (B2H) assay, which was developed for use in E. coli cells. There are several key features of this B2H system, which are described below. </p> <h3>Reporter Genes </h3> <p> The B2H system employs a specially engineered strain of E. coli, called B2H SELECT, that encodes 2 reporter genes: bla, which encodes for beta-lactamase, an enzyme that breaks down beta-lactam antibiotics such as penicillin or carbenicillin, and lacZ, which encodes for beta-galactosidase, an enzyme that breaks down lactose. When these reporter genes are expressed in the bacterial cell, beta-lactamase enables that bacterial cell to grow on media supplemented with the antibiotic carbenicllin, while beta-galactosidase enables that bacterial cell to degrade a chemical analog of lactose, called X-gal, which then turns the bacterial cell blue. Thus, expression of the reporter genes in bacterial cells is easily detected because those cells are able to form colonies, which turn blue, on medium supplemented with carbenicllin and X-gal. On the other hand, cells in which the reporter genes are not expressed should not be able to form colonies due to their sensitivity to carbenicllin.</p> <h3>Test Promoter for the Regulation of Reporter Genes</h3> <p> In order for the B2H assay to act as a reliable indicator of protein-binding in the cell, expression of the reporter genes must be regulated such that it is only activated as a result of a PPI occurring in the cell. This is achieved through the use of a specially engineered promoter, known as a 'test' promoter, to control the transcription of both reporter genes. The test promoter is considered a ‘weak’ promoter because by itself, RNA polymerase is not able to stably bind and initiate transcription; thus, only negligible reporter gene expression can occur. However, this test promoter was engineered to contain an additional regulatory feature that can stabilize RNA polymerase at the test promoter, thereby allowing for transcription of the reporter genes to occur in a PPI-dependent manner. </p> <h3> Regulating the Test Promoter </h3> <p> The regulatory feature of the B2H test promoter is a binding site, or operator, for a specific DNA-binding protein, the CI (pronounced ‘see one’) immunity repressor protein of phage lambda. The CI operator is situated upstream of the test promoter, such that when CI is bound to its operator site, it is in close proximity to any RNA polymerase molecule that contacts the test promoter. If operator-bound CI and RNA polymerase are able to physically interact, RNA polymerase binding at the test promoter is stabilized, thereby activating transcription of the reporter genes. Importantly, though, at this test promoter, CI and RNA polymerase are not normally able to interact with one another. However, if CI and RNA polymerase are each fused to proteins that are able to interact with one another, then an interaction between the CI-fusion protein and RNA polymerase-fusion protein can bridge the gap between these proteins, stabilizing RNA polymerase at the test promoter, and turning on expression of bla and lacZ. Thus, to test if two proteins can interact with one another in the B2H assay, one protein is fused to CI and the other to RNA polymerase. If cells expressing these fusion proteins grow and turn blue in the presence of carbenicllin and X-gal, then the two proteins likely interact; if no growth is observed, it is likely that those two proteins do not interact with one another. </p> <h3> Expressing Fusion Proteins </h3> To test for PPIs, the proteins of interests must first be expressed as fusions to CI or to a subunit of RNA polymerase. Each fusion protein is expressed as a single polypeptide encoded by two different genes that have been joined together. For the purposes of generating these fusion proteins, two plasmids were specifically engineered: <ul role = 'list'> <li> p2H&alpha; plasmid, cloned using the protocols outlined in Chapter 2, such that it can be expressed as a fusion to a portion of the &alpha; subunit of RNA polymerase, and</li> <li> pCI plasmid, which encodes random fragments of M. smegmatis genes that will be produced as a fusion to the CI protein. </li> </ul> <p> Both p2H&alpha; and pCI plasmids contain inducible promoters to regulate the production of the fusion proteins they encode. Note that both plasmids contain the same negatively regulated inducible promoter, ‘Lac*’. Like the pExTra 'Tet' promoter, which is maintained in the 'off' state when bound by the Tetracycline-repressor protein, TetR, the Lac* promoter, is maintained in the 'off' state when bound by the transcriptional regulator LacI. The Lac* promoter is turned “on” by the addition of the inducer lactose or a lactose mimic known as Isopropyl b-D-1-thiogalactopyranoside, or IPTG. When either inducer binds LacI, it causes a conformational change that releases the regulator from the DNA and allows RNA polymerase to bind and activate transcription of the downstream fusion genes. The reporter strain B2H SELECT encodes the LacI regulator, which is constitutively produced, ensuring that fusion proteins under the control of the Lac* promoter are not expressed until the IPTG inducer is introduced into the experimental system. </p> <h3> Selecting for PPIs </h3> <p> To perform the B2H assay, a p2Ha plasmid is used to transform B2H SELECT E. coli cells containing a pCI plasmid. Once transformed, IPTG is added to the cells, inducing expression of both fusion proteins encoded by these plasmids: a phage gene of interest is expressed from the p2H&alpha; plasmid as a fusion to a portion of the &alpha; subunit of RNA polymerase (&alpha;-phage gene) and a M. smegmatis gene fragment fused to CI (CI-host gene fragment) is expressed from the pCI plasmid. If the phage protein of interest is able to interact with a particular M. smegmatis protein fragment, that interaction will activate expression of the reporter genes in that particular cell. In the absence of an interaction, no reporter gene expression occurs. </p> <p> The M. smegmatis genome encodes ~ 9,000 different proteins. Assessing whether a phage gene product is able to interact with each of these host proteins, one-by-one, would be very labor and time-intensive. Instead, to determine which of these proteins, if any, interact with a phage protein of interest, the B2H assay has been developed to rapidly test for pairwise interactions between the phage protein of interest and a collection of ~ 1 million different M. smegmatis protein fragments cloned into a pCI gene fusion library. The B2H SELECT strain containing this pCI library is transformed with a p2H&alpha; plasmid; in the presence of IPTG, each E. coli cell produces the same &alpha;-phage gene fusion but a different CI-host gene fragment fusion. The bla reporter gene used in the B2H assay enables us to perform a genetic selection; a million transformed cells can be plated on selective medium containing IPTG and carbenicillin, and only the subset of those cells in which bla is being expressed from the test promoter due to a phage-host PPI should be able to survive. Thus, we can quickly ‘select’ for candidate interaction partners from a large pool of host protein fragments. The second reporter gene, lacZ, acts as an additional indicator that expression is occurring from the test promoter in surviving colonies as these should turn blue on selection plates containing X-gal. </p> <h3> Validating Hits </h3> <p> Ideally, after performing a B2H selection assay we will isolate carbenicillin-resistant colonies in which a biologically relevant PPI triggers expression of the bla and lacZ reporter genes. However, 2-hybrid assays can produce both false negative and false positive results. Some proteins may not be properly folded or stable when produced in the non-native context of the E. coli cytoplasm, or fusion to another large protein domain may interfere with the ability of a protein to bind its normal interaction partner. Thus, even if a phage protein does have a biologically important binding partner in the M. smegmatis proteome, we may not be able to identify it in our system. Furthermore, some colonies isolated in the B2H selection may be resistant to carbenicillin for reasons other than the presence of a true PPI. Three possible types of false positive results are described below. </p> <ul role = 'list'> <li> Some B2H Select E. coli cells may have acquired a random mutation on the chromosome that confers resistance to carbenicillin. In most cases, this class of mutants should be easily identifiable since they will not be expressing the reporter genes and therefore form colonies that are white rather than blue.</li> <li> Some B2H Select E. coli cells may have acquired a mutation in the test promoter, such that expression of the reporter genes is constitutive, and happens in the absence of a PPI.</li> <li> Some B2H Select E. coli cells may express ‘sticky’ CI protein fusions that are able to interact non-specifically with the phage protein or even RNA polymerase itself. </li> </ul> <p> After isolating a carbenicillin-resistant colony in our B2H selection, we must perform downstream analyses to try to distinguish between a true PPI candidate and potential false positives. We can use a diagnostic PCR to detect ‘sticky’ CI fusion fragments that appear repeatedly in selections with different phage proteins and eliminate them from our pool of candidate interaction partners. To further validate a PPI, the p2H&alpha; and pCI plasmids can be extracted from a colony that survives the B2H selection. Doing so isolates the plasmids expressing the two candidate interacting proteins away from chromosomal or test promoter mutations that may be present in that particular carbenicillin-resistant cell. When a new batch of B2H SELECT cells is transformed with the isolated plasmids, a genuine PPI should result in all transformants being able to survive a B2H selection. </p> <p> Once we have confirmed that growth on carbenicillin is due to an interaction between a phage gene product fusion and host protein fusion, we can sequence the pCI plasmid to identify the host interaction partner. Gaining a full understanding of the biological significance of this PPI and how it relates to the function of the phage gene product will take additional work, beyond the scope of one semester. Importantly, though, these interaction analyses will undoubtedly reveal important clues in our quest to understand gene function and will create new lines of research for phage biologists to explore! </p>"},"4-2":{content:"<h3>Objective:</h3> <p>To perform a bacterial 2-hybrid selection in order to identify potential protein-protein interactions between a phage gene product of interest and host proteins.</p> <h4>Rationale: </h4> <p>In this protocol, a bacterial two-hybrid system is used to rapidly assay the pairwise interactions between a phage gene product of interest and a plasmid gene library encoding for roughly a million different fragments of the ~7,000 proteins in the <em>M. smegmatis </em>proteome. The 2-hybrid analysis is performed in a special <em>E. coli </em>reporter strain, B2H SELECT, containing a test promoter that controls the expression of two reporter genes, <em>bla</em>, which encodes for the carbenicillin-resistance factor beta-lactamase, and <em>lacZ</em> , which encodes for the enzyme beta-galactosidase; this strain is transformed with two plasmids encoding the two 'hybrid' fusion proteins. The phage gene is expressed from the p2H&alpha; plasmid as a fusion to a subunit of <em>E. coli</em> RNA polymerase while each host protein fragment is expressed from a different pCI plasmid as a fusion to the CI DNA-binding protein. This library of pCI plasmids was previously transformed into the B2H SELECT reporter strain to generate the B2H SELECT/pCI-SMEG Library electrocompetent cells used in this protocol. In each aliquot of these cells there are about a billion different cells, each transformed with one of the million different pCI plasmids. You will trasnform these cells with the p2H&alpha; plasmid of interest by electroporation and plate them on special selective medium containing antibiotics to maintain selection of the plasmids, IPTG to induce production of the fusion proteins, X-gal to monitor<em> lacZ</em> expression, and a high concentration of carbenicillin; out of the fraction of competent cells that took up the p2H&alpha; plasmid, those that express two interacting fusion proteins will be able to activate expression of <em>bla</em> and survive in the presence of carbenicillin. These colonies should also be blue due to the activation of<em> lacZ</em> expression and the presence of the colored substrate X-gal in the selective medium; note, though, that the amount of blueness observed can vary from colony to colony. This 'selection' of colonies in which a potential protein-protein interaction is occuring works best when about a million transformed cells are grown on a plate, thus ensuring that many or most of the possible ~1 million different pairwise interactions have been tested. </p>  <h4>Supplies:</h4></p> <ul role = 'list'> <li>B2H SELECT/pCI-SMEG Library electrocompetent cells</li> <li>Plasmid DNA for the p2H&alpha; plasmid expressing the phage gene fusion of interest</li> <li>Ice</li> <li>Microcentrifuge tubes</li> <li>Recovery broth (LB broth without any antibiotics)</li> <li>IPTG solution (10 mM)</li> <li>B2H Selection plates</li> <li>B2H Growth plates</li> <li>Rattler beads or spreader, with 100 % EtOH</li> <li>Shaking incubator set to 37 <b>°</b>C</li> <li>Electroporation cuvettes (gap: 0.1 cm)</li> <li>Electroporator (voltage: 1.8 kV)</li> <li>Paper towel or KimWipe</li> <li>Plate incubator set to 30 ºC </li> </ul> <h4>Procedure :</h4> <strong><b style='color:#b22222'> Important : </b></strong>Competent cells should be kept on ice throughout Steps A and B. </p> <ol role = 'list' style='list-style-type:upper-alpha;'> <li>Set up the transformation reaction.<br> <strong>Note:</strong> The number of transformation reactions and the volume of competent cells used in the transformation reaction will depend on the competency of your bacterial cells, and should be discussed with your instructor. <ol role = 'list'> <li>Place the followwing three items on ice: <ul role = 'list'> <li>an aliquot of frozen B2H SELECT/pCI-SMEG Library electrocompetent cells </li> <li>recombinant p2H&alpha; plasmid DNA encoding your gene of interest on ice to chill</li> <li>an electroporation cuvette</li> </ul> </li> <li><p>The transformation reaction is optimized for 100 ng of plasmid DNA. Based on the concetration of your p2H&alpha; plasmid DNA, use the formula below to calculate the volume of sample needed to obtain 100 ng of plasmid DNA.</p><p> Formula: volume needed, &micro;l = (amount needed, ng) / (concentration of stock, ng/&micro;l)</p> <p><em>Example: For plasmid backbone DNA at 50 ng/&micro;l, </em></p><p> <em>volume needed, &micro;l = (100 ng) / (50 ng/&micro;l) </em><em>= 2 &micro;l</em></p></li> <li>Add the volume of p2H&alpha; plasmid DNA calculated in Step A3 to a pre-chilled microcentrifuge tube.<br> <strong><b style='color:#b22222'> Important : </b></strong> The maximum volume of p2H&alpha; plasmid DNA that should be added to the competent cells is 3 &micro;l.</li> <li>Once the competent cells have thawed, transfer 50 &micro;l of competent cells to the microcentrifuge tube containing the plasmid DNA, and keep on ice.</li> <li >Pipette the mixture up and down gently, one time.<br> <strong><b style='color:#b22222'> Important : </b></strong> Act quickly to ensure that the mixture remains cold. Repeated pipetting can introduce bubbles, which may interfere with electroporation. </li> <li >Transfer the mixture into the pre-chilled electroporation cuvette.</li> <li > <strong><b style='color:#b22222'> Important : </b></strong> Pipette the mixture into the thin gap between two metal electrodes at the bottom of the cuvette. Gently tap the cuvette on the bench to evenly settle the mixture at the bottom of the cuvette.<br> <strong><b style='color:#b22222'> Important : </b></strong> Be careful not to introduce any air bubbles in the mixture. </li> </ol> </li> <li >Electroporate the cells to allow for DNA uptake. <ol role = 'list'> <li >Adjust the settings on the electroporator according to your instructor's directions.</li> <li >Remove the cuvette from the ice, and using a paper towel, wipe off any moisture on the outside of the cuvette. Proceed quickly and carefully through the remaining steps.</li> <li >Place the cuvette into the electroporator holder.</li> <li >Pulse the cells, taking note of the pulse length value. It should be between 4 - 6 msec. <ol role = 'list' start='1' style='list-style-type: lower-alpha;'> <li >If a flash of light is seen or a loud popping sound is heard during the pulse, an electric arc has occured due to contaminating salts in the mixture. Repeat the procedure from the begining, this time using DNA that has been diluted in water to reduce the salt concentration in the final mixture.</li> </ol> </li> <li >Proceed immediately to Step C. </li> </ol> </li> <li >Allow cells to recover. <ol role = 'list'> <li >Using aseptic technique, immediately add 1 ml of the LB broth to the electroporation cuvette. Carefully pipette the liquid up and down 3-4 times to mix, then transfer the entire volume from the elctroporation cuvette into a clean and labelled microcentrifuge tube.</li> <li >Incubate the microcentrifuge tube at 37 <b>°</b>C, with shaking for 1 hour. </li> </ol> </li> <li >Prepare for plating. <ol role = 'list'> <li >After 1 hour of recovery at 37 <b>°</b>C, add 2 &micro;l of inducer (IPTG) to the cells in the microcentrifuge tube.</li> <li >Incubate the microcentrifuge tube at 37 <b>°</b>C, with shaking, for an additional 1 hour. During the 1 hour incubation period, perform Step D3.</li> <li >Obtain three B2H Selection plates per transformation reaction, and label the plates with your name, date, incubation temperature, and experimental information according to your instructor's direction. Warm the selection plate(s) by placing them in a 30 <b>°</b>C incubator.</li> </ol> </li> <li >Spread the transformation reaction onto selection plate(s). <ol role = 'list'> <li >For each transformation reaction, transfer 25 &micro;l of the recovery mix to one of the B2H Selection plates, 50 &micro;l to the second plate, and 100 &micro;l to the final plate.<br> <strong><b style='color:#b22222'> Important : </b></strong> Save the remaining recovery mix for Step F.</li> <li >Using Rattler beads or a glass spreader that has been decontaminated using EtOH, spread the mixture to even distribute the cells on the agar surface of each selection plate.</li> <li >With the lids closed, allow the plates to sit on the bench until all the liquid has been fully absorbed into the agar.</li> </ol> </li> <li >Setup and spot a dilution series of the recovery mix on plates without Carbenicillin. <ol role = 'list'> <li >Obtain 7 microcentriuge tubes, and arrange these microcentrifuge tubes in a rack, and label them 10<sup>-1</sup>, 10<sup>-2</sup>, 10<sup>-3</sup>, 10<sup>-4</sup>, 10<sup>-5</sup>.10<sup>-6</sup>, and 10<sup>-7</sup>.</li> <li value='2'>Add 180 &micro;l of LB liquid medium to each of the tubes</li> <li >Transfer 20 &micro;l of the remaining recovery mix from Step E1 to the “10<sup>-1</sup>” tube.</li> <li >Cap the tube and vortex briefly to mix. <ol role = 'list' start='1' style='list-style-type: lower-alpha;'> <li > <strong>Note:</strong> This solution in this “10<sup>-1</sup>” tube contains 1/10<sup>th</sup> the number of cells as your undiluted sample. It is also referred to as a 1:10 dilution.</li> </ol> </li> <li >Using a new pipette tip, transfer 20 &micro;l of the “10 <sup>-1</sup>” sample to the “10<sup>-2</sup>” tube and vortex well. <ol role = 'list' start='1' style='list-style-type: lower-alpha;'> <li >This solution contains 1/100<sup>th</sup> as many cells as your undiluted sample. It can also be referred to as your 1:100 dilution.</li> </ol> </li> <li >Continue each successive dilution until you get to your last tube, using a fresh tip for each dilution.</li> <li >For each transformation reaction, obtain four B2H Growth plates, and label the plate with your name, date, and incubation temperature. Divide the plate into 8 sections, and label the sections according to each dilution, including a section for the undiluted recovery mix.</li> <li >One at a time, aseptically transfer 10 &micro;l of each dilution onto the appropriate position on a labeled B2H Growth plate, using a new pipette tip for each spot.</li> <li >On the remaining three B2H Growth plates, plate 100 &micro;l of the 10<sup>-3</sup>, 10<sup>-4</sup>, and 10<sup>-5 </sup>dilutions, using a new pipette tip for each plate. </li> <li >Using Rattler beads or a glass spreader that has been decontaminated using EtOH, spread the mixture to even distribute the cells on the agar surface of each selection plate.</li> <li >With the lids closed, allow the four plates to sit on the bench for approximately 30 min for the liquid to be absorbed into the agar.</li> </ol> </li> <li >Place all the plates from Step E and Step F in a 30 <b>°</b>C incubator for 48 hours.</li> <li >Evaluate outcomes of the 2-Hybrid selection. <ol role = 'list'> <li >After 48 hours, the B2H Selection plates should be photographed and evaluated for the number, color, and size of colonies formed. This information should be recorded in your lab notebook. <br> <strong>Note:</strong> It is strongly recommended that plates be refrigerated for 2 - 24 hours prior to evaluating colony color. This low incubation temperature enhances the blue colony color on B2H Selection plates, making it easier to distinguish pale colonies from blue colonies.</li> <li >Calculate the fraction of transformed cells that were able to grow on carbenicillin. <ol role = 'list' start='1' style='list-style-type: lower-alpha;'> <li >Evaluate the B2H Growth plates. <ol role = 'list' start='1' style='list-style-type: lower-roman;'> <li >Of the plates from Step F9 on which 100 &micro;l was plated, identify one that has a countable number of colonies (i.e. between 20 and 200 colonies). </li> <li><p>Calculate the number of transformed cells in cfu/ml using the formula:</p><p> (cfu/ml) = (# cfu/ volume used in &micro;l) x (10<sup>3</sup> &micro;l/ml) x dilution factor*</p><p> <em>*For a 10<sup>-4</sup> dilution, the dilution factor is </em><em>10<sup>4 </sup></em>.</p><p> <strong>Example</strong>: If 60 colonies are observed on a plate with 100 &micro;l of the 10<sup>-5</sup> dilution plated, the calculation would look like this:</p><p> cfu/ml = (60 cfu/100 &micro;l) x (10<sup>3</sup> &micro;l/ ml) x (10<sup>5</sup>)</p> <p>= (6.0 x 10<sup>3</sup> x 10<sup>5</sup>) cfu/ml</p><p> = 6.0 x 10<sup>7 </sup>cfu/ml</p></li> <li>The cfu/ml represents the total number of transformed cells in your B2H transformation. Calculate the number of transformants plated on each selection plate by multiplying the cfu/ml by the fraction of the total 1 ml volume that was plated. <p><strong>Example</strong>: 50 &micro;l of the recovery mix represents 1/20th of the total 1 ml volume. Thus, on the B2H Selection plate on which 50 &micro;l of recovery mix was plated, 1/20th of the total cfu were plated.</p> <p> cfu plated = (6.0 x 10<sup>7 </sup>cfu/ml) x (1/20) = 3.0 x 10<sup>6 </sup>cfu/ml</p></li> </ol> </li> <li>Evaluate the spot dilution plate. <ol role = 'list' start='1' style='list-style-type: lower-roman;'> <li>Do the numbers of colonies in each spot make sense? For example, is there a 10-fold reduction in number of colonies as you move through the spots? If so, choose the dilution(s) that contain a countable number of colonies. </li> <li>Do the colony numbers agree with the results from the plates on which 100 &micro;l was plated? For example, is there a 10-fold increase in the number of colonies where 10-fold more volume was plated?</li> <li>If none of the plates on which 100 &micro;l was plated have a countable number of colonies, the cfu/ml can be approximated using this plate.<br> <strong>Note:</strong> B2H Selection plates on which fewer than 500,000 transformants or greater than 5 million transformants were plated should not be considered for candidate selection in Protocol 4.2. </li> </ol> </li> <li>The surviving fraction of transformed cells able to grow on Carbenicillin can be calculated by dividing the number of colonies on each B2H Selection plate by the number of cfu calculated in step H3. Typically this number is around 10<sup>-5 </sup>. <p> <strong>Example: </strong>If 25 colonies are observed on a plate on which 1.5 x 10<sup>6 </sup>cfu were plated, the surviving fraction would be: </p> <p>fraction of surviving cells = 25 colonies/1.5 x 10<sup>6 </sup>cfu = 1.7x 10<sup>-5 </sup>cfu</p> </li> </ol> </li> </ol> </li> <li>For B2H Selection plates containing colonies and on which 5 x 10<sup>5 </sup> - 5 x 10<sup>6 </sup>cfu were plated, proceed to Protocol 4.2 Selecting Candidate 2-Hybrid Hits. </li> </ol> <p><strong>Helpful tips:</strong></p>  "},"4-3":{content:"<h3>Objective:</h3> <p>To isolate and process candidate hits from a bacterial 2-hybrid selection assay for downstream verification. </p> <h4>Rationale: </h4> <p>After performing a bacterial 2-hybrid selection assay, surviving colonies should be isolated and cultured such that the plasmids they harbor can be purified for further analysis. In addition to the putative protein-protein interaction colonies, these selection plates may also contain colonies that are carbenicillin-resistant for less interesting reasons. For example, some cells may have randomly acquired a chromosomal mutation that confers resistance to carbenicillin; colonies that are completely white or very tiny may fall into this class. Furthermore, some survivng cells may harbor a mutation in the test promoter that allows for constitutive expression of the bla and lacZ reporter genes. Finally, some protein fragments may be 'sticky' and able to interact with many different phage proteins or even E. coli RNA polymerase itself. These last two classes of carbenicillin-surviving colonies are harder to distinguish as they will lead to healthy, blue colonies just like a true protein-protien interaction. In this protocol, colonies will be screened by colony PCR for known 'sticky' CI fusion proteins such that they can be eliminated from the pool of hits. In Protocol 4.3, the remaining candidates will be analyzed further to determine whether the ability to grow on carbenicillin is due to the expression of interacting partners or some type of mutation in the reporter strain. </p> <p> <h4>Supplies:</h4></p> <ul role = 'list'> <li>B2H Selection plates on which 500,000 to 5 million transformed cells were plated and incubated at 30 ºC for 48 h </li> <li>LB broth supplemented with 50 μg/ml Kanamycin, 25 μg/ml Chloramphenicol, 50 μg/ml Spectinomycin</li> <li>Sterile test tubes with lids</li> <li>Sterile innoculation or wooden sticks</li> <li>Temperature-controlled shaking incubator or rotor</li> <li>5 ml serological pipettes</li> <li>Pipette aid</li> <li>Microcentrifuge tubes</li> <li>Microcentrifuge</li> <li>Plasmid mini-prep Kit (Zymo)</li> <li>Sterile, nuclease-free water</li> <li>PCR tubes</li> <li>2X One Taq Polymerase Master Mix</li> <li>DNA Primers (10 μM)</li> <li>Ice</li> <li>Sterile, nuclease-free water</li> <li>Thermocycler</li> </ul> <h4>Procedure :</h4> <ol role = 'list' style='list-style-type:upper-alpha;'> <li> Inoculate liquid cultures. <ol role = 'list'> <li> After recording observations on the number, color, and size of colonies on the B2H Selection plate, follow Protocol 2.11, to inoculate a culture for each colony on the plate. <ol role = 'list' start='1' style='list-style-type: lower-alpha;'> <li>Colonies should be grown in LB broth supplemented with 50 μg/ml Kanamycin, 25 μg/ml Chloramphenicol, and 50 μg/ml Spectinomycin.</li> </ol> </li> <li>Cultures should be grown at 30 ºC, with shaking, for 16-24 h. </li> </ol> </li> <li>Screen candidates for known 'sticky' fragments by PCR <ol role = 'list'> <li>Prepare template DNA. <ol role = 'list' start='1' style='list-style-type: lower-alpha;'> <li>Obtain and label a new microcentrifuge tube for each sample to be tested.</li> <li>Aseptically transfer 100 &micro;l of each culture to the labelled tubes. These are your DNA templates.</li> </ol> </li> <li>Perform the PCR. <ol role = 'list' start='1' style='list-style-type: lower-alpha;'> <li>Consult your instructor to determine which primers to use for detection of 'sticky fragments' and the appropraite annealing temperature for each primer pair.</li> <li>Follow the instructions in Protocol 2.2: Polymerase Chain Reaction</a>, setting up a reaction for each culture to be tested. <ol role = 'list' start='1' style='list-style-type: decimal;'> <li>2X Taq Polymerase Master Mix should be used. </li> </ol> </li> </ol> </li> <li>Determine whether amplification was successful. <ol role = 'list' start='1' style='list-style-type: lower-alpha;'> <li>Follow instructions in Protocol 2.3: Gel Electrophoresis to visualize and determine the size of any amplified products for each colony PCR reaction.</li> <li>The presence of a band indicates the pCI plasmid encodes the 'sticky' fragment. Consult with the instructor to determine if the amplified product is the expected size. </li> <li>Only proceed to Step C with those cultures that do not yield an amplified product. </li> </ol> </li> <li>Note: Cultures can be left on the bench for several hours while performing the PCR analysis before proceeding to Step C. If PCR analysis and plasmid prepping will be done on separate days, cultures can be pelleted by centrifuging for 5 min at <em>10,000 x g</em> and pellets stored at -20 ºC. </li> </ol> </li> <li>Purify p2H&alpha; and pCI-SMEG plasmids. <ol role = 'list'> <li>Follow Protocol 2.12 to prepare plasmid from each culture. </li> <li>This is a mixed plasmid prep, containing both the p2H&alpha; and pCI-SMEG plasmids. </li> </ol> </li> </ol> <p><strong>Helpful tips:</strong></p>"},"4-4":{content:"<h3>Objective:</h3> <p>To verify that growth on carbenicillin is due to the PPI of fusion proteins encoded by p2H&alpha; and pCI-SMEG plasmids in the B2H SELECT strain.</p> <h4>Rationale: </h4> <p> In order to distinguish between a true PPI and a false positive colony that has acquired the ability to grow on carbenicillin due to some mutation in the original cell, the plasmids isolated from carbenicillin-resistant candidates in Protocol 4.2 must be transformed again into a new batch of B2H SELECT cells. These transformed cells will be incubated in the presence of IPTG in order to induce expression of the 2-hybrid fusion genes and plated again on both B2H Selection plates containing carbenicillin and B2H Growth plates without carbenicillin. If the original colony was isolated on carbenicillin due to the interaction of the two plasmid-encoded fusion proteins, then these new transformed cells should be able to grow on carbenicillin again. However, if the original colony was isolated on carbenicillin due to a resistance mutation, then these new transformed cells should not be able to grow on carbenicillin. Those candidates that are confirmed to grow on carbenicillin due to a plasmid-encoded PPI can be analyzed further to sequence and identify the M. smegmatis protein fragment that is able to interact with your phage gene product of interest. </p> <p> <h4>Supplies:</h4></p> <ul role = 'list'> <li>B2H SELECT chemical competent cells </li> <li>Plasmid DNA from isolated from each B2H Selection candidate.<br> Note: Each sample will contain both p2H&alpha; and pCI-SMEG plasmids</li> <li>LB agar plates, supplemented with 50 μg/ml Kanamycin, 25 μg/ml Chloramphenicol, and 50 μg/ml Spectinomycin</li> <li>LB broth, supplemented with 50 μg/ml Kanamycin, 25 μg/ml Chloramphenicol, 50 μg/ml Spectinomycin, and 20 μM IPTG</li> <li>Recovery broth (LB broth with no antibiotics)</li> <li>B2H Selection plates</li> <li>B2H Growth plates</li> <li>Ice</li> <li>Microcentrifuge tubes</li> <li>Microcentrifuge</li> <li>Heat block set to 42 <b>°</b>C.</li> <li>Shaking incubator set to 37 <b>°</b>C</li> <li>Plate incubator set to 30 ºC </li> <li>Rattler beads or spreader, with 100 % EtOH</li> <li>Sterile innoculation or wooden sticks</li> <li>A printed plate template, available for download <a href='https://dzf8vqv24eqhg.cloudfront.net/userfiles/11001/22461/ckfinder/files/B2H%20re-test%20Plate_Template.pdf' target = '_blank'> here </a>.</li> </ul> <h4>Procedure :</h4> <h5> Day 1</h5> <ol role = 'list' style='list-style-type:upper-alpha;'> <li>Transform B2H SELECT cells with p2H&alpha; and pCI-SMEG plasmids purified from B2H selection candidates. <ol role = 'list'> <li>Follow Protocol 2.9 to perform a chemical transformation of B2H SELECT chemical competent cells for each candidate plasmid prep generated in Protocol 4.2. <ol role = 'list' start='1' style='list-style-type: lower-alpha;'> <li>Transform 25 &micro;l of competent cells with 1 &micro;l of the mixed plasmid prep. This should be around 50 ng total of DNA. </li> <li>After the recovery step, spin the 1 ml of recovery mix in a microcentrifuge at 2,000 x g for 5 min.</li> <li>Remove 900 &micro;l of the supernatant fraction, being careful to not disturb the pellet. Discard the supernatant in an appropriate liquid waste container.</li> <li>Resuspend the pellet in the remaining liquid, approximately 100 &micro;l, by pipetting up and down to mix.</li> <li>Plate 20 &micro;l of this resuspension on an LB agar plate supplemented with 50 μg/ml Kanamycin, 25 μg/ml Chloramphenicol, and 50 μg/ml Spectinomycin. </li> <li>Incubate the plate at 30 ºC for 24 h.  </li> </ol> </li> </ol> </li> </ol> <h5> Day 2 </h5> <ol role = 'list' start='2' style='list-style-type:upper-alpha;'> <li>Pick and resuspend two colonies for re-evaluation of plasmid-dependent growth on B2H Selection plates. <ol role = 'list'> <li>Observe the transformation plate(s) from step A. </li> <li>On each transformation plate, identify 2, well-isolated colonies <ol role = 'list' start='1' style='list-style-type: lower-alpha;'> <li>On the bottom of the plate, circle and label the 2 colonies to be tested.</li> <li>These 2 colonies represent replicates A and B for each candidate. </li> </ol> </li> <li>Obtain and label a clean microcentrifuge tube for each of the 2 colonies to be tested. <ol role = 'list' start='1' style='list-style-type: lower-alpha;'> <li>Fill each labelled microcentrifuge tube with 500 &micro;l LB broth supplemented with 50 μg/ml Kanamycin, 25 μg/ml Chloramphenicol, 50 μg/ml Spectinomycin, and 20 μM IPTG.</li> <li>Using a sterile wooden stick, transfer each selected colony into the corresponding microcentrifuge tube. Shake the stick so transfer as much of the colony on the stick into the liquid medium.</li> <li>Cap the microcentrifuge tube, and vortex to disperse the colony. This is your undiluted cell suspension. </li> </ol> </li> </ol> </li> <li>Induce expression of the &alpha; and CI fusion genes in each cell suspension. <ol role = 'list'> <li>Incubate the tubes from step B at 37 ºC with shaking for 1 h. </li> </ol> </li> <li>Set up 10-fold serial dilutions (Figure 4.3a) <ol role = 'list'> <li>For each of the cell suspensions, obtain 5 additional microcentriuge tubes, arrange these microcentrifuge tubes in a rack, and label them 10<sup>-1</sup>, 10<sup>-2</sup>, 10<sup>-3</sup>, 10<sup>-4</sup>, and 10<sup>-5</sup>.</li> <li value='2'>Add 900 &micro;l of LB broth supplemented with 50 μg/ml Kanamycin, 25 μg/ml Chloramphenicol, 50 μg/ml Spectinomycin, and 20 μM IPTG to each of the tubes</li> <li>Perform the Steps D 4 - 7 for each of your undiluted cell suspensions.</li> <li>Transfer 100 &micro;l of your undiluted cell suspension to the “10<sup>-1</sup>” tube.</li> <li>Cap the tube and vortex briefly to mix.<br> Note: This solution in this “10<sup>-1</sup>” tube contains 1/10<sup>th</sup> the number of cells as your undiluted sample. It is also referred to as a 1:10 dilution.</li> <li>Using a new pipette tip, transfer 100 &micro;l of the “10 <sup>-1</sup>” sample to the “10<sup>-2</sup>” tube and vortex well.<br> Note: This solution contains 1/100<sup>th</sup> as many cells as your undiluted sample. It can also be referred to as your 1:100 dilution.</li> <li>Continue each successive dilution until you get to your last tube, using a fresh tip for each dilution.</li> </ol> </li> </ol> <div class='img-wrap'> <div class='img-container'><img src='./assets/images/figure-3-2-1.jpg' alt='An illustration of 6 partially-filled tubes, each containing 900 microliters, with a progressively increasing lighter color in each from left to right. The first tube on the left is labeled undiluted cell suspension and the rest are labeled 10 to the negative one through 10 to the negative 5. An arrow labeled 100  microliters is drawn between each tube.'></div> <div class='img-hr'></div> <div class='img-hr'></div><div class='title-container'>  Figure 4.3a. Serial dilutions. A volume of the undiluted cell suspension (100 &micro;l) is transferred into 900 &micro;l of LB broth with the appropriate supplements in the 10-1 tube. After the tube is mixed, 100 &micro;l of the 10-1 sample is transferred into the 10-2 tube, and the tube is mixed. This serial dilution continues until the desired dilutions are obtained. The last tube will contain a final volume of 1000 &micro;l.Figure 3.2b. The printable 'plate template' serves as a guide for spotting the samples on the agar plate, such that when plates are viewed from the bottom, the samples are distrubuted as above.</div></div> <ol role = 'list' start='5' style='list-style-type:upper-alpha;'> <li> <p>Spot each dilution series on B2H Selection and B2H Growth plates. </p> <ol role = 'list'> <li> <p>Obtain B2H Selection and B2H Growth plates. </p> <ol role = 'list' start='1' style='list-style-type: lower-alpha;'> <li> <p>A total of 6 dilution series can be spotted on each plate (Figure 4.3b).</p> <ol role = 'list'> <li> <p>Dilution series for up to 3 transformed candidates can be tested on each plate. </p> </li> <li> <p>For each candidate, the dilution series for the two replicate cell suspensions should be spotted next to each other on the same plate. </p> </li> <li> <p>Each dilution will be spotted onto one B2H Selection plate and one B2H Growth plate. <br> Note: The pattern of spotting is identical both plates.</p> </li> </ol> </li> </ol> </li> <li>Label the bottom edge of each agar plate with your name, the date, and whether or not carbenicillin is present. </li> <li>Divide the plate into columns for each candidate to be tested, and label the candidate number above each column. </li> <li>Place the agar plate on a printed copy of the plate template.<br> Note: The printable plate template is a mirror image of Figure 4.3b. This is to ensure that, when viewed from the bottom of the plate, the negative control is placed in the leftmost column, as in Figure 4.3b.</li> <li>One at a time, aseptically transfer 5 &micro;l of each dilution onto the appropriate position on the agar plates, using a new pipette tip for each spot.</li> <li>Allow the plates to sit on the bench for approximately 30 min for the liquid to be absorbed into the agar.</li> <li>Once the liquid has been absorbed, invert the plates and incubate at 30 <b>°</b>C for a total of ~36-48 hrs.</li> <li> Note: A positive and negative control should be spotted on B2H Selection and B2H Growth plates for each class. Consult your instructor for information on how these control plates should be prepared. <br> </li> </ol> </li> </ol> <div class='img-wrap'> <div class='img-container'><img src='./assets/images/plate-template.png' alt='A circle circumscribing a 6 by 6 grid. The rows from top to bottom are labeled: undiluted, 10 to the negative 1 through 10 to the negative 5. The columns are labeled candidate 3 rep B, candidate 3 rep A, candidate 2 rep B, candidate 2 rep A, candidate 1 rep B, and candidate 1 rep A'></div> <div class='img-hr'></div><div class='title-container'>  Figure 3.2b. The printable 'plate template' serves as a guide for spotting the samples on the agar plate, such that when plates are viewed from the bottom, the samples are distrubuted as above.</div></div> <ol role = 'list' start='6' style='list-style-type:upper-alpha;'> <li>Evaluate the ability of B2H SELECT cells transformed with candidate plasmids to grow on carbenicillin. <ol role = 'list'> <li>Note: It is strongly recommended that plates be refrigerated for 2 - 24 hours prior to evaluating colony color. This low incubation temperature enhances the blue colony color on B2H Selection plates, making it easier to distinguish pale colonies from blue colonies.</li> <li>All colonies transformed with candidate plasmids should grow on B2H Growth plates, and may or may not grow on B2H Selection plates. <ol role = 'list' start='1' style='list-style-type: lower-alpha;'> <li>For each candidate, observe and record which, if any, dilutions on the B2H Selection plate have growth. <ol role = 'list'> <li>In those dilutions that have single colonies, observe the size and color of those colonies.</li> <li>Does this colony morphology match the original candidate isolated in Protocol 4.1? </li> </ol> </li> </ol> </li> <li>Photograph your plates, following your instructor’s directions.</li> </ol> </li> </ol> <p><strong>Helpful tips:</strong></p>"},"14-1":{content:""},"14-2":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><strong>AD Supplement (0.5 L)</strong></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p><em>M. smegmatis </em>mc<sup>2</sup> 155 only</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>NaCl</p><p>&nbsp;</p><p>Albumin (Fraction V)</p></br><p>ddH<sub>2</sub>O</p></br><p>Dextrose</p></td><td colspan = '2'><p>4.25 g</p><p>&nbsp;</p><p>25 g</p></br><p>To 0.5 L</p></br><p>10 g</p></td><td><p>145 mM</p></br><p>5.0 %</p><p>2.0 %</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Add 400 mL of ddH<sub>2</sub>O to a 1-L Erlenmeyer flask.</li><li>Weigh out NaCl and albumin, and slowly, while stirring, add to the ddH<sub>2</sub>O in the Erlenmeyer flask.</li><li>Stir until dissolved.</li><li>Once dissolved, continue stirring and slowly add the dextrose.</li><li>Once the dextrose is dissolved, transfer the solution to a 1-L &nbsp;OR 0.5 L graduated cylinder, and bring the volume to 0.5 L with ddH<sub>2</sub>O.</li><li>Sterilize.</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Filter-sterilize; <span>do not autoclave.</span></p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At 4 ˚C, indefinitely if not contaminated</p></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Recipe:</p><ul><li>7H9 liquid medium complete</li></ul></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>Adding the dextrose last helps keep it from recrystallizing on the bottom of the flask.</li><li>Make sure the albumin is completely in solution before the filter-sterilization step. To dissolve the albumin, use a large stir bar spinning at a moderate to high speed. Very vigorous stirring will cause the albumin to denature and collect at foam/bubbles at the surface of the solution.</li><li>Standard medium for the growth of mycobacteria (Middlebrook) generally contains ADC (for albumin, dextrose and catalase) or OADC (ADC plus oleic acid). This guide uses the recipe for ADC but without catalase, hence the term “AD supplement”.</li><li>0.5 L of AD supplement is require to prepare 5 L of 7H9 liquid media .</li><li>Approximately 2.5 L of 7H9 liquid media is used per semester for a class of 20 students.</li></ol></td></tr></tbody></table>"},"14-3":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '6'><p><strong>1 M CaCl<sub>2</sub> (100 mL)</strong></p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '5' rowspan = '1'>General</td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td colspan = '4'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>CaCl<sub>2</sub></p><p>&nbsp;</p><p>ddH<sub>2</sub>O</p></td><td colspan = '4'><p>14.7 g of CaCl<sub>2</sub>.2H<sub>2</sub>0 <strong>or</strong></p><p>11.1 g of CaCl<sub>2</sub> (anhydrous)</p><p>see instructions below</p></td><td><p>1 M</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '5'><p>Note: You will need stock solutions of CaCl<sub>2</sub> at 1 M and 100 mM.</p><p>&nbsp;</p><p>For 1 M CaCl<sub>2</sub>:</p><ol><li>Add 90 mL of ddH<sub>2</sub>O to a large beaker.</li><li>Weigh out CaCl<sub>2</sub> and slowly, while stirring, add to the ddH<sub>2</sub>O until the CaCl<sub>2 </sub>is dissolved.</li><li>Transfer the solution to a graduated cylinder, and add ddH<sub>2</sub>O to 100 mL.</li><li>Sterilize.</li></ol><p>&nbsp;</p><p>For 100 mM CaCl<sub>2</sub>:</p><ol><li>Add 50 mL of 1 M CaCl<sub>2</sub> to a large beaker.</li><li>Add ddH<sub>2</sub>O to 500 mL</li><li>Sterilize*</li></ol><p>*if steps 1 &amp; 2 are performed using sterile ddH<sub>2</sub>O and sterile 1M calcium chloride, as well as using aseptic technique, there is no need to sterilize the solution.</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '5'><p>Filter-sterilize; <span>do not autoclave.</span></p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '5'><p>At room temperature, indefinitely. <span>Do not refrigerate</span></p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '5' rowspan = '1'><p>In Recipe:</p><ul><li>&nbsp;7H9 liquid medium complete and top agar</li><li>PYCa liquid medium, agar, and top agar.</li><li>Phage buffer</li></ul></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '5'><ol><li>CaCl<sub>2 </sub>should not be autoclaved as it can precipitate out of solution at high temperatures.</li><li>Due to the high concentration of salts in Middlebrook 7H9 media, CaCl<sub>2 </sub>has a tendency to precipitate out of solution. As such, CaCl<sub>2</sub> will ideally be added to 7H9 media immediately before the solution is used.</li><li>100 mL of both 1 M CaCl<sub>2 </sub>and 100 mM CaCl<sub>2</sub> should be sufficient for preparing all reagents requiring CaCl<sub>2</sub> for ~ 20 students per semester.</li></ol></td></tr></tbody></table>"},"14-4":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>0.8 M Ca(NO<sub>3</sub>)<sub>2</sub> (100 mL)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p><em>Streptomyces spp. </em>only</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p><strong>Ca(NO<sub>3</sub>)<sub>2</sub></strong></p><p><strong>ddH<sub>2</sub>O</strong></p></td><td colspan = '2'><p>18.89 g of Ca(NO<sub>3</sub>)<sub>2</sub>.4H<sub>2</sub>0</p><p>To 100 mL</p></td><td><p>0.8 M</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Add 80 mL of ddH<sub>2</sub>O to a large beaker.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</li></ol><ol><li value = '2'>Weigh out Ca(NO<sub>3</sub>)<sub>2</sub> and slowly, while stirring, add to the ddH<sub>2</sub>O until the Ca(NO<sub>3</sub>)<sub>2 </sub>is dissolved</li></ol><ol><li value = '3'>Transfer the solution to a graduated cylinder, and add ddH<sub>2</sub>O to 100 mL.</li></ol><ol><li value = '4'>Sterilize.</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Autoclave.</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, indefinitely. <span>Do not refrigerate</span></p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Recipe:</p><ul><li>Media Supplement for <em>Streptomyces</em> phage</li></ul></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><p>&nbsp;</p></td></tr></tbody></table>"},"14-5":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '6'><p><strong>Carbenicillin (CB) at 1000X</strong></p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '5' rowspan = '1'><p><em>M. smegmatis </em>mc<sup>2</sup>155 only</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td colspan = '3'><p><strong>Amount</strong></p></td><td colspan = '2'><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>Carbenicillin (powder)</p></td><td colspan = '3'><p>50 mg/ml</p></td><td colspan = '2'><p>stock = 1000X</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '5'><ol><li>Weigh out 100 to 600 mg of carbenicillin powder and place it in a 10-15 mL tube.</li></ol><ol><li value = '2'>Divide the number of milligrams of carbenicillin by 50 to determine the volume (in milliliters) of ddH<sub>2</sub>O to add (e.g., for 477 mg of CB, add 9.5 mL ddH<sub>2</sub>O).</li></ol><ol><li value = '3'>Stir or shake until dissolved.&nbsp;</li><li>Filter-sterilize (0.22&micro;m pore size) into a fresh sterile tube.</li><li>Prepare 1 mL aliquots and freeze at -80 ˚C</li></ol></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '5'><p>Filter-sterilize (0.22&micro;m pore size) into a fresh sterile tube.</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '5'><p>At -80 ˚C for 3 years.</p><p>At 4 ⁰C for ≤ 60 days.</p></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '5' rowspan = '1'><p>In Recipe: 7H9 Liquid Medium</p></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>Label Stock Solution </strong></p></td><td colspan = '5'><p>CB, 1000X, date, initials</p></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare Final Solution </strong></p></td><td colspan = '5'><ol><li>Remove a frozen aliquot, allow to thaw, and use as follows. Do not return thawed aliquot to -80 ˚C. Instead, store thawed aliquots at 4 ˚C</li><li>When using, add stock solution at 1:1000. For example, for 1 L of medium, add 1 mL of stock.</li></ol></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '5'><ol><li>If your host bacteria is resistant to CB, it can be added to prevent growth of other bacteria.</li></ol><ol><li value = '2'>Mechanism of action: CB is a semi-synthetic penicillin that interferes with bacterial cell-wall synthesis.</li></ol><ol><li value = '3'>50 mL of carbenicillin should be sufficient per semester for a class of ~ 20 students.</li></ol></td></tr></tbody></table>"},"14-6":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '5'><p><strong>Cycloheximide (CHX)</strong></p></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '4' rowspan = '1'><p>General</p></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td colspan = '3'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>Cycloheximide (powder)</p></td><td colspan = '3'><p>10 mg/ml</p></td><td><p>stock = 1000X</p></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '4'><ol><li>Weigh out 50 to 100 mg of cycloheximide powder and place it in a 10-15 mL tube.</li></ol><ol><li value = '2'>Divide the number of milligrams of cycloheximide by 10 to determine the volume (in milliliters) of ddH<sub>2</sub>O to add (e.g., for 51 mg of CHX, add 5.1 mL ddH<sub>2</sub>O).</li></ol><ol><li value = '3'>Stir or shake until dissolved.&nbsp;</li><li>Filter-sterilize (0.22&micro;m pore size) into a fresh sterile tube.</li><li>Prepare 1 mL aliquots and freeze at -80 ˚C.</li></ol></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '4'><p>Filter-sterilize (0.22&micro;m pore size) into a fresh sterile tube.</p></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '4'><p>At -80 ˚C for 1 year.</p><p>At 4 ⁰C for ≤ 60 days.</p></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '4' rowspan = '1'><p>In Recipe:</p><ul><li>7H9 Liquid Medium</li><li>PYCa Liquid Medium</li><li>Luria Agar</li><li>PYCa Agar</li></ul></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '4'><p>&nbsp;</p></td></tr><tr><td><p><strong>Label Stock Solution </strong></p></td><td colspan = '4'><p>CHX, 1000X, date, initials</p></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare Final Solution </strong></p></td><td colspan = '4'><ol><li>Remove a frozen aliquot, allow to thaw, and use as follows. Do not return thawed aliquot to -80 ˚C. Instead, store thawed aliquots at 4 ˚C.</li></ol><ol><li value = '2'>When using, add stock solution at 1:1000. For example, for 1 L of medium, add 1 mL of stock.</li></ol></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '4'><ol><li>CHX is added to media to prevent growth of fungi and yeast.</li></ol><ol><li value = '2'>Mechanism of action: CHX inhibits protein biosynthesis in eukaryotic organisms. It interferes with the peptidyl transferase activity of the 60S ribosome, thus blocking translational elongation.</li></ol><ol><li value = '3'>50 mL of cycloheximide should be sufficient per semester for a class of ~ 20 students.</li></ol></td></tr></tbody></table>"},"14-7":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>40 % Dextrose (200 mL)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>Dextrose anhydrous</p></br><p>ddH<sub>2</sub>O</p></td><td colspan = '2'><p>80 g</p></br><p>To 200 mL</p></td><td><p>40 %</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><p>1. Add 100 mL of ddH<sub>2</sub>O to a beaker, and then add the dextrose while stirring.</p><p>2. Top up with ddH<sub>2</sub>O to 200 mL, and continue stirring until dextrose has dissolved. If the dextrose does not fully dissolve, gently warm the solution on the stir plate (or in a microwave).</p><p>3. Filter-sterilize the solution. Do not autoclave.</p></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Filter-sterilize. <span>Do not autoclave.</span></p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, indefinitely if not contaminated.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Recipe:&nbsp;</p><ul><li>PYCa media</li><li>PYCa agar plates&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</li><li>PYCa top agar</li><li>Media Supplement for <em>Streptomyces</em> phage</li></ul></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>100 mL of 40 % dextrose should be sufficient for preparing all reagents and media requiring dextrose, per semester for a class of ~ 20 students.</li></ol></td></tr></tbody></table>"},"14-8":{content:"<table border = '1' cellpadding = '0' cellspacing = '0' width = '431'><tbody><tr><td colspan = '4'><p><strong>Enrichment Broth (100 mL)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p><em>M. smegmatis </em>mc<sup>2</sup>155 only</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td><p><strong>Amount</strong></p></td><td colspan = '2'><p><strong>Final Concentration</strong></p></td></tr><tr><td><p><strong>7H9 liquid media neat</strong></p></br><p><strong>100 mM CaCl<sub>2 </sub>stock</strong></p></td><td><p>99 mL</p></br><p>1 mL</p></td><td colspan = '2'><p>1X</p></br><p>1 mM</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'>All ingredients must be added to a sterile bottle or flask using aseptic technique. The final medium can be filter-sterilized.</td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>All ingredients are sterile. The final medium can be filter-sterilized.</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, for up to 1 year. Discard if solution is discolored, contaminated or has precipitation.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3'><p>In Protocol: 5.2, 5.5</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>If antibiotics or antimicrobials are added, the medium must be stored at 4 ⁰C.</li><li>A good rule of thumb is that CaCl<sub>2</sub> should be added to all solutions that come in contact with phage, and ideally, this should happen immediately before the solution is used. This is because CaCl<sub>2</sub> will precipitate out when cooling after autoclaving or when refrigerated.</li><li>30 ml of Enrichment broth is required per student per soil sample tested. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</li></ol></td></tr></tbody></table>"},"14-9":{content:"<table border = '1' cellpadding = '0' cellspacing = '0' width = '431'><tbody><tr><td colspan = '4'><p><strong>EDTA 0.5 M (100 mL)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td><p><strong>Amount</strong></p></td><td colspan = '2'><p><strong>Final Concentration</strong></p></td></tr><tr><td><p><strong>EDTA disodium salt dihydrate</strong></p><p><strong>ddH<sub>2</sub>O</strong></p></td><td><p>18.61 g</p><p>&nbsp;</p><p>To 100 mL</p></td><td colspan = '2'><p>0.5 M</p><p>&nbsp;</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Weigh out EDTA and place in a large beaker.</li><li>Add approximately 80 mL of ddH<sub>2</sub>O and stir until dissolved.</li><li>Adjust the pH to 8.0 with NaOH.</li><li>Transfer the solution to a graduated cylinder and add ddH<sub>2</sub>O to 100 mL.</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Filter-sterilize or autoclave</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, indefinitely if not contaminated.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3'><p>In Protocol: 9.1a, 9.1b, 9.2a, 9.2b</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>For protocol 9.2b, prepare a 0.1 M EDTA solution by diluting the 0.5 M EDTA solution in water.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</li></ol></td></tr></tbody></table>"},"14-10":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>40 % Glycerol (500 mL)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>ddH<sub>2</sub>O</p></br><p>Glycerol (also known as glycerin or glycerine)</p></td><td colspan = '2'><p>To 500 mL</p></br><p>200 ml</p></td><td><p>&nbsp;</p><p>40 %</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Measure 250 mL ddH<sub>2</sub>O and place in a large beaker.</li><li>Slowly add 200 mL of glycerol while stirring.</li><li>When the glycerol has completely mixed with the water, transfer the solution to a graduated cylinder.</li><li>Bring the volume to 500 mL with ddH<sub>2</sub>O. Mix.</li><li>Sterilize</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Filter-sterilize or autoclave.</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, indefinitely if not contaminated.</p></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Recipe:</p><ul><li><p>7H9 liquid medium<br>&nbsp;</p></li></ul><p>In Protocol:</p><ul><li><p>Making glycerol stocks</p></li></ul></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><p>100 mL of 40 % glycerol should be sufficient for all reagents and media requiring glycerol per semester.</p></td></tr></tbody></table>"},"14-11":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>80 % Isopropanol (50 ml)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>99% Isopropanol</p><p>ddH<sub>2</sub>O</p></td><td colspan = '2'><p>40 ml</p><p>To 50 ml</p></td><td><p>80 %</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Measure 40 ml isopropanol and place in a sterile conical tub&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</li><li>Slowly add ddH<sub>2</sub>O to 50 ml.</li><li>Mix using a vortex or by inversion.</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Not required</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p><span>Important: </span>Do Not Store.<br>80 % isopropanol should be prepared and used within the same day.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Protocol: 9.1, 9.2</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>Prepare 80 % isopropanol only as needed. Isopropanol will evaporate with time, resulting in a solution with reduced isopropanol concentration. Isopropanol solutions below 80 % will result in premature elution of DNA during phage DNA extraction.</li></ol><ol><li value = '2'>6 mL of isopropanol is used per attempt at DNA extraction from phage lysates.</li></ol></td></tr></tbody></table>"},"14-12":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>1 M MgCl<sub>2</sub> (100&nbsp; mL)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p><em>Streptomyces spp. </em>only</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p><strong>MgCl<sub>2</sub></strong> hexahydrate*</p><p><strong>ddH<sub>2</sub>O</strong></p><p>&nbsp;</p><p><span><em><span>*<strong>MgCl<sub>2</sub></strong> hexahydrate can be substituted with 9.5 g of&nbsp; <strong>MgCl<sub>2</sub></strong> anhydrous in this recipe</span></em></span></p></td><td colspan = '2'><p>20.33 g</p><p>To 100 mL</p></td><td><p>1 M</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Weigh out MgCl<sub>2</sub> and place in a large beaker containing approximately 75 mL of ddH<sub>2</sub>O, and stir until MgCl<sub>2 </sub>is dissolved.</li><li>Transfer the solution to a graduated cylinder, and add ddH<sub>2</sub>O to 100 mL.</li><li>Sterilize</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Autoclave or filter-sterilize</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, for a year.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Recipe:</p><ul><li>Media Supplement for <em>Streptomyces</em> phage</li></ul></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><p>&nbsp;</p></td></tr></tbody></table>"},"14-13":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '6'><p><strong>1 M MgSO<sub>4</sub> (100 mL)</strong></p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td colspan = '2'><p><strong>For bacterial host</strong></p></td><td colspan = '4' rowspan = '1'><p>General</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td colspan = '3'><p><strong>Amount</strong></p></td><td colspan = '2'><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>MgSO<sub>4 </sub>(anhydrous)</p></br><p>ddH<sub>2</sub>O</p></td><td colspan = '3'><p>12.04 g</p></br></br><p>To 100 mL</p></td><td colspan = '2'><p>1 M</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '5'><ol><li>Weigh out MgSO<sub>4</sub> and place in a large beaker containing approximately 95 mL of ddH<sub>2</sub>O, and stir until MgSO<sub>4 </sub>is dissolved.</li><li>Transfer the solution to a graduated cylinder, and add ddH<sub>2</sub>O to&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 100 mL.</li><li>Sterilize.</li></ol></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '5'><p>Autoclave or filter-sterilize</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '5'><p>At room temperature, for a year.</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '5' rowspan = '1'><p>In Recipe:</p><ul><li>Phage buffer</li></ul></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '5'><ol><li>10 mL of MgSO<sub>4 </sub>is required to prepare 1 L of phage buffer.</li><li>100 mL of MgSO<sub>4</sub> is sufficient for 1 – 5 semesters for a class of ~ 20 students.</li></ol></td></tr></tbody></table>"},"14-14":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>Nuclease Mix (5 mL)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>NaCl</p><p>ddH<sub>2</sub>O</p><p>DNase 1</p><p>RNase A</p><p>Glycerol</p></td><td colspan = '2'><p>0.09 g</p><p>4.25 mL</p><p>250 &micro;L of 5-mg/mL stock</p><p>125 &micro;L of 10-mg/mL stock</p><p>2.5 mL</p></td><td><p>150 mM</p><p>&nbsp;</p><p>0.25 mg/mL or 500 KU/ml</p><p>0.25 mg/mL or 12.5 U/mg</p><p>50 %</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><p><strong>Note</strong>: Begin by preparing the RNase A and DNase 1 stock solutions according to the manufacturer’s protocol. RNAse A stock solutions are typically prepared in Tris-Cl (10 mM; pH 7.5) and NaCl (15 mM), and includes a 5-min heating step at 100 ˚C.</p><ol><li>Dissolve the NaCl in 4.25 mL of ddH<sub>2</sub>O in a sterile 15-mL conical tube.</li><li>Transfer 2.1 mL of the NaCl solution into a new 15-mL conical tube.<br><span><span>Note: the remaining 2.1 mL can be stored indefinitely and used at a later date to prepare more nuclease mix.</span></span></li><li>Add the glycerol to the NaCl solution, and mix with gentle inversion.</li><li>Add the RNase and DNase stock solutions, and mix with gentle inversion.</li><li>Add ddH<sub>2</sub>O to a final volume of 5 mL, and mix with gentle inversion until the solution is homogeneous.</li><li>Aliquot into microcentrifuge tubes.</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Not required</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At -20 ⁰C, for up to 2 years.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Protocol: 9.1</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>Be extra careful not to contaminate pipettes, benches, or other reagents with this nuclease mixture, because it will degrade any DNA samples.</li><li>Prepare DNase I and RNase A stock solutions according to the instructions provided by the manufacturer.</li><li>5 &micro;L of nuclease mix is used per attempt at DNA extraction from phage lysates.</li><li>Estimate 1-2 attempts at DNA extraction per student.</li></ol></td></tr></tbody></table>"},"14-15":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '7'><p><strong>Phage Buffer (PB; 1 L)</strong></p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '6' rowspan = '1'><p>General</p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td colspan = '3'><p><strong>Amount</strong></p></td><td colspan = '3'><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>1 M Tris stock (pH 7.5)</p><p>1 M MgSO<sub>4</sub>&nbsp;stock</p><p>NaCl</p><p>ddH2O</p><p>100 mM CaCl<sub>2 </sub>stock</p><p>Glycerol (100 %), if using*</p></td><td colspan = '3'><p>10 mL</p><p>10 mL</p><p>4 g</p><p>up to 1 L</p><p>10 mL</p><p>100 mL</p></td><td colspan = '3'><p>10 mM</p><p>10 mM</p><p>68 mM</p><p>&nbsp;</p><p>1 mM</p><p>10 %</p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '6'><ol><li>Add 800 mL of&nbsp; ddH<sub>2</sub>O to a large flask or beaker.</li></ol><ol><li value = '2'>Add all ingredients to the water, while stirring. Glycerol should be added at this stage, if using (see notes*).</li></ol><ol><li value = '3'>Stir until all components have completely dissolved.</li></ol><ol><li value = '4'>Add ddH<sub>2</sub>O to 1 L.</li></ol><ol><li value = '5'>Filter-sterilize.</li></ol><ol><li value = '6'>Aliquot into sterile bottles, flasks, or tubes in 50-mL or 100-mL (or other) portions as needed.</li></ol></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '6'><p>Filter-sterilize</p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '6'><p>At room temperature, for up to 3 years. Discard if solution is discolored, contaminated or has precipitation.</p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '6' rowspan = '1'><p>In Protocol:</p><p>5.3, 5.4, 5.6, 6.1, 6.2, 6.3, 6.4, 6.5, 7.1, 8.1a, 8.1b, 11.1, 11.4</p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '6'><ol><li>*If your phage is particularly unstable in phage buffer, you may consider supplementing your phage buffer with glycerol to 10 %.</li><li>1 L of phage buffer is sufficient per semester for a class of ~ 20 students.</li></ol></td></tr></tbody></table>"},"14-16":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>Phage Precipitation Solution (100 mL)</strong></p><p><span><strong>[If using, prepare as necessary]</strong></span></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>ddH<sub>2</sub>O</p><p>Polyethylene glycol 8000</p><p>(PEG 8000)</p><p>NaCl</p></td><td colspan = '2'><p>To 100 mL</p><p>30 g</p><p>&nbsp;</p><p>19.3 g</p></td><td><p>&nbsp;</p><p>30 %</p><p>&nbsp;</p><p>3.3 M</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Add 60 mL of ddH<sub>2</sub>O, followed by 19.3 g NaCl, to a glass bottle with a screw cap, and swirl until the NaCl is completely dissolved.</li></ol><ol><li value = '2'>Slowly add 30 g of PEG 8000. Add only a few grams at a time, alternately swirling and gently heating in a microwave (e.g. by heating only for 10 sec at a time) until all the PEG is dissolved.</li></ol><ol><li value = '3'>Add sterile water to 100 mL.</li></ol><ol><li value = '4'>Add a clean magnetic stir bar and stir until homogeneous.</li></ol><ol><li value = '5'>Filter-sterilize.</li></ol><ol><li value = '6'>Dispense into sterile bottles or tubes.</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Filter-sterilize.</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, indefinitely. Discard if solution is deteriorating, discolored, contaminated or has precipitation.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Protocol: 9.1 of Instructors Guide</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>The NaCl and the PEG 8000 are difficult to dissolve. Several iterations of heating and stirring may be required. If there are precipitates in the solution, do not use it! Rather, attempt to dissolve particulates by alternately heating the solution in a microwave and swirling. Allow the solution to cool before using. Use this solution only at room temperature.</li></ol><ol><li value = '2'>Up to 4 mL of phage precipitation buffer is used per attempt at DNA extraction using the PEG precipitation protocol.</li></ol></td></tr></tbody></table>"},"14-17":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '5' rowspan = '1'><p>General</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td colspan = '3'><p><strong>Amount</strong></p></td><td colspan = '2'><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>SDS powder</p><p>ddH<sub>2</sub>O</p></td><td colspan = '3'><p>1 g</p><p>up to 10 mL</p></td><td colspan = '2'><p>10 %</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '5'><p><span>Important:</span> SDS may cause respiratory irritation if inhaled. Is using loose powder of SDS (instead of micropellets), a mask should be used when weighing out the powder.<br>&nbsp;</p><ol><li>Weigh out 1g of SDS powder and place it in a 50 mL tube.</li></ol><ol><li value = '2'>Add ddH<sub>2</sub>O to 10 mL.</li></ol><ol><li value = '3'>Invert the tube gently and repeated until the SDS is fully dissolved. Vigorous shaking will lead to excessive foaming.</li><li>Filter-sterilize (0.22&micro;m pore size) into a fresh sterile tube.</li><li>Prepare 1 mL aliquots.</li></ol></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '5'><p>Filter-sterilize (0.22&micro;m pore size) into a fresh sterile tube.</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '5'><p>At room temperature, indefinitely if not contaminated.</p></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '5' rowspan = '1'><p>In Protocol: 9.1a, 9.1b, 9.2a, 9.2b</p></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>Label Stock Solution </strong></p></td><td colspan = '5'><p>10 % SDS</p></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '5'><ol><li>2 mL of 10 % SDS should be sufficient per semester for a class of ~ 20 students.</li></ol></td></tr></tbody></table>"},"14-18":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>1X TBE</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>10X TBE</p><p>ddH<sub>2</sub>O</p></td><td colspan = '2'><p>100 mL</p><p>900 mL</p></td><td><p>1X (89-mM Tris, 89-mM boric acid, and 2-mM EDTA)</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><p>Dilute the 10X TBE to 1X with the volumes above (i.e., 100 and 900 mL). There is no need to check or adjust the pH of this solution, and it does not have to be sterilized before use.</p></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Not required</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, indefinitely. Discard is discolored, contaminated or has precipitation.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Protocol: 10.2</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><p>Volume of buffer will depend on size of gel and gel tank</p></td></tr></tbody></table>"},"14-19":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>1 M Tris (base), pH 7.5 (100 mL)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>Trizma base</p><p>ddH<sub>2</sub>O</p></td><td colspan = '2'><p>12.11 g</p><p>To 100 mL</p></td><td><p>1 M</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Add approximately 95 mL of ddH<sub>2</sub>O to a large beaker.</li></ol><ol><li value = '2'>Add Trizma base to the breaker while stirring.</li></ol><ol><li value = '3'>When the Trizma base is dissolved, bring the pH to 7.5 with HCl.</li></ol><ol><li value = '4'>Transfer the solution to a graduated cylinder, and add ddH<sub>2</sub>O to 100 mL</li><li value = '5'>&nbsp;Sterilize</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Autoclave or filter-sterilize</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, for up to a year.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Recipe:</p><ul><li>Phage buffer</li></ul></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>10 mL of 1M Tris is required to prepare 1 L of phage buffer.</li></ol><ol><li value = '2'>100 mL of 1M Tris is sufficient for 1 – 5 semesters for a class of ~ 20 students.</li></ol></td></tr></tbody></table>"},"14-20":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>20 % Tween80 (100 ml)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>ddH<sub>2</sub>O</p><p>Tween80</p></td><td colspan = '2'><p>To 100 ml</p><p>20 ml</p></td><td><p>20 %</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Measure 60 mL of ddH<sub>2</sub>O and place in a medium-size beaker.</li></ol><ol><li value = '2'>Slowly add 20 mL of Tween80 while stirring.</li></ol><ol><li value = '3'>Once the Tween80 has completely dissolved, transfer the solution to a graduated cylinder and bring to 100 mL with ddH<sub>2</sub>O.</li></ol><ol><li value = '4'>Sterilize</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Filter-sterilize. Do not autoclave</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, indefinitely if not contaminated</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Recipe:</p><ul><li>7H9 liquid medium</li></ul></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>It may take a while for Tween80 to dissolve. Add Tween 80 slowly and alternate between stirring and warming the solution. For example, alternate the beaker between the stir plate and at 55 ⁰C water bath to dissolve the Tween80.</li><li>100 mL of 20 % Tween80 is sufficient for 1 – 10 semesters for a class of ~ 20 students.</li></ol></td></tr></tbody></table>"},"14-21":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>Uranyl Acetate Stain</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>Uranyl acetate</p><p>ddH<sub>2</sub>O</p></td><td colspan = '2'><p>0.1 g</p><p>10 mL</p></td><td><p>1 % w/v</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Weigh out the uranyl acetate and place in a conical containing approximately 9 mL of ddH<sub>2</sub>O, and stir</li><li>When the uranyl acetate is dissolved, bring the final volume up to 10 mL with ddH<sub>2</sub>O.</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Prepare a 0.22-&micro;m filter by passing 3 mL of ddH<sub>2</sub>O through the filter. Discard the filtered ddH<sub>2</sub>O. Filter-sterilize the uranyl acetate solution and dispense in 1-mL aliquots.</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>Cover each aliquot in foil to protect from light. Store at room temperature, for up to 3 years. Discard appropriately if precipitation occurs.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Protocol:</p><p>8.1a, 8.1b</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>This is a highly toxic compound. Always wear gloves when preparing or working with it.</li></ol><ol><li value = '2'>5 &micro;L of uranyl acetate stain is used per sample to be imaged.</li></ol></td></tr></tbody></table>"},"14-22":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>Water, sterile (200 mL)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>ddH<sub>2</sub>O</p><p>&nbsp;</p></td><td colspan = '2'><p>200 mL</p><p>&nbsp;</p></td><td><p>&nbsp;</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Measure 200 mL ddH<sub>2</sub>O (ideally deionized water) and place bottle with a cap.</li><li>Sterilize</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Filter-sterilize or autoclave.</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, indefinitely if not contaminated.</p></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Recipe:</p><ul><li><p>80 % Isopropanol<br>&nbsp;</p></li></ul><p>In Protocol:</p><ul><li><p>DNA Extraction</p></li><li><p>Restriction Enzymes Digests</p></li></ul></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><p>500 mL of sterilized water should be sufficient for all reagents and experiments per semester.</p></td></tr></tbody></table>"},"14-23":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p>Zinc Chloride (2 M)</p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>ZnCl<sub>2</sub></p><p>ddH<sub>2</sub>O</p></td><td colspan = '2'><p>10.9 g</p><p>To 40 mL</p></td><td><p>2 M</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Weigh out ZnCl<sub>2</sub> and place in a conical tube containing approximately 35 mL of ddH<sub>2</sub>O, and vortex until ZnCl<sub>2 </sub>is dissolved.</li><li value = '2'>Add ddH<sub>2</sub>O to 40 mL.</li><li>Sterilize.</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Filter-sterilize.</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p><span>Important:</span> Do Not Store.</p><p>*Prepare immediately before use.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Protocol:</p><p>9.2b</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><p>* &nbsp;When prepared at this high concentration, ZnCl<sub>2 </sub>will precipitate after ~ 1 hour at room temperature.</p></td></tr></tbody></table>"},"14-24":{content:""},"14-25":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'> <tbody> <tr> <td colspan = '4'> <p><strong>Middlebrook 7H9 Liquid Medium: Neat (900 mL)</strong></p></td></tr><tr> <td colspan = '4'> <p>&nbsp;</p></td></tr><tr> <td> <p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'> <p><em>M. smegmatis </em>mc<sup>2</sup>155 only</p></td></tr><tr> <td colspan = '4'> <p>&nbsp;</p></td></tr><tr> <td> <p><strong>Ingredients</strong></p></td><td colspan = '2'> <p><strong>Amount</strong></p></td><td> <p><strong>Final Concentration</strong></p></td></tr><tr> <td> <p>7H9 broth base</p><p>40% glycerol stock&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>ddH<sub>2</sub>O</p></td><td colspan = '2'> <p>4.7 g</p><p>5 mL</p><p>To 900 ml</p></td><td> <p>&nbsp;</p></td></tr><tr> <td colspan = '4'> <p>&nbsp;</p></td></tr><tr> <td> <p><strong>To Prepare</strong></p></td><td colspan = '3'> <ol> <li>Add broth base to 850 mL of ddH2O in an Erlenmeyer flask.</li><li>While stirring, add the glycerol. Stir until the broth base powder is completely dissolved.</li><li>Bring up to 900 mL with ddH2O in a graduated cylinder.</li><li>Aliquot into 90-mL, 450-mL, or 900-mL portions as needed.</li><li>Sterilize</li></ol> </td></tr><tr> <td colspan = '4'> <p>&nbsp;</p></td></tr><tr> <td> <p><strong>To Sterilize</strong></p></td><td colspan = '3'> <p>Autoclave.</p></td></tr><tr> <td colspan = '4'> <p>&nbsp;</p></td></tr><tr> <td> <p><strong>To Store</strong></p></td><td colspan = '3'> <p>At room temperature, indefinitely if not contaminated or discolored.</p></td></tr><tr> <td colspan = '4'> <p>&nbsp;</p></td></tr><tr> <td> <p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'> <p>In Recipe:</p><ul> <li>Growing <em>Mycobacterium smegmatis</em> mc<sup>2</sup>155</li><li>1X 7H9 top agar</li></ul> </td></tr><tr> <td colspan = '4'> <p>&nbsp;</p></td></tr><tr> <td> <p><strong>Notes</strong></p></td><td colspan = '3'> <ol> <li>This medium will be diluted by the addition of 10 % (v/v) AD supplement (see Middlebrook 7H9 Liquid Medium: Complete). This is why medium is dispensed in 9/10 volumes (e.g., 450 mL).</li></ol> <ol> <li value = '2'>5 L of 7H9 liquid media should be sufficient per semester for a class of 20 students. </li></ol> </td></tr></tbody></table>"},"14-26":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>Middlebrook 7H9 Liquid Medium: Complete (100 mL)</strong></p><p><strong><span>(For bacterial growth without Tween 80)&nbsp;</span>&nbsp;&nbsp;&nbsp;&nbsp;</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p><em>M. smegmatis </em>mc<sup>2</sup>155 only</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>7H9 liquid medium: neat</p><p>AD supplement</p><p>CB stock</p><p>CHX stock</p><p>100 mM CaCl<sub>2</sub>&nbsp;stock</p></td><td colspan = '2'><p>89 ml</p><p>10 ml</p><p>100 &micro;l</p><p>100 &micro;l</p><p>1 ml</p></td><td><p>1X</p><p>10 %</p><p>50 &micro;g/ml</p><p>10 &micro;g/ml</p><p>1 mM</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><p>All ingredients must be added to a sterile bottle or flask using aseptic technique. The final medium cannot be autoclaved or filter-sterilized.</p></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>All ingredients are already sterile</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At 4 ⁰C, for up to 30 days. Discard if media is discolored or contaminated.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Recipe:</p><ul><li>Growing <em>Mycobacterium smegmatis</em> mc2155</li></ul></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>Make complete 7H9 medium as needed; refrigeration may cause the CaCl<sub>2</sub> to precipitate. If antimicrobials and AD supplement have been added, the medium must be stored at 4 °C.</li><li>A good rule of thumb is that CaCl<sub>2</sub> should be added to all solutions that come in contact with phage, and ideally, this should happen immediately before the solution is used. This is because CaCl<sub>2</sub> will precipitate out when cooling after autoclaving or when refrigerated.</li></ol></td></tr></tbody></table>"},"14-27":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>Middlebrook 7H9 Liquid Medium: Complete with Tween80</strong></p><p><strong><span>(For bacterial growth with Tween 80)&nbsp;&nbsp;&nbsp;</span>&nbsp;&nbsp;</strong>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p><em>M. smegmatis </em>mc<sup>2</sup>155 only</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>7H9 liquid medium</p><p>AD supplement</p><p>CB stock</p><p>CHX stock</p><p>100 mM CaCl<sub>2</sub>&nbsp;stock</p><p>20 % Tween80 stock</p></td><td colspan = '2'><p>90 ml</p><p>10 ml</p><p>100 &micro;L</p><p>100 &micro;L</p><p>1 ml</p><p>250 &micro;L</p></td><td><p>1X</p><p>10 %</p><p>50 &micro;g/ml</p><p>10 &micro;g/ml</p><p>1 mM</p><p>0.05 %</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><p>All ingredients must be added to a sterile bottle or flask using aseptic technique. The final medium cannot be autoclaved or filter-sterilized.</p></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>All ingredients are already sterile</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At 4 ⁰C, for up to 30 days. Discard if media is discolored or contaminated.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Recipe:</p><ul><li>Growing <em>Mycobacterium smegmatis</em> mc2155</li></ul></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>In liquid culture, <em>Mycobacterium smegmatis</em> tends to clump. When liquid cultures are grown from bacterial colonies, Tween80 should be added to 0.05% to prevent clumping.</li></ol><ol><li value = '2'>This culture is then sub-cultured into medium without Tween80 for use in phage infections.</li></ol><ol><li value = '3'>Make complete 7H9 medium as needed; refrigeration will cause the calcium to precipitate. If antimicrobials and AD supplement have been added, the medium must be stored at 4 °C.</li></ol><ol><li value = '4'>A good rule of thumb is that CaCl<sub>2</sub> should be added to all solutions that come in contact with phage, and ideally, this should happen immediately before the solution is used. This is because CaCl<sub>2</sub> will precipitate out when cooling after autoclaving or when refrigerated.</li></ol></td></tr></tbody></table>"},"14-28":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '8'><p><strong>1X Middlebrook Top Agar (TA; 100mL)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong></p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td colspan = '2'><p><strong>For bacterial host&nbsp;&nbsp; </strong></p></td><td colspan = '6' rowspan = '1'><p><em>M. smegmatis </em>mc<sup>2</sup>155 only</p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td colspan = '4'><p><strong>Amount</strong></p></td><td colspan = '3'><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>100 mM CaCl<sub>2 </sub>stock</p><p>7H9 liquid media neat</p><p>2X TA</p></td><td colspan = '4'><p>1 mL</p><p>50 mL</p><p>50 mL</p></td><td colspan = '3'><p>1 mM</p><p>&nbsp;</p><p>1 X</p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '7'><ol><li>Using aseptic technique, add 1 mL of CaCl<sub>2</sub> stock to the 50 mL of 7H9 medium.</li><li>Place into a 55 °C water bath.</li><li>Melt the 2X TA in a microwave. Make sure the cap is loose! The agar must be completely melted, so carefully swirl the bottle and check to make sure there are no clumps. The 2X TA should come to a boil but not boil over.</li><li>Once the agar is completely melted, place the 2X TA into a 55 °C water bath. When both solutions are at 55 °C, aseptically add the medium-calcium mixture to the 2X TA and swirl.</li></ol></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '7'><p>All ingredients must be added using aseptic technique. The final medium cannot be autoclaved or filter-sterilized.</p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '7'><p>In the 55 °C water bath ≤ 7days. Cooling to room temperature will cause the CaCl<sub>2</sub> to precipitate out of solution.</p><p>Discard if media is discolored or contaminated.</p><p>Repeated melting and solidifying of top agar will cause the agar to deteriorate and not solidify properly on plates.</p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '7' rowspan = '1'><p>In Protocol:</p><p>5.3, 5.6, 6.1, 6.4, 6.5, 7.1, 11.1, 11.2, 11.3, 11.4, 12.1,12.2</p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '7'><ol><li>The final agar concentration will be 0.4%</li></ol><ol><li value = '2'>A good rule of thumb is that CaCl<sub>2</sub> should be added to all solutions that come in contact with phage, and ideally, this should happen immediately before the solution is used. This is because CaCl<sub>2</sub> will precipitate out when cooling after autoclaving or when refrigerated.</li></ol><p>3. 3 – 5 mL of top agar is used per plating.</p></td></tr></tbody></table>"},"14-29":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '5'><p ><strong>2X Middlebrook Top Agar (2X TA; 1 L)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong></p></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td colspan = '2'><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p><em>M. smegmatis </em>mc<sup>2</sup>155 only</p></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td colspan = '2'><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td colspan = '2'><p><strong>7H9 broth base</strong></p><p><strong>Agar</strong></p><p><strong>ddH<sub>2</sub>O</strong></p></td><td colspan = '2'><p>4.7 g</p><p>8 g</p><p>1 L</p></td><td><p>&nbsp;</p><p>0.8 %</p></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '4'><ol><li>Place ddH<sub>2</sub>O and a stir bar, then broth base and agar, into an Erlenmeyer flask; put onto a stir plate.</li><li>Stir until the broth base powder is completely dissolved (the agar will not dissolve until it is heated).</li><li>Autoclave the flask, and cool to 55 °C in a 55 °C water bath.</li><li>Gently swirl the solution to evenly distribute the agar.</li><li>In a hood, using aseptic technique, dispense into sterile bottles using only half the available volume (e.g., for a 100 mL bottle, add 50 mL of 2X TA; this will allow you to add 50 mL of media directly to the molten 2X TA to prepare 1X TA). Keep any lids loose until the agar has solidified.</li><li>Tighten lids for storage.</li><li>To melt stored top agar for use, heat in a microwave using 30 second intervals until agar has completely melted.</li><li>Repeated melting and solidifying of top agar will cause the agar to deteriorate and not solidify properly on plates.</li></ol></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td colspan = '2'><p><strong>To Sterilize</strong></p></td><td colspan = '3' ><p>Autoclave</p></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td colspan = '2'><p><strong>To Store</strong></p></td><td colspan = '3' ><p>As a solid, store at room temperature, indefinitely. Discard if media is discolored or contaminated.</p></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td colspan = '2'><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Recipe:</p><ul><li>1X Middlebrook Top Agar<br>&nbsp;</li></ul><p>In Protocol:</p><ul><li>12.1</li></ul></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td colspan = '2'><p><strong>Notes</strong></p></td><td colspan = '3' ><p>2.5 L of 2X top agar is sufficient for per semester for ~ 20 students</p></td></tr></tbody></table>"},"14-30":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '8'><p><strong>Luria Agar (L-Agar) Plates (1 L) &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong></p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td colspan = '2'><p><strong>For bacterial host</strong></p></td><td colspan = '6' rowspan = '1'><p><em>M. smegmatis </em>mc<sup>2</sup>155 only</p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td colspan = '4'><p><strong>Amount</strong></p></td><td colspan = '3'><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>Luria broth base</p><p>Agar</p><p>ddH<sub>2</sub>O</p><p>CB stock (if using)</p><p>CHX stock</p></td><td colspan = '4'><p>15.5 g</p><p>15 g</p><p>To 1 L</p><p>1.0 mL</p><p>1.0 mL</p></td><td colspan = '3'><p>&nbsp;</p><p>&nbsp;</p><p>&nbsp;</p><p>50 &micro;g/mL</p><p>10 &micro;g/mL</p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '7'><ol><li>To 1 L of ddH<sub>2</sub>O in a large flask, add 15.5 g of Luria broth base and 15 g agar. Mix well with a magnetic stir bar.</li></ol><ol><li value = '2'>Autoclave the flask.</li></ol><ol><li value = '3'>Cool to 55 °C in a 55 °C water bath.</li></ol><ol><li value = '4'>Once the agar has cooled to 50-55 °C, aseptically add antimicrobials as necessary.</li></ol><ol><li value = '5'>Mix well by swirling, and avoid bubbles.</li></ol><ol><li value = '6'>Using aseptic technique, pour agar into Petri dishes.</li><li value = '7'>Allow the agar to set overnight.*</li></ol></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '7'><p>Autoclave (see above)</p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '7'><p>At 4 °C, for up to 3 months. Plates can still be used beyond 3 months, although the antimicrobials may have been partially degraded. Discard if plate is discolored or contaminated.</p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '7' rowspan = '1'><p>In Protocol:</p><p>5.3, 5.4, 5.6, 6.1, 6.4, 6.5, 7.1, 11.1, 11.2, 11.3, 11.4, 12.2</p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '7'><ol><li>Prior to adding antibiotics or antimicrobials, the cooled medium can be aliquoted into sterile bottles, allowed to solidify, and stored at room temperature. The solidified agar can be melted by autoclaving for 5 mins, or in a microwave. Antimicrobials should only be added once the freshly melted agar has cooled to 55 ˚C.</li><li>Prior to use, agar plates stored at 4 ˚C should be allowed to warm to room temperature.</li><li>1 L of agar is sufficient to prepare 50 agar plates.</li><li>* Though the agar will set within 1 – 2 hours, allowing the plates to sit at room temperature overnight helps remove some water content within the agar. Drying plates in this manner helps prevent top agar, when used, from slipping on the surface of the agar.</li><li>Approximately 1,500 agar plates are used per semester per class of 20 students. Consider preparing plates in batches of 500 plates.</li></ol></td></tr></tbody></table>"},"14-31":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>Media Supplement for <em>Streptomyces</em> phage (32.5 mL)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p><em>Streptomyces </em>spp. only</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td><p><strong>Amount</strong></p></td><td colspan = '2'><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>1 M MgCl<sub>2</sub></p><p>0.8 M Ca(NO<sub>3</sub>)<sub>2</sub></p><p>40% Dextrose</p></td><td><p>10 mL</p><p>10 mL</p><p>12.5 mL</p></td><td colspan = '2'><p>10 mM (after addition to culture media)</p><p>&nbsp; 8 mM (after addition to culture media)</p><p>0.5% (after addition to culture media)</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Aseptically add each component to a sterile container to mix or add each directly to cooled, sterile culture media (per 1 L).</li><li>Mix by gently swirling.</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Autoclave each component separately.</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, for up to 1 year. Discard if solution is discolored, contaminated or has precipitation.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3'><p>Nutrient Broth with Media Supplement for <em>Streptomyces</em> phage, Nutrient Agar with Media Supplement for <em>Streptomyces</em> phage</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><p>If using a different species of <em>Streptomyces</em> for phage isolation, a different base media may be used with the same supplement.</p></td></tr></tbody></table>"},"14-32":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '3'><p><strong>Nutrient Broth (1 L)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </strong></p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '2' rowspan = '1'><p><em>Streptomyces </em>spp. only</p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p><strong>Nutrient broth powder</strong></p><p><strong>ddH<sub>2</sub>O </strong></p></td><td><p>8 g</p><p>&nbsp;</p><p>To 1 L</p><p>&nbsp;</p></td><td><p>1X</p><p>&nbsp;</p><p>&nbsp;</p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '2'><ol><li>Mix 8 g of Nutrient broth powder with 1 L of ddH<sub>2</sub>O in a large flask and mix well with a magnetic stir bar.</li><li>Aliquot as needed.</li><li>Sterilize</li></ol></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '2'><p>Autoclave</p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '2'><p>At room temperature or 4 °C.</p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '2'><p><em>Streptomyces</em> liquid culture growth for starter cultures.</p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '2'><ol><li>Because <em>Streptomyces</em> cultures are prone to clumping when grown in liquid, it is highly recommended that all cultures contain 5% w/v PEG 8000, which serves as a physical dispersant, and that cultures be shaken vigorously in a baffled flask.&nbsp; PEG can be added to mixture prior to autoclaving.</li></ol><ol><li value = '2'>5 L of 7H9 liquid media should be sufficient per semester for a class of 20 students.</li></ol></td></tr></tbody></table>"},"14-33":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '3'><p><strong>Nutrient Broth with Media Supplement for <em>Streptomyces</em> phage (1 L)</strong></p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '2' rowspan = '1'><p><em>Streptomyces </em>spp. only</p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p><strong>Nutrient broth powder</strong></p><p><strong>ddH<sub>2</sub>O </strong></p><p><strong>Media Supplement for <em>Streptomyces</em> phage</strong></p></td><td><p>8 g</p><p>&nbsp;</p><p>To 1 L</p><p>&nbsp;</p><p>&nbsp;</p><p>32.5 mL</p></td><td><p>1X</p><p>&nbsp;</p><p>&nbsp;</p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '2'><ol><li>Mix 8 g of Nutrient broth powder with 1 L of ddH<sub>2</sub>O in a large flask and mix well with a magnetic stir bar.</li><li>Aliquot as needed.</li><li>Autoclave.</li><li>Add Media Supplement for <em>Streptomyces</em> phage (32.5 mL/1L) after media has cooled.</li></ol></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '2'><p>Autoclave</p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '2'><p>At room temperature or 4 °C.</p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '2'><p><em>Streptomyces</em> liquid culture growth in which phage infection will occur (including enrichment cultures)</p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '2'><p>Because <em>Streptomyces</em> cultures are prone to clumping when grown in liquid, it is highly recommended that all cultures contain 5% w/v PEG 8000, which serves as a physical dispersant, and that cultures be shaken vigorously in a baffled flask.&nbsp; PEG can be added to mixture prior to autoclaving.</p></td></tr></tbody></table>"},"14-34":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>Nutrient Agar with <em>Streptomyces </em>phage supplement (NA+) Plates (1 L)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p><em>Streptomyces </em>spp. only</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td><p><strong>Amount</strong></p></td><td colspan = '2'><p><strong>Final Concentration</strong></p></td></tr><tr><td><p><strong>Nutrient agar powder</strong></p><p><strong>ddH<sub>2</sub>O</strong></p><p><strong>Media Supplement for <em>Streptomyces</em> phage</strong></p></td><td><p>23 g</p><p>To 967.5 L</p><p>&nbsp;</p><p>&nbsp;</p><p>32.5 mL</p></td><td colspan = '2'><p>1X</p><p>&nbsp;</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Mix 23 g of Nutrient agar powder with ddH<sub>2</sub>O to 967.5 mL total volume in a large flask and mix well with a magnetic stir bar.</li><li>Autoclave the flask.</li><li>Cool to 55 °C in a 55 °C water bath.</li><li>Once the agar has cooled to 50-55 °C, aseptically add 32.5 mL of sterile Media Supplement for <em>Streptomyces</em> phage.</li><li>Mix well by swirling, and avoid bubbles.</li><li>Using aseptic technique, pour agar into Petri dishes.</li><li>Allow the agar to set overnight.*</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Autoclave (see above)</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature or 4 °C, for up to 3 months. Discard if plate is discolored or contaminated.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3'><p>In <em>Streptomyces</em> plate growth protocols.</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>Prior to adding media supplements, the cooled medium can be aliquoted into sterile bottles, allowed to solidify, and stored at room temperature. The solidified agar can be melted by autoclaving for 5 mins, or in a microwave. Supplements should only be added once the freshly melted agar has cooled to 55 ˚C.</li><li>Prior to use, agar plates stored at 4 ˚C should be allowed to warm to room temperature.</li><li>* Though the agar will set within 1 – 2 hours, allowing the plates to sit at room temperature overnight helps remove some water content within the agar. Drying plates in this manner helps prevent top agar, when used, from slipping on the surface of the agar.</li><li>1 L of agar is sufficient to prepare 50 agar plates.</li><li>Approximately 1,500 agar plates are used per semester per class of 20 students. Consider preparing plates in batches of 500 plates.</li></ol><p>&nbsp;</p></td></tr></tbody></table>"},"14-35":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '7'><p>&nbsp;</p><p><strong>PYCa Agar Plates (1 L)</strong></p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '6' rowspan = '1'><p>General (excluding <em>M. smegmatis</em> and <em>Streptomyces </em>spp.)</p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td colspan = '4'><p><strong>Amount</strong></p></td><td colspan = '2'><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>Agar</p><p>Yeast extract</p><p>Peptone</p><p>1 M Cacl<sub>2</sub></p><p>Dextrose (40 %)</p><p>ddH<sub>2</sub>O</p><p>CHX stock</p></td><td colspan = '4'><p>15.0 g</p><p>1 g</p><p>15 g</p><p>4.5 mL</p><p>2.5 mL</p><p>990 mL</p><p>1.0 mL</p></td><td colspan = '2'><p>1.5 %</p><p>1 g/L</p><p>15 g/L</p><p>4.5 mM</p><p>0.1 %</p><p>&nbsp;</p><p>10 &micro;g/mL</p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '6'><p><strong>Note</strong>: A video protocol for this recipe card is available <a href = 'https://seaphages.org/video/55' target = '_blank'><span><strong>here</strong></span></a>.<br>&nbsp;</p><ol><li>Place water and a stir bar, followed agar, peptone and yeast extract, into an Erlenmeyer flask. Put onto a stir plate.</li></ol><ol><li value = '2'>Stir until the yeast extract and peptone are completely dissolved (the agar will not dissolve until it is heated).</li></ol><ol><li value = '3'>Autoclave the flask, and cool to 60 °C in a 60 °C water bath.</li></ol><ol><li value = '4'>In a hood, aseptically add dextrose and CaCl<sub>2</sub>.</li></ol><ol><li value = '5'>Add antibiotics/antimicrobials stock solutions, if using.</li></ol><ol><li value = '6'>Mix well by swirling, and avoid bubbles.</li></ol><ol><li value = '7'>Using aseptic technique, pour agar into Petri dishes.</li><li value = '8'>Allow the agar to set overnight.*</li></ol></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '6'><p>Autoclave</p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '6'><p>At 4 °C, for up to 3 months. Plates can still be used beyond 3 months, although the antimicrobials may have been partially degraded. Discard if plate is discolored or contaminated.</p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '6' rowspan = '1'><p>In Protocol: 5.3, 5.4, 5.6, 6.1, 6.4, 6.5, 7.1, 11.1, 11.2, 11.3, 11.4, 12.2</p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '6'><ol><li>Prior to adding antibiotics or antimicrobials, the cooled medium can be aliquoted into sterile bottles, allowed to solidify, and stored at room temperature. To melt many batches of solidified agar,&nbsp; autoclave for ~ 10 -15 mins. Alternatively, melt the agar in small bathes using a microwave. Antimicrobials should only be added once the freshly melted agar has cooled to 55 ˚C.</li><li>Prior to use, agar plates stored at 4 ˚C should be allowed to warm to room temperature.</li><li>* Though the agar will set within 1 – 2 hours, allowing the plates to sit at room temperature overnight helps remove some water content within the agar. Drying plates in this manner helps prevent top agar, when used, from slipping on the surface of the agar.</li><li>1 L of agar is sufficient to prepare 50 agar plates.</li><li>Approximately 1,500 agar plates are used per semester per class of 20 students.<br>- Consider preparing plates in batches of 500 plates.<br>- Often, a limiting factor to preparing plates in large quantities is adequate space at 4 ˚C to store those plates.</li></ol></td></tr></tbody></table>"},"14-36":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>PYCa Liquid Medium (1 L)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3'><p>General (excluding <em>M. smegmatis</em> and <em>Streptomyces </em>spp.)</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>Yeast extract</p><p>Peptone</p><p>ddH<sub>2</sub>O</p><p>1 M CaCl<sub>2</sub>&nbsp;stock</p><p>Dextrose (40 %)</p><p>CHX stock</p></td><td colspan = '2'><p>1.0 g</p><p>15.0 g</p><p>990 ml</p><p>4.5 ml</p><p>2.5 ml</p><p>1 ml</p></td><td><p>1 g/L</p><p>15 g/L</p><p>&nbsp;</p><p>4.5 mM</p><p>0.1 %</p><p>10 &micro;g/ml</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Add 990 mL of ddH<sub>2</sub>O into a large flask. Then weigh out and add peptone and yeast extract to the flask.</li></ol><ol><li value = '2'>Stir until completely dissolved.</li></ol><ol><li value = '3'>Autoclave.</li></ol><ol><li value = '4'>Once cool, aseptically add dextrose and CaCl<sub>2</sub>.</li></ol><ol><li value = '5'>Aliquot as 100-mL, 500-mL, or 1000-mL portions, in sterile vials, &nbsp;as needed.*</li></ol><ol><li value = '6'>If using CHX, add prior to use.</li></ol><p>* It may be more convenient to aliquot the media (lacking dextrose and calcium chloride) into vials prior to sterilization by autoclaving. In this case, the vials do not need to be sterile, but dextrose and calcium chloride stock solutions should still only be added after sterilization. You will also need to adjust the volume of dextrose and calcium chloride stock solutions added to each vial.</p></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Autoclave.</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, indefinitely, or at 4 °C for up to 30 days if antimicrobials have been added.</p><p>Discard if media is discolored or contaminated.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Protocol: 5.2, 5.5, &amp; 12.1</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>If antimicrobials have been added, the medium must be stored at 4 °C.</li></ol><ol><li value = '2'>5 L of PYCa liquid media should be sufficient per semester for a class of 20 students.</li></ol></td></tr></tbody></table>"},"14-37":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>PYCa Top Agar (PYCa TA at 1X concentration; 1 L)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3'><p>General (excluding <em>M. smegmatis</em> and <em>Streptomyces </em>spp.)</p></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '2'><p>&nbsp;</p></td><td><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>Agar</p><p>Yeast extract</p><p>Peptone</p><p>1 M Cacl<sub>2</sub></p><p>Dextrose (40 %)</p><p>ddH<sub>2</sub>O</p></td><td colspan = '2'><p>4.0 g</p><p>1 g</p><p>15 g</p><p>4.5 mL</p><p>2.5 mL</p><p>990 mL</p></td><td><p>0.4 %</p><p>1 g/L</p><p>15 g/L</p><p>4.5 mM</p><p>0.1 %</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><p><strong>Note</strong>: You may choose to prepare a large volume of media and then transferring aliquots into smaller vials after sterilization (as described below), or preparing the media in smaller vials before sterilization. If choosing the latter, adjust the weight and volumes of each media component accordingly.<br>&nbsp;</p><ol><li>Add water and a stir bar, followed by agar, peptone and yeast extract, into an Erlenmeyer flask; put onto a stir plate.</li><li value = '2'>Stir until the yeast extract and peptone is completely dissolved (the agar will not dissolve until it is heated).</li></ol><ol><li value = '3'>Autoclave the flask.</li></ol><ol><li value = '4'>Once autoclaved and cooled slightly (not below 60 ˚C), add dextrose and CaCl<sub>2</sub>, using aseptic technique. Gently swirl the flask to mix the solution.</li></ol><ol><li value = '5'>In a hood, using aseptic technique, dispense into 50 mL or 100 mL sterile bottles.</li></ol><ol><li value = '6'>If using immediately, maintain top agar in the molten form by placing it at 55 ˚C. If not, keep bottle lids slightly loose until the agar has solidified. Once solidified, tighten lids for storage.</li></ol><ol start = '7'><li>To melt stored top agar for use, make sure the cap is loose, and heat in microwave using 30 second intervals. The top agar should come to a boil but not boil over. As the agar must be completely melted, <span><strong>carefully</strong></span> and gently swirl the bottle with the loose cap facing away from you and others. Check to make sure there are no clumps.</li></ol><ol start = '8'><li>Once the agar is completely melted, place the top agar into a 55 °C water bath to cool before use.</li></ol><ol start = '9'><li>Repeated melting and solidifying of top agar will cause the agar to deteriorate and not solidify properly on plates.</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Autoclave</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><ul><li>As a solid, store at room temperature, indefinitely. Discard if media is discolored or contaminated.</li><li>As molten agar, store for up to 10 days. Agar stored in its molten form deteriorates over time and may not solidify properly on plates.</li></ul></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Protocol:<br>5.3, 5.6, 6.1, 6.4, 6.5, 7.1, 11.1, 11.2, 11.3, 11.4, 12.1, 12.2</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><p>5 L of PYCa top agar is sufficient for per semester for ~ 20 students .</p></td></tr></tbody></table>"},"14-38":{content:""},"14-39":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '2'><p><strong>Growing bacteria from a frozen stock (Streak Plate)</strong></p></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td><p><span>Note</span>: A video demonstrating this protocol is available at <a href = 'https://seaphages.org/video/54/' target = '_blank'><strong>here</strong></a>.<br>&nbsp;</p><ol><li>Refer to Host Basics in the Phage Discvery Guide for specific guidelines when working with a particular bacterial host. In particular, take note of the appropriate media (agar) and incubation temperature.</li></ol><ol><li value = '2'>Label an appropriate agar plate.</li></ol><ol><li value = '3'>Once you are ready to streak a plate, obtain your frozen stock from the freezer and complete Step 4 quickly&nbsp; to minimize the amount of thawing of your frozen stock.</li><li>Using an inoculation loop or sterile wooden stick, scrape the surface of a frozen stock, and immediately return the stock to the freezer.</li><li>Aseptically spread the bacteria at the end of the loop or wooden stick onto the agar plate by streaking for single colonies.</li><li>Incubate for 2 – 3 days at the appropriate temperature.</li><li>Monitor growth over time to ensure bacterial growth parameters are as expected (e.g. time for colony formation and color/texture of colonies.</li><li>Once single colonies are visible and large enough for picking (see image of plate in “Host Basics” of the Phage Discovery Guide), plates should be stored either at room temperature or at 4 ˚C. To prevent over drying, the plates should be wrapped in parafilm.</li></ol></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td><p>At room temperature or at 4°C.</p></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td><p>In Protocol: Streak plate</p></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td><ol><li>This is your P0FF (passage 0 from frozen) stock. Depending on the host bacteria, this stock culture can be stored at room temperature or at 4°C for several weeks. We recommend storage for no more than a couple of weeks.</li><li>We suggest that you start P0FF stock cultures every week or every other week.</li></ol></td></tr></tbody></table>"},"14-40":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '3'><p><strong>Growing liquid cultures of bacteria from a single colony</strong></p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '2'><p><span>Note</span>: A video demonstrating this protocol is available at <a href = 'https://seaphages.org/video/56/' target = '_blank'><strong>here</strong></a>.</p><p>&nbsp;</p><ol><li>Refer to Host Basics in the Phage Discovery Guide for specific guidelines when working with a particular bacterial host. In particular, take note of the appropriate media (liquid) and incubation temperature.</li></ol><ol><li value = '2'>Add 50 mL of the appropriate liquid medium to a 250-mL baffled flask. If using, add antibiotics or antimicrobials.</li></ol><ol><li value = '3'>Using an inoculation loop or sterile wooden stick, pick a small piece from the middle of a single colony of bacteria on an agar plate. Avoid using colonies from plates older than 2 weeks.</li></ol><ol><li value = '4'>Aseptically transfer the bacteria from the loop or wooden stick into the medium in the flask.</li></ol><ol><li value = '5'><strong>Tween80 should be added to 0.05% when culturing bacteria that tend to clump (e.g. <em>M. smegmatis </em>mc<sup>2</sup>155). </strong>Cultures prepared with Tween80 must be subcultured to remove Tween80. See <a class = 'chap-linking' tabindex = '0'    data-link-to = '14-41' >instructions for subculturing from cultures prepared with Tween80</a>.</li></ol><ol><li value = '6'>Incubate, with shaking (250 rpm), at the recommended temperature.</li></ol><ol><li value = '7'>Monitor growth over time to ensure culture growth parameters are as expected (e.g. duration before culture is expected to become saturated)</li></ol><ol><li value = '8'>Once the culture is saturated, store at room temperature.</li></ol><p><span>Important</span>: Whenever a liquid culture of bacteria is prepared, the culture should be tested as follows:</p><ul><li>Prepare a streak plate from the culture, and monitor for the expected growth rate and colony morphology of the bacterial strain. If the growth rate or colony morphology is not as expected, the culture may be contaminated.</li><li>Prepare a top agar lawn using an aliquot of the bacterial culture, and test the ability of a host-specific phage to infect the top agar lawn using a spot test <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-8' >Protocol 5.6</a>. If the host-specific phage is unable to infect the bacterial lawn, the bacterial culture may be contaminated.</li></ul></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '2'><p>At room temperature or at 4°C, for 1 – 2 weeks.</p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '2' rowspan = '1'><p>In Protocol:</p><ul><li>Growing liquid cultures for use in phage experiments</li></ul></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '2'><ol><li>This is your P1FF (passage 1 from frozen) stock. This stock culture can be stored at room temperature, the duration for which will depend on the bacterium. Check Host Basics in this Guide for information about a specific bacterium.</li></ol><ol><li value = '2'>It can be aseptically aliquoted into smaller volumes for storage (this minimizes the potential for contamination).</li></ol><ol><li value = '3'>We recommend that you start P1FF stock cultures weekly, depending on use and the size of a given class. Regardless, because of the possibility of contamination, a backup P1FF culture should be available at all times.</li></ol><ol><li value = '4'>Single colonies from an agar plate can be used to inoculate multiple P1FF cultures for as long as the plate remains uncontaminated.</li></ol><ol><li value = '5'>If your PIFF does<strong> <span>NOT </span></strong>contain Tween80, this culture can be used for experiments with phage.</li></ol><ol><li value = '6'>Tween80 is added to cultures for some bacteria to minimize clumping during bacterial growth. <strong>The presence of Tween80, however, will likely inhibit phage infection.<br>For this reason:</strong><br>To grow liquid cultures for use in phage experiments, sub-inoculate from a P1FF culture into liquid media without Tween80.&nbsp; For instructions, see the recipe card “Growing liquid cultures of bacteria for use in phage experiments.”</li></ol></td></tr></tbody></table>"},"14-41":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '2'><p><strong>Preparing a culture without Tween80 from one with Tween80</strong></p></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td><p>Note: This protocol assumes a saturated liquid culture has been prepared, and contains Tween80.<br>&nbsp;</p><ol start = '1'><li>Add 50 mL of the appropriate liquid media (i.e., without Tween80) to a 250-mL baffled flask. For larger volumes, adjust accordingly (see notes). If using, add antibiotics or antimicrobials.</li><li >Aseptically, add a 1:1000 volume of the saturated culture containing Tween80. For example, for a 50-mL culture, add 50-&micro;L of your culture that contains Tween80.</li><li >Incubate, with shaking (250 rpm), at the recommended temperature.</li></ol></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td><p>At room temperature or at 4 °C.</p></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td><p>In Protocol: 5.2 - 7.2</p></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p><p>&nbsp;</p></td><td><p>Note: A culture prepared from a single colony is known as a P1FF. A new culture prepared from a PIFF is known as a P2FF.<br>&nbsp;</p><ol start = '1'><li>The addition of small (≤1:1000) volumes of the PIFF culture (containing Tween80) when preparing a sub-culture without Tween80 reduces the likelihood of bacterial clumping in subculture. If clumping does occur, continue incubating the subculture up to an additional 48 hours.</li><li>This culture can be stored at room temperature, the duration for which will depend on the bacterium. Check Host Basics in this guide for information about a specific bacterium.</li><li>To prepare additional cultures without Tween80, it is best to go back to the initial P1FF stock culture, so long as the P1FF culture is not contaminated, is not clumpy, or has not been stored for beyond the recommended time (see Host Basics in this guide)</li><li>The volume of a culture should be approximately one-fifth the size of the flask. For example, for 200 mL of bacterial culture, use a 1-L baffled flask. This maximizes aeration when the bacteria are grown on a shaker.</li><li>Tween80 will inhibit phage infection. It is advisable to double-check that any medium that will be used in phage experiments does not include Tween80.</li></ol></td></tr></tbody></table>"},"14-42":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '2'><p><strong>Preparing frozen stocks of bacteria</strong></p></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td><p><span>Note</span>: A video demonstrating this protocol is available at <a href = 'https://seaphages.org/video/57/' target = '_blank'><strong>here</strong></a>.</p><p>&nbsp;</p><ol><li>Label the appropriate number of cryo-tubes with the name of the bacteria, the date of preparation, and initials of the person preparing the stock.</li></ol><ol><li value = '2'>Add 300 &micro;L of 40 % glycerol to each tube.</li></ol><ol><li value = '3'>Add 700 &micro;L of bacterial culture to each tube.</li></ol><ol><li value = '4'>Mix by inversion.</li><li>Store at -80 °C.</li></ol></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td><p>at -80 °C, indefinitely.</p></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td><ol><li>To use frozen bacterial stocks, refer to the recipe card “Growing bacterial cultures from frozen a stock.”</li><li>It is important that only one person access any given&nbsp; glycerol stock. When a glycerol stock is first used, it should be labelled with the name of the person who is using it.</li></ol></td></tr></tbody></table>"},"14-43":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '2'><p><strong>Preparing Additional Phage Lysate &amp; Long-term Storage</strong></p></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td><ol><li>First, determine the titer of your phage lysate by performing a full plate titer (<a class = 'chap-linking' tabindex = '0'    data-link-to = '6-7' >Protocol 6.5</a>).</li><li>Based on the calculated titer, prepare webbed plates (<a class = 'chap-linking' tabindex = '0'    data-link-to = '7-2' >Protocol 7.1</a>).<br><em>Note: You can flood those plates from Step.1 that are webbed or have very high densities of plaques. This lysate should also be titered as part of Step.4.</em></li><li>Prepare additional lysate by flooding the webbed plates (<a class = 'chap-linking' tabindex = '0'    data-link-to = '6-5' >Protocol 6.3</a>). Proceed to Step 4, ideally as soon as Protocol 6.3 is complete.</li></ol><ol start = '4'><li>Determine the titer of the freshly prepared phage lysate by performing a full plate titer (<a class = 'chap-linking' tabindex = '0'    data-link-to = '6-7' >Protocol 6.5</a>). Ideally, the titer of the lysate will be at 10<sup>9 </sup>pfu/ml or greater. If the titer is below 10<sup>9</sup> pfu/ml, return to Step 2 and attempt the following modification: When webbed plates have been identified, incubate those webbed plates for an additional 24 - 48 hrs such that the bacteria remaining within the 'webbing' have also been lysed by phage before proceeding to Step 3.</li><li style  =  'text-align :start;'>Next:</li></ol><ol style  =  'list-style-type : lower-roman '><li><strong>To prepare freezer stocks</strong>, add 1 ml of the high-titer phage lysate to a labeled cryotube, then add 70 &micro;L of DMSO. Mix by inversion and store at -80 °C. Ideally, 3 - 4 freezer stocks will be prepared.</li><li><strong>To maintain a working stock,</strong> store the lysate at 4 °C.<br><span><strong>Important</strong></span>: The stability of phage particles in lysate stored at 4 °C will vary from phage to phage. Generally, phage stability parallels the titer, with phages more stable at higher titers. Regardless, the titer of phage lysates can drop from one day to another, or remain stable for weeks. It is therefore important to recalculate the titer of your lysate periodically.</li><li><strong>To archive phage lysates,</strong> refer to the archiving protocol (<a class = 'chap-linking' tabindex = '0'    data-link-to = '7-4' >Protocol 7.3</a>).<br><strong><span>Note</span>:</strong> You will receive bar-coded cryo-tubes and DMSO from the program in the fall of Year 1.&nbsp; All samples for archiving at the University of Pittsburgh should be prepared as described in the Phage Discovery Guide.</li></ol></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td><p>Freezer stocks -80 °C, indefinitely. Working stocks at 4 °C, with the titer checked periodically (e.g. biweekly)</p></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td><ol><li>To use frozen phage lysates, scrape a few microliters from the surface of the frozen lysate using a sterile micropipette tip, and immediately return the stock to the freezer to prevent thawing of the frozen stock. Repeated freeze-thaw cycles can damage phage particles.</li></ol><ol><li value = '2'>Transfer the scraped lysate to a microcentrifuge tube pre-filled with 100 &micro;L of phage buffer.</li><li value = '3'>The resuspension can then be used to prepare a working stock of phage lysate.</li><li value = '4'>Avoid thawing the frozen stock.</li></ol></td></tr></tbody></table>"},"14-44":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '2'><p><strong>Received Biologicals from the Program? Follow these Next Steps.</strong></p></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>Bacteria</strong></p></td><td><p>Bacterial strains are typically shipped as glycerol/freezer stocks to program members, and are intended for use in preparing additional freezer stocks for long-term storage.</p><p>Upon receiving these glycerol/freezer stocks, store these stocks at -80 °C.</p><p>&nbsp;</p><p>To prepare additional glycerol/freezer stocks:</p><ol><li>Streak out bacteria to obtain single colonies. Follow the instructions provided in ''<a class = 'chap-linking' tabindex = '0'    data-link-to = '14-39' >Growing bacteria from a frozen stock</a>''.</li><li>Within days of obtaining single colonies, prepare a liquid culture of the bacterial strain. Follow the instructions provided in ''<a class = 'chap-linking' tabindex = '0'    data-link-to = '14-40' >Growling liquid stock of cultures from a single colony</a>''.</li><li>Once the liquid culture is saturated, prepare additional frozen stocks following the instructions provided in ''<a class = 'chap-linking' tabindex = '0'    data-link-to = '14-42' >Preparing frozen stocks of bacteria</a>''.</li></ol><p>&nbsp;</p><p><span>Important</span>: Whenever a liquid culture of bacteria is prepared, the culture should be tested as follows:</p><ul><li>Prepare a streak plate from the culture, and monitor for the expected growth rate and colony morphology of the bacterial strain. If the growth rate or colony morphology is not as expected, the culture may be contaminated.</li><li>Prepare a top agar lawn using an aliquot of the bacterial culture, and test the ability of a host-specific phage to infect the top agar lawn using a spot test <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-8' >Protocol 5.6</a>. If the host-specific phage is unable to infect the bacterial lawn, the bacterial culture may be contaminated.</li></ul></td></tr><tr><td colspan = '2'>&nbsp;</td></tr><tr><td><p><strong>Phage</strong></p></td><td><p>Phage are typically shipped as working stocks (i.e. the stock lacks a cryo-protectant and should not be placed in a freezer). Upon receiving phage lysate from the program, store the lysate at 4 °C. Within weeks of receiving the lysate, prepare additional lysate for long-term storage and for use in research following instruction provided in ''<a class = 'chap-linking' tabindex = '0'    data-link-to = '14-43' >Preparing Additional Phage Lysate &amp; Long-term Storage</a>''.</p></td></tr></tbody></table>"},"14-45":{content:""},"14-46":{content:"<p><span><h2>Notes About Recipes:</h2></span></p><p><h3>About ddH2O </h3></p><p>Water should be double-distilled at a minimum. Filtered water (e.g., Millipore) is ideal. Regardless of water availability, consistency of preparation is essential (i.e., always use the same water).<br>&nbsp;</p><p><h3>About Labeling</h3></p><p>Clearly label all media and solutions, and date the additions of supplements. Sterile solutions should also be clearly labeled as such, and any autoclave tape (indicating sterility) should be left on the bottles.<br>&nbsp;</p><p><h3>About Adjusting the pH of Solutions</h3></p><p>Be sure to check each recipe for the proper acid or base solution that should be used to adjust the pH (if needed) of each solution or reagent.</p>"},"14-47":{content:"<p>&nbsp;</p><p><span>When using an autoclave, selecting the correct settings is crucial.</span></p><p><span><h2>Cycle Type:</h2></span></p><p><span>Most autoclaves are equipped with 3 settings: Liquid, Wrapped (or Vacuum), andUnwrapped (or Gravity). </span></p><ul><li><span><strong>Liquid Cycle</strong>: When sterilizing liquids by autoclave, the LiquidCycle should be selected. The Liquid Cycle is programmed to release the chamber pressure slowly at the endof the cycle, which prevents your super-heated liquids from rapidly bubbling over.</span></li><li><span><strong>Wrapped (or Vacuum) Cycle</strong>: Most consumables and glassware used,as well as biohazard waste generated, should be sterilized in an autoclave using the Wrapped (or Vacuum)Cycle. This cycle mechanically replaces the air in the chamber with steam, which allows the steam to accessareas where air may otherwise be trapped (e.g. pipette tip boxes). Note that this cycle is notpre-programmed to release chamber pressure slowly, and should not be used for liquids.</span></li><li><span><span><strong>Unwrapped (or Gravity) Cycle</strong>:Unwrapped utensils can be sterilized using this cycle. Unlike the Wrapped Cycle, the Unwrapped cycleuses gravity to displace the air in the chamber. As such, air trapped in containers (e.g. pipette tipboxes) may not be replaced with steam, limiting sterilization.</span></span></li></ul><p>&nbsp;</p><p><span><h2>Cycle temperature and duration:</h2></span></p><ul><li><strong>Liquids:</strong> Liquids should be autoclaved at 121 ˚C (250 ˚F). The cycle duration varies dependingon the amount of liquid being autoclaved. As a rule, 500 ml of liquid should be autoclaved for 40 mins. An extra5 minutes should be added to the overall duration for each additional 500 ml of liquid.</li><li><strong>Non-liquids:</strong> Most consumables and glassware used, as well as biohazard waste generated, shouldbe sterilized in an autoclave set at 121 ˚C (250 ˚F) for 20 mins.</li></ul>"}}},PhageDiscovery:{title:"Phage Discovery Guide",search:"Search This Guide",home:"app/homebtn.svg",searchicon:"app/searchbtn.svg",help:"app/helpicon.svg",popupSearchImg:"app/popup-search.svg",download:"app/downloadbtn.svg",headerimg:"homepage.png",close:"app/closebtn.svg",collapse:"app/collapse-bt.svg",hhmi:"hhmi-logo.png",youtube:"youtube.png",facebook:"facebook.png",twitter:"twitter.png",helpPopupContent:"<div class = 'pop-items'> <div class = 'popup-box'> <div class = 'popup-box-heading'> <img class = 'home-icon' src = './assets/images/app/homebtn.svg' alt = 'home button' /> <span class = 'text'> Home </span> </div> <div class = 'popup-box-content'> <p class = 'popup-box-content-spacing'>On clicking on home button user will go to the landing page. User can access this button from every screen of this   guideThis butto is placed at top most left side of the header. On clicking on home button user will go to the landing page. User can access this butoon from every screen of this guideThis butto is placed at top most left side of the header. </p> </div> </div> <div class = 'popup-box'> <div class = 'popup-box-heading'> <img class = 'home-icon' src = './assets/images/app/collapse-bt.svg' alt = 'home button' /> <span class = 'text'> Collapse </span> </div> <div class = 'popup-box-content'> <p class = 'popup-box-content-spacing'>On clicking on collapse button our left side navigation will shift left and go out from the screen, and at the same time right side content will will shihft left to fill the white gap. </p> </div> </div> <div class = 'popup-box'> <div class = 'popup-box-heading'> <img class = 'home-icon' src = './assets/images/app/helpicon.svg' alt = 'Help' /> <span class = 'text'> Help </span> </div> <div class = 'popup-box-content'> <p class = 'popup-box-content-spacing'> On Clicking on the help button, a pop up nwindow will open and give you a brief about buttons and on screens. This button is placed at top right side of the header. On Clicking on the help button, a pop up window will open and give you a brief about buttons and their functions.  </p> </div> </div> <div class = 'popup-box'> <div class = 'popup-box-heading'> <img class = 'home-icon' src = './assets/images/app/downloadbtn.svg' alt = 'download' /> <span class = 'text'> Download </span> </div> <div class = 'popup-box-content'> <p class = 'popup-box-content-spacing'> On clicking on download button user will be abble to download the whole onlinne guide. This button is placed at top most right side of the header. </p> </div> </div> <div class = 'popup-box'> <div class = 'popup-box-heading'> <img class = ' search-bg' src = './assets/images/app/popup-search.svg' alt = 'search' /> <span class = 'text'> Search </span> </div> <div class = 'popup-box-content'> <p class = 'popup-box-content-spacing'>  Clicking on search bar user can search any keyword and search any chapter or related content. This button is placed at centre the header. </p> </div> </div> </div></div>",footer:[{id:"1",contenttype:"icon",title:"Facebook",content:"facebook.png",link:"https://www.facebook.com/HowardHughesMed"},{id:"2",contenttype:"icon",title:"Youtube",content:"youtube.png",link:"https://www.youtube.com/channel/UCUWJVAgUcpvWwKJXba5XoxQ"},{id:"3",contenttype:"icon",title:"Twitter",content:"twitter.png",link:"https://twitter.com/hhminews"},{id:"4",contenttype:"popup",title:"Contact Us",content:"Contact Us",popupContent:"<div class = 'contact-content'> <div>Contact Information:</div> <div> Email: <a href = 'mailto:info@seaphages.org'>info@seaphages.org</a> </div> Website: <a href = 'https://seaphages.org/' target  =  '_blank'>seaphages.org</a> </div> "},{id:"5",contenttype:"popup",title:"Copyright",content:"Copyright",popupContent:"<div class = 'copyright-content'> <p>    Marianne Poxleitner1, Welkin Pope2, Deborah Jacobs-Sera2, Viknesh    Sivanathan3, Graham Hatfull2 Gonzaga University1, University of Pittsburgh2, Howard Hughes Medical Institute3 The Science Education Alliance-Phage Hunters Advancing Genomics and Evolutionary Science Supported by the Howard Hughes Medical Institute <br /> <br /> <br /> Contact Information: Email:<a href = 'mailto:info@seaphages.org'>info@seaphages.org</a> | Website:<a href = 'https://seaphages.org/' target  =  '_blank'>seaphages.org</a> <br /> <br /> <br /> With contributions from: <br /> <br /> Ilzat Ali, Aleem Mohamed, Emily Davis, and Ariel Egbunine, University of Maryland Baltimore County Priscilla Kobi3, William Biederman3, Danielle Heller3 <br /> <br />Kevin Bradley, Infectious Disease Programs at the Association of Public Health Laboratories<br /><br />Steven Cresawn, James Madison University<br /><br />Dan Russell, University of Pittsburgh <br /><br />    Nicholas M. Jaramillo, Graphic Artist: Figures 3.0-2, 3.0-3,    3.0-4, 3.0-5,8.0-2 <br /><br /><br /><br />    <br />    Some text and figures adapted from the SEA-PHAGES Resource Guide ©    2012, Howard Hughes Medical Institute, and the NGRI In Silico    Phage Resource Guide, Analyze © 2008, Howard Hughes Medical    Institute, including Figures 13.0-4, 13.0-5, and 13.0-8 by Lucia    Barker  </p></div>"},{id:"6",contenttype:"image",title:"HHMI",content:"hhmi-logo.png",link:"https://www.hhmi.org/"}],navigation:[{id:"1",pageId:"1",templateId:"chapterContent",name:"Chapter 1: Welcome",imgtext:"Welcome",img:"welcome.svg",child:!1},{id:"2",pageId:"2",templateId:"toc",name:"Chapter 2: Lab Basics",img:"lab-basics.svg",imgtext:"Lab Basics",child:!0,childElements:[{id:"1",pageId:"2-1",templateId:"chapterContent",name:"Chapter 2: An Overview"},{id:"2",pageId:"2-2",templateId:"chapterContent",name:"Protocol 2.1: Aseptic Technique Exercise"},{id:"3",pageId:"2-3",templateId:"chapterContent",name:"Guidelines for working with bacteria"}]},{id:"3",pageId:"3",templateId:"chapterContent",name:"Chapter 3: Phage Basics",img:"phage-basics.svg",imgtext:"Phage Basics",child:!1},{id:"4",pageId:"4",templateId:"toc",name:"Chapter 4: Host Basics",img:"host-basics.svg",imgtext:"Host Basics",child:!0,childElements:[{id:"1",templateId:"chapterContent",pageId:"4-1",name:"Chapter  4: An Overview"},{id:"2",templateId:"chapterContent",pageId:"4-2",name:"Chapter  4: Tips and Hints"},{id:"3",templateId:"chapterContent",name:"Arthrobacter globiformis",pageId:"4-3"},{id:"4",templateId:"chapterContent",pageId:"4-4",name:"Arthrobacter sp."},{id:"5",templateId:"chapterContent",pageId:"4-5",name:"Gordonia rubripertincta"},{id:"6",templateId:"chapterContent",pageId:"4-6",name:"Gordonia terrae"},{id:"7",templateId:"chapterContent",pageId:"4-7",name:"Microbacterium foliorum"},{id:"8",templateId:"chapterContent",pageId:"4-8",name:"Microbacterium testaceum"},{id:"9",templateId:"chapterContent",pageId:"4-9",name:"Mycobacterium smegmatis"}]},{id:"5",templateId:"toc",pageId:"5",name:"Chapter 5: Isolation",img:"isolation.svg",imgtext:"Isolation",child:!0,childElements:[{id:"1",templateId:"chapterContent",pageId:"5-1",name:"Chapter  5: An Overview"},{id:"2",templateId:"chapterContent",pageId:"5-2",name:"Chapter  5: Tips and Hints"},{id:"3",templateId:"chapterContent",pageId:"5-3",name:"Protocol 5.1: Collecting Environment Samples"},{id:"4",templateId:"chapterContent",pageId:"5-4",name:"Protocol 5.2: Direct Isolation"},{id:"5",templateId:"chapterContent",pageId:"5-5",name:"Protocol 5.3: Plaque Assay"},{id:"6",templateId:"chapterContent",pageId:"5-6",name:"Protocol 5.4: Picking a Plaque"},{id:"7",templateId:"chapterContent",pageId:"5-7",name:"Protocol 5.5: Enriched Isolation"},{id:"8",templateId:"chapterContent",pageId:"5-8",name:"Protocol 5.6: Spot Test"}]},{id:"6",templateId:"toc",pageId:"6",name:"Chapter 6: Phage Purification",img:"purification.svg",imgtext:"Purification",child:!0,childElements:[{id:"1",templateId:"chapterContent",pageId:"6-1",name:"Chapter 6: An Overview"},{id:"2",templateId:"chapterContent",pageId:"6-2",name:"Chapter 6: Tips and Hints"},{id:"3",templateId:"chapterContent",pageId:"6-3",name:"Protocol 6.1: Plaque Assay for Purification"},{id:"4",templateId:"chapterContent",pageId:"6-4",name:"Protocol 6.2: Serial Dilutions"},{id:"5",templateId:"chapterContent",pageId:"6-5",name:"Protocol 6.3: Collecting Plate Lysates"},{id:"6",templateId:"chapterContent",pageId:"6-6",name:"Protocol 6.4: Spot Titer"},{id:"7",templateId:"chapterContent",pageId:"6-7",name:"Protocol 6.5: Full Plate Titer"}]},{id:"7",templateId:"toc",pageId:"7",name:"Chapter 7: Phage Amplification",img:"amplification.svg",imgtext:"Amplification",child:!0,childElements:[{id:"1",templateId:"chapterContent",pageId:"7-1",name:"Chapter 7: An Overview"},{id:"2",templateId:"chapterContent",pageId:"7-2",name:"Protocol 7.1: Making Webbed Plates from a Lysate of Known Titer"},{id:"3",templateId:"chapterContent",pageId:"7-3",name:"Protocol 7.2: Entering a Phage into the Actinobacteriophage Database"},{id:"4",templateId:"chapterContent",pageId:"7-4",name:"Protocol 7.3: Archiving your Phage Sample"},{id:"5",templateId:"chapterContent",pageId:"7-5",name:"Chapter:7 Tips & Hints"}]},{id:"8",templateId:"toc",pageId:"8",name:"Chapter 8: Electron Microscopy",img:"microscopy.svg",imgtext:"Microscopy",child:!0,childElements:[{id:"1",templateId:"chapterContent",pageId:"8-1",name:"Chapter  8: An Overview"},{id:"2",templateId:"chapterContent",pageId:"8-2",name:"Protocol 8.1a: Mounting Phage Samples for TEM and Staining with Uranyl Acetate"},{id:"3",templateId:"chapterContent",pageId:"8-3",name:"Protocol 8.1b: Mounting Phage Samples for TEM and Staining with Uranyl Acetate (using Pelco Tabs)"},{id:"4",templateId:"chapterContent",pageId:"8-4",name:"Protocol 8.1c: Mounting Phage Samples for TEM and Staining using the Parafilm Drop Method"},{id:"5",templateId:"chapterContent",pageId:"8-5",name:"Chapter:8 Tips & Hints"}]},{id:"9",templateId:"toc",pageId:"9",name:"Chapter 9: Extracting Phage DNA",img:"DNA-extraction.svg",imgtext:"DNA Extraction",child:!0,childElements:[{id:"1",templateId:"chapterContent",pageId:"9-1",name:"Chapter  9: An Overview"},{id:"2",templateId:"chapterContent",pageId:"9-2",name:"Protocol 9.1: Phage DNA Extraction"},{id:"3",templateId:"chapterContent",pageId:"9-3",name:"Protocol 9.2a: Phage DNA Extraction Following Precipitation with PEG"},{id:"4",templateId:"chapterContent",pageId:"9-4",name:"Protocol 9.2b: Phage DNA Extraction Following Precipitation with Zinc Chloride"},{id:"5",templateId:"chapterContent",pageId:"9-5",name:"Chapter:9 Tips & Hints"}]},{id:"10",templateId:"toc",pageId:"10",name:"Chapter 10: Characterization",img:"characterization.svg",imgtext:"Characterization",child:!0,childElements:[{id:"1",templateId:"chapterContent",pageId:"10-1",name:"Chapter  10: An Overview"},{id:"2",templateId:"chapterContent",pageId:"10-2",name:"Chapter:10 Tips & Hints"},{id:"3",templateId:"chapterContent",pageId:"10-3",name:"Protocol 10.1: Setting Up Restriction Enzyme Digests"},{id:"4",templateId:"chapterContent",pageId:"10-4",name:"Protocol 10.2: Casting Agarose Gels"},{id:"5",templateId:"chapterContent",pageId:"10-5",name:"Protocol 10.3: Gel Electrophoresis of Restriction Enzyme Digests"},{id:"6",templateId:"chapterContent",pageId:"10-6",name:"Protocol 10.4: Analyzing Restriction Enzyme Gels"}]},{id:"11",templateId:"toc",pageId:"11",name:"Chapter 11: Further Discovery",img:"furthur-discovery.svg",imgtext:"Furthur Discovery",child:!0,childElements:[{id:"1",templateId:"chapterContent",pageId:"11-1",name:"Chapter  11: An Overview"},{id:"2",templateId:"chapterContent",pageId:"11-2",name:"Protocol 11.1: Creating and Testing Lysogens: Streaking from a High Titer Spot"},{id:"3",templateId:"chapterContent",pageId:"11-3",name:"Protocol 11.2: Testing and Verification of Potential Lysogens: Patch Assay"},{id:"4",templateId:"chapterContent",pageId:"11-4",name:"Protocol 11.3: Testing and Verification of Potential Lysogens: Liquid Phage Release"},{id:"5",templateId:"chapterContent",pageId:"11-5",name:"Protocol 11.4: Sensitivity Assay"},{id:"6",templateId:"chapterContent",pageId:"11-6",name:"Protocol 11.5: Host Range Assay"}]},{id:"12",templateId:"toc",pageId:"12",name:"Chapter 12: Toolbox",img:"toolbox.svg",imgtext:"Toolbox",child:!0,childElements:[{id:"1",templateId:"chapterContent",pageId:"12-1",name:"Chapter  12: An Overview"},{id:"2",templateId:"chapterContent",pageId:"12-2",name:"Protocol 12.1: Making Top Agar (1X Middlebrook Top Agar)"},{id:"3",templateId:"chapterContent",pageId:"12-3",name:"Protocol 12.2: Plaque Streak Plates"},{id:"4",templateId:"chapterContent",pageId:"12-4",name:"Protocol 12.3: Taking Plaque Pictures"}]},{id:"13",templateId:"chapterContent",pageId:"13",name:"Preview - Introduction to Bioinformatics",img:"bioinformatics.svg",imgtext:"Bioinformatics",child:!1},{id:"14",templateId:"recipe-toc",pageId:"14",name:"Recipe Cards",img:"recipe-cards.svg",imgtext:"Recipe Cards",child:!1,appendix:!0,appendixElements:[{id:"1",templateId:"toc",pageId:"14-1",name:"Recipes for Solutions",img:"recipe-cards.svg",imgtext:"Recipe Cards",child:!0,childElements:[{id:"1",templateId:"chapterContent",pageId:"14-2",name:"AD Supplement"},{id:"2",templateId:"chapterContent",pageId:"14-3",name:"Calcium Chloride (1 M)"},{id:"3",templateId:"chapterContent",pageId:"14-4",name:"Calcium Nitrate (0.8 M )"},{id:"4",templateId:"chapterContent",pageId:"14-5",name:"Carbenicllin Stock Solution"},{id:"5",templateId:"chapterContent",pageId:"14-6",name:"Cycloheximide Stock Solution"},{id:"6",templateId:"chapterContent",pageId:"14-7",name:"Dextrose (40 %)"},{id:"7",templateId:"chapterContent",pageId:"14-8",name:"Enrichment Broth"},{id:"8",templateId:"chapterContent",pageId:"14-9",name:"EDTA"},{id:"9",templateId:"chapterContent",pageId:"14-10",name:"Glycerol (40 %)"},{id:"10",templateId:"chapterContent",pageId:"14-11",name:"Isopropanol (80 %)"},{id:"11",templateId:"chapterContent",pageId:"14-12",name:"Magnesium Chloride (1 M)"},{id:"12",templateId:"chapterContent",pageId:"14-13",name:"Magnesium Sulphate (1 M)"},{id:"13",templateId:"chapterContent",pageId:"14-14",name:"Nuclease Mix"},{id:"14",templateId:"chapterContent",pageId:"14-15",name:"Phage Buffer"},{id:"15",templateId:"chapterContent",pageId:"14-16",name:"Phage Precipitation Solution"},{id:"16",templateId:"chapterContent",pageId:"14-17",name:"SDS (10 %)"},{id:"17",templateId:"chapterContent",pageId:"14-18",name:"Tris Borate EDTA (TBE)"},{id:"18",templateId:"chapterContent",pageId:"14-19",name:"Tris, pH 7.5 (1 M)"},{id:"19",templateId:"chapterContent",pageId:"14-20",name:"Tween80 (20 %)"},{id:"20",templateId:"chapterContent",pageId:"14-21",name:"Uranyl Acetate Stain"},{id:"21",templateId:"chapterContent",pageId:"14-22",name:"Water"},{id:"22",templateId:"chapterContent",pageId:"14-23",name:"Zinc Chloride"}]},{id:"2",templateId:"toc",pageId:"14-24",name:"Recipes for Media",img:"recipe-cards.svg",imgtext:"Recipe Cards",child:!0,childElements:[{id:"1",templateId:"chapterContent",pageId:"14-25",name:"7H9 Liquid Medium(Neat)"},{id:"2",templateId:"chapterContent",pageId:"14-26",name:"7H9 Liquid Medium(Complete)"},{id:"3",templateId:"chapterContent",pageId:"14-27",name:"7H9 Liquid Medium(Complete with Tween80)"},{id:"4",templateId:"chapterContent",pageId:"14-28",name:"7H9 Top Agar (1X)"},{id:"5",templateId:"chapterContent",pageId:"14-29",name:"7H9 Top Agar (2X)"},{id:"6",templateId:"chapterContent",pageId:"14-30",name:"Luria Agar (L-Agar) Plates"},{id:"7",templateId:"chapterContent",pageId:"14-31",name:"Media Supplement for Streptomyces phage"},{id:"8",templateId:"chapterContent",pageId:"14-32",name:"Nutrient Broth"},{id:"9",templateId:"chapterContent",pageId:"14-33",name:"Nutrient Broth(with Supplements for S.griseus)"},{id:"10",templateId:"chapterContent",pageId:"14-34",name:"Nutrient Agar Plates(with Supplements for S.griseus)"},{id:"11",templateId:"chapterContent",pageId:"14-35",name:"PYCa Agar Plates"},{id:"12",templateId:"chapterContent",pageId:"14-36",name:"PYCa Liquid Medium"},{id:"13",templateId:"chapterContent",pageId:"14-37",name:"PYCa Top Agar"}]},{id:"3",templateId:"toc",pageId:"14-38",name:"Propagating Bacteria and Phage",img:"recipe-cards.svg",imgtext:"Recipe Cards",child:!0,childElements:[{id:"1",templateId:"chapterContent",pageId:"14-39",name:"Growing bacteria from a frozen stock"},{id:"2",templateId:"chapterContent",pageId:"14-40",name:"Growing liquid cultures of bacteria from a single colony"},{id:"3",templateId:"chapterContent",pageId:"14-41",name:"Preparing a culture without Tween80 from one with Tween80"},{id:"4",templateId:"chapterContent",pageId:"14-42",name:"Preparing frozen stocks of bacteria"},{id:"5",templateId:"chapterContent",pageId:"14-43",name:"Preparing Additional Phage Lysate & Long-term Storage"},{id:"6",templateId:"chapterContent",pageId:"14-44",name:"Received Biologicals from the Program? Follow these Next Steps."}]},{id:"4",templateId:"toc",pageId:"14-45",name:"Notes",img:"recipe-cards.svg",imgtext:"Recipe Cards",child:!0,childElements:[{id:"1",templateId:"chapterContent",pageId:"14-46",name:"Notes about Recipes"},{id:"2",templateId:"chapterContent",pageId:"14-47",name:"Autoclave Setting"}]}]}],chData:{1:{content:"<p>Welcome to the SEA-PHAGES (Science Education Alliance-Phage Hunters Advancing Genomics and Evolutionary Science) program, an international project hosted by the Howard Hughes Medical Institute in collaboration with HHMI Professor Graham Hatfull at the University of Pittsburgh. You are joining an adventure to explore the bacteriophage population and discover the evolutionary mechanisms that give rise to such a delightfully diverse and dazzling array of viruses. </p> <p>The phage discovery and genomics platform has two primary components. The first is the isolation, purification, and amplification of new phages from environmental samples by using a specific bacterial host, coupled with electron microscopy and DNA isolation and analysis. The second component is the bioinformatic dissection of phage genomes using computational programs to identify genes, regulatory elements, and other genomic features. The ensuing insights typically present a variety of hypotheses than can be tested computationally or with molecular genetic approaches.</p> <p>This phage discovery guide contains a complete set of detailed protocols for each of the approaches required for the de novo isolation and characterization of bacteriophages, along with supporting material explaining the experimental approaches and biological reasoning.</p> <h2>Expanding Scientific Knowledge</h2> <p>The bacteriophage population is vast, dynamic, and old, so it is not surprising that it is so diverse. There are estimated to be 1031 phage particles in the biosphere, but fewer than about 3,000 have been characterized genomically. As you venture to discover new phages, you will experience one of the hallmarks of authentic scientific research—scientists don’t know what the answer is before doing the experiment! Therefore, as a scientist, you will need to formulate hypotheses, design experiments, and interpret the data. You will learn that not every experiment works the first time. Sometimes they never work, and you need to practice your problem-solving skills to advance your research.</p> <p>This may sound daunting, but the SEA-PHAGES program provides many resources to help you, including this detailed guide. These resources include databases and websites, scientific articles and reviews, access to scientists with experience in this research area and, of course, the thousands of other faculty and students in the SEA-PHAGES program who are sharing in this journey of discovery.</p> <h2>How to Use This Phage Discovery Guide</h2> <p>This guide is organized into chapters that focus on individual stages of your quest to isolate and characterize new bacteriophages. Each chapter contains step-by-step protocols, lists of required reagents, and troubleshooting information. The chapters and protocols are organized in the order that you will commonly use them. Because each protocol corresponds to a stand-alone module, you may find it useful to sometimes pick and choose among the protocols most pertinent to the task you are undertaking. You will use some protocols more than once. In these cases the protocol is located in the chapter where it is first encountered, and you will have to return to it when needed. Chapter 12, the Toolbox, contains some additional protocols you may or may not need. When used as an online document, the protocols are connected by hyperlinks in the text, in the table of contents, and in the flow charts.</p>We strongly recommend that when you start a protocol for the first time you read the entire protocol before you begin. There are helpful tips and troubleshooting options at the end of each protocol.<p> <h2>Getting Started</h2> </p> <p>Now it’s time to get started with your phage isolation! We wish you luck and hope that you embrace the inevitable challenges and enjoy the thrill of discovery and contributing to the body of scientific knowledge. The flow chart below provides a framework for how the component pieces of the program fit together, and we encourage you to refer back to this frequently as a reminder of where you are in the big picture.</p> <div class = 'img-wrap'> <div class = 'img-container'> <img  src = './assets/images/figure-1-0-1.jpg' alt  = 'A flowchart with six phases, each with their own flowchart: The isolation phase has an arrow pointing to the purification phase which has an arrow pointing to the amplification phase which points to both the extraction phase and microscopy phase. The  extraction has an arrow pointing to the characterization phase.' /> </div>   <div class = 'img-hr'></div> <div class = 'title-container'>Figure 1.0-1. Flow chart of Phage Discovery experiments </div> </div>"},2:{content:"<ul><li class = 'list'>Chapter 2: An Overview</li><li class = 'list'>Protocol 2.1: Aseptic Technique Exercise</li><li class = 'list'>Guidlines for working with bacteria</li></ul>"},3:{content:" <h2>Phages Are Viruses</h2> <p>Bacteriophages (also known as phages) are viruses that infect bacterial hosts and have properties shared by all viruses. First, they are unable to replicate by themselves. They require a bacterial host to reproduce, taking advantage of the host’s cellular machinery and redirecting it toward viral reproduction. Second, like all viruses, phages are specific for particular hosts. The host range can be restricted to a single bacterial strain, or it can span different bacterial species, even genera. However, there are few—if any—phages with a host range that spans different bacterial orders, and having a host range that spans different phyla is even less likely.</p> <h2>The Bacteriophage World</h2> <p>Bacteriophage particles comprise the majority of all biological entities in the biosphere. The global phage population is not only vast (an estimated 1031 particles) but also highly dynamic and very old. It is also highly genetically diverse. We understand a few phages in great detail, but our knowledge  of the phage population at large is murky at best. So, three general questions about phages emerge: What are they? How are they related to each other? How did they get to be that way? Exploring their viral diversity and the evolutionary mechanisms that give rise to this diversity are the central scientific issues addressed by the SEA-PHAGES program.</p> <h2>100 Years and Counting</h2> <p>Bacteriophages were discovered about 100 years ago (1915–1917), and two of the major contributors to that discovery were Frederick Twort and Felix D’Herrelle. Others may have observed bacteriophage behavior before this, and even Twort did not fully recognize the nature of the viral properties he witnessed. D’Herrelle characterized the particulate nature of bacteriophages in part through the plaque assay he developed, which is still in widespread use today.</p> <p>This assay provides a simple way to visualize the presence of phage particles in a sample. In brief, a single phage particle present within a thin layer, or “lawn,” of bacteria growing in a petri dish can propagate in the bacteria to form a clearing, called a plaque (Figure 3.0-1). In essence, a plaque is caused by the destruction of bacteria cells by phage particles. Plaque formation requires multiple rounds of phage infection and growth, and each plaque can contain 108 or more particles by the time the bacterial cells have formed a confluent lawn. These early studies by D’Herrelle and colleagues were abstract but experimentally sound, and in the late 1940s the morphologies of various phage types were finally visualized with the invention of the electron microscope.</p>  <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-3-1-1.png' alt = 'Picture of a an opaque circular dish covered with transparent round spots.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 3.0-1. Bacterial lawn with visible plaques.</div></div><h2>Simple Is Beautiful</h2><p>Bacteriophage particles come in many shapes and sizes, but the majority are tailed viruses containing double-stranded DNA (dsDNA) genomes (the order Caudovirales). In these phages the linear DNA genome is contained within a protein shell (the <strong>head</strong> or <strong>capsid</strong>), which is attached to a <strong>tail</strong> (Figure 3.0-2). Phages in other orders may have either single-stranded DNA (ssDNA) or RNA and can be enveloped with a lipid membrane, have one or more than one chromosome, or have a variety of other configurations. Phages may have as few as a handful of genes or more than 500, but the dsDNA-tailed phages typically have 50–250 genes. (Note that this is much fewer than their bacterial hosts, which may have more than 6,000 genes.)</p><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-3-0-2.jpg' alt = 'Illustration of a phage with a multi-faceted geometric head with a long strand extending below labeled the tail and a cone shape at the bottom of the tail labeled tail fibers.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 3.0-2. Phage structure. Phages are composed of a head and tail, with tail fibers that are used for host cell attachment.</div></div><h2>Eating Away at Bacteria?</h2><p>The term “bacteriophage” was coined from the Greek word phaegin (φαγεῖν), meaning “to eat,” because phages appear to “eat” bacteria when they infect them. This is not strictly true, although many phages do repurpose and recycle cellular components and use them to make more of themselves. Typically, infection of a single bacterial cell by a single phage results in the generation of many progeny phage particles. This is accompanied by a complete breakdown of cell integrity, called lysis, that releases the new phage so they can disperse and find new cells to infect. The process by which a phage generates new phage particles via infection and lysis of the host cell is called lytic growth and is explained in greater detail below.</p><h2>Lytic Growth of Bacteriophages</h2><p>The process of lytic growth can be broken down into several phases (see Figure 3.0-3). The first phase is adsorption, in which the phage recognizes the bacterial host cell and attaches to its surface using its tail fibers. Next, the phage’s DNA is injected into the host cell while the phage’s protein structure (head and tail) remains attached to the outside of the cell. The phage’s DNA passes from the head, through the tail, and across the host’s cell wall and cell membrane into the cytoplasm. After the linear phage DNA has completely entered the cytoplasm, it circularizes to avoid the host cell’s defenses against linear DNA.</p><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-3-0-3.jpg' alt = 'A circular diagram with 6 illustrated phases with arrows pointing from phase 1 to 2, 2 to 3, 3 to 4, 4 to 5, 5 - 6, and 6 back to 1. The phagses are labeled  1. phage adsorption and D-N-A injection, 2. phage D-N-A circularization, 3. phage D-N-A replication, 4. phage heads and tails production and more D-N-A replication, 5. D-N-A packaging and phage assembly, 6. cell lysis and phage release'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 3.0-3. The lytic life cycle of phage. (1) A phage attaches and adsorbs to a host cell using its tail fibers and injects its linear chromosome into the bacterium. (2) The phage chromosome circularizes inside the host cell. (3) The phage DNA is replicated. (4) Phage heads and tails are produced. (5) New virions are assembled. (6) The bacterial host cell is lysed and the new virions released.</div></div><p>Once the phage DNA reaches the cytoplasm, expression of some of the phage genes begins. In some phages, this can happen before the genome has even fully entered the cell! These genes that are transcribed and translated first (see Preview: Introduction to Bioinformatics for a detailed description of these processes) are called “early” genes. Proteins expressed from early genes may include polymerases necessary for phage DNA replication, transcription factors necessary for the expression of other phage genes, and enzymes that degrade the host cell’s DNA.</p><p>Following the early gene expression, a new set of phage genes is expressed. These are known as “late” genes. Late-gene products include the structural proteins that comprise the new phage particles, or<strong> virions</strong>. The protein composition of each phage particle includes approximately 20 different types of proteins, each of which is incorporated into the virion particle in a specific order and copy-number. The copy-number is the total number of molecules of the same protein found in a single virion particle. Phage capsid proteins have a very high copy number. A phage with a genome size of 50 kb will have ~400 copies of the capsid protein per head, while the copy-number of the tail fiber protein can be quite low, with only 3–18 per tail. Usually, phage heads and tails are assembled as separate substructures, then DNA is packaged into the heads, and tails are attached (Figure 3.0-3). Finally, cell lysis occurs when phage-encoded enzymes degrade the cell wall and the cell membrane breaks. A single infected cell may yield 100 or more phage particles upon lysis, a parameter defined as the ”burst size” of the phage.</p> <h2>Virulent Phages Versus Temperate Phages</h2><p>Virulent phages (also known as <strong></strong>lytic phages</strong>) are those that use the lytic cycle, and only the lytic cycle, to reproduce. However, some phages have the ability to choose a different life cycle—the lysogenic cycle. These phages are called <strong> temperate phages</strong>. Temperate phages differ from lytic phages in that following DNA injection there are two possible outcomes of infection (Figure 3.0-4). In approximately 80–90 % of single-cell infections, the phage will proceed through the lytic cycle and create many progeny phages. However, in the other 10–20 % of infections, the phage will enter the lysogenic cycle (Figure 3.0-5). In the lysogenic cycle, the genes required for the lytic cycle are repressed, rather than expressed, and the phage genome becomes stably maintained within the cell, even through many subsequent generations of cell division. The phage genome within the cell is called a <strong>prophage</strong>, and a cell that carries a prophage is called a <strong>lysogen</strong>. A common mechanism used by a prophage to ensure its stability is by integrating into the host chromosome through a process known as homologous recombination.</p><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-3-0-4.jpg' alt = 'A flowchart of 4 illustrated phases depicting the two paths of a temperate phage. A red phage is shown injecting red linear D-N-A into a cell with coiled D-N-A, with an arrow pointing to the next phase in which the red D-N-A is circular inside the cell. Next, two arrows point to two different phases.In one phase, 4 red phages are bursting out of a cell. In the other phase, the red D-N-A becomes part of the coiled D-N-A of an intact cell.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 3.0-4. Decision of a temperate phage. Following DNA injection into a host cell a temperate phage may enter the lytic cycle or become a prophage (shown in red) and repress its lytic functions. This decision is controlled through a set of genes and regulatory sequences that are collectively called a “genetic switch”.</div></div><p>Once integrated, the prophage is replicated by the host DNA polymerases and passed to both daughter cells during normal cell division. In some temperate phages, the prophage does not integrate into the host genome but instead is established as a large extrachromosomal plasmid. These prophages encode and express several “partitioning” proteins that ensure each daughter cell receives a copy of the prophage plasmid.</p><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-3-0-5.jpg' alt = 'An alternative flowchart with 7 illustrated phases describing the life cycle of a temperate phage:  1. phage adsorption and D-N-A injection, 2. phage D-N-A circularization. An alternate path outside of the lifecycle leads to the maintenance of lysogeny. 3. phage D-N-A replication, 4. phage heads and tails production and more D-N-A replication, 5. D-N-A packaging and phage assembly, 6. cell lysis and phage release.  Arrows pointing from phase 1 to 2, 2 to 3 and to the maintenance of lysogeny, 3 to 4, 4 to 5, 5 - 6, and 6 back to 1.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 3.0-5 Life cycle of a temperate phage. (1) A phage attaches to a host bacterium using its tail fibers and injects its linear chromosome. (2) The phage chromosome circularizes and it is either maintained in the host as a prophage (shown here integrated into the bacterial chromosome in red) or enters the lytic cycle. The expression of the lytic genes is prevented through constant expression of the immunity repressor protein (shown as yellow rectangles). When the lysogen is stressed, the prophage may excise and begin the lytic cycle.  (3) The phage DNA is replicated. (4) New tail and capsids are produced. (5) New virions are assembled. (6) The bacterial cell is lysed and the new virions are released.</div></div><p>In the lab, it is sometimes possible to distinguish between lytic and temperate phages by examining the differences in the appearance of the plaques that the phages form on bacterial lawns. Lytic phages form clear plaques (Figure 3.0-6 A) as a consequence of all of the bacterial cells being infected and killed. In contrast, temperate phages form cloudy or “turbid” plaques (Figure 3.0-6 B) that contain a mixture of killed cells and lysogens. However, it is important to note that other factors play a role in the appearance or “morphology” of a plaque, and there are phages that are NOT temperate that also produce cloudy plaques. Likewise, mutants of temperate phages sometimes lose their ability to form lysogens and will only produce clear plaques. Without genomic information, it can be difficult to distinguish between all these possibilities. See Chapter 11 for more detailed information.</p><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-3-0-6.png' alt = 'An image with two pictures labeled A and B. Picture A shows a circular opaque container covered with transparent round spots. Picture B shows a circular opaque container covered with less opaque round spots.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 3.0-6. Two plaque morphologies, one clear (A) and the other cloudy (B). Frequently, lytic phages will produce clear plaques (as in A) while temperate phages produce cloudy or “turbid” plaques (as in B). Note that plaque appearance or “morphology” is influenced by many factors, including media type, number of cells and phage present, and incubation time and temperature.  In the mycobacteriophage collection there are many examples of phages that produce plaques that are not clear and are not able to form lysogens.</div></div><p> While only a fraction of the global phage population has been sequenced and examined, genomic analysis of thousands of bacterial host strains shows that many carry prophages. And some strains contain dozens of different prophages simultaneously, all integrated at different locations in the chromosome. Some of these prophages encode toxins or other genes that confer pathogenicity to the host bacterium (cholera is a good example). This evidence suggests that the temperate phage life cycle is possibly the “norm.”</p><h2>Phage Host Range</h2><p>The host range of a particular phage is a description of the range of bacterial types that can be infected by that phage. Many phages have narrow host ranges and can infect only a single strain of bacteria and no other strains in the same bacterial species. Other phages have broad host ranges and can infect bacteria of different species, or even different genera.</p><p>There are many factors that influence whether a particular phage can infect a particular bacterium. These determinants of host specificity include the presence of a receptor in the host’s cell wall that the phage can recognize and adsorb to. The phage also has to be able to interact with the host’s cellular machinery, including the host transcription and translation systems. Furthermore, it is relatively easy for phages to alter their host ranges, sometimes with as little as a single base mutation in their genome sequence. So unsurprisingly, host range mutants occur frequently in some phages (as often as 1 in 100,000 or 1 in 1,000,000 particles in a lysate). There is usually no easy way to predict with confidence the host range from a phage’s genomic sequence alone, so the host range needs to be determined experimentally.</p><h2>The Great War</h2><p>The phage population is not only vast but also highly dynamic, with the entire global phage population predicted to turn over in a matter of days. The phage-bacteria relationship involves constant selective pressure for bacterial hosts to become insensitive to phage infection, along with the necessity for phages to coevolve so they always have hosts that can support their replication. It is this struggle, perhaps going on for several billion years, that gives rise to such a diverse population. There are many known ways in which bacteria can become insensitive to infection, and there are likely many more ways to be discovered. The known ways include loss or alteration of the host receptor, host restriction–modification systems, superinfection immunity, CRISPR systems, and surface exclusion. There are also many examples of abortive infection systems in which a bacterial host cell sacrifices itself to prevent a phage particle from reproducing, thus sparing other bacterial cells nearby. These are described in more detail below and in Chapter 11.</p><h2>Mechanisms of Insensitivity: Immunity, Exclusion, and Resistance</h2><h3><i>Immunity and Exclusion</i></h3><p>Superinfection <strong> immunity</strong> refers to the ability of a lysogen to resist infection from an invading phage by repressing the invader’s lytic genes. A lysogen’s ability to resist infection is mediated through three processes: the constitutive expression of the immunity repressor protein from the prophage, its subsequent binding to the prophage DNA, and the repression of transcription of the lytic genes. The immunity repressor protein can also bind to the DNA of phages that attempt to infect the lysogen (“superinfection”) and repress expression of the lytic genes from the invader’s injected DNA, granting the lysogen “immunity.”</p> <p>In addition to repressor-mediated superinfection immunity, a prophage in a host cell can confer insensitivity to phage infection though exclusion. <strong> Exclusion </strong> refers to when prophage-encoded and expressed proteins alter the host cell surface in such a way that other phages are “excluded” from absorption and DNA injection. </p> <h3><i>Resistance</i></h3><p>Bacterial resistance to phage infection describes the insensitivity of a host cell to infection due to mutation or loss of the phage receptor or through the action of one or more bacterial defense systems. The best characterized systems are restriction, abortive infection, and CRISPR, however, there are likely many more yet to be discovered.</p><p>Bacterial restriction systems were first discovered through the observation that a particular phage was “restricted” from infecting a related strain of its preferred host. Upon investigation, it was discovered that the host encoded an endonuclease that cut double-stranded DNA at a specific base pair sequence. The host’s DNA was protected from digestion at these sequences by the addition of methyl groups to its DNA. Subsequent investigations revealed that there are a vast number of different restriction systems that recognize different sequences; each system contains both an endonuclease and a DNA modification enzyme that recognize the same DNA sequence. Typically, these sequences are 4 to 6 bp in length. The nucleases have since been dubbed “restriction endonucleases” or just “restriction enzymes,” and they are commonly used tools in molecular biology labs.</p><p>Abortive-infection mechanisms allow cells to deliberately sacrifice themselves when infected by a phage. One example is the toxin-antitoxin system. During normal cell growth, both the toxin and the antitoxin are expressed, and the antitoxin prevents the toxin from killing the cell, usually by binding to it and down-regulating its expression. Phage infection can lead to cessation of host cell gene expression, including that of the toxin and antitoxin. However, in many of these systems, the antitoxin is less stable than the toxin and degrades more rapidly. The degradation of the antitoxin releases  the toxin and causes the death of the cell. In this way the cell sacrifices itself to prevent the phage from multiplying, sparing other bacterial cells in the population.</p><p>Clustered Regularly-Interspaced Palindromic Repeats (CRISPRs) and their Cas proteins (Cas, for CRISPR– associated) are found in prokaryotic genomes and are a means of phage defense through acquired immunity. The CRISPR array is constructed of short (30–40 bp), repeated palindromic sequences that are interspersed with short (~40 bp) “‘spacer’” pieces of DNA. These spacer pieces of DNA are identical to short DNA segments present in bacteriophage genomes. Through the actions of the CRISPR-Cas system, a host can use the library of spacer sequences present in the CRISPR array to prevent infection from a bacteriophage whose genome contains one of the protospacer sequences. Recently, CRISPR-Cas systems have been used for easy, specific editing of eukaryotic and prokaryotic gene sequences. Consequently, they have enormous potential as molecular tools in areas such as human health and food production. </p><p>Prophage genomes can also carry restriction systems, toxin-antitoxin genes, exclusion genes, and even CRISPR arrays. Because phages also have to counteract resistance mechanisms, they can encode antirestriction genes and anti-CRISPR genes, among many others.</p><h2>All the World’s a Phage</h2><p>When we compare the genomes of two phages that infect phylogenetically distant bacterial hosts, we find they typically share little or no sequence information. As more phage genomes are sequenced, the genetic connections between phages begin to emerge, suggesting that phages have shared, but have had unequal access to, a common gene pool. But our knowledge of phage genomics remains scant. Currently, (as of summer 2016) the sequences of only about 2,000 dsDNA phage genomes are available. This number is dwarfed by the more than 50,000 sequenced bacterial genomes.</p><p>Nonetheless, the genetic connections among phages can be elucidated by deepening our knowledge of the phages that infect particular hosts and extending this knowledge to phylogenetically-proximal hosts. This is the strategy employed by the SEA-PHAGES program, which has dug deeply into the phages that infect Mycobacterium smegmatis mc2155 and is now exploring the phages of other hosts within the phylum Actinobacteria.</p><h2>Is Any of This Useful?</h2><p>Bacteriophages have played central roles in what we know today about the fundamental basics of molecular biology. Because bacteriophages are simple, both structurally and genetically, and propagate quickly to large numbers, mutants with specific phenotypes can be isolated, mapped, and characterized. Then using the awesome power of phage genetics, we can address fundamental questions about gene structure, organization, regulation, and function. Because of the extensive biological novelty associated with this great genetic diversity, phage genetics continues to uncover new, interesting, and useful insights.</p><p>Phages have many potential applications, either as whole particles or as individual genetic components. For example, the therapeutic use of phages—which has captured the imagination of scientists for the past 95 years or so—remains an area of interest, especially in light of the emergence of antibiotic resistance among bacterial pathogens. Phages have also proved useful in disease diagnosis—from the early studies in phage typing (using phage host range to identify clinical isolates) to the use of reporter phages for rapid drug-susceptibility testing of clinical samples. The use of phage-derived tools in bacterial genetics is also extensive, including in the development of vector systems, the development of microbial computers, mutant construction, and transposon delivery.</p><h2>The Road Ahead</h2><p>The phage population is vast (1031 particles) and its genetic diversity is huge. With only about 2 x 103 genomes sequenced to date, we’ve yet to make significant progress in defining viral diversity. And because of its extensive genetic novelty and number of genomes, the phage population harbors the greatest reservoir of undiscovered genetic information in the biosphere. Elucidating the functions of all these yet-to-be-discovered genes remains a substantial challenge, but their potential for advances in biotechnology and medicine is beyond question. Get to work!</p>"},4:{content:"<div class = 'lab-basic-img'><img src = './assets/images/host-basics.svg' alt = 'Background image'/></div> <div class = 'title-border2'><p class = 'title2'>Chapter 4: Host Basics</p></div> <ul> <li class = 'list'>An Overview: Host Basics</li>  <li class = 'list'>Arthrobacter globiformis</li>  <li class = 'list'>Arthrobacter sp.</li>  <li class = 'list'>Gordonia rubripertincta</li>  <li class = 'list'>Gordonia terrae</li>  <li class = 'list'>Microbacterium foliorum</li>  <li class = 'list'>Microbacterium testaceum</li>  <li class = 'list'>Mycobacterium smegmatis</li>  <li class = 'list'>Tips & Hints</li> </ul>"},5:{content:"<div class = 'welcome-img'><img src = './assets/images/isolation.svg' alt = 'Background image'/></div>  <div class = 'title-border lab-basic'><p class = 'title'>Chapter 5: Isolation</p> </div> <ul> <li class = 'list'>An Overview: Phage Isolation</li>  <li class = 'list'>Protocol 5.1: Collecting Environment Samples</li>  <li class = 'list'>Protocol 5.2: Direct Isolation</li>  <li class = 'list'>Protocol 5.3: Plaque Assay</li>  <li class = 'list'>Protocol 5.4: Picking a Plaque</li>  <li class = 'list'>Protocol 5.5: Enriched Isolation</li>  <li class = 'list'>Protocol 5.6: Spot Test</li>  <li class = 'list'>Tips & Hints</li> </ul>"},6:{content:"<div class = 'welcome-img'><img src = './assets/images/purification.svg' alt = 'Background image'/></div>  <div class = 'title-border lab-basic'><p class = 'title'>Chapter 5: Isolation</p> </div> <ul> <li class = 'list'>An Overview: Phage Purifications</li>  <li class = 'list'>Protocol 6.1: Plaque Assay for Purification</li>  <li class = 'list'>Protocol 6.2: Serial Dilutions</li>  <li class = 'list'>Protocol 6.3: Collecting Plate Lysates</li>  <li class = 'list'>Protocol 6.4: Spot Titer</li>  <li class = 'list'>Protocol 6.5: Full Plate Titer</li>  <li class = 'list'>Tips & Hints</li> </ul>"},7:{content:"<div class = 'welcome-img'><img src = './assets/images/purification.svg' alt = 'Background image'/></div>  <div class = 'title-border lab-basic'><p class = 'title'>Chapter 5: Isolation</p> </div> <ul> <li class = 'list'>An Overview: Phage Amplification</li>  <li class = 'list'>Protocol 7.1: Making Webbed Plates from a Lysate of Known Titer</li>  <li class = 'list'>Protocol 7.2: Entering a Phage into the Actinobacteriophage Database</li>  <li class = 'list'>Protocol 7.3: Archiving your Phage Sample</li>  <li class = 'list'>Tips & Hints</li> </ul>"},8:{content:"<div class = 'lab-basic-img'><img src = './assets/images/purification.svg' alt = 'Background image'/></div> <div class = 'title-border2'><p class = 'title2'>Chapter 6: Phage Purification</p></div>This is for 8"},9:{content:"This is for 9"},10:{content:"This is for 10"},11:{content:" <h2> Chapter 10: Characterizing Phage Genomes by Restriction Enzyme Digests </h2>     <h3>Tips and Hints</h3> <ul> <li  > If uncut DNA from Protocol 10.1 appears to be degraded when run on an agarose gel (i.e. a smear without the discrete band typical of uncut phage genomic DNA), there may be nucleases persisting in your DNA prep. Nuclease in your DNA prep, if present, will be inactive because of the lack of cofactors required for activity. However, upon the addition of restriction enzyme buffer (i.e. the uncut control in Protocol 10.1), the necessary cofactors for nuclease activity are introduced. To test for the presence of nucleases in your DNA prep, include an additional control in Protocol 10.1 in which neither restriction enzyme nor buffer are included. </li> <li  > The percentage of agarose can be adjusted to resolve different sized bands: 0.8 % will resolve larger bands (greater than 10 kb), 2.5 % may help resolve the size of very small bands (less than 500 bp). </li> <li  > The voltage of the gel can be adjusted to run the gels faster or slower. If you run a gel above 100 Volts, the bands may smear. </li> <li  > Stains other than EtBr can be used for visualization of DNA bands and may be less toxic, as well as eliminate the use of a UV light box. These dyes are generally more expensive but the benefits may outweigh the cost. </li> </ul>"},12:{content:"This is for 13"},13:{content:"<p>Bioinformatics uses computers to understand biological data. It is a field that is rapidly becoming a critical component in all areas of biology and medicine—from ecologists who study populations and migration patterns to epidemiologists who study emergent diseases and assess threatsto public health. At the most basic level, the field of bioinformatics is aboutrecognizing patterns. We use bioinformatics in the SEA-PHAGES course to find patternsin the sequence of nucleotides in a phage’s genome, so we can predict where genes arelocated and better understand phage genetics.</p><p>Before we learn how bioinformaticscan help us study phage biology, we need to learn some basic information about genomes.The <strong> genome</strong> of an organism is the sum total of all its genetic information (its DNA). Therefore, we need to understand the molecular structure of DNA, how an organism uses its DNA,and how DNA is organized. Once we have these basics down, we can move on to a discussion of how bioinformatics can help us understand phage genomes.</p><h2><i>Components of dsDNA</i></h2><p>An organism’s genome is organized into discrete units called chromosomes. Unlike our own cells, which have two copies of each chromosome, bacteria and viruses have only one chromosome, or one set of chromosomes. In addition, viral genomes can be made of RNA or DNA and be double-stranded or single-stranded. To date, all phages isolated on hosts of the phylum Actinobacteria have a single chromosome constructed of double-stranded DNA (dsDNA), and the entire genome is contained within a single chromosome.</p><p>DNA is made of polymers of the nucleotides adenine (A), thymine (T), guanine (G), and cytosine (C). A nucleotide is a basic unit of DNA, composed of a sugar molecule, a phosphate group, and a base (A, T, G, or C), covalently bonded together (Figure 13.0-1). Individual nucleotides are connected to the nucleotides above them by a covalent phosphodiester bond, forming long polymers called nucleic acids. In dsDNA chromosomes, two of these long nucleic acid strands are connected by hydrogen bonds between bases to form a twisted structure known as a double helix (Figure 13.0-1). Adenines are always paired with thymines and guanines are always paired with cytosines (and vice versa) in a phenomenon known as complementary base pairing.</p> <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-12-1-1.png' alt = 'An illustration of the structure of D-N-A. The top of the illustration shows a twisted ladder with double-colored rungs. The lower part of the illustration zooms in on one section of the ladder to show the hydrogen bonds between the bases of adenine, guanine, thymine and cytosine and the covalent bonds with the sugars and phosphates.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 13.0-1. Structural model of DNA. In dsDNA the sugar and phosphate molecules form the “backbone,” while the bases (A, T, G, C) are hydrogen-bonded together down the center. The entire structure twists into a structure known as theDNA double helix.</div></div> <h2><i>The Central Dogma</i></h2><p>The central dogma of molecular biology describes the flow of biological information through the macromolecules of DNA, RNA, and protein (Figure 13.0-2). It has been a major tenet of molecular biology since the discovery of DNA’s structure in 1950s. First outlined by Francis Crick, the central dogma describes the varying ways in which information flows between nucleic acids and from nucleic acids to proteins, and it critically states that information does not flow from proteins to nucleic acids. Thus, information is typically stored as sequences of As, Ts, Gs, and Cs in a DNA genome, transcribed into an intermediary RNA molecule, and translated into a sequence of amino acids in proteins that perform either structural (e.g., phage tail fiber) or enzymatic (e.g., DNA polymerase) activities (Figure 13.0-3).</p> <p>While the main principles of the dogma—DNA acting as a template for RNA that in turn codes for proteins—remains true, discoveries over the past decades have expanded our understanding of the roles RNA plays in the cell. These include RNA’s ability to store information that can be used to generate new DNA during reverse transcription, to catalyze reactions as ribozymes, to regulate gene expression, and to aid in bacterial defense mechanisms against viruses.</p> <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-12-1-2.jpg' alt = 'A flowchart starting with D-N-A and a thick arrow pointing to R-N-A labeled transcription and a thick arrow pointing from R-N-A to protein labeled translation. There is a small arrow pointing back to D-N-A and an arched arrow above D-N-A  labeled replication. '/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 13.0-2. The central dogma of molecular biology.</div></div> <p><strong>Transcription</strong> is the process by which single-stranded RNA is generated using a DNA template and an enzyme called <strong>RNA polymerase</strong>. The resultant RNA is complementary in sequence to the template DNA, but in RNA the thymine base (T) found in DNA is replaced with the base uracil (U), and the sugar molecules in the backbone are different. </p><p>RNA is used during <strong> translation</strong> as a template for making proteins (messenger RNA [mRNA]), as a molecule to match a specific amino acid with a specific nucleotide sequence (transfer RNA [tRNA]), or as a key component of the ribosome (ribosomal RNA [rRNA]). In addition to RNA’s informational and structural roles, RNA’s chemistry supports a variety of important enzymatic activities. Many small RNA molecules also play key regulatory roles.</p><p>Ribosomes are macromolecular structures composed of proteins and rRNAs. They are highly abundant cellular complexes found in all domains of life. <strong>Translation</strong> is the synthesis of proteins by a ribosome using the mRNA sequence as a template. Only specific regions of DNA known as genes are transcribed and translated; other regions of DNA play regulatory roles controlling whether genes are transcribed or not.</p> <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-12-1-3.png' alt = 'A detailed illustration of with three rows labeled D-N-A, R-N-A , and protein, with D-N-A, R-N-A and protein sequence illustrated in each row. There is a circular arrow above the D-N-A sequence with the labeled replication and D-N-A polymerase. There is an arrow from the D-N-A sequence to the R-N-A sequence labelled transcription and R-N-A polymerase. There is an arrow from the R-N-A sequence to the protein sequence labelled translation and robosome complex.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 13.0-3. The central dogma. In biology DNA is transcribed to RNA, which is translated into proteins.</div></div> <h2><i>Genes</i></h2><p>A <strong>gene</strong> is defined as a unit of DNA on a chromosome that codes for a protein or for RNA. Transcription of a gene results in one of three types of RNA molecules: mRNAs, which will be translated into proteins; rRNAs, which are incorporated into ribosomes; and tRNAs, which become charged with specific amino acids for use in protein synthesis. Much of the information about genes presented in this chapter is based on what is known about bacterial genes, because they are well understood and share similarities with phage genes. After all, phages must hijack host bacteria transcriptional and translational machinery for the synthesis of their own RNA and proteins.</p> <p>For a gene to be transcribed and translated, it must contain the following:</p><ol><li>DNA sequences that signal the start and stop of <strong>transcription</strong>. These regulatory sequences are called promoters and terminators.</li><li>DNA sequences that signal the start and stop of<strong> translation</strong>. These are known as start (initiation) and stop codons.</li><li>The DNA sequence that will be translated into a protein. This sequence is called an open reading frame (ORF).</li></ol><p>The basic structure of a gene includes the promoter, the terminator, and the ORF (see Figure 13.0-4).</p> <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-12-1-4.png' alt = 'An illustration of a gene structure showing a thick, long oblong shape labeled O-R-F.  There is a thin black line to the left with blue stripes labeled promoter and a thin black line to the right with an orange stripe labeled terminator.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 13.0-4. Basic gene structure.</div></div> <h2><i>The Promoter and Terminator</i></h2><p>The <strong>promoter</strong> region of a gene is a sequence of nonprotein-coding DNA that is upstream (to the left) of the protein-coding region of the gene. The promoter is where<strong> RNA polymerase</strong>, the enzyme responsible for translating DNA into an RNA transcript, is recruited to the DNA. Bacterial promoters (and some phage promoters) contain two hexameric (six base pair) sequences referred to as the -10 and -35 motifs. These DNA sequences are approximately 10 and 35 base pairs upstream of the <strong>transcription start site (TSS)</strong> (Figure 13.0-5). The -10 and -35 sequences vary among different promoters, but they are all variations of a “consensus” sequence that reflects the most common bases at each of the positions. Typically, agreement of any specific promoter with the consensus sequence is a reflection of promoter strength (the frequency with which mRNA synthesis is initiated); that is, the closer the -10 and -35 sequences are to the consensus, the stronger the promoter.</p><p>Bacteria generally have several different types of promoters, which are recognized by different forms of the RNA polymerase containing variations of one of its subunits, called the sigma factor. In Escherichia coli, the predominant promoter types are recognized by sigma 70–associated RNA polymerases and have -10 and -35 consensus sequences of 5’-TATAAT and 5’-TTGACA, respectively. Many other bacteria, including Actinobacteria, share a similar set of promoters with identical, or very similar, consensus sequences. However, other promoters, including phage promoters, are still not well defined.</p><p>The <strong> terminator </strong>is the specific sequence where the RNA polymerase detaches from the template DNA strand, thus stopping transcription and releasing the new RNA.</p>  <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-12-1-5.png' alt = 'A more detailed illustration of the gene structure with the promoter region expanded to show the blue stripes as negative 35 consensus and negative 10 consensus and the black area next to the blue stripes as the transcriptional start site and the end of the black area as the shine-dalgarno box with sequence A-G-G-A-G-G. The yellow oblong shape labeled O-R-F is has a green stripe on the left labeled start codon and a red stripe on the right labeled stop codon. The orange stripe in the black section to the right of the O-R-F is labeled terminator.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 13.0-5. Detailed gene structure.</div></div> <h2><i>The Open Reading Frame</i></h2><p>The <strong>open reading frame (ORF)</strong> is the portion of the gene that encodes the protein. Proteins are composed of long polymers of amino acids. Each of the 20 different amino acids is indicated in DNA and RNA sequences by codons. A <strong>codon </strong>is a triplet of nucleotide bases that, when translated, indicates which specific amino acid should be added to a protein chain. The genetic code (Figure 13.0-6) is universal; that is, the same codons are used for the same amino acids across all organisms (with some exceptions). Each of the 20 common amino acids occurring in proteins is coded for by at least one triplet codon, but in most cases two, four, or six different triplet codons.</p><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/openreadingframe.png' alt = ''/></image></div><div class = 'title-container'><div class = 'img-hr'> </div>Figure 13.0-6. The genetic code. This table shows all of the possible codons and their corresponding amino acids with the single-letter amino acid abbreviations. Note that the three start codons are typed in red: ATG, GTG, and TTG. </div></div> <p>Because mRNA is translated using a triplet code and read by the ribosome three nucleotides at a time, the nucleotide sequence in mRNA (which is reflected in the DNA sequence) can be interpreted in three separate ways, or <strong>reading frames</strong>, depending on which nucleotide is used to begin translation. As illustrated in Figure 13.0-7A, if you begin with nucleotide 1 (indicated by the blue arrow), the codons can be grouped as shown in Row 1. If you begin with nucleotide 2 (indicated by the green arrow), the codons would be those in Row 2. And if you begin with nucleotide 3 (indicated by the red arrow),the codons would be those in Row 3</p> <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-12-1-7.png' alt = 'An image with two parts. Part A shows a strand of a D-N-A sequence at the top and three more rows of the same in which the sequence in writen in groups of 3 letters.  The first row has a blue arrow pointing to the first letter in the sequence, the second row has a green arrow pointing to the second letter in the sequence, and the third row has a red arrow pointing to the third letter of the sequence. Part B shows the resulting protein sequence based on these three readings.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 13.0-7. DNA reading frames. (A) Each strand of DNA has three different reading frames based on which nucleotide is used to start translation. (B) The resulting protein depends on which reading frame is used.</div></div><p>Remember that DNA is double-stranded, so if you have three ways to translate a top strand of DNA in the forward direction (left to right), you also have three ways to translate the complementary strand in the reverse direction (right to left) (Figure 13.0-7B). This means that dsDNA has six possible reading frames! </p> <p>If each strand of mRNA has three possible reading frames, how does a ribosome know which is the “right” reading frame to make a given protein? Translation begins when a ribosome is recruited to a strand of mRNA and recognizes a start codon. The <strong>start codon</strong> is the site at which the ribosome starts synthesizing the protein by adding amino acids in the order indicated by the mRNA template.</p> <p>  Bacteria and phages share with eukaryotes the use the codon AUG to initiate protein synthesis. But unlike eukaryotes, bacteria and phages sometimes use GUG and UUG (and very occasionally, other codons) to initiate translation. The frequencies with which AUG, GUG, and UUG are used varies enormously; in E. coli, AUG is predominantly used, whereas in mycobacteria, GUG is used almost as frequently as AUG, with UUG used in only about 5 % of genes. All three of these codons are used not only for translation initiation but also as “regular” codons within the gene bodies. However, translation initiation uses a special tRNA (tRNAfMet) that is devoted to initiation. As a consequence, the first amino acid in newly synthesized proteins is always methionine, regardless of whether AUG, GUG, or UUG is used. In some instances the N-terminal methionine is often removed.</p> <p> How does the protein synthesis machinery distinguish between a codon (such as AUG) used for initiation from one located within a gene body? The answer is that additional sequences are used to promote binding of ribosomes so that initiation can begin. In bacteria and phages there is a specific sequence present just upstream (3–12 nucleotides) of the initiation codon called the <strong>ribosome binding site (RBS)</strong> (also called the <strong>Shine-Dalgarno site</strong>). (See Figure 13.0-5.) It is a short sequence (~8 nucleotides) to which the mRNA can base pair with the 3’ end of the rRNA in the small ribosomal subunit (16S rRNA).</p> <p> Translation stops when the ribosome encounters a <strong>stop codon</strong>, which signals the end of the ORF (Figure 13.0-5). The stop codons UAA, UAG, or UGA (TAA, TAG, or TGA in the DNA sequence) do not code for amino acids, thereby arresting translation at the last codon before the stop codon. The stop codon should not be confused with the transcription terminator. An mRNA molecule is typically longer than just the coding sequence and so extends to the 3’ side of the stop codon.</p> <h2><i>Operons</i></h2> <p> Unlike eukaryotes, bacteria and bacteriophages organize many (but not all) of their genes into operons. An operon consists of two or more open reading frames (genes) that share a common promoter and a terminator to make up a single transcriptional unit (Figure 13.0-8). This means that one mRNA transcript will include two or more ORFs. These ORFs are then translated as separate proteins by ribosomes. The genes in bacteriophage genomes typically are organized into long operons, with only five or so promoters serving the approximately 100 genes in a 50 kb genome. This allows for the close packing of bacteriophage genes, with little intergenic space allocated for regulatory regions or transcriptional signals.</p> <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-12-1-5.png' alt = 'An illustration of a gene structure showing a thick, long oblong shape labeled O-R-F,  a thin black line to the left with blue stripes labeled promoter and a thin black line to the right with an orange stripe labeled terminator. The thick and long oblong shape labeled O-R-F is  divided into three sections labeled labeled O-R-F 1, O-R-F 2, and O-R-F 3. The blue stripes of the left are labeled promoter core and the black line to the left in labeled TSS.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 13.0-8. Operons contain multiple genes under the control of a single promoter, and they have a single TSS and terminator.</div></div> <h2><i>Coupled Transcription-Translation </i></h2>  <p>In eukaryotes, transcription occurs in the nucleus and mRNA molecules are transported across the nuclear membrane to the cytoplasm where translation occurs. In bacteria, there is no nuclear membrane, so translation begins as soon as the 5’ end of the mRNA is synthesized. Transcription and translation are therefore tightly coupled processes. An important consequence of this is that interrupting the process of translation can result in the premature arrest of transcription. This leads to the phenomenon called <strong>genetic polarity</strong> in which a mutation in one gene appears to inactivate the activity of additional downstream genes. This phenomenon only occurs within operons, usually as a result of a nonsense mutation (a base change converting a codon into a stop codon), an insertion, or a frameshift mutation, that interferes with translation of an upstream gene.</p> <h2><i>Protiens</i></h2> <p>Like nucleic acids, proteins are macromolecules. Within a living organism, proteins are involved in nearly every life process. Proteins have many different roles, including catalysis of reactions, structure, transport of other molecules, regulation, and cell signaling. Not all genes in an organism’s genome are constantly transcribed and translated, nor are the proteins expressed at the same level. Rather, organisms control which genes are expressed and to what degree, based on the organism’s needs at a given time.</p> <h2><i>Proteins Expressed from Phage Genomes</i></h2> <p>Bacteriophages vary widely, and like bacteria, they have genes for many different types of proteins, depending on what hosts they infect, what environments they exist in, and what processes they engage in. The phages that infect bacteria of the phylum Actinobacteria carry genes that encode proteins involved in</p> <ul><li>Bacterial infection (attachment and insertion)</li><li>Phage DNA synthesis</li> <li> Virion head and tail assembly</li><li> DNA packaging into phage particles</li><li>Bacterial lysis</li> <li>Establishment and maintenance of lysogens (temperate phages only)</li></ul> <p>Of course, phages carry many genes in addition to these, including tRNAs, nucleic acid modification enzymes, toxins, and a multitude of genes whose functions we do not yet understand. In fact, the majority of the genes in the genome of every phage (~75 %) encode proteins with no known function.</p> <h2><i> Phage Genomics</i></h2> <p>The study of genomics begins with a complete genome sequence. Phage genomes sizes vary from 15 to 250 kilobases (kb), or base pairs, long. When you receive your phage’s genome sequence from the DNA sequencing center, it is a long sequence of adenines, thymines, guanines, and cytosines. Although bacteriophage genomes are relatively small compared with bacterial or eukaryotic genomes, even 50,000 bases of sequence are too large and too complicated for a person to analyze easily without the aid of a computer. Computer algorithms make it possible to analyze a genome and predict gene locations and other interesting genome characteristics.</p> <h2><i>Genome Annotation</i></h2> <p>Genome <strong>annotation</strong> is the prediction of the genes and other features coded within a phage genome and their potential functions. This is typically the first task in genome analysis. Several guiding principles of annotation are used to predict genome features, and these will be discussed in much greater detail in the bioinformatics portion of the course.</p> <p>Although it is possible to use computer algorithms to predict the location of ORFs and their possible start and stop codons, each ORF needs to be carefully evaluated. Likewise, predicting the possible functions of the proteins encoded by the proposed genes—and their role in the biology of the phage—requires further study. Functional characterization must be performed in the lab and can involve genetic approaches such as constructing and characterizing mutants, isolating mutants with specific phenotypes, and mapping the mutations. Biochemical strategies can also be performed. These strategies include expressing and purifying proteins to determine their activities and using structural approaches to provide insights into structure-function relationships.</p> Bioinformatic analyses using computer programs can determine patterns in sequences and similarities to other known proteins. These programs are extremely powerful and many are quick and simple to use. For example, by using sequence alignment tools, it is possible to rapidly screen sequences for similarity to the many thousands of sequences in public databases. These similarities can then be used to predict functions to phage-encoded proteins. Predicted protein functions can provide substantial information about the biology and the life cycles of the phage.</p> <h2><i>Comparative Genomics</i></h2> <p> In addition to finding the features of a particular phage genome, we use comparative tools to examine the relationships between genomes, thereby gaining considerable insight into both gene function and genome evolution. Some phages are similar in terms of nucleotide sequence similarity, shared gene content (as determined by similar protein sequences of predicted genes), and genome architecture (genome size, transcriptional organization, and location of genes). These similar phages frequently (but not always) share biological properties such as host range or lifestyle. Careful examination of genome-alignment maps of similar phages can provide clues to genetic novelty that can be explored experimentally. Exploring the relationships between the groups of related genomes yields insights into gene flow, phage evolution, and the global population of bacteriophages.</p> <h2><i>Bioinformatics Tools</i></h2> <br><p> It is helpful to remember that the many computer programs available for genomic analyses are just tools in a large toolbox. You will need to decide which programs to use to address different issues and questions that arise. None of the programs are perfect, so a useful strategy is to use, whenever possible, multiple tools and then test whether they give consistent predictions. Keep in mind that when annotating genomes you are not establishing facts, you are making predictions about genome organization and function. There is no right or wrong answer, but there are ”good,”’ well-supported solutions and ”poor” solutions that lack strong support. Also, you should not assume that all the data available in either local or public databases are reliable. Much of the data may be good and well-supported, but some may not be. Information about a sequence may have been supported when the sequence was first characterized, but because databases have grown with new data and sequences, the information about a sequence in the database warrants reinterpretation.</p> <h2><i> Resources:</i></h2> <p>See <a href = 'https://seaphages.org/' target = '_blank'>seaphages.org </a>and<a href = 'https://phagesdb.org/' target = '_blank'> phagesdb.org</a> for more information on the software used in the bioinformatics portion of the course and instructions on how to download and use it.</p> "},14:{content:""},"2-1":{content:"<p    > Performing scientific research in a laboratory can be an exciting and rewarding experience. As a research scientist in this program, you will learn scientific techniques and protocols that are often used in microbiology and molecular biology research. Before you begin, however, you will need to understand the fundamentals of working in a lab that are critical to your safety and success. </p> <h2>Lab Safety</h2> <p    > While every laboratory has its own specific rules and regulations, there are some universal guidelines that you should always follow. They are designed to keep you and your lab mates safe and to comply with state and federal regulations. Please read the list below carefully and talk to your instructor about any specific rules that must be adhered to in your workspace. </p> <h3> <i>General Rules</i> </h3> <ul> <li>No eating or drinking.</li> <li>No open-toe shoes.</li> <li>Never work alone in a lab.</li> <li>Tie back long hair.</li> <li>Avoid wearing baggy clothing.</li> <li>Never leave a lit Bunsen burner unattended.</li> <li>Keep workspace clear of clutter.</li> <li>Minimize loud talking and distractions.</li> <li>Know the location of fire extinguishers, emergency eye wash stations, and emergency showers.</li> <li>Properly dispose of waste generated from an experiment. Not everything can go down the drain or in a garbage can.</li> <li>No mouth pipetting!</li> <li>Wear safety glasses at all times.</li> <li>Wear gloves when working with bacteria.</li> <li>Wash your hands before leaving lab.</li> <li>If you are not sure, ask first!</li> </ul> <h3> <i>Guidelines for working with bacteria </i> </h3> <p    > Exposure to the Actinobacteria species recommended for use in this course do not pose health risks to individuals with healthy immune systems. If an individual is immuno-compromised or pregnant, the individual should consult a physician to determine the appropriate level of participation in lab activities. The primary hazard of working with the Actinobacteria used in this course is from exposure via mucosal membrane routes (eyes, mouth and nose), broken skin, or ingestion. Therefore, you should wear safety glasses at all times, wear gloves when handling bacteria, avoid generating aerosols of bacterial cultures, and avoid touching your face, applying cosmetics, adjusting contact lenses, biting your nails etc., while in lab. Consult your instructor if you have health concerns about working with bacteria. </p> <p    > Most importantly, work responsibly! There is no way to predict all dangerous lab situations, but most can be avoided simply by using common sense, being thoughtful and deliberate, and paying attention to your surroundings. In the event of an accident, be sure to alert your instructor immediately so the proper course of action can be followed. </p> <h2>Record Keeping</h2> <h3> <i>Laboratory Notebooks</i> </h3> <p    > When performing scientific research it is important to keep a record of your work that follows the principle of autonomous replication. Basically, anyone should be able to repeat your experiments by using your notes and obtain the same results. Therefore, maintaining a detailed and descriptive laboratory notebook is a key part of laboratory research. To fulfill the requirement for the principle of autonomous replication, your notebook should follow these basic guidelines: </p> <ol style = 'list-style-type:upper-alpha;' > <li  > The level of detail should be high enough that you can go back at a later time and troubleshoot your procedures if, for some reason, a procedure does not work or yields questionable results. </li> <li  > Jargon, personal shorthand, and personal abbreviations should be kept to a minimum. </li> <li  > Drawings, figures, and tables are encouraged. They must contain enough information so that another scientist can interpret them without extensive reference to the text. </li> </ol> <p    > There are endless ways to format a good scientific notebook and different disciplines have different conventions. In fact, every professional scientist develops a personal notebook style over time. Below is an outline of the minimum information required in a laboratory notebook. You may be asked to follow your instructor's stylistic preferences, or you may be able to organize your notebook in your own way. Regardless of the format you follow, it is imperative that you write in your lab notebook as you perform the procedure every time you are in the lab. Memory is not reliable, and experimental data and results without sufficient documentation are not trustworthy and must be discarded. </p> <h3> <i>Laboratory Notebook Outline</i> </h3> <ol style = 'list-style-type:upper-alpha;'> <li  > <strong>Table of Contents.</strong> You should leave 1–3 pages at the beginning of the notebook blank to accommodate a table of contents. </li> <li  > <strong>Date.</strong> Date each page of the notebook at the beginning of the entry. It is imperative that you record everything in your notebook as you do it. </li> <li  > <strong>Title.</strong> This should explain, in three or four words, the focus of the experiment. The title of the experiment is not necessarily the title of the protocol being followed, since multiple procedures may be used to complete a single experiment. </li> <li  > <strong> Aims and Purpose.</strong> This is a thumbnail sketch of the reason you are performing the protocols and experiments for the day. It should stand alone as your rationale. In other words, anyone should be able to open your notebook to any experiment and understand the “why” of the laboratory work performed. </li> <li  > <strong>Procedures and protocols.</strong> This should be a comprehensive, accurate, and detailed step-by-step accounting of your procedures. Even if you are repeating a protocol without any changes, write out the exact steps. Use a bulleted list that outlines the action items used to complete the experiment. It is recommended that you write out this list before beginning the procedure so you know what steps you will be doing, and so you can check them off as you proceed and note any changes. </li> <li  > <strong>Results.</strong> <ol> <li  >Primary data (e.g., numerical values, photographs, printouts, observations, drawings) are results of an experiment.</li> <li  >Negative results are results too! Be sure to include them.</li> <li  >All results must be described comprehensively and accurately.</li> <li  >Any loose data (e.g., photographs, printouts) must be securely taped into the notebook, dated, and properly labeled.</li> <li  >Anyone should be able to read the results of an experiment and know exactly what happened.</li> <li  >Data tables are labeled with a descriptive legend above the table.</li> <li  >Figures are accompanied by a descriptive legend below the figure.</li> </ol> </li> <li  > <strong>Analysis and interpretation.</strong><ol> <li>The results of an experiment must be analyzed and an objective interpretation documented in the notebook. Even if the results indicate a simple “yes” or “no” answer, it is important to state that explicitly in this section.</li> <li>Report any unexpected findings or problems during the course of the experiments. This can act as a rationale for additional experiments, for changing the protocol or materials, or for altering the path your research has been following.</li> <li>Interpretation of data can sometimes be subjective, but your reasons for a particular interpretation <i>must</i> be stated here.<br> </li> </ol> </li> <li  > <strong>Future plans.</strong> On the basis of your interpretation of the data, you should end each experiment or day of recording in your notebook with a short outline of your next steps. The rationale for this should be clear in your “analysis” section and doesn’t need to be restated. This section will lead you and the reader to the next procedure with a clear idea of “what’s next,” and why you are proceeding in that direction. </li> </ol> <p    > As you keep your lab notebook for this course, it is important to note that when the scientific data (such as where your phage was found and how it was purified) are published, the source of the data (namely, your notebook) belongs to and must be retained by the institution for at least seven years. Your notebook should, therefore, be a document that can be read and understood for years to come, especially if the science surrounding your phage progresses from the point of discovery outlined in your notebook. </p> <h3> <i>Labeling in the Laboratory</i> </h3> <p    >Maintaining a good laboratory notebook isn’t the only important record keeping that happens in a lab. You also need to be able to identify your experimental samples when they are placed in an incubator or freezer with hundreds of other identical-looking samples. Just as with keeping a notebook, every scientist has his or her preferred way of labeling things like Petri plates, microcentrifuge tubes, and reagents—but here are some general strategies to follow.</p> <ol style = 'list-style-type:upper-alpha;' > <li  >Each set of tubes, plates, and bottles should be labeled with the following (examples in Figure 2.0-1) <ul type = 'disc'> <li  >Your name or initials</li> <li  >The date</li> <li  >The contents (i.e., DNA)</li> </ul> </li> <li  >Any prepared bottles of media or other solutions, tubes, cultures, or stocks that are exclusively yours should be labeled with <ul type = 'disc'> <li  >Your name or initials</li> <li  >The contents</li> <li  >The date(s) received, prepared, and opened</li> </ul> </li> </ol>    <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-2-1-1.jpg' alt = 'An image with two pictures labeled A and B. Picture A shows a circular covered opaque container with writing around the edge. Picture B shows a small tapered tube with a cap with writing on the cap and on the side of the tube.'/></image></div> <div class = 'title-container'>Figure 2.0-1. Examples of proper labeling. (A) Petri plates are labeled on the bottom (agar-side) to ensure that the label accompanies the experiment even if the lid is lost, displaced, or swapped with another plate. By labeling around the edge of the plate you can see the results without looking through the label. (B) Microcentrifuge tubes are labeled with an identifier on the top of the tube so it can be located when it is in a tube rack. Additional information should be written on the side of the tube, in the designated area, where the label will not rub off.</div></div><h2>Preventing Contamination</h2> <p    >Microbes are everywhere: on lab surfaces and equipment, on human skin and hair, and in the air. In a microbiology laboratory we use materials and procedures to ensure that we grow only the bacteria and phages we are interested in. If we pick up other bacteria, fungi, or phages from our surrounding environment we say that we have “contamination.”Sterilization and aseptic technique help avoid contamination.</p> <p    >“Sterile” materials have been rendered free of biological organisms—including phages,bacteria, and fungi. The most common method for sterilizing commonlab materials and reagents, such as phage buffer, bacterial growth media, plastic tips, and glass tubes, is to use an autoclave, a machine that uses pressure and steam to heat materials to 121 °C and 18 psi. Materials that have been autoclaved are frequently marked with special tape that appears white before autoclaving, and striped afterward. Alternatively, some liquids that do not tolerate extreme heat can be passed through a filter (“filter-sterilized”) to physically remove cellular organisms.</p> <p    >Sterilization of larger or sensitive surfaces (such as you!) requires alternative methods. Lab benches and equipment are routinely treated with “disinfectants”—chemicals that kill or inhibit growth of contaminants on surfaces. Common lab disinfectants used to remove bacteria are 70 % alcohol, 10 % bleach, or a phenolic like CiDecon™. Many phages are not sensitive to 70 % ethanol, so for disinfection of phage-contaminated materials bleach and phenolics are better choices. In addition, you should wash your hands thoroughly with hot water and soap before and after each lab class since you are probably the biggest source of contamination in the lab!</p> <p    >Aseptic technique refers to the best lab practices that minimize contamination of your sterile working materials and experiments. Many of these practices are just common sense: for example, make sure you declutter and disinfect your working space before you start an experiment. The classic key components of aseptic technique are described below and can be practiced in the <a class = 'chap-linking' tabindex = '0'    data-link-to = '2-2' >Using Aseptic Technique exercise in protocol 2.1</a>. Read this section and then do the exercise to learn the specific techniques.</p> <h3> <i>Work Area Preparation</i> </h3> <p    >When preparing for an experiment requiring aseptic technique, you want to organize your bench first. Remove any unnecessary clutter, and disinfect your workspace with 70 % ethanol. As the ethanol dries it dehydrates bacteria on the bench surface the same way hand sanitizer kills bacteria on your skin. Be mindful not to try to speed up the drying process by waving your hands around or blowing on the surface because this will only introduce additional contaminants. When working with certain bacteria and viruses, it is a good idea to use a second disinfectant containing phenol or quaternary ammonia compounds as well. Because the active ingredients in these supplementary disinfectants denature proteins, it is important to always wear gloves when handling them.</p> <h3> <i>Bunsen Burner Use</i> </h3> <p    >Classic aseptic technique requires the use of a Bunsen burner. Once lit, the flame should be adjusted so that blue “cone” can be seen in the flame. The tip of this cone is the hottest part of the flame. The function of the flame is to provide an updraft by heating the air (Figure 2.0-2). Because heat rises, airborne bacteria, spores, fungi, and dust particles are forced upward and away from your work area. Therefore, it is important to work inside this “zone,” being careful not to disturb the updraft with rapid movements that disturb the air around your bench. You also need to arrange your supplies to be close enough to the Bunsen burner to take advantage of the updraft. Until you are familiar with using a Bunsen burner, it is important to constantly be mindful of the flame. Even with an apparent blue cone the flame is hard to see and can be overlooked. Never leave a flame unattended in case a lab mate or instructor doesn’t see it and inadvertently reaches over or through it.</p>     <div class = 'img-wrap'><div class = 'img-container'><image src = './assets/images/figure-2-1-2.png' alt = 'An illustration of a vertical tube with a blue flame on top with arrows pointing away labeled updraft. A blue circle around the base of the tube is labeled aseptic area around flame.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 2.0-2. How a Bunsen burner creates an updraft to keep the area aseptic</div></div> <h3> <i>Aseptic Transfer of Materials and Reagents</i> </h3> <p    > When you move liquids from one vessel to another, follow three simple rules to maintain an aseptic environment. First, always flame the opening of glass bottles and test tubes every time you open them and again before you close them. The goal of flaming a glass vessel is not to heat the glass itself, but to heat the air inside the neck of the vessel. This will cause the air to rise and prevent airborne contaminants from falling inside. This requires one or two quick passes through the flame only because extended exposure to the flame can cause the glass to break. This is an example of classic aseptic technique. With today’s plastic components, however, flaming is not in widespread use. </p> <p    > The second rule is to never lay down a lid or a cap on the bench top. Because you will most likely be holding a pipette in your dominant hand, you will have to figure out how to hold the bottle and the lid in one hand, or some other arrangement. This is a feat of dexterity that has nothing to do with the size of your hands, and it comes with practice. Finally, the third rule to good aseptic technique is to never use a pipette for more than one transfer. After you pipette from one sterile vessel into another, ALWAYS get a new pipette or pipette tip, regardless of how sure you are that you didn’t contaminate it the first time. </p> <h3> <i>Bench Top Management</i> </h3> <p    > As noted above, you want to keep the supplies you are using within the aseptic field created by the Bunsen burner. In addition, you want to minimize clutter and keep your workspace well organized. If you have supplies you know you will need, keep them close to the aseptic zone, but out of the way until needed. Keep pipette tip boxes and microcentrifuge tubes closed at all times to prevent contamination. Wipe up any spills immediately with disinfectant. </p> <h3> <i>Disposal, and Cleanup</i> </h3> <p    > After you finish your work for the day and start cleaning your lab bench, you need to follow proper disposal protocols. Bacteria, either in liquid or on plates, should not be discarded in the sewer or garbage until treated properly. Follow the protocol outlined by your instructor for the disposal of old plates and culture tubes. When working with microorganisms, always disinfect your workstation with 70 % ethanol and a disinfectant like CiDecon, before leaving and wash your hands thoroughly. </p>"},"2-2":{content:" <p> <h2>Objective :</h2> Practicing aseptic technique </p> </div> <div  > <p> <hr><br><h2>Rationale:</h2> Aseptic technique is used to prevent contamination in microbiology experiments. The growth of contaminating bacteria, fungi, and viruses can be avoided by disinfecting your bench, creating an updraft with a Bunsen burner, and moving with care and deliberation. </p> </div> <div > <hr><br><h2>Supplies:</h2> <ul> <li >70 % EtOH</li> <li  >CiDecon™ or other disinfectant</li> <li  >Bunsen burner and striker</li> <li   >10 ml serological pipettes and pipettor</li> <li   >Bottles, culture tubes, conical tubes, microcentrifuge tubes, and water to practice sterile transfers</li> </ul> </div> <div  > <hr><br><h2>Procedure:</h2>   <ol style = 'list-style-type:upper-alpha;'> <li>Prepare your work area. <ol> <li>Tidy up your workbench by removing clutter, papers, bottles, etc.</li> <li>Using a squeeze bottle containing CiDecon™, dispense enough disinfectant to dampen the entire work surface. Be certain to wear gloves if the disinfectant is known to denature proteins, as it can be hazardous to your health.</li> <li>Using a paper towel, gauze pad, or Kimwipes™, wipe the entire surface, starting at the back, to moisten the entire work surface.</li> <li>Let the disinfectant evaporate—do not wipe dry.</li> <li>Repeat steps 2–4 with 70 % EtOH. There is no need to wear gloves when handling EtOH.</ol></li>  <li>Ignite the Bunsen burner. <ol> <li>Remove your gloves before working with an open flame.  <br> <strong>Important</strong>: It is generally recommended that gloves not be worn while working with an open flame. This is because gloves, when worn and accidentally placed too close to or within an open flame, may melt and cause severe burns to the skin.  <br><strong>Important</strong>: Note that your gloves are never sterile.</li> <li>Light the Bunsen burner, but only after the disinfectant has completely dried.</li> <li>Adjust the flame so that a blue “cone” can be seen in the flame. (The tip of this cone is the hottest part of the flame.)</li> </ol> </li> <li>Transfer 10 ml from a small sterile bottle to a sterile culture tube (Figure 2.1-1) <ol> <li>Arrange a 50 ml or 100 ml bottle containing sterile water and a sterile culture tube in your aseptic field.</li> <li>Loosen (but do not remove) the tube and bottle caps.</li> <li>Peel down the wrapper on the 10 ml serological pipette (as you would a banana) from the top (opposite the tip). Hold the flaps against the pipette in your nondominant hand. (If you write with your right hand, that hand is your dominant hand.)</li> <li>Using your other (dominant) hand, place the pipette on the pipettor and remove the wrapper. Do not let the tip of the pipette—which is sterile—touch any surface! Using your nondominant hand, unscrew the top of the bottle and leave it between your forefinger and middle finger (Figure 2.1-1G).</li> <li>Use you nondominant hand to pick up the bottle and pass the opening through the Bunsen burner flame two to three times. Do not linger over the flame, and do not heat the bottle or lip. This technique is called “flaming.”</li> <li>Immediately place the serological pipette into the bottle, and using the pipettor, draw up 10 ml.</li> <li>Flame the top of the bottle again and replace the cap, and set the bottle down.</li> <li>Using your nondominant hand, pick up the culture tube. Using the smallest finger on the hand holding the pipette, remove the top of the culture tube. (Alternatively, lift the cap off the tube with the fourth and middle fingers of your nondominant hand and then pick up the open tube with your index finger and thumb).</li> <li>Flame the opening of the culture tube and dispense the liquid from the pipette into the tube.</li> <li>Flame the culture tube again, and replace the cap, and set the tube down.</li> <li>Remove the pipette from the pipettor, and then discard the pipette. <div class = 'img-wrap'> <div class = 'img-container'> <img src = './assets/images/figure-2.1-1.jpg' alt = 'An illustration with 14 tiles labeled A through N showing the transfer of liquid samples using aseptic technique.'></div>   <div class = 'img-hr'></div> <div class = 'title-container'>Figure 2.1-1. Transferring liquid samples using aseptic technique. Before beginning, dispense ethanol or other disinfectant onto the surface of your bench (A) and wipe from back to front (B). Light your Bunsen burner (C). Prepare your materials, and loosen all bottle and tube caps (D). Peel down both sides of the pipette package (E) and hold both flaps while attaching the pipettor (F). Using your first and second fingers, remove the bottle top (G) and pick up the bottle with the same hand and “flame” (H). Remove your sample (I) and reflame and recap the bottle (J). Using the smallest finger of your pipettor hand, remove the top of the tube (K and L) and add your sample to the tube (M). Recap the tube before discarding the pipette (N). When finished, turn off your Bunsen burner, tidy up, and wipe down your bench. </div> </div></li> </ol> <li> Transfer 100 &micro;l from a microcentrifuge tube into a culture tube.<ol> <li>Arrange a microfuge tube filled with water and a culture tube in your aseptic field.</li> <li>Adjust the p200 pipettor to transfer 100 &micro;l. Then, holding your p200 in your dominant hand, open the pipette tip box and retrieve a sterile pipette tip. </li> <li>Using your nondominant hand, pick up the microcentrifuge tube and open it.</li> <li>Insert the pipette into the microfuge tube and draw 100 &micro;l into the pipette (to the first stop).</li> <li>Close the microfuge tube and return it to the rack.</li> <li>Using your nondominant hand, remove the cap from the culture tube and pick up the tube. (This can be done a number of different ways, practice with what works for you.) Be careful not to invert the pipettor or touch the tip of the pipette to anything in the process.</li> <li>Flame the culture tube.</li> <li>Insert the pipettor into the culture tube only as far as the sterile tip. Touch the tip to the side of the tube and expel the 100 &micro;l (depressing the plunger to the second stop) allowing the liquid to run down the side of the tube.</li> <li>Flame the culture tube and replace the cap.</li> <li>Eject the tip into the proper receptacle.</li> </ol></li>  <li>Practice the techniques above with a partner. Things to watch: <ul type = 'disc'> <li>Hand and finger placement</li> <li>Movement of pipette and tips</li> <li>Aseptic field location</li> <li>Placement and handling of tube and bottle caps</li> </ul> </li> </ol> </li> </ol> </div> <div  > <p> <hr><br><h3>Helpful Tips:</h3> </p> <ul> <li   >Never pass your hands or fingers over the tops of open containers within a sterile field (such as open bottles or flasks, the inside of tubes and bottle caps, and agar plates).</li> <li   >Plastic conical tubes and microcentrifuge tubes cannot be flamed, so it is doubly important to work with an open flame and take care not to pass fingers or hands over any tube openings.</li> <li   >Never wear gloves when working with an open flame.</li> <li   >The microcentrifuge tube should be kept closed until liquid is transferred and it should be closed immediately when you are done.</li> <li   >Never set a bottle or tube cap or Petri dish lid on a bench top.</li> <li  >Never go into a sterile solution with a used pipette or tip. In other words, never reuse a pipette, even if great care has been taken to keep it sterile.</li> <li  >Never leave an open flame unattended. </li> <li  >Always work with an open flame when opening sterile tubes or bottles.</li> <li  >Hold open tubes and bottles in the aseptic field at an angle to reduce the chance of airborne contamination.</li> <li  >Never have more than one tube, bottle, or flask open on the bench at one time.</li> <li  >Even if someone else has recently used the bench and the bench top has been wiped down with disinfectant, always begin your laboratory time by wiping down the bench top.</li> </ul> </div> </div>"},"2-3":{content:" <p> The following are recommendations that should be used to supplement general lab practices and established safety rules at your institution when working with bacteria. Please review these guidelines carefully and adhere to them to ensure a safe laboratory environment for everyone.Before working with bacteria: </p> <h2>Before working with bacteria:</h2> <ol> <li  > Ensure that you are aware of lab practices and safety rules, which should include the recommendations listed in this document. </li> <li  > Make sure you are aware that: <ul type = 'disc'> <li  > Actinobacteria recommended for use in this project are safe for those with a normal immune system. If an individual has any health concerns related to exposure to bacteria, including being immuno-compromised or pregnant, the individual should consult a physician to determine the appropriate level of participation in lab activities. </li> <li  > The primary hazards of working with the Actinobacteria used in this course are from exposure via mucosal membrane routes (eyes, mouth and nose), parenteral inoculation (broken skin), or ingestion. </li> </ul> </li> <h2 class = 'sub-heading' > While working with bacteria: </h2> <li  > Gloves and lab coats should be used. Change gloves when contaminated or when integrity is compromised. Do not touch face, apply cosmetics, adjust contact lenses, bite nails, etc. </li> <li  > Care should be taken to avoid generating aerosols of cultures or suspensions of bacteria; do not sonicate or vortex bacterial cultures or suspensions; transfer bacterial cultures and suspensions with care and deliberation. Safety glasses and surgical masks, or a biosafety cabinet, are recommended if there is risk of generating aerosols. </li> <li  > Label cultures, tubes and plates containing bacteria appropriately. </li> <li  > Lab work surfaces should be wiped-down with an appropriate phenolic disinfectant before and after use. An example of an appropriate disinfectant for any Actinobacterial strain is CiDecon Detergent Disinfectant Concentrate, available from multiple vendors including Fisher Scientific and VWR. Disinfectants should be used according to the manufacturer instructions. Once cleaned, the work surface should then be wiped-down with 70 % ethanol to remove residual disinfectant from the surface. </li> <li  > Materials that have come in contact with bacteria and that are intended for reuse (e.g. glass flasks) should be treated with CiDecon before washing. Spent CiDecon can be disposed of in sinks with tap-water running. </li> <li  > Materials for disposal that have come in contact with bacteria (e.g. microcentrifuge tubes or agar plates) should be disposed of in biohazard bins. </li> <li  > Sticks or loops used for streaking or inoculating bacteria should not be flamed after use. Instead, sticks should be disposed of in a biohazard bin, while metal loops should be treated with CiDecon followed by 70 % ethanol. </li> <h2 class = 'sub-heading'> After working with bacteria: </h2> <li  >Wash hands thoroughly.</li> </ol><h2  > Contamination </h2>  <ul> <li  > As a disinfectant, 70 % EtOH is sufficient to kill most bacterial contaminants on surfaces. However, it is not effective on Mycobacteria because of the mycolic acid wall. Use of an additional disinfectant like CiDecon will remove mycobacterial contaminants. </li> <li  > Many phages are resistant to 70 % EtOH, so daily decontamination of lab spaces and equipment with CiDecon or 10 % bleach will help prevent cross contamination of phages. </li> <li  > Cross contamination of phages within a classroom can be a big problem. To reduce phage contamination, each researcher should have their own small aliquots of phage buffer, clean up phage spills immediately, and keep pipette barrels clean. </li> </ul> <h2  >Disposal</h2> <ul> <li  > Dispose of all unused bacterial cultures appropriately by mixing the contents with CiDecon, bleach, or autoclaving. </li> <li  > Materials for disposal that have come in contact with bacteria (e.g. microcentrifuge tubes or agar plates) should be disposed of in biohazard bins. </li> <li  > Waste collected in biohazard bins should be inactivated using an autoclave, or removed by a licensed waste removal service provider. </li> <li  > Materials that have come in contact with bacteria and that are intended for reuse (e.g. glass flasks) should be treated with bleach or CiDecon before washing. Spent bleach and CiDecon can be disposed of in sinks with tap water running. </li> <li  > Unused soil samples can be returned to the outside, and the plastic bags they were collected in can be discarded in the regular trash. </li> <li  > Soil that has been incubated during enriched isolation experiments should be autoclaved in their flasks or conical tubes prior to discarding. </li>  </ul>"},"4-1":{content:" <h2>An Introduction to Bacteria</h2> <p>Bacteria are prokaryotes and among the oldest organisms on the planet. They are also among the simplest free-living life-forms known. Like all prokaryotes, they are single-cell organisms and lack a nuclear membrane and other internal membrane-bound organelles. Bacteria are small, usually in the range of 1 to 5 microns (&micro;m) across (a human red blood cell is 8 &micro;m).</p> <p>All bacteria have a cytoplasmic plasma membrane made of lipids that enclose the inner workings of the bacterial cell. A cell wall constructed of sugars and proteins surrounds the plasma membrane and gives the bacteria structural strength. Bacteria can have an additional lipid membrane and a slimy capsule that protects them from harsh environments or performs other functions. Some bacteria have appendages called “pili” and “flagella” (singular are “pilus” and “flagellum,” respectively). These hair-like structures protrude from the bacteria and perform specialized functions. Inside the plasma membrane all bacteria contain a DNA genome (typically 2–10 megabase pairs Mbp). But unlike the genome in eukaryotic cells, the bacterial genome is not enclosed by a lipid membrane (Figure 4.0-1). Genes are transcribed by RNA polymerase to make messenger RNAs (mRNA), which are used by ribosomes as substrates for protein synthesis. Because bacteria lack a nuclear membrane, the processes of transcription (mRNA synthesis) and translation (protein synthesis) are tightly coupled events.</p><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-4-1-1.png' alt = 'Illustration of an oval diagram with hairs protruding around the edge labeled pili and one long tail extending from one end labeled flagellum. The outside of the oval has three layers: the outide is labeled capsule and the middle is labeled cell wall and the inside is labeled cytoplasmic membrane. The medium inside the oval is labeled cytoplasm. Red dots inside the oval are labeled ribosomes and green tangled string-like structures are labeled bactierial D-N-A. '/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 4.0-1. Bacterial structure.</div></div><h2>The Bacterial World</h2><p>Bacteria are extremely abundant and by mass outweigh all plants and animals on the planet, combined. They are present in almost all habitats and naturally reside in and on you. Bacteria are also tremendously diverse, and different bacteria thrive in different environments. In each environment, bacteria have rich interactions with other forms of life. For example, the many different bacteria that live within the human gastrointestinal tract contribute to our health by helping us digest food and build our immune system. There are, however, bacteria that cause devastating human diseases like cholera (Vibrio cholerae) and tuberculosis (Mycobacterium tuberculosis). Wherever bacteria live, their environment is hardly ever theirs alone and can be a hostile place. They must deal with toxins and antibiotics naturally produced by fungi and other bacteria, as well as bacteriophages that can infect them. Over time, bacteria have evolved a variety of strategies to combat these assaults, making them incredibly successful organisms despite their simplicity.</p><p>All free-living organisms contain ribosomes for protein synthesis, and these ribosomes contain large RNA molecules that contribute to catalytic function. Different bacterial species have slightly different ribosomal RNA (rRNA) sequences, and these can be used to construct models—or a phylogeny—of their evolutionary relationships (Figure 4.0-2).</p><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-4-1-2.png' alt = 'Diagram of a phylogentic tree showing the evolutionary relationships of the ribosomal R-N-A sequences of bacteria, archaea, and eukaryota.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Fig. 4.0-2.  Phylogeny showing evolutionary relationships between the three phyla. (Photo courtesy of Wikipedia Commons, https://commons.wikimedia.org/wiki/File:Phylogenetic_Tree_of_Life.png_)</div></div> <h2>The Bacterial Life Cycle</h2><p>It is common for bacteria to reproduce asexually by dividing in half through a process known as binary fission. Bacterial growth is thus exponential, with one cell dividing to produce two cells, each of these dividing to produce four cells, then each of those dividing to produce eight cells, and so on. Fortunately, various influences, including resource limitations, lead to growth cessation. Otherwise, the world soon would be engulfed in bacteria! In the lab, when bacteria are growing in a liquid medium, this stage of growth is referred to as “stationary phase” (Figure 4.0-3) and the culture is sometimes referred to as being “saturated.” The number of bacterial cells in a saturated cultured is typically 2–4 x 109 cells/ml. </p><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-4-1-3.png' alt = 'A line graph with the x-axis representing time and the y-axis representing the log of cell number. A red line begins with a  short horizontal plateau near zero labeled A, following by a steep positive slope labeled B, followed by a horizontal plateau labeled C, followed by a steep negative slope labeled D.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 4.0-3. Bacterial growth over time. When inoculated into fresh media, bacteria will take some time recover without growth. This is called the “lag phase” (A). Bacteria will then enter a logarithmic (or exponential) growth phase (B), where the number of bacteria doubles once every time period (the period depends on the bacterial species). Once nutrients are used up, the bacterial number will remain constant in the stationary phase (C), and eventually decline in the death phase (D).</div></div><p>The growth rate of bacteria can be measured as the “doubling time,” that is, the time it takes for the number of bacterial cells to double. The doubling time varies greatly among different bacteria, and it also fluctuates widely depending on environmental aspects such as temperature and nutrient availability. Some bacteria can grow with doubling times as fast as 15 –20 minutes, whereas others may take days or even weeks to double.</p><p>In the lab we normally study a bacterial strain in isolation from other types of bacteria. We provide the bacteria with growth media, either in the form of a solid (agar) or a liquid, that is rich in nutrients to encourage their growth. This is called “culturing bacteria.” When a bacterial sample is streaked onto solid media, we can observe individual bacterial colonies. All the bacteria in a colony are genetically identical (apart from any in which rare mutations have occurred) because they arose from a single bacterium. The number of cells in a bacterial colony varies enormously depending on how large the colony is, but it is generally in range of 107–108 cells.</p><p> The morphology of bacterial colonies—that is, their color, size, and shape—differs greatly from species to species, ranging from yellow to purple and from smooth to rough. If many bacterial cells (e.g., 105) are added to solid media, individual colonies do not form. Instead, the bacteria will grow as a lawn that encompasses the entire surface area of the plate. When bacteria are cultured in liquid, they diffuse in the solution, and as they accumulate, the liquid becomes increasingly cloudy as it forms a saturated culture.</p>"},"4-2":{content:" <h2>Growing and maintaining cultures.</h2> <ul> <li>Always determine the temperature range within which your bacteria will grow. Once you have selected a growth temperature to use, for reproducibility of experiments, it is important that cultures used for any given phage or set of experiments be prepared at the same temperature.</li> <li>Use freshly prepared cultures, as older cultures do not form bacterial lawns reproducibly. In order to maintain a supply of fresh cultures for experiments, it is best to start new cultures <strong>weekly or biweekly </strong>throughout the semester.</li> <li>It is a good idea to check your working cultures by performing a spot test with a phage known to infect this host.  A nice smooth lawn is indicative of a good working culture, and a clearing where the known phage was spotted is a good indicator that the culture remains susceptible to phage infection. If the control phage does not form the expected clearing, you might not be growing your bacterial host, but a contaminant instead.  Note that the quality of a lawn often deteriorates as a culture ages, with older or dying cultures often producing embedded colonies in the top agar that can make it difficult to observe plaques.</li> <li >Try to have a dedicated space with dedicated pipettes where you prepare media and cultures. Avoid working with phage in this area, and with these pipettes.</li> <ul>"},"4-3":{content:" <h2>An Introduction to the Host Bacteria, <i>Arthrobacter globiformis</i></h2> <p>As a network of researchers, SEA-PHAGES explores the genetic diversity and relationships of populations of bacteriophages that infect Actinobacteria, incluidng the bacterium Arthrobacter globiformis.  There are many different strains of Arthrobacter globiformis. (isolated from different environmental samples). The Arthrobacter globiformis strain widely used in SEA-PHAGES is called Arthrobacter globiformis B2979-SEA, and is made available to you via the USDA culture collection. If you intend to isolate phages using this bacterium, then this will be your ‘host’ for phage isolation. </p><p> <i> Arthrobacter globiformis </i> is one of over 70 species that make up the genus Arthrobacter, classified within the phylum Actinobacteria. Arthrobacter globiformis is a gram-positive soil organism, whose relatives are well equipped to breakdown various hydrocarbons. Arthrobacter arilaitensis, for instance, grows on the surface of cheese and catabolizes the fatty acids, amino acids, and lactic acid present, contributing to the color and flavor of the cheese. Other Arthrobacter species are capable of breaking down compounds such as hexavalent chromium, which is a carcinogen, suggesting a potential use in bioremediation (i. e. the breakdown of pollutants). The specific Arthrobacter strain you will use, alongside its relatives, has been found to produce penicillin derivatives and aspartic decarboxylase.</p> <p><i> Arthrobacter globiformis </i> will grow and divide on many nutrient-rich media. In your research, you will grow Arthrobacter sp. on PYCa media and at 30 - 37 ˚C. Under these conditions, Arthrobacter globiformis grows rapidly, dividing approximately once every 2 - 3 hours. This means that it typically takes about 3 days for an individual cell to form a colony on an agar plate. However, when preparing a bacterial lawn, you can start by adding many bacterial cells (e. g. 107 or more) so that a lush lawn grows within 1 – 2 days. For reproducibility of experiments, it is important that cultures used for a given set of experiments be prepared at the same temperature. Arthrobacter globiformis colonies are white to tan in color, smooth and glossy. A liquid culture of Arthrobacter globiformis inoculated from a single colony will take 2 days to form a saturated liquid culture, exhibiting a tan color. The strain you will work with is not inhibited by the antifungal cyclohexamide.  Therefore, this can be added to the growth media to prevent other microorganisms from growing in your cultures. </p><p><i>Arthrobacter globiformis</i> colonies are white to tan in color, smooth and glossy. A liquid culture of Arthrobacter globiformis inoculated from a single colony will take 2 days to form a saturated liquid culture, exhibiting a tan color. The strain you will work with is not inhibited by the antifungal cyclohexamide.  Therefore, this can be added to the growth media to prevent other microorganisms from growing in your cultures.</p><p>On average, 30 % of attempts to isolate phage from soil by enrichment with Arthrobacter globiformis yield phage. </p><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-4-3-1.png' alt = 'Picture of a circular covered container covered with an off-white smear along the left perpendicular side, and off-white round spots extending outward from the smear.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 4.0-4. Arthrobacter globiformis. growing on an agar plate.</div></div> <p>If you are using this bacterium as your host for phage isolation, refer back to the list below for the specific growth and culture requirements for your experiments. </p><p><strong>Growth media:</strong> PYCa  </p><p><strong>Growth temperature:</strong> 22 - 37 ˚C. </p><p><strong>Antimicrobials: </strong>cycloheximide (10 &micro;g/ml) </p><p><strong>Phage Buffer:</strong> 10 mM Tris (pH 7.5), 10 mM MgSO<sub>4</sub>, 68 mM NaCl, 1 mM CaCl<sub>2</sub>, (10% glycerol, optional) </p><p> <strong>Restriction Enzymes:</strong> BamHI, ClaI, MseI, NspI, and SacII. </p><p> Note: Isoschizomers, which are different restriction enzymes that recognize the same DNA sequence, may be used in place of any of the enzymes listed above.</p>"},"4-4":{content:"<h2>An Introduction to the Host Bacteria, <i>Arthrobacter sp.</i></h2> <p>As a network of researchers, SEA-PHAGES explores the genetic diversity and relationships of populations of bacteriophages that infect Actinobacteria, incluidng the bacterium Arthrobacter sp.  There are many different strains of Arthrobacter sp. (isolated from different environmental samples). The Arthrobacter sp. strain widely used in SEA-PHAGES is called Arthrobacter sp. KY3901. This strain is from the American Type Culture Collection (catalog # 21022). If you intend to isolate phages using this bacterium, then this will be your ‘host’ for phage isolation.</p><p> Arthrobacter sp. is one of over 70 species that make up the genus Arthrobacter, classified within the phylum Actinobacteria. Arthrobacter sp. is a gram-positive soil organism, whose relatives are well equipped to breakdown various hydrocarbons. Arthrobacter arilaitensis, for instance, grows on the surface of cheese and catabolizes the fatty acids, amino acids, and lactic acid present, contributing to the color and flavor of the cheese. Other Arthrobacter species are capable of breaking down compounds such as hexavalent chromium, which is a carcinogen, suggesting a potential use in bioremediation (i. e. the breakdown of pollutants). The specific Arthrobacter strain you will use, alongside its relatives, has been found to produce penicillin derivatives and aspartic decarboxylase. </p><p>Arthrobacter sp. will grow and divide on many nutrient-rich media. In your research, you will grow Arthrobacter sp on PYCa media and at 22 - 28˚C. Under these conditions, Arthrobacter sp grows rapidly, dividing approximately once every 2 - 3 hours. This means that it typically takes about 3 days for an individual cell to form a colony on an agar plate. However, when preparing a bacterial lawn, you can start by adding many bacterial cells (e. g. 107 or more) so that a lush lawn grows within 1 – 2 days. For reproducibility of experiments, it is important that cultures used for a given set of experiments be prepared at the same temperature. </p><p>Arthrobacter sp colonies are tan to yellow in color, smooth and glossy (Figure 4.0-4). When incubated for prolonged periods, the color of the colonies intensifies. A liquid culture of Arthrobacter sp inoculated from a single colony will take 1 – 2 days to form a saturated liquid culture, exhibiting a tan color. The strain you will work with, Arthrobacter sp. KY3901, is not inhibited by the antifungal cyclohexamide or the antibiotic spectinomycin.  Therefore, this can be added to the growth media to prevent other microorganisms from growing in your cultures.</p><p> On average, 40 % of attempts to isolate phage from soil by enrichment with Arthrobacter sp. yield phage </p><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-4-4-1.png' alt = 'Picture of a circular covered container covered with an yellow smear along one side, and yellow round spots extending outward from the smear.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 4.0-4. Arthrobacter sp. growing on an agar plate.</div></div> <p>  If you are using this bacterium as your host for phage isolation, refer back to the list below for the specific growth and culture requirements for your experiments.</p><p> <strong>Growth media:</strong> PYCa  </p><p><strong>Growth temperature:</strong>22˚C - 28˚C (this host does not grow at 37˚C)</p><p> <strong>Antimicrobials: </strong>cycloheximide (10 &micro;g/ml), spectinomycin (optional)</p> <strong>Phage Buffer:</strong>10 mM Tris (pH 7.5), 10 mM MgSO4, 68 mM NaCl, 1 mM CaCl2, (10% glycerol, optional)</p><p> <strong>Restriction Enzymes:</strong> BamHI, ClaI, EcoRI, HaeIII, HindIII, MseI. </p><p>Note: Isoschizomers, which are different restriction enzymes that recognize the same DNA sequence, may be used in place of any of the enzymes listed above.</p>"},"4-5":{content:"<h2>An Introduction to the Host Bacteria: <i> Gordonia rubripertincta</i></h2> <p>  As a network of researchers, SEA-PHAGES explores the genetic diversity and relationships of populations of bacteriophages that infect Actinobacteria, incluidng the bacterium Gordonia rubripertincta.  There are many different strains of Gordonia rubripertincta (isolated from different samples). The Gordonia rubripertincta strain widely used in SEA-PHAGES is called Gordonia rubripertincta B16540-SEA, and is made available to you via the USDA culture collection. If you intend to isolate phages using this bacterium, then this will be your ‘host’ for phage isolation. </p><p>G. rubripertincta is one of over 19 species that make up the genus Gordonia, classified within the phylum Actinobacteria. G. rubripertincta is a gram-positive soil organism and, like its relatives, is well equipped to breakdown various hydrocarbons. This ability makes Gordonia attractive bacteria for use in bioremediation (i. e. the breakdown of pollutants) and in industrial biotechnology. For example, Gordonia sp. MTCC 4818 is able to breakdown phthalic acid esters, which can act as hormones that disrupt normal body function.  Another Gordonia species, CYSK-1, is able to remove sulfur from fossil fuels, which will allow for the production of cleaner fuels, such as ultra-low-sulfur diesel. </p><p>  G. rubripertincta will grow and divide on many nutrient-rich media. In your research, you will grow G. rubripertincta on PYCa media and at 30 ˚C. Under these conditions G. rubripertincta grows rapidly, and it typically takes about 2 - 3 days for an individual cell to form a colony on an agar plate. However, when preparing a bacterial lawn, you can start by adding many bacterial cells (e. g. 107 or more) so that a lush lawn grows within 1 – 2 days. For reproducibility of experiments, it is important that cultures used for a given set of experiments be prepared at the same temperature. </p><p>G. rubripertincta colonies are tan to orange in color, glossy and smooth in texture, and round or slighly irregular in shape (Figure 4.0-4). When incubated for prolonged periods, the color of the colonies intensifies to a brilliant orange. A liquid culture of G. rubripertincta inoculated from a single colony will take 2 – 3 days to form a saturated liquid culture, exhibiting a tan to orange color. The strain you will work with, G. rubripertincta B16540-SEA, is not inhibited by the antifungal cyclohexamide, or the antibiotic carbenicillin at low concentrations.  Therefore, this can be added to the growth media to prevent other microorganisms from growing in your cultures.</p><p> On average, 20 % of attempts to isolate phage from soil by enrichment with G. rubripertincta yield phage. </p> <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-4-5-1.png' alt = 'Picture of a circular covered container covered with an orange smear along the top and right sides, and orange round spots extending outward from the right smear.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 4.0-4. Gordonia rubripertincta growing on an agar plate..</div></div> <p>If you are using this bacterium as your host for phage isolation, refer back to the list below for the specific growth and culture requirements for your experiments. </p><p><strong>Growth media:</strong> PYCa, supplemented with Tween80 as necessary (see <a class = 'chap-linking' tabindex = '0'    data-link-to = '14-40' > instructions for culturing bacteria that require Tween80</a>) </p><p> <strong>Growth temperature:</strong>30 ˚C </p><p><strong>Antimicrobials: </strong> cycloheximide (10 &micro;g/ml), carbenicillin (not recommended) </p><p><strong>Phage Buffer:</strong>10 mM Tris (pH 7.5), 10 mM MgSO4, 68 mM NaCl, 1 mM CaCl2, (10% glycerol, optional) </p><p><strong>Restriction Enzymes:</strong> BamHI, HaeIII, HindIII, MseI, NspI, and SacII. </p><p>Note: Isoschizomers, which are different restriction enzymes that recognize the same DNA sequence, may be used in place of any of the enzymes listed above.</p>"},"4-6":{content:"<h2>An Introduction to the Host Bacteria, <i>Gordonia terrae</i></h2> <p>As a network of researchers, SEA-PHAGES explores the genetic diversity and relationships of populations of bacteriophages that infect Actinobacteria, incluidng the bacterium Gordonia terrae.  There are many different strains of Gordonia terrae (isolated from different  samples). The Gordonia terrae strain widely used in SEA-PHAGES is called Gordonia terrae CAG3 that was originally isolated from oil-contaminated soil in China. If you intend to isolate phages using this bacterium, then this will be your ‘host’ for phage isolation. </p><p>G. terrae. is one of over 19 species that make up the genus Gordonia, classified within the phylum Actinobacteria. G. terrae. is a gram-positive soil organism, whose relatives are well equipped to breakdown various hydrocarbons. This ability makes Gordonia attractive bacteria for use in bioremediation (i. e. the breakdown of pollutants) and in industrial biotechnology. For example, Gordonia sp. MTCC 4818 is able to breakdown phthalic acid esters, which can act as hormones that disrupt normal body function.  Another Gordonia species, CYSK-1, is able to remove sulfur from fossil fuels, which will allow for the production of cleaner fuels, such as ultra-low-sulfur diesel.</p><p> G. terrae. will grow and divide on many nutrient-rich media. In your research, you will grow G. terrae on PYCa media and at 30 ˚C. Under these conditions G. terrae grows rapidly, dividing approximately once every 2 - 3 hours. This means that it typically takes about 3 days for an individual cell to form a colony on an agar plate. However, when preparing a bacterial lawn, you can start by adding many bacterial cells (e. g. 107 or more) so that a lush lawn grows within 2 – 3 days. For reproducibility of experiments, it is important that cultures used for a given set of experiments be prepared at the same temperature.</p><p> G. terrae colonies are tan to orange in color, round, rough and dry in texture (Figure 4.0-4). When incubated for prolonged periods, the color of the colonies intensifies. A liquid culture of G. terrae inoculated from a single colony will take 2 – 3 days to form a saturated liquid culture, exhibiting a tan to orange color. The strain you will work with, G. terrae CAG3, is not inhibited by the antifungal cyclohexamide, or the antibiotic ampicillin at low concentrations.  Therefore, this can be added to the growth media to prevent other microorganisms from growing in your cultures.</p><p> On average, 33 % of attempts to isolate phage from soil by enrichment with Gordonia terrae yield phage. </p><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-4-6-1.png' alt = 'Picture of a circular covered container covered with an orange smear along one side, and orange round spots extending outward from the smear.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 4.0-4. Gordonia terrae growing on an agar plate.</div></div><p>If you are using this bacterium as your host for phage isolation, refer back to the list below for the specific growth and culture requirements for your experiments.</p><p> <strong>Growth media:</strong>  PYCa, supplemented with Tween80 (see <a class = 'chap-linking' tabindex = '0'    data-link-to = '14-40' >instructions for culturing bacteria that require Tween80</a>) </p><p> <strong>Growth temperature:</strong>30 ˚C </p><p><strong>Antimicrobials: </strong> cycloheximide (10 &micro;g/ml), ampicillin (optional, and up to 10 mg/ml)</p><p> <strong>Phage Buffer:</strong>10 mM Tris (pH 7.5), 10 mM MgSO4, 68 mM NaCl, 1 mM CaCl2, (10% glycerol, optional) </p><p><strong>Restriction Enzymes:</strong> BamHI, ClaI, EcoRI, HaeIII, HindIII, MseI.</p><p> Note: Isoschizomers, which are different restriction enzymes that recognize the same DNA sequence, may be used in place of any of the enzymes listed above.</p>"},"4-7":{content:"<h2>An Introduction to the Host Bacteria, <i> Microbacterium foliorum</i></h2> <p>As a network of researchers, SEA-PHAGES explores the genetic diversity and relationships of populations of bacteriophages that infect Actinobacteria, incluidng the bacterium Microbacterium foliorum (M. foliorum).  There are many different strains of M. foliorum (isolated from different environmental samples). The M. foliorum strain widely used in SEA-PHAGES is called M. foliorum SEA B-24224 that was isolated from grass in Germany, and made available to you from the Agricultural Research Service of USDA. If you intend to isolate phages using this bacterium, then this will be your ‘host’ for phage isolation.</p><p> M. foliorum is one of more than 90 species that make up the genus Microbacterium, classified within the phylum Actinobacteria. Microbacteria are gram-positive rod-shaped bacteria that are slightly smaller than typical rod-shaped bacteria, hence the name microbacteria. Microbacteria have been isolated from soil, water, and plants. Microbacteria have also been isolated from cheese rinds, where it may contribute to the flavor profile.</p><p> M. foliorum will grow and divide on many nutrient-rich media. In your research, you will grow M. foliorum on PYCa media at 28 – 30 ˚C. Under these conditions, M. foliorum grows rapidly, dividing approximately once every 90 minutes. This means that it typically takes about 2 – 3 days for an individual cell to form a colony on an agar plate. However, when preparing a bacterial lawn, you can start by adding many bacterial cells (e. g., 107 or more) so that a lush lawn grows within 24 hours. M. foliorum will also grow at room temperature (~ 22 ˚C), but it divides more slowly and therefore requires extra time to form either a colony or a lawn, typically an additional 24 hours. M. foliorum does not grow at 37 ˚C. For reproducibility of experiments, it is important that cultures used for a given set of experiments be prepared at the same temperature.</p><p> M. foliorum colonies are tan to yellow in color, smooth and glossy (Figure 4.0-4). When incubated for prolonged periods, or when grown at room temperature, the color of the colonies are a brighter yellow. A liquid culture of M. foliorum inoculated from a single colony will take 24 hours to form a saturated liquid culture, exhibiting a tan to yellow color, which intensifies with prolonged incubation or when grown at room temperature. The strain you will work with, M. foliorum, is not inhibited by the antifungal cyclohexamide. Therefore, cycloheximide can be added to the growth media to prevent other microorganisms from growing in your cultures. </p><p>On average, 20 % of attempts to isolate phage from soil by enrichment with M. foliorum yield phage. </p> <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-4-7-1.png' alt = 'Picture of a circular covered container covered with a yellow smear along the left and top sides, and yrllow round spots extending outward from the top smear.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 4.0-4. Microbacterium foliorum growing on an agar plate.</div></div><p>If you are using this bacterium as your host for phage isolation, refer back to the list below for the specific growth and culture requirements for your experiments.</p><p> <strong>Growth media:</strong>PYCa  </p><p><strong>Growth temperature:</strong>optimal at 28 ˚C - 30 ˚C; grows well at room temperature (~ 22 ˚C); this host does not grow at 37˚C. </p><p><strong>Antimicrobials: </strong>  cycloheximide (10 &micro;g/ml) </p><p><strong>Phage Buffer:</strong>10 mM Tris (pH 7.5), 10 mM MgSO4, 68 mM NaCl, 1 mM CaCl2, (10% glycerol, optional)</p><p> <strong>Restriction Enzymes:</strong>  HaeIII, MseI, NspI, SacII, and SalI.</p><p> Note: Isoschizomers, which are different restriction enzymes that recognize the same DNA sequence, may be used in place of any of the enzymes listed above.</p>"},"4-8":{content:"<h2>An Introduction to the Host Bacteria: <i>Microbacterium testaceum</i></h2> <p>In this course, you will be isolating bacteriophage that infect the bacterium Microbacterium testaceum (M. testaceum). You will use a strain called M. testaceum SEA B-24232 that is made available to you from the Agricultural Research Service of USDA. This is your “host” for phage isolation.</p><p> M. testaceum is one of more than 90 species that make up the genus Microbacterium, classified within the phylum Actinobacteria. Microbacteria are gram-positive rod-shaped bacteria that are slightly smaller than typical rod-shaped bacteria, hence the name microbacteria. Microbacteria have been isolated from soil, water, and plants. Microbacteria have also been isolated from cheese rinds, where it may contribute to the flavor profile.</p><p> M. testaceum will grow and divide on many nutrient-rich media, and at temperatures ranging from room temperature (22 ˚C) to 37 ˚C. In your research, you will grow M. testaceum on PYCa media at 30 ˚C. Under these conditions, M. testaceum grows rapidly and it typically takes about 2 – 3 days for an individual cell to form a colony on an agar plate. However, when preparing a bacterial lawn, you can start by adding many bacterial cells (e. g., 107 or more) so that a lush lawn grows within 24 hours. At room temperature (~ 22 ˚C), M. testaceum divides more slowly and typically requires an additional 24 hours to form either a colony or a lawn on plates. For reproducibility of experiments, it is important that cultures used for a given set of experiments be prepared at the same temperature. </p><p>M. testaceum colonies are tan to yellow in color, smooth and glossy (Figure 4.0-4). When incubated for prolonged periods, or when grown at room temperature, the color of the colonies are a brighter yellow. A liquid culture of M. testaceum inoculated from a single colony will take 24 - 48 hours to form a saturated liquid culture, exhibiting a tan to yellow color, which intensifies with prolonged incubation or when grown at room temperature. The strain you will work with, M. testaceum SEA B-24232, is not inhibited by the antifungal cyclohexamide. Therefore, cycloheximide can be added to the growth media to prevent other microorganisms from growing in your cultures.</p><p> On average, 20 % of attempts to isolate phage from soil by enrichment with M. testaceum yield phage. </p> <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-4-8-1.png' alt = 'Picture of a circular covered container covered with an yellow smear along one side, and yellow round spots extending outward from the smear.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 4.0-4. Microbacterium testaceum growing on an agar plate.</div></div><p> If you are using this bacterium as your host for phage isolation, refer back to the list below for the specific growth and culture requirements for your experiments. </p><p><strong>Growth media:</strong>PYCa </p><p> <strong>Growth temperature:</strong>30 ˚C</p><p> <strong>Antimicrobials: </strong>  cycloheximide (10 &micro;g/ml) </p><p><strong>Phage Buffer:</strong>10 mM Tris (pH 7.5), 10 mM MgSO4, 68 mM NaCl, 1 mM CaCl2, (10% glycerol, optional)</p><p> <strong>Restriction Enzymes:</strong>  HaeIII, MseI, NspI, and SacII. </p><p> Note: Isoschizomers, which are different restriction enzymes that recognize the same DNA sequence, may be used in place of any of the enzymes listed above.</p>"},"4-9":{content:"<h2>An Introduction to the Host Bacteria, <i>Mycobacterium smegmatis</i></h2> <p>As a network of researchers, SEA-PHAGES explores the genetic diversity and relationships of populations of bacteriophages that infect Actinobacteria, incluidng the bacterium Mycobacterium smegmatis (M. smegmatis, or colloquially known as “smeg”).  There are many different strains of M. smegmatis (isolated from different environmental or clinical samples). The M. smegmatis strain widely used in SEA-PHAGES is called M. smegmatis mc2155. If you intend to isolate phages using this bacterium, then this will be your ‘host’ for phage isolation.</p><p> M. smegmatis is one of more than 100 species that make up the genus Mycobacterium, classified within the phylum Actinobacteria. Mycobacteria were named this because they sometimes have a fungus-like appearance under the microscope—although they are definitely not fungi! The Mycobacterium genus contains many species that affect human health, including M. tuberculosis and M. leprae, which are the causative infectious agents of tuberculosis and leprosy, respectively. M. smegmatis, however, is a saprophyte and is found in environmental samples, including soil and compost. Although they share the same genus, M. smegmatis and M. tuberculosis have many other differences, including vastly different doubling times: approximately 3 hours for M. smegmatis and 24 hours for M. tuberculosis. Some bacteriophages are capable of infecting both species, whereas others infect only one or the other.</p><p>  M. smegmatis will grow and divide on many nutrient-rich media. In your research, you will grow M. smegmatis on 7H9 media supplemented with albumin and dextrose (AD) at 37 ˚C. Under these conditions, M. smegmatis grows at its fastest, dividing approximately once every 3 hours. This means that it typically takes about 3–4 days for an individual colony growing from a single cell on an agar plate. However, when preparing a bacterial lawn, you can start by adding many bacterial cells (e. g., 107 or more) so that a lush lawn grows within 20–24 hours. M. smegmatis will also grow at 30 ˚C, but it divides more slowly and therefore requires more time to form either a colony or a lawn. For reproducibility of experiments, it is important that cultures used for a given set of experiments be prepared at the same temperature.</p><p> M. smegmatis mc2155 colonies are tan-white in color and dry, rough, and wrinkled in texture (Figure 4.0-4). A liquid culture of M. smegmatis inoculated from a single colony will take 3–5 days to form a saturated liquid culture, exhibiting a tan-white color. The strain you will work with, M. smegmatis mc2155, is not inhibited by the antibiotic ampicillin or the antifungal cyclohexamide. Therefore, these can be added to the growth media to prevent other microorganisms from growing in your cultures. </p><p>On average, 60 % of attempts to isolate phage from soil by enrichment with M. smegmatis yield phage. </p> <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-4-9-1.png' alt = 'Picture of a circular covered container covered with an off-white smear along one side, and off-white smear and round spots extending outward from the smear.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 4.0-4. Mycobacterium smegmatis growing on an agar plate.</div></div> <p>If you are using this bacterium as your host for phage isolation, refer back to the list below for the specific growth and culture requirements for your experiments.</p> <p><strong>Growth Media:</strong><ul><li>Liquid: 7H9, supplemented with albumin dextrose (AD) and Tween80 (see <a class = 'chap-linking' tabindex = '0'    data-link-to = '14-40' >instructions for culturing bacteria that require Tween80</a>)</li><li>Agar: L-agar plates</li><li>Top Agar: 7H9</li></ul></p><p> <strong>Growth temperature:</strong>37˚C, (or 30˚C)</p><p> <strong>Antimicrobials: </strong>carbenicillin (50 &micro;g/ml), cycloheximide (10 &micro;g/ml) </p><p><strong>Phage Buffer:</strong>10 mM Tris (pH 7.5), 10 mM MgSO4, 68 mM NaCl, 1 mM CaCl2, (10% glycerol, optional)</p><p> <strong>Restriction Enzymes:</strong>  BamHI, ClaI, EcoRI, HaeIII, HindIII, MseI.. </p><p>Note: Isoschizomers, which are different restriction enzymes that recognize the same DNA sequence, may be used in place of any of the enzymes listed above.</p>"},"5-1":{content:" <p> Identifying and characterizing new bacteriophages continues to be an important and rewarding avenue of research. Bacteriophages are the most abundant life forms on the planet and therefore are a rich source of novel genetic information. Although several thousand unique bacteriophages have been discovered that infect various bacterial hosts, new phages are continually being found that are remarkably different genetically from any previously discovered. Sequencing the genomes of these new phages may reveal similarities to previously identified phages, but even small differences can reveal important insights about the biology and evolution of viruses.</p><p> The road to discovering a new phage begins with collecting an environmental sample. All environmental samples, including soil, compost, and water, contain phages. But not all samples contain phages specific to your bacterial host. A good strategy to use for finding new phages is to learn about your host and where it lives in the environment. Where you find host bacteria, you will find phages.</p><p> In the lab, you will extract phages present in your environmental sample and test for their ability to infect your bacterial host by using one of two techniques: direct isolation or enriched isolation (or maybe both). The steps you use to detect phage in these samples take advantage of the bacteriophage infection cycle.</p><p> <h2> Direct Isolation </h2> <p> <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-4' >Direct Isolation (protocol 5.2) </a>performed in combination with a <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-5' > Plaque Assay(protocol 5.3)</a> is a quick method to detect phages in both solid and liquid environmental samples. This method provides a snapshot of all of the phages present in your sample that may infect your host. Indeed, using direct isolation, you may discover many different phages with distinct plaque morphologies. However, since not every phage in your sample will succeed at infecting your host bacteria, you may find as few as one plaque on your plate, or you may not find any plaques at all. </p><h2>Enriched Isolation</h2><p><a class = 'chap-linking' tabindex = '0'    data-link-to = '5-7' >The Enriched Isolation (protocol 5.5)</a> method involves amplifying the number of phages in your environmental sample before identifying plaques on solid media with a <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-8' > Spot Test (protocol 5.6)</a>. An enriched isolation is accomplished by mixing the soil sample with bacterial growth media to extract phages, followed by filtering out unwanted soil microbes. Bacterial host cells are then added to the filtrate. Any phage particles present that recognize the specific host can then grow and amplify through multiple rounds of lytic growth. A single phage particle can multiply to become many millions of particles. It is important to note that one particular type of phage may outcompete other phages present, so that phages you recover at the end are not necessarily representative of those present in your starting sample. </p><h2>Plaque Assay</h2><p> Because individual phage particles are too small to be seen directly with the naked eye, you have to look for evidence of phage growth in the form of plaques, which are the result of bacterial cell death. <strong><a class = 'chap-linking' tabindex = '0'    data-link-to = '5-5' >The Plaque Assay</strong> (protocol 5.3)</a> allows you to detect plaques on a “lawn” of host bacteria. Lawns of bacteria are made by mixing bacteria with liquid media and agar in a concentration which allows bacteria to form a smooth opaque layer. As the bacteria grow, phages present in your sample will infect and replicate within the bacterial host, killing the bacteria at the end of each lytic cycle. Each phage infection can produce tens to hundreds of new phage particles. The new phage that escape from each lysed bacterium diffuse through the agar and infect nearby bacteria, repeating the process of replication and lysis. Given enough time, a circular zone of cleared bacteria called a plaque will be visible, indicating the presence of phage! </p><p>Whereas some plaques will be large and clear and therefore easy to identify, some may be difficult to see because they are small or turbid. What’s more, air bubbles or improperly set top agar can often be mistaken for plaques. Therefore, ambiguous-looking plaques should always be tested to confirm that they were caused by phages. The methods presented—<a class = 'chap-linking' tabindex = '0'    data-link-to = '5-6' >Picking a Plaque (protocol 5.4)</a> followed by the <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-8' >Spot Test </a>(protocol 5.6 and also described below)—allow you to test multiple putative plaques quickly by using a single agar plate. If you are sure that your plaques are genuine, your instructor may encourage you to skip the spot test. However, it can be gratifying to test multiple plaques that differ in morphology by using the spot test to see how many different phages you may have isolated.</p> <h2>Spot Test</h2> <p>Enriched isolation cultures are screened for the presence of phages by spreading a thin layer of bacteria, media, and agar on a plate and pipetting small aliquots of the sample to be tested on top. As the lawn of bacteria grows, phage present in the sample will complete the infection and lysis cycle, creating large cleared areas or “spots” composed of multiple overlapping plaques. Consequently, Spot tests are a way to screen multiple samples for phage on the same plate.</p> <p>Congratulations! Once you have a positive spot test you have successfully isolated at least one phage and you are ready to move on to Chapter 6,<a class = 'chap-linking' tabindex = '0'    data-link-to = '6' > Phage Purification</a>.</p><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-5-1-1.jpg' alt = 'A flowchart for isolation: collecting environmental samples branches in two directions. One direction leads to direct isolation, then plaque assay. The other direction leads to enriched isolation then spot test. Both branches then point to the question: do you have phage? If no, then return to collecting environmental samples. If yes, then proceed to purification.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 5.0-1. Flow chart of experiments for Phage Isolation.</div></div>"},"5-2":{content:"<h2>Enriched and Direct Isolation</h2> <p><ul><li>To extract phage from soil, the initial soil “washing” step is critical in overcoming electrostatic interactions between phages and the soil matrix. While one hour of shaking at 250 rpm may be okay, two to three hours is better, and substantially increases the phage yield per sample.</li>  <li>Because phage yields may be lower with the new enrichment protocol, you may want to set up multiple enriched isolation cultures at once.</li>  <li>The phage extraction step (washing) and enriched isolation can be performed in 50 ml conical tubes. This allows for additional samples to be incubated at once (Figure 1.0). In such instances, it may be necessary to use vented conical tubes as some bacteria can be sensitive to low oxygen levels (Figure 1.1).</li>  </ul></p> <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-5-2-1.png' alt = 'A picture inside of an incubator with 3 groupings of conical tubes with white labels and filled with material.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 1.0. Arrangement of 50 ml conical tubes in a shaker equipped for flasks</div></div><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-5-2-2.png' alt = 'A picture of two gloved hands, each holding conical tubes: the right is covered with a green cap with holes, the left is covered with a red cap without holes.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Fig 1.1. Right: A 50 ml conical tube with a vented cap. Left: A 50 ml conical tube without a vented cap, shown for comparison.</div></div>     <ul> <li>Direct plating/isolation is used to avoid the bias introduced by enrichment cultures, where fast-replicating phages quickly outnumber/outcompete slower-replicating phages. Although yields from direct isolation might be lower than with enriched isolation, the direct isolation procedure is relatively fast.</li> <li>The greater the volume of direct isolation sample assayed for plaques, the better your chances of finding a phage. As such, you can vary the total volume you plate by varying the concentration of agar in your top agar. <br/> <strong>Example:</strong> <br/> If you washed your soil in medium, you can plate 5 ml total on your plate by using 2.5 ml direct filtrate and 2.5 ml 2X top agar. By increasing the agar concentration in your top agar to 4x (without altering the concentration of other media components), you can still plate 5 ml total on your plate, but now using 4 ml of direct filtrate and 1 ml of your new top agar.</li> <li>It is possible to vary the length of time and the temperature for incubating enrichment cultures. Peak yield of phages from cultures seems to be between 24 - 48 hours, though on occasion this can extend out to 4 days. If you want to test an enriched isolation multiple times over a long incubation period, a 1.5 ml sample can be removed using a sterile serological pipette and the culture returned to the incubator. Phage yields will change as the cultures are allowed to incubate longer. Incubations can also occur between room temperature and the highest growth temperature recommended for a given bacterial host. In addition, the phage profile tends to change with time, as well as with different temperatures.</li> <li>Excess enriched cultures should be treated as a saturated bacterial culture for disposal by treating with bleach or autoclaving.</li> <li>You may want to incubate the spot plates without inverting initially, and return after 24 hours to invert. This will prevent slippage of the top agar.</li> </ul><h2>Plaque Assay</h2> <ul> <li>If your plaques are very small, try reducing the amount of cells used in plating, decreasing the temperature of plate incubation, and/or decreasing the percentage of agar in the top agar. You can go as low as 0.2 % top agar and your plates will still set. For example, if you are using 2X top agar, you can use 1 part 2X and 3 parts liquid media. You will want to empirically test that your plates will solidify in your atmospheric conditions.<br> To increase plaque size: <ol> <li>Temperature: decrease incubation temperature below 37 °C to room temperature (or 30°C to room temperature for non-smegmatis hosts.)</li> <li>Cell concentration: decrease number of host cells to as low as 100 ml of saturated cell culture (~ 10<sup>7</sup> to 10<sup>8 </sup>cfu/ml)</li> <li>Top agar concentration: decrease to 0.35% to 0.18% (you can go lower if you use agarose but that gets expensive).<br> </li> </ol> </li> <li>Time of pre-adsorption of phage to cells prior to mixing with top agar and plating can be varied. Some phages absolutely require at least 15 minutes of pre-adsorption with host cells prior to mixing with top agar and some do not. In a pinch, this step can be shortened or skipped—usually the result is an apparently lower titer (rather than no infection), and more irregularly sized plaques. However, lengthening the time of pre-absorption beyond 15 minutes does not really change any experimental outcomes unless you go past one infection cycle, ( ~ three hours in mycobacteriophages).</li> </ul>"},"5-3":{content:" <h2>Objective:</h2> <p>To obtain an environmental sample containing bacteriophage <hr><br><h2>Rationale:</h2> By collecting soil, compost, water, or other samples rich in bacteria, you will aim to collect a bacteriophage that can infect those bacteria. To increase your chances of gathering a variety of phages that can infect your specific host bacteria, you should consider environments where your host bacteria thrive. Therefore, collecting samples from a variety of environments is ideal.</p>  <hr><br><h2>Supplies:</h2><ul><li>Plastic sandwich bags for collecting soil samples and a tool for digging</li>  <li>Clean plastic bottle for liquid samples</li>  <li>Labeling pen</li>  <li>Smart phone or tablet with GPS capabilities or computer</li></ul> <hr><br><h2>Procedure:</h2> <ol style = 'list-style-type:upper-alpha;'><li>Collect the specified number of samples as directed by your instructor. For each sample perform the following steps:     <ol><li>For solid samples, turn a clean plastic sandwich bag inside out and insert your hand into the bag as if it were a glove. Grab a handful of soil, keeping the plastic bag between your hand and the sample. Remove your hand, inverting the bag with the soil to the inside, and seal the bag.</li><li>For liquid samples rinse the plastic bottle with your sample water by filling it one-third full, capping it, shaking it vigorously, and then dumping the sample water back out. After doing this three times, fill the bottle with your sample and cap the bottle.</li></ol> </li><li>Label the sample bag or bottle appropriately (e.g., initials, location) so you can identify where the sample was collected.</li> <li>Record important aspects of the sample and collection site.<ol><li>Name the sample something that will identify the location where it was collected.</li>  <li>Record the GPS coordinates of your sample collection site.  <ul><li>If you have a smartphone or tablet during sample collection, determine the GPS coordinates and record this information.</li><li>If you do not have a smartphone or tablet during sample collection, determine the GPS coordinates when you have access to a computer. Record this information.</li></ul></li><li>Record the physical characteristics of your sample.<ul><li>For soil samples: Was the soil wet or dry? Was it sandy or full of organic matter? Approximately how far below the surface was the soil collected?</li>  <li>What was the ambient temperature?</li>  </ul> </li>  <li>Repeat steps 1 - 3 for each sample collected.</li></ol></li>  <li>Back in the lab, make certain that all the sample information you collected, including collection site and physical charatceristics, are recorded in your lab notebook. <ul type = 'disc'><li>Be as specific as possible when recording this information. You will need to log this information on the program database. </li> </ul></li> <li>As directed by your instructor, your next step will be to process your environmental sample(s) using the <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-4' >Direct Isolation (5.2)</a> or <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-7' >Enriched Isolation (5.5) </a>protocol, or both.</li></ol></ul>  <hr><br><h3>Helpful Tips:</h3>  <ul><li>While soil samples can be collected several days ahead of processing, it is best to use the freshest samples possible.</li><li>If you collect soil samples ahead of time, store the soil samples in a cool place and do not let them dry out.</li></ul></p> "},"5-4":{content:"  <h2>Objective:</h2> <p>To extract phages from an environmental sample  <hr><br><h2>Rationale:</h2> This method extracts phages from microbes and particulate matter in a solid or liquid environmental sample. The extracted sample is then used to infect your host bacteria by using the plaque assay. This method offers a snapshot of all the phages present in your environmental sample that may infect your host.</p>  <hr><br><h2>Supplies:</h2>  <ul><li>Environmental sample</li>  <li>Liquid media* (5 ml/sample)</li>  <li>Sterile 3 ml or 5 ml syringe</li>  <li>0.22 &micro;m syringe filter</li>  <li>5 ml serological pipettes</li>  <li>Microcentrifuge tubes</li>  <li>15 ml conical tube</li> </ul> <hr><br><h2>Procedure:</h2>  <ol style = 'list-style-type:upper-alpha;'><li>Prepare your bench for aseptic work and assemble your supplies.</li>  <li>You will need an environmental sample collected using the protocol <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-3' >Collecting Environmental Samples (5.1)</a>.</li>  <li>Extract phage from solid environmental samples, such as soil or compost. If you collected a liquid sample, proceed directly to Step D.     <ol><li>Using a 15 ml conical tube, fill it approximately one-third to one-half full with soil.</li><li>Add liquid media until the sample is submerged beneath 2–3 ml of liquid.</li><li>Cap the tube and invert several times to mix thoroughly.</li><li>Incubate the tube while shaking vigorously in a shaking incubator at 250 rpm for 1–2 hours.</li><li>Allow the sample to sit until particulate matter has mostly settled. This may take as few as 2 minutes or as many as 20 minutes. Alternatively, you can centrifuge these samples to hasten this process. To do this, balance the tubes and centrifuge at 2,000 x g for 10 minutes to pellet (i.e., force to the bottom of the tube) most of the soil.</li></ol></li><li>Prepare a phage filtrate using aseptic technique.<ol><li>Open the package of a syringe filter (0.22 &micro;m), leaving the filter in the packaging.</li>  <li>Using a syringe, remove approximately 2 ml of liquid from the top of the flooded sample.<ol style = 'list-style-type:lower-alpha;'><li>Avoid withdrawing any solid material from the bottom of the tube to prevent clogging the filter during filtration.</li></ol></li><li>Attach the syringe to the top of the filter, and then remove the filter from the package. Be careful not to contaminate the filter in the process.<ol style = 'list-style-type:lower-alpha;'><li>Make sure the filter is screwed firmly into place.</li></ol></li><li>Depressing the syringe plunger, dispense a minimum of 0.5 ml of filtrate into a labeled microcentrifuge tube.<ol style = 'list-style-type:lower-alpha;'><li>Because debris can clog the filter, you may encounter resistance. Do not continue to force liquid through the filter or it will break. If your filter clogs, remove the clogged filter, replace it with a new one, and continue filtering.</li><li>Cap the tube immediately.</li></ol></li><li>Discard the syringe and filter.</li>  <li>Proceed directly to the <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-5' >Plaque Assay (5.3) </a>protocol.</li></ol></li> </ol>   <hr><br><h3>Helpful Tips:</h3>  <ul><li>Soils are composed of many different types of particles, each with its own surface charge. Phages stick to these charged soil particles through electrostatic interactions. Vigorously shaking your soil sample in media for 1-2 hours will help break these interactions, releasing the phages from the soil. If you have your sample shaken overnight, make sure you use phage buffer instead of media so as not to accidentally grow pathogenic soil microbes.</li><li>This method will also work well with liquid samples.</li><li>After being shaken, some parts of your soil sample may never settle (bark floats!). Avoid adding the floating particles to your syringe before filtering.</li><li>These samples cannot be stored longer than 24 hours at 4 °C. The concentration of infectious phage decreases rapidly when stored at low concentrations.</li><li>You can do a direct isolation at the same time as an enriched isolation without having to set up a separate experiment. See the <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-7' >Enriched Isolation (5.5)</a> protocol for details.</li><li>A brief video demonstrating this protocol is available <a href = 'https://seaphages.org/video/49/' target = '_blank'>here</a>.</li></ul> *If you are using M. smegmatis as the host for phage isolation, you may use Enrichment Broth as the liquid media in this experiment, as opposed to 7H9 media.</p> "},"5-5":{content:"<h2>Objective:</h2><p>Detecting the presence of phages on bacterial lawns</p><hr><br><h2>Rationale:</h2><p> The plaque assay allows you to visually confirm the presence of phage particles in a sample. It is a versatile assay that can be used for phage isolation, purification, and titering. In a plaque assay, host bacteria are mixed with a phage sample and grown as a lawn on agar. If phages are present, they will infect and replicate within the bacterial host, killing the host in the process. Newly replicated phages diffuse within the agar and repeat the process of infection, replication, and lysis of nearby host bacteria. As a result, a visible circular zone of clearing/killing called a plaque will become apparent on the bacterial lawn. Note that each plaque arises from a single phage particle in the original phage sample.</p><hr><br><h2>Supplies:</h2><ul> <li>Phage samples for isolation, purification, or titering</li><li>Host bacteria (250 &micro;l/plate)</li><li>Agar plates</li><li>Phage buffer</li><li>Top agar, molten (between 55 - 60 ˚C)</li><li>Microcentrifuge tubes</li><li>5 ml serological pipettes</li><li>Positive control phage (optional)</li></ul><hr><br><h2>Procedure:</h2><h3>Day 1</h3><ol style = 'list-style-type:upper-alpha;'> <li>Prepare your bench for aseptic work and assemble your supplies.</li><li>Assemble the samples you want to assay.<p> <strong>Important:</strong>Your phage samples could be taken from your direct isolation, your filtered enriched isolation, or a picked plaque. It is probable that your samples include serial dilutions.</p></li><li>Inoculate the host bacteria with your phage sample.<ol> <li>Obtain the same number of aliquots of 250 &micro;l host bacterial cultures as you have phage samples, plus one for a positive control and one for a negative control.<ol style = 'list-style-type:lower-alpha;'> <li>Label the culture tubes accordingly.</li></ol> </li><li>Use a micropipettor and aseptic technique to<ol style = 'list-style-type:lower-alpha;'> <li>Dispense each phage sample into the appropriate culture tube containing 250 &micro;l of host bacteria according to Table 5.3-1.</li></ol> </li><li>Mix each inoculated host culture by gently tapping the tube.<p><strong>Important:</strong> Make sure your sample makes contact with the bacteria. When you pipette a volume as small as 10 &micro;l sometimes your sample may stick to the side of the tube.</p></li><li>Let your sample sit undisturbed for 5–10 minutes to allow for attachment.<table> <caption> <p style = 'text-align: left;'><strong><span style = 'font-size:14px;'>Table 5.3-1. Volumes of phage samples added to bacterial cultures for a plaque assay.</span></strong></p></caption> <thead><tr><th>Sample Type</th><th>Sample Volume</th></tr></thead> <tbody><tr> <td> <p>Direct isolation sample</p></td><td> <p>500 &micro;l</p></td></tr><tr> <td> <p>Enriched culture</p></td><td> <p>10 &micro;l</p></td></tr><tr> <td> <p>Serial dilutions of picked plaques</p></td><td> <p>10 &micro;l</p></td></tr><tr> <td> <p>Lysates for titering</p></td><td> <p>10 &micro;l</p></td></tr><tr> <td> <p>Negative control</p></td><td> <p>10 &micro;l phage buffer</p></td></tr><tr> <td> <p>Positive control</p></td><td> <p>10 &micro;l provided phage sample</p></td></tr></tbody> </table></li> </ol> </li><li>Plate the samples with top agar. For this part of the experiment you will need 3 ml of molten top agar per sample. Your instructor may provide this for you or you may need to make it according to the protocol found in the <a class = 'chap-linking' tabindex = '0'    data-link-to = '12' >Toolbox</a>. <ol> <li>Obtain the same number of agar plates as you have samples. (Don’t forget your positive and negative control samples.) Label these plates accordingly.</li><li>Remove a bottle of top agar from the 55 °C bath.<br><span><strong>Important</strong></span>: You want to keep the top agar in the 55 °C bath for as long as possible to prevent it from prematurely solidifying on your bench.</li><li>For each sample, including controls <ol style = 'list-style-type:lower-alpha;'> <li>Using a sterile 5 ml pipette, aseptically transfer 3 ml of top agar to an inoculated host tube (i.e., the tube containing bacterial host and phage sample).<br><span><strong>Important</strong></span>: Try to avoid making or withdrawing bubbles, as they can look like plaques on plates.</li><li>Immediately aspirate (suck-up) the mixture back into the pipette and transfer it to the appropriate plate and discard the pipette.<br><span><strong>Important</strong></span>: The top agar should not sit in the pipette for more than a few seconds because the agar will begin to solidify.</li><li>Gently, but quickly, tilt the plate in multiple directions until the top agar mixture evenly coats the agar plate.<br> </li></ol> </li></ol> </li><li>Repeat this process for each of your samples.</li><li>Incubate plates to allow bacterial growth and phage infection.<ol> <li>Let the plates sit undisturbed for ~20 minutes until the top agar solidifies.</li><li>After the top agar has solidified, gently invertthe plates and place in the proper incubator.</li><li>Incubate the plates at the specified temperature for 24–48 hours.</li><li>Record the incubation time and temperature in your notebook.</li></ol> </li></ol><h3>Day 2</h3><ol style = 'list-style-type:upper-alpha;'> <li>Check the plates for plaques. <ol> <li>Remove the plates from the incubator and hold them up to a light source to look for plaques. (It is easier to see them if you remove the lid.) <ol style = 'list-style-type:lower-alpha;'> <li>If you do not see plaques, return the plates to the incubator for further incubation and check again.</li> </ol> </li> <li>Record your results in your lab notebook. <ol style = 'list-style-type:lower-alpha;'> <li>Be thorough. What do you see on your plates? Count the number of plaques and take note of the size, shape, and other distinctive features of the plaques. Remember, negative results are important too.<br> <strong>Important</strong>: The morphology of a plaque is an important characteristic. Simply noting “small” or “round” is not an adequate descriptor of plaque morphology. Try to be specific in your plaque descriptions. A good description will include size, turbidity, margin type, etc.</li> </ol> </li> </ol> </li> </ol><hr><br><h3>Helpful Tips:</h3><ul> <li>When you pipette hot top agar onto the plate, avoid introducing bubbles because they will look like plaques later.</li><li>It is important that the top agar is neither too hot (it will kill bacteria and phages) nor too cold (it sets too quickly).</li><li>A common problem is inverting the plate before the top agar has properly set, resulting in top agar and bacteria sliding off the plate onto the lid! How quickly the top agar sets depends on many factors, including room temperature and humidity. You can check the top agar by gently tapping the side of dish and seeing if it moves. Also, when you pick up the plates, watch what happens when you start to tilt the plate.</li><li>When observing plates after incubating, you may see as few as one plaque and it may be as tiny as a pinpoint. </li><li>If your plates have excessive condensation on the lid, be mindful of drips when viewing your plates.</li><li>A brief video demonstrating this protocol is available <strong><a href = 'https://seaphages.org/video/43/' target = '_blank'>here.</strong></li>"},"5-6":{content:" <h2>Objective:</h2><p>To retrieve phage particles from a plaque and create a liquid sample</p><hr><br><h2>Rationale:</h2><p>A plaque is a zone of clearing on a bacterial lawn that is formed when a single phage particle infects, replicates, and lyses bacteria. As a result, a plaque is filled with millions of identical phage particles. To retrieve phage from a plaque, a plaque is “picked” and phage particles from the plaque are resuspended in phage buffer. You can use the resulting liquid sample of phage for multiple follow-up experiments, including a spot test and plaque purification. </p><hr><br><h2>Supplies:</h2><ul><li>Agar plates with plaques of interest</li><li>Phage buffer</li><li>Microcentrifuge tubes</li></ul><hr><br><h2>Procedure:</h2><ol style = 'list-style-type:upper-alpha;'><li>Prepare your bench for aseptic work and assemble your supplies.</li><li>Label the plaques.<ol><li>Using a labeling pen, mark the plaques you intend to pick by drawing a small circle around the plaque on the bottom of the plate. If picking multiple plaques, label each plaque with a unique letter or number, or with some other identifier.<ol style = 'list-style-type:lower-alpha;'><li>It is possible that you will be picking plaques from more than one agar plate. Be sure to label plaques in a way that will allow you to keep track of them and record the details in your lab notebook.</li></ol></li></ol></li><li>Record the detailed morphology of each plaque (e.g., size, cloudy or clear, margin type) you have circled.</li><li>Label and prepare microcentrifuge tubes.<ol><li>Obtain as many tubes as the number of plaques you intend to pick.</li><li>Label each tube according to the identifier you used for each plaque.</li><li>Using aseptic technique, aliquot 100 &micro;l of phage buffer into each microcentrifuge tube.</li></ol></li> <li>“Pick” a plaque.<ol><li>Place a sterile tip onto a p200 micropipettor.</li><li>Holding the pipettor perpendicular to the agar surface, gently stab the top agar in the center of the plaque (Figure 5.4-1).<ol style = 'list-style-type:lower-alpha;'><li>Avoid touching the bacteria surrounding the plaque.</li></ol></li><li>Place the end of the tip into the phage buffer in the corresponding microcentrifuge tube. Then tap the tip on the wall of the tube and pipette up and down to dislodge phage particles. Discard the tip.</li><li>Mix well by vortexing.</li><li>Repeat Steps 1–4 for each plaque you are picking.</li></ol></li><li>As directed by your instructor, your next step will be to perform the <a class = 'chap-linking' tabindex = '0'    data-link-to = '6-4' >Serial Dilutions (6.2) </a>protocol or the <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-8' >Spot Test (5.6)</a> protocol.</li></ol><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-5-6-1.jpg' alt = 'A picture of a person bracing a petri dish containing transparent circles on an an opaque substrate with one hand and the tip of a pipette entering the substrate over one of the circles. An adjacent illustration depicts a cross-section of the petri dish showing the bottom agar layer, the top agar layer with bacterial lawn growth, a cleared area of bacterial lawn that is a plaque and a pipette tip vertical to and entering the middle of the cleared area.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 5.4-1. Picking a plaque from an agar plate.</div></div><hr><br><h3>Helpful Tips:</h3><p><ul><li>During incubation of the plate, the phage particles are located primarily within the visible plaque, which stops enlarging once the cells stop growing. But after the cells stop growing, phage particles can continue to diffuse into the surrounding areas.  Therefore, it is helpful to pick from reasonably fresh plates. Also, remember that the farther apart plaques are, the more likely they are to be free of contaminating phages from neighboring plaques.</li><li>When you purify a phage, pick from a plate with a low number (2–30) of plaques. The farther away a plaque is from adjacent plaques, the better your chance of picking only one type of phage.</li><li>These samples should be used immediately because concentration of infectious particles decreases rapidly in phage buffer.</li><li>A brief video demonstrating this protocol is available <a href = 'https://seaphages.org/video/44/' target = '_blank'> here</a>.</li></ul></p>"},"5-7":{content:"<h2>Objective:</h2><p>To amplify phages present in your environmental samples</p><hr><br><h2>Rationale:</h2><p> Enriched cultures are used to amplify phages present in samples by providing favorable conditions for phage replication. Phages are first extracted from a solid environmental sample into bacterial growth media. Bacteria, including those that are potentially dangerous, are removed via filtration. The filtered sample is seeded with host bacteria and the cultures are incubated to encourage growth and replication. As bacteria replicate, phages present in the sample go through multiple rounds of infection and lysis. The result is a culture with an exponentially larger concentration of phages specific to your bacterial host than your initial environmental sample.</p><hr><br><h2>Supplies:</h2><ul><li>Solid environmental sample</li><li>0.22 &micro;m Corning ® Tube-Top Vacuum Filter Systems or syringe filters</li><li>Liquid media*</li><li>10X liquid media (if using liquid environmental samples)</li><li>Baffled Erlenmeyer flask, Erlenmeyer flask autoclaved with pipette tips in the bottom, or 50 ml sterile conical tubes</li><li>Sterile 5 ml syringes (if needed)</li><li>0.22 &micro;m syringe filters (if needed)</li><li>Microcentrifuge tubes</li><li>Host bacteria (500 &micro;l)</li></ul><hr><br><h2>Procedure:</h2><h3>Day 1</h3><ol start = '1' style = 'list-style-type: upper-alpha;'> <li>You will need the solid environmental samples you collected using the protocol <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-3' >Collecting Environmental Samples (5.1)</a>.<br> </li> <li>Extract phages from a soil sample. <ol> <li>Fill a 50 ml conical tube with your sample to the 15 ml mark.</li> <li>Add liquid media to the 35 ml mark and vortex.</li> <li>Shake the sample at ~250 rpm for 1–2 hour<sup>1</sup>.</li> <li>Balance the tubes and centrifuge at 2,000 x g for 10 minutes to pellet (i.e., force to the bottom of the tube) most of the soil<sup>2</sup><a>.</a><br> </li> </ol> </li> <li>Prepare your bench for aseptic work and assemble your supplies. <ol> <li>Filter the supernatant through a 0.22 &micro;m filter to remove unwanted bacteria and soil particles<sup>3</sup>. <ol start = '1' style = 'list-style-type: lower-alpha;'> <li>Collect the flow through in a sterile baffled Erlenmeyer flask or a 50 ml sterile conical tube.</li> <li>Recovered volumes will range between 20 and 25 ml.<br> </li> </ol> </li> </ol> </li> <li>Seed the culture with host bacteria. <ol> <li>Add 0.5 ml of bacterial host culture to the flask or conical tube.</li> <li>Incubate the flask or conical tube at the proper temperature, shaking at 220 rpm for 2–5 days. <ol start = '1' style = 'list-style-type: lower-alpha;'> <li>If you are using a 50 ml conical tube, you must ensure that the culture will be properly aerated. To do so, screw the cap on one-quarter of a turn so that the conical tube is only loosely capped, and then secure the cap with a short piece of lab tape to ensure it does not fall off. Check to make sure that the conical tube remains only loosely capped. Tubes must remain upright while being shaken, and care taken to avoid spillage.</li> <li>If using a liquid environmental sample, you must add the appropriate volume of 10X liquid media as a source of nutrients for your host bacteria.</li> </ol> </li> </ol> </li> </ol> <h3>Day 2</h3> <p>After the enriched culture has been allowed to incubate for 2–5 days, you can continue with the protocol.</p> <ol start = '1' style = 'list-style-type: upper-alpha;'> <li>Prepare your bench for aseptic work and assemble your supplies.<br> </li> <li> Filter the enriched culture. <ol> <li>Using an appropriate pipette, transfer 1.4 ml of your enriched culture from the Erlenmeyer flask to a microcentrifuge tube.</li> <li>Repeat this procedure so that you have two microcentrifuge tubes, each with 1.4 ml of enriched culture.</li> <li>Spin the tubes at high speed in the microcentrifuge for 1 minute to pellet the bacteria.</li> <li>If your supernatant is not clear or if you suspect your enrichment contains non-host bacteria, filter the supernatant through a 0.22 &micro;m filter as described below. Otherwise, proceed directly to Step 5. <ol start = '1' style = 'list-style-type: lower-alpha;'> <li>Remove the plunger from a syringe.</li> <li>Open a sterile filter and attach it to the barrel of the syringe.</li> <li>Pipette 1 ml of supernatant from each microcentrifuge tube into the syringe barrel (for a total of 2 ml).</li> <li>Place the tip of the filter/syringe over a sterile microcentrifuge tube and insert the plunger into the syringe.</li> <li>Depress the plunger and collect the sterile filtrate.</li> </ol> </li> <li>Transfer the supernatant into a clean microcentrifuge tube, avoiding the bacterial pellet.</li> <li> Immediately cap the microfuge tube containing your supernatant or filtrate and label it appropriately. It should be stored at 4 °C.</li> <li>Either return your culture to the incubator, or dispose of your enriched culture as directed by your instructor.<br> </li> </ol> </li> <li>As directed by your instructor, your next step will be to test your supernatant for phages by using a <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-8' >Spot Test (5.6)</a>.</li> </ol><hr><br><h3>Helpful Tips:</h3></p><ul><li>If you want to do a direct isolation at the same time as your enriched isolation you can remove 500 &micro;l of filtered sample prior to Step D and set up a plaque assay directly. Be sure to work aseptically.</li><li>If you want to test your culture after 24 or 48 hours of shaking, feel free to remove a 1.4 ml aliquot with a serological pipette, filter it, and proceed with a spot test. Be sure to work aseptically and replace your culture on the shaker while you await your results.</li><li>This protocol is designed for solid soil samples only. If you want to isolate phages from a liquid sample use the Direct Isolation protocol.</li><li>50 ml conical tubes can be incubated with the lids tightly sealed as long they are opened every 24 hours to allow access to oxygen. Alternatively, large sized culture tubes with loose fitting lids can be used when incubating enriched isolations.</li></ul><p><strong>References:</strong></p><p><ol><li>If you do not have a shaker that accommodates 50 ml conical tubes this can be done in an Erlenmeyer flask.</li><li>If you do not have a centrifuge to accommodate 50 ml conical tubes, the wetted soil can be filtered through Whatman filter paper folded to fit into the top of a conical tube before sterile filtering.</li><li>Corning ® Tube-Top Vacuum Filter Systems or sterile syringes with syringe filters can be used.</li><li>* If you are using<em>M. smegmatis</em> as the host for phage isolation, you may use Enrichment Broth as the liquid media in this experiment, as opposed to 7H9 media.</li><li>A brief video demonstrating this protocol is available <a href = 'https://seaphages.org/video/50/' target = '_blank'> here</a>.</li></ol>"},"5-8":{content:" <h2>Objective:</h2><p>To test a sample for the presence of phage</p><hr><br><h2>Rationale:</h2><p>Plaques are zones of clearing on a bacterial lawn that are formed when phages infect, replicate, and lyse bacteria. Spot tests can be used to screen enriched cultures or verify the presence of phage in putative plaques. By “spotting” a sample onto a fresh lawn of bacteria, you can confirm the presence of phage if a zone of clearing occurs following incubation.  </p><hr><br><h2>Supplies:</h2><ul><li>Liquid phage sample (either a picked plaque from a direct isolation, or an enriched isolation)</li><li>Agar plates</li><li>Host bacteria (250 &micro;l/plate)</li><li>Top agar, molten (between 55 - 60 ˚C)</li><li>Phage buffer</li><li>5 ml serological pipette</li></ul><hr><br><h2>Procedure:</h2><ol style = 'list-style-type:upper-alpha;'><li>Prepare your bench for aseptic work and assemble your supplies.</li><li>Collect the liquid phage samples that need testing.</li><li>Prepare a bacterial lawn by using aseptic technique.<ol><li>Label the bottom of an agar plate.<ol style = 'list-style-type:lower-alpha;'><li>Divide the plate into as many sections as you have samples by drawing on the bottom of the plate. (See Figure 5.6-1 for examples of how to best divide the plate into sections.) Label each section according to your samples.</li></ol></li><li>Obtain a 250 &micro;l culture of host bacteria.</li><li>For this part of the experiment you will need 3 ml of molten top agar per plate. Your instructor may provide this for you or you may need to make it according to the protocol found in the <strong><a class = 'chap-linking' tabindex = '0'    data-link-to = '12' >Toolbox</a></strong>.<ol style = 'list-style-type:lower-alpha;'><li>Using a sterile 5 ml pipette, transfer 3 ml of molten top agar to a culture tube containing host bacteria and then immediately draw the solution back into the same pipette.<p><strong>Important:</strong> Try to avoid making or withdrawing bubbles, as they can look like plaques on plates.</p></li></ol></li><li>Dispense the top agar-bacteria mixture onto an agar plate.<ol style = 'list-style-type:lower-alpha;'><li>The top agar should not sit in the pipette for more than a few seconds because the agar will begin to solidify.</li><li>Gently, but quickly, tilt the plate in multiple directions until the top agar mixture evenly coats agar plate.</li></ol></li><li>Allow the plate and to sit undisturbed for 20 minutes or until the top agar solidifies completely.</li></ol><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-5-8-1.jpg' alt = 'An illustration of two circles representing agar plates with different spot patterns. The image on the left shows spots around the edge of the circle and one in the middle, the image on the right shows spots lined up in a 3 by 3 grid.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 5.6-1. Possible spot plate patterns.</div></div></li><li>Spot the liquid phage samples and controls on the prepared bacterial lawn.<ol><li>Aseptically transfer 10 &micro;l of each sample, one at a time, onto the proper location on the bacterial lawn<ol style = 'list-style-type:lower-alpha;'><li>Hold the tip slightly above the agar and expel the drop slowly to avoid splattering.</li><li>Be sure to spot your samples in the right place! Remember that labels on the bottom of a plate are mirror images (i.e., they will appear backward) of your labeling scheme when the plate is turned over.</li></ol></li><li>Spot 10 &micro;l of sterile phage buffer on the plate as a negative control.</li><li>Allow the liquid from the spots to absorb into the agar (generally 10–15 minutes).</li><li>Without inverting the plates, incubate them at the proper temperature for 24–48 hours.</li></ol></li><li>Check spot plates for clearing.<ol><li>After at least 24 hours, check each spot on the agar plate. If you see a zone of clearing for any of your spotted samples, congratulations! Your original sample contained phage!</li><li>Make sure that your negative control does NOT show signs of phages.</li></ol></li><li>Record the details of your spot plate in your laboratory notebook.</li><li>You can now proceed to Chapter 6, <a class = 'chap-linking' tabindex = '0'    data-link-to = '6' >Purifying Your Phage!</a></li></ol><hr><br><h3>Helpful Tips:</h3><ul><li>You should include no more than  10 samples on a single plate.</li><li>A zone of clearing in a spot test is NOT a plaque; rather, it is many overlapping plaques that appear as a single large zone of clearing.</li><li>It is not recommended to pick a phage for purification from a spot if multiple phages were spotted on the same plate.</li><li>A brief video demonstrating this protocol is available<a href = 'https://seaphages.org/video/45/' target = '_blank'> here</a>.</li></ul>"},"6-1":{content:"<p>Now that you have isolated phages from your environmental sample, you are ready to start the purification process. It is possible that your environmental sample yielded more than one kind of phage (Figure 6.0-1). The goal of phage purification is to ensure that you end up with a phage sample that contains only one kind of phage. In other words, once purified, all the phage particles in your sample should be genetic clones of each other. This is called a “homogenous population” or a “clonal population” of phage.</p><p>Having a pure phage population is very important because it will enable you to characterize your phage and discover its biological properties, including morphology, plaque size, and genome sequence. Imagine how difficult it would be to ascribe these properties to your phage if your sample contained many kinds of phages, each with its own morphologies and traits!</p><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-6-1-1.png' alt = 'The image contains three pictures of opaque plates, labeled A, B and C. Picture A contains large and small transparent circles, picture B contains small uniform sized transparent circles , and picture C contains large uniform sized transparent circles.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 6.0-1. Phage purification. Before purification a heterogeneous phage sample may have more than one plaque morphology (A). Following purification, the phage with the small (B) and large (C) plaques are homogeneous populations.</div></div>  <h2>Picking a Plaque</h2><p>The best method to ensure that your phage sample is a single clonal population is with plaque purification. This is accomplished by separating the phages from each other by diluting the samples so that only a few phage particles are placed on a plate. This will enable you to obtain well-isolated plaques.</p><p>Because each plaque on a plate originated from a single phage particle, a sample of genetically identical phage can be prepared by touching a single plaque with a stick or pipette tip to pick up phage and then swirling the tip in phage buffer. This is called “picking a plaque”. This liquid sample of phage particles can then be serially diluted and examined by plaque assay for consistency of plaque morphology according to the<a class = 'chap-linking' tabindex = '0'    data-link-to = '6-3' > Plaque Assay for Purification protocol 6.1</a>. This process is repeated until all plaque morphologies are consistent. Usually, no more than three rounds of purification are needed.</p><p>When purifying a phage it is important to understand that phage particles diffuse within the agar plate; therefore, phages from a nearby plaque could diffuse into your selected plaque. To minimize the presence of contaminating phages, you should pick from a well-isolated plaque taken from a highly dilute phage sample. Likewise, if multiple positive spot tests are present on the same agar plate, phages can diffuse from one spot to another. For this reason, you should not pick from a spot plate that has multiple positive spots (unless they all came from the same direct isolation plate). You would not have any way to guarantee that the phage you eventually isolate originated from the spot that you picked from, rather than from an adjacent spot. Therefore, when you perform phage purification, use the original direct isolation plaque or the enriched culture instead of picking phages from the positive spots.</p>   <h2>Collecting a Lysate and Calculating Titer</h2><p>The last steps in phage purification are collecting a lysate and calculating the concentration, or titer, of that lysate. To do this, collect a lysate from a plate densely packed with plaques from your last round of purification (<a class = 'chap-linking' tabindex = '0'    data-link-to = '6-5' >protocol 6.3, Collecting Plate Lysates</a>). The best lysates come from plates that are so densely packed that the plaques cover the entire plate, leaving remnants of bacteria in a pattern that looks like a spiderweb. Plates like this, with confluent plaques, are called “webbed” plates. When phage buffer is added directly to the plate surface, phage particles from the plaques diffuse into the buffer, which can then be collected and filtered to separate out the phage. Finally, the titer, or concentration, of phage particles can be calculated.</p> <h2>Calculating the Titer</h2><p><a class = 'chap-linking' tabindex = '0'    data-link-to = '6-6' > Spot Titer (protocol 6.4)</a> followed by a <a class = 'chap-linking' tabindex = '0'    data-link-to = '6-7' > Full Plate Titer (protocol 6.5)</a>. The spot titer is typically used to provide a quick titer estimate by spotting multiple dilutions of a phage sample on a single plate. The titer is calculated by counting the number of plaques in a specific spot. This can be tricky and inaccurate, therefore providing only an estimate. The full plate titer requires separate plates for each dilution. Although this method uses more plates, it allows you to accurately count plaques and calculate more precise titer values.</p><p>By performing a spot titer and a full plate titer consecutively, you can use the estimate from the spot titer to predict which dilutions (i.e., how many phage) will yield a countable number of plaques, thereby minimizing the number of plates needed for a full plate titer. Because the titer of phage lysates is determined by counting plaques, a titer is typically presented as the number of plaque-forming units, or pfu, in every milliliter of lysate, or pfu/ml. In practice, phage lysates typically have titers in the range of 108 pfu/ml to 1011 pfu/ml. </p><p>Once you have an initial lysate you can amplify your phage to generate enough lysate for collecting DNA and archiving as described in  <a class = 'chap-linking' tabindex = '0'    data-link-to = '7' >Chapter 7: Phage Amplification</a>.</p><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-6-1-2.jpg' alt = 'A flowchart for purification: plaque assay for purification branches in two directions. One direction leads to direct isolation plaque, then picking a plaque. The other direction leads to positive enriched isolation sample. Both branches then point to serial dilutions then plaque assay, then the question are your plaque morphologies consistent and have you done more than one round of purification? If no, then repeat plaque assay for purification. If yes, then collect plate lysates, spot titer, full plate titer and proceed to amplification.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 6.0-2. Flow chart of phage purification experiments.</div></div>"},"6-2":{content:"<h2>How many rounds of purification?</h2><ul><li>First, it is important to note that phage will continue to diffuse across an agar plate beyond the plaque, diffusing further with time. It is therefore important to pick plaques for purification soon after plaques are visible. Don’t allow diffusion of phages from surrounding plaques to complicate the purification process.</li><li>According to classic microbiology techniques, three rounds of purification should be performed to ensure a single clonal phage population.</li><li> Additional rounds of purification are discouraged because they provide your phage with the opportunity to accumulate mutations when its genome is replicated within the bacterial host. This leads to genome truncations and other issues that are problematic during annotation.</li><li>The chances of a well-isolated plaque from a direct isolation experiment containing multiple types of phages is extremely low. Therefore, one or two rounds of purification may be sufficient, especially if there is only a single plaque on the plate. Regardless, always take into account the consistency of plaques formed on plates.</li><li>Some phages can have plaques with different morphologies, typically varying in plaque size, and no amount of purification will change this. Should you encounter a phage that forms different plaque morphologies even after 3 rounds of purification, consider picking a plaque for each morphology, serially dilute each sample, and plate dilutions for each sample using a plaque assay to see if these morphologies persist to can be separated. If the morphologies separate (i.e. each plate only has plaques with one morphology), you’ve shown that there are two phages. If the multiple morphologies persist (i.e. each plate continues to contain all the morphologies), it indicates that your phage likely has multiple plaque morphologies and you have reciprocal replicates confirming this.</li><li>The process of phage isolation is complete when a lysate is collected and its titer calculated. This initial lysate is collected from the last round of purification, and once plaque morphologies are consistent on all of the dilution plates. The plate used to collect this lysate will probably not be a perfect webbed plate, which is okay. Note: this is no longer called a Medium Titer Lysate since it could easily have as high a titer as the final lysate.</li></ul><h2>Warning about mutations.</h2><ul><li>When picking a plaque, make sure you are picking a plaque that is representative of your phage sample. If you have thirty turbid plaques on a plate and one clear plaque. Pick a turbid plaque.</li><li>Make sure that your plaque morphology remains consistent over time. If you have a cloudy plaque that suddenly turns clear you may have isolated a mutant and should go back and pick another turbid plaque.</li><li>If you have a dilution series that has two morphologies on a low dilution plate, you should make an attempt to separate the plaque morphologies as described above, but do not continue to do so for many rounds of purification.</li></ul><h2>Changes from the Resource Guide</h2><ul><li>We are now recommending no more than three rounds of plaque purification to prevent the accidental creation of mutants in the lab.</li><li>Using plaque streaks for purification is strongly discouraged as it seems to contribute to mixed phage samples.</li></ul>"},"6-3":{content:"<h2>Objective:</h2><p>To generate well-isolated plaques</p><hr><br><h2>Rationale:</h2><p>Before you begin characterizing a phage, you need to ensure that you have a clonal phage population. The process involves serially diluting either a positive enriched culture or after rounds of purification, a picked plaque, and then performing a plaque assay with the diluted samples. By repeating this process one or two times, and always picking a plaque that is well isolated from adjacent plaques, you will obtain a clonal phage population.</p><hr><br><h2>Supplies:</h2><ul><li>Phage samples for purification</li><li>Phage buffer</li><li>Microcentrifuge tubes</li><li>Host bacteria</li><li>Agar plates</li><li>Top agar</li><li>5 ml serological pipettes</li></ul><hr><br><h2>Procedure:</h2><p>This is not actually a new procedure, but a combination of protocols that you have already used. The protocols referenced here are found in previous chapters as well as the Toolbox.</p><ol style = 'list-style-type:upper-alpha;' > <li>Prepare your bench for aseptic work and assemble your supplies.<br>  </li> <li>Gather your phage samples. <ol> <li>If you are starting from an enriched isolation that tested positive for phage, skip to    Step D.</li> <li>If your phage sample is a plaque, draw a circle around the plaque of interest on the bottom of the plate and label it. Record the details in your lab notebook.<br>  </li> </ol> </li> <li>Follow the <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-6' >Picking a Plaque(5.4)</a> protocol to pick the plaque(s) you want purify. <ol> <li>It is best to pick only well-isolated plaques (at least 1.5 cm apart) from a fresh plate.</li> <li>At the end of the “Picking a Plaque” protocol, you will have liquid phage samples.<br>  </li> </ol> </li> <li>Dilute your liquid phage samples according to the <a class = 'chap-linking' tabindex = '0'    data-link-to = '6-4' >Serial Dilutions (6.2)</a> protocol.<br>  </li> <li>Prepare your agar plates. <ol> <li>Obtain as many agar plates as you have phage samples to purify.</li> <li>Label the bottom of the plate with your name, the date, and a reference to the sample.<br>  </li> </ol> </li> <li>Plate your dilutions according to the <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-5' >Plaque Assay (5.3)</a> protocol.<br>  </li> <li>After incubating the plates for 24–48 hours, record your results in your lab notebook. Consider the following questions:<ul  style = 'list-style-type:disc' ><li>How many plaques are on each plate? Do the number of plaques on your plates follow the expected pattern of your serial dilutions?</li><li>Do all your plaques share the same morphology?</li><li>If you have more than one morphology, how do they differ?</li><li>If you think you still have more than one kind of phage in your sample, what should you do next?</li></ul></li><li>Once you are certain you have a single phage population, you can move on to the <a class = 'chap-linking' tabindex = '0'    data-link-to = '6-5' >Collecting Plate Lysates (6.3)</a> protocol as directed by your instructor.<ol><li>To collect plate lysates, use the best webbed plate from your final round of phage purification.</li></ol></li> </ol><hr><br><h3>Helpful Tips:</h3><ul><li>It is not uncommon for a phage to form plaques that have more than one morphology. If you continue to observe plaques with multiple morphologies, even after carefully performing one or two rounds of purification, you may have such a phage.</li><li>Phage morphologies are also known to change based on concentration. It is not uncommon for plaques to look smaller when they are closer together at higher concentrations.</li><li>When purifying a phage, its size is not the most reliable morphological trait since the size can sometimes vary. Instead, the presence or absence of haloes or defined borders, as well as the turbidity of plaques, is more dependable.</li></ul>"},"6-4":{content:"<h2>Objective:</h2><p> To prepare liquid phage samples of decreasing concentrations</p><hr><br><h2>Rationale:</h2><p> Serial dilution is a method used to manipulate the number of phages in a sample. You will use serial dilutions to purify, amplify, and titer your phage. This protocol uses 10-fold serial dilutions, meaning that the concentration of phage in each tube is 10 times less than the previous tube, allowing for easy mathematical calculations.</p><hr><br><h2>Supplies:</h2><ul><li>Phage buffer</li><li>Samples requiring diluting (e.g., picked plaques or lysate)</li><li>Microcentrifuge tubes</li></ul><hr><br><h2>Procedure:</h2><ol style = 'list-style-type:upper-alpha;'> <li>Prepare your bench for aseptic work and assemble your supplies.<br>  </li> <li>Set up 10-fold serial dilutions. <ol> <li>Arrange the proper number of microcentrifuge tubes in a rack and label them 10<sup>-1</sup>, 10<sup>-2</sup>, 10<sup>-3</sup>,….10<sup>-8</sup>.<br> <span ><strong>Important</strong></span>: See “Helpful Tips” to determine how many dilutions to make.</li> <li >Add 90 &micro;l of phage buffer to each of the tubes.<br>  </li> </ol> </li> <li>Perform 10-fold serial dilutions (Figure 6.2-1). <ol> <li>Add 10 &micro;l of your undiluted phage sample to the “10<sup>-1</sup>” tube and vortex well. <ol style = 'list-style-type:lower-alpha;'> <li>The solution in this “10<sup>-1</sup>” tube contains 1/10<sup>th</sup> the number of phage particles as your undiluted sample. It is also referred to as a 1:10 dilution.</li> <li>Make sure to use a clean pipette tip for each transfer and pipette carefully, vortexing your sample well before making each dilution. Otherwise, you will not make accurate 10-fold dilutions.</li> </ol> </li> <li>Transfer 10 &micro;l of the “10 <sup>-1</sup>” sample to the “10<sup>-2</sup>” tube and vortex well. <ol style = 'list-style-type:lower-alpha;'> <li>This solution contains 1/100<sup>th</sup> as many phage particles as your undiluted sample. It can also be referred to as your 1:100 dilution.</li> <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-6-4-1.jpg' alt = 'An illustration of 9 partially-filled tubes, each containing 90 microliters, with a progressively increasing lighter color in each from left to right. The first tube on the left is labeled original sample and the rest are labeled 10 to the negative one through 10 to the negative 8. An arrow labeled 10 microliters is drawn between each tube.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 6.2-1. Serial dilutions. The serial dilution of phage in preparation for a plaque assay. A volume of the undiluted phage sample (10 &micro;l) is transferred into 90 &micro;l of phage buffer in the 10<sup>-1</sup> tube. After the tube is mixed, 10 &micro;l of the 10<sup>-1</sup> sample is transferred into the 10<sup>-2</sup> tube, and the tube is mixed. This serial dilution continues until the desired dilutions are obtained. The last tube will contain a final volume of 100 &micro;l.</div></div> </ol></li> <li>Continue each successive dilution until you get to your last tube.</li> </ol> </li> <li>Return to the protocol requiring the diluted samples; for example, <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-5' >Plaque Assay (5.3)</a>,<a class = 'chap-linking' tabindex = '0' data-link-to = '6-6'> Spot Titer (6.4) </a>, <a class = 'chap-linking' tabindex = '0'    data-link-to = '6-7' >Full Plate Titer (6.5)</a>, and<a class = 'chap-linking' tabindex = '0'    data-link-to = '7-2' > Making Webbed Plates (7.1)</a>.</li></ol><hr><br><h3>Helpful Tips:</h3><ul><li>The number of dilutions you make depends on the experiment you are doing and the phage you are working with.</li><li>If you are diluting an enriched culture for the plaque assay, diluting to 10<sup>-5</sup> is usually sufficient.</li><li>If you are diluting to titer a plate lysate, you may need to dilute to 10<sup>-8</sup> to obtain a countable number of plaques. You want to plate a dilution that can produce 20–200 plaques.</li><li>Phages at low concentration tend to lose viability over time when they are stored in phage buffer. Therefore, storing serial dilutions is not recommended.</li><li>A brief video demonstrating this protocol is available <a href = 'https://seaphages.org/video/47/' target = '_blank'>here</a>.</li></ul><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-6-4-2.jpg' alt = 'Image is a picture of 6 opaque petri dishes marked 10 to the negative one through 10 to the negative 6. The first two plates are fully opaque. The third has numerous lightly transparent small circle. The forth shows numerous transparent circles. The fifth shows fewer transparent circles and the sixth shows one transparent circle. '/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 6.2-2. Plaque assay of dilution series. The phage StarStuff was serially diluted and plated for a plaque assay. The most concentrated samples (10<sup>-1</sup> and 10<sup>-2</sup>) create cleared plates where no bacteria are visible. The 10<sup>-3</sup> plate shows a very high concentration of plaques, but individual plaques are not visible until the 10<sup>-4</sup> plate, which is webbed plate. The 10<sup>-5</sup> and 10<sup>-6</sup> plates have countable numbers of plaques, but the 10<sup>-5</sup> plate offers the most accurate number for counting.</div></div>"},"6-5":{content:"<h2>Objective:</h2><p> To generate a highly concentrated liquid phage sample</p><hr><br><h2>Rationale:</h2><p>Depending on the phage, each plaque on a plate may contain between 103 and 107 phage particles. These phage are collected to generate a concentrated phage solution called a lysate. Lysates are collected by “flooding” a plate with a high concentration of phage, such as a webbed plate (Figure 6.3-1). Flooding is accomplished by adding phage buffer to cover the surface of the plate and allowing the phage to diffuse into the buffer. The buffer is then harvested (removed from the plate by syringe) and filtered to remove agar particles and bacteria. Lysates are used for many basic phage biology experiments, including phage DNA extraction and transmission electron microscopy, as well as for long-term phage storage.</p><hr><br><h2>Supplies:</h2><ul><li>Webbed plate(s) with clonal phage population</li><li>Phage buffer (8 ml/plate)</li><li>0.22 &micro;m filter</li><li>5 ml syringe</li><li>15 ml sterile conical tube for lysate storage</li></ul><hr><br><h2>Procedure:</h2><ol style = 'list-style-type:upper-alpha;'><li>Prepare your bench for aseptic work and assemble your supplies.</li><li>Identify one or more plates for lysate collection.<ol><li>Your webbed plate may be one prepared from your last round of plaque purification or from the protocol <a class = 'chap-linking' tabindex = '0'    data-link-to = '7-2' >Making Webbed Plates (7.1)</a>.</li></ol><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-6-5-1.png' alt = 'A picture of an opaque plate with numerous lightly transparent circles.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 6.3-1. Webbed plate with confluent plaques.</div></div>      </li><li>Flood the webbed plate(s).<ol><li>Apply 8 ml of sterile phage buffer to the webbed plate.</li><li>Let the plate(s) sit at room temperature for 2–4 hours OR store the plates overnight (12–14 hours) at 4 °C.</li><li>Swirl the phage buffer gently, taking care not to splash.</li></ol></li><li>Harvest a plate lysate<ol><li>When the incubation time is complete, remove the lid from the plate and place it on the bench. Tilt the plate slightly by placing one edge of the plate on the lid, allowing the lysate to pool to one side (Figure 6.3-2).</li><li>Prepare a 0.22 &micro;m filter by opening the packaging but not removing the filter. Set aside.</li><li>Using a 5 ml syringe aspirate (suck up) the lysate from the plate.</li><li>Carefully attach the syringe to the filter. Depress the syringe plunger and collect the filtrate in a 15 ml sterile conical tube.</li><li>If you still have unfiltered lysate remaining on your plate, remove the filter and store it in its plastic packaging to maintain sterility. Aspirate the remaining lysate, reattach the used filter, and filter the remaining lysate, collecting the filtrate in the same sterile conical tube.</li><li>Label the tube appropriately.<div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-6-5-2.jpg' alt = 'A picture of a plate with a  yellow liquid floating on top of a pale yellow substrate. The plate is tipped with one end resting on the lid. '/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 6.3-2. Collecting a plate lysate. After flooding a plate with phage buffer, the lysate can be collected by tipping the plate and allowing the liquid to pool to one side. It can then be aspirated into a syringe.</div></div></li></ol></li><li>Pool the lysates. <ol> <li>If you have multiple webbed plates for a single phage sample, repeat Step D for each plate. The same filter can be used for all plates if it remains sterile and unclogged. However, if it has been contaminated or becomes clogged, use a new filter.</li> <li>Combine all of the filtered lysates into the same sterile conical tube.</li> <li>Record the final volume of lysate collected.<br>  </li> </ol> </li><li>Store the lysates at 4 °C. <ol> <li>This lysate can be stored for months at 4 °C. For longer-term storage, follow the protocol <a class = 'chap-linking' tabindex = '0'    data-link-to = '7-4' >Archiving Your Phage Sample (7.3)</a>.<br>  </li> </ol> </li><li>According to your instructor’s directions, calculate the titer of your lysate using the <a class = 'chap-linking' tabindex = '0'    data-link-to = '6-6' >Spot Titer (6.4) </a>protocol, the <a class = 'chap-linking' tabindex = '0'    data-link-to = '6-7' >Full Plate Titer (6.5) protocol</a>, or both.</li></ol><hr><br><h3>Helpful Tips:</h3><ul><li>Phages remain viable in lysates because compounds like albumin and glycerol diffuse into the phage buffer and stabilize the capsid and tail proteins.</li><li>It is OK if your last round of purification did not yield a perfect webbed plate. Use the plate that contains the highest number of phage. </li><li>A brief video demonstrating this protocol is available <a href = 'https://seaphages.org/video/48/' target = '_blank'>here</a>.</li></ul>"},"6-6":{content:"<h2>Objective:</h2><p>To determine the concentration of phage particles in a lysate using a spot test</p><hr><br><h2>Rationale:</h2><p> The total number of phage particles in your lysate can be calculated by performing serial dilutions, followed by a spot test for each dilution and then counting the number of plaques formed. By taking into account the volume and dilution used to generate those plaques, you can calculate the concentration, or titer, of your phage lysate. Because each plaque is formed from a single phage particle, that original phage particle is called a plaque-forming unit (pfu). The concentration of a lysate is presented as the number of plaque-forming units in every milliliter of lysate, or pfu/ml. The spot titer is a modified plaque assay that uses just one agar plate for as many as eight lysate dilutions, which allows you to calculate the titer of your lysate. The drawback of using a spot titer is that the number of countable plaques per spot is small, introducing a large margin of error. Therefore, a spot titer is used to predict which dilutions you should use when performing a full plate titer.</p><hr><br><h2>Supplies:</h2><ul><li>Lysate for titering</li><li>Agar plate</li><li>Host bacterial culture</li><li>Top agar, molten (between 55 - 60 ˚C)</li><li>Phage buffer</li><li>Microcentrifuge tubes</li><li>5 ml serological pipettes</li></ul><hr><br><h2>Procedure:</h2><ol style = 'list-style-type:upper-alpha;'><li>Prepare your bench for aseptic work and assemble your supplies.</li><li>Label one agar plate for a spot titer.<ol><li>Label the bottom of an agar plate with your name, the date, and the notation “spot titer.”</li><li>Using a labeling pen, divide the plate into as many sections as you have samples. Remember to include a section for a negative control. See Figure 6.4-1 for examples of how to best divide the plate into sections. Label each section according to your samples.</li></ol><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-6-6-1.jpg' alt = 'A picture of two opaque plates. The plate on the right is marked with a 3 by 3 grid with number notations from negative one through negative 8. Five areas of transparent spots fill the cells marked negative one through negative 5. The plate on the left has 5 large transparent spots around the edge of the plate. The plate is marked with number notations negative one through negative 8 and the spots are next to notations negative one through negative 5. '/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 6.4-1. Two spot plate examples.</div></div></li><li>Prepare a bacterial lawn using aseptic technique. (For this part of the experiment you will need 3 ml of molten top agar per plate. Your instructor may provide this for you or you may need to make it according to the protocol found in the <a class = 'chap-linking' tabindex = '0'    data-link-to = '12' >Toolbox</a>).<ol><li>Using a sterile 5 ml pipette, transfer 3 ml of molten top agar into a 5 ml sterile pipette.</li><li>Transfer the top agar to a culture tube containing 250 &micro;l of host bacteria and immediately draw the solution back into the same pipette.</li><li>Dispense the top agar-bacteria mixture onto an agar plate.</li><li>Gently, but quickly, tilt the plate in multiple directions until the top agar mixture evenly coats the agar plate.</li><li>Allow the plate to sit undisturbed until it solidifies completely. This will take at least 10 minutes.</li></ol></li><li>Perform the <a class = 'chap-linking' tabindex = '0'    data-link-to = '6-4' >Serial Dilutions (6.2)</a> protocol on the sample you wish to titer.<ol><li>Lysates should be diluted to 10<sup>-1</sup>, 10<sup>-2</sup>, 10<sup>-3</sup>, 10<sup>-4</sup>, 10<sup>-5</sup>, 10<sup>-6</sup>, 10<sup>-7</sup>, and 10<sup>-8</sup> in phage buffer.</li></ol></li><li>Spot the dilutions and controls on the prepared bacterial lawn.<ol><li>One at a time, aseptically transfer 3 &micro;l of all samples onto the proper location on the bacterial lawn.</li><li>Use 3 &micro;l of sterile phage buffer as the negative control.</li><li>Allow the liquid from the spots to absorb into the agar for 30 minutes or longer.</li><li>Incubate plates (right-side up, not inverted) at the proper temperature for 24– 48 hours.</li></ol></li><li>After incubating the plates, count the number of plaques.<ol><li>After at least 24 hours, check each spot on the agar plate.</li><li>Do the numbers of plaques in each spot make sense? For example, is there a 10-fold reduction in number of plaques as you move through the spots? If so, choose the dilution(s) that contain a countable number of plaques.</li></ol></li><li>Calculate the titer in pfu/ml using the formula:<p>Titer (pfu/ml)  =  (# pfu/ volume used in &micro;l) x (10<sup>3</sup> &micro;l/ml) x dilution factor*</p><p>*The dilution factor is the reciprocal of the dilution used. For a 10<sup>-7</sup> dilution, the dilution factor is 10<sup>7</sup>.</p><p>Example: If 12 plaques were observed on a spot created by 3 &micro;l of a 10<sup>-7</sup>  dilution.</p><p>Titer (pfu/ml)  =  (12 pfu/3 &micro;l) x  (10<sup>3</sup> &micro;l/ ml) x (10<sup>7</sup>)</p><p>     =  (4 x 10<sup>3</sup> x 10<sup>7</sup>) pfu/ml</p><p>     =  4 x 10<sup>10</sup> pfu/ml</p><p>Essentially, you are dividing the number of plaques by the volume of sample plated. Then you multiply that number by the reciprocal of the dilutions used to make that plate and convert &micro;l to ml to obtain the titer in pfu/ml.</p></li><li>See the <a class = 'chap-linking' tabindex = '0'    data-link-to = '6-7' >Full Plate Titer (6.5)</a> protocol for how to use your spot titer to decide which dilutions to use to determine a more accurate titer.</li></ol><hr><br><h3>Helpful Tips:</h3><ul><li>When exponential numbers are multiplied, the exponents are added together. For example, 10<sup>5</sup> x 10<sup>4</sup>  =  10<sup>9</sup>.</li><li>Bacterial lawns can be spread in advance, refrigerated overnight, and “spotted” the next day.</li><li>When making your spots avoid touching the agar with the pipette tip. Hold the tip slightly above the agar and expel the drop slowly to avoid splattering.</li><li>Remember that labels on the bottom of a plate are mirror images (i.e., they will appear backward) of your labeling scheme when the plate is turned over. Be sure to spot your samples in the right place!</li><li>A high-titer lysate has a titer of at least 5 x10<sup>9</sup> pfu/ml. (If you have at least three plaques in your 10<sup>-7</sup> spot, you have a high enough titer to isolate DNA and archive your lysate.)</li></ul><strong>Sample Problem:</strong><p>Calculation of titer from a spot test.</p><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-6-6-2.jpg' alt = 'A picture of a opaque plate marked with a number notation in a grid ranging from negative one through negative 10. Large transparent spots are located by the numbers negative one through negative 5. The spot marked negative six is noticably less transparent  and negative seven and negative eight show only partial spots.'/></image></div></div><p>The above spot plate contains spots from 10 &micro;l of 10<sup>–1</sup> through 10<sup>–10</sup> dilutions. The 10<sup>–1</sup> through 10<sup>–5</sup> dilutions form cleared spots, and no plaques are evident on the 10<sup>–9</sup> or the 10<sup>–10</sup> spots. However, individual plaques are visible on the 10<sup>–6</sup>, 10<sup>–7</sup>, and 10<sup>–8</sup> dilution spots. The 10<sup>–7</sup> dilution spot above shows 28 plaques and the 10<sup>–8</sup> dilution shows 3. That both spots yield approximately the same titer as the difference between 2.8 x 10<sup>10</sup> and 3 x 10<sup>10</sup> is a reflection of the inherent error between counting 28 vs 3 plaques.</p><p>28 pfu/10 &micro;l x 1000 &micro;l/1 ml x 10<sup>7</sup>  =  2.8 x 10<sup>10</sup> pfu/ml</p><p>or</p><p>3 pfu/10 &micro;l x 1000 &micro;l/1 ml x 10<sup>8</sup>  =  3.0 x 10<sup>10</sup> pfu/ml</p>"},"6-7":{content:"<h2>Objective: </h2> <p>To determine the concentration of phage particles in a lysate by using a plaque assay</p> <hr><br><h2>Rationale:</h2> <p>The total number of phage particles in your lysate can be calculated by performing serial dilutions, followed by a plaque assay and then counting the number of plaques formed. By taking into account the volume and dilution used to generate these plaques, you can calculate the concentration, or titer, of your phage lysate. Because each plaque is formed from a single phage particle, that original phage particle is called a plaque-forming unit (pfu). The concentration of a lysate is presented as the number of plaque-forming units in every milliliter of lysate, or pfu/ml. The full plate titer is the most accurate method of determining the number of infectious phage in a liquid sample.</p> <hr><br><h2>Supplies:</h2> <ul> <li>Liquid phage sample requiring titering</li> <li>Phage buffer</li>  <li>Agar plates for plating dilutions</li>  <li>Top agar, molten (between 55 - 60 ˚C)</li>  <li>Host bacteria</li>  <li>Microcentrifuge tubes</li> <li>5 ml serological pipettes</li> </ul> <hr><br><h2>Procedure:</h2><ol style = 'list-style-type:upper-alpha;'><li>Prepare your bench for aseptic work and assemble your supplies.<br>  </li> <li>Perform the <a class = 'chap-linking' tabindex = '0'    data-link-to = '6-4' >Serial Dilutions (6.2)</a> protocol on the sample you wish to titer and set up a <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-5' >Plaque Assay (5.3)</a> for the dilutions. <ol  > <li>If you are titering a highly concentrated phage sample, like a plate lysate, dilute and plate the   10<sup>-1</sup>, 10<sup>-2</sup>, 10<sup>-3</sup>, 10<sup>-4</sup>, 10<sup>-5</sup>, 10<sup>-6</sup>, 10<sup>-7</sup>, and 10<sup>-8</sup> dilutions according to the protocols for <a class = 'chap-linking' tabindex = '0'    data-link-to = '6-4' >Serial Dilutions (6.2)</a> and the <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-5' >Plaque Assay(5.3)</a>.<br> OR</li> <li>If you have already performed a <a class = 'chap-linking' tabindex = '0'    data-link-to = '6-6' >Spot Titer (6.4)</a> on the sample and determined its estimated titer, plate only the dilutions predicted to yield a countable number of plaques (typically 20 –200) on a plate from a <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-5' >Plaque Assay (5.3)</a>. <ol style = 'list-style-type:lower-alpha;'> <li>For example, if the 10<sup>-7</sup> dilution from your spot titer resulted in 12 plaques, you should “bracket” this dilution by plating one dilution on either side to ensure you obtain a countable number of plaques on at least one plate. This means you should plate the 10<sup>-6</sup>, 10<sup>-7</sup>, and 10<sup>-8</sup> dilutions. If the spot titer is ambiguous, plate additional dilutions; for example, the 10<sup>-5</sup> 10<sup>-6</sup>, 10<sup>-7</sup>, 10<sup>-8</sup>, and 10<sup>-9 </sup>dilutions.<br>  </li> </ol> </li> </ol> </li> <li>The next day check the plates and confirm that your dilutions are valid (i.e., a 10-fold decrease in plaques). <ol> <li>Use a plate with 20–200 isolated plaques and count the number of plaques.</li> </ol> </li>            <li>Calculate the titer in pfu/ml using the formula:<br> Titer (pfu/ml)  =  (# pfu/ volume used in &micro;l) x (10<sup>3</sup> &micro;l/ml) x dilution factor*<br> <em>*For a 10<sup>-7</sup> dilution, the dilution factor is 10<sup>7</sup>.</em>   <p ><strong>Example</strong>: If 24 plaques were observed on a plate from a culture infected with 10 &micro;l of a 10<sup>-5</sup> dilution (Figure 6.5-1), the calculations look like this:<br> Titer (pfu/ml)  =  (24 pfu/10 &micro;l) x (10<sup>3</sup> &micro;l/ ml) x (10<sup>5</sup>)</p> <p >    =  (2.4 x 10<sup>3</sup> x 10<sup>5</sup>) pfu/ml<br>     =  2.4 x 10<sup>8 </sup>pfu/ml</p> <p ><em>Essentially, you are dividing the number of plaques by the volume of sample plated. That number is multiplied by the reciprocal of the dilutions used to make that plate and then converted from </em><em>&micro;l to ml to obtain the titer in pfu/ml.  </p></li> </ol>  </li> </ol> <p > </p>                               <hr><br><h3>Helpful Tips:</h3>  <ul> <li>When exponential numbers are multiplied, the exponents are added together. For example, 10<sup>5</sup> x 10<sup>4</sup>  =  10<sup>9</sup>.</li>  <li>A high-titer lysate has a titer of 5 x10<sup>9</sup> pfu/ml or more. We recommend that you try to attain at least this  titer for future experiments and long-term storage.</li> <li>If there are 24 plaques on the 10<sup>-7</sup> plate, you should expect 240 plaques on the 10<sup>-6</sup> plate. However, if when counting the plaques you count 200 plaques, the titer equals 2 x 10<sup>10 </sup>pfu/ml. Realize that there is no practical difference between the two titers, but counting more plaques is more accurate.<div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-6-7-1.jpg' alt = 'Image is a picture of 6 opaque petri dishes marked 10 to the negative one through 10 to the negative 6. The first two plates are fully opaque. The third has numerous lightly transparent small circle. The forth shows numerous transparent circles. The fifth shows fewer transparent circles and the sixth shows one transparent circle. '/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 6.5-1. Plaque assay of dilution series. The phage StarStuff was serially diluted and plated for a plaque assay. The most concentrated samples (10<sup>-1</sup> and 10<sup>-2</sup>) create cleared plates where no bacteria are visible. The 10<sup>-3</sup> plate shows a very high concentration of plaques, but individual plaques are not visible until the 10<sup>-4</sup> plate, which is a webbed plate. The 10<sup>-5</sup> and 10<sup>-6</sup> plates have countable numbers of plaques, but the 10<sup>-5</sup> plate offers the most accurate number for counting</div></div></li></ul>"},"7-1":{content:"<p>You completed the process of phage isolation when you made and titered a lysate. If you collected your initial lysate from a single plate with a high density of plaques during your last round of plaque purification, you probably have only five milliliters of lysate, which is not a sufficient volume for DNA extraction, transmission electron microscopy (TEM), and archiving. Therefore, you need to make more lysate to increase your phage stock. Indeed, your will need to collect 8–10 ml of lysate with a titer of 5 x 109 pfu/ml or higher. This can be accomplished by making additional plates that have a very high density of plaques, known as webbed plates (Figure 7.0-1), and flooding these plates to harvest a lysate. A lysate prepared from webbed plates is often of a high titer and is called a high-titer lysate (HTL). HTLs are the starting point for many basic phage biology experiments and long-term storage.</p><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-6-5-1.png' alt = 'A picture of an opaque plate with numerous lightly transparent circles.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 7.0-1 Webbed plate with confluent plaques.</div></div> <p>The trick to making webbed plates that will yield a lysate with a titer of 5 x 109 or above is getting the right number of phage on the plate to start with. If too few phages are added to the top agar before incubation, the plate will be sparsely covered with plaques and yield too few phage when flooded and the lysate harvested. Likewise, if too many phages are added to the top agar before incubation, they will kill all of the bacteria in an early round of infection and the resulting phage yield will be low. Therefore, you have to start with the number of phages that will allow for maximum rounds of infection and lysis before running out of available host bacteria. This can be accomplished by determining the number of plaques formed by your phage that will result in a webbed plate, as described below.</p>  <p>Plaque size varies from phage to phage (Figure 7.0-2), and so will the number of plaques needed to form a webbed plate. The smaller the plaque size, the more plaques you will need to make a webbed plate. The larger the plaque size, the fewer plaques you will need to make a webbed plate.</p>  <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-7-1-4.jpg' alt = 'The image contains three pictures of opaque plates, labeled A, B and C. Picture A contains very small transparent circles, picture B medium sized transparent circles, and picture C contains large transparent circles.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 7.0-2. Examples of small (A), medium (C), and large (C) plaques.</div></div> <p>You can calculate the number of plaques needed to make a webbed plate based on data you already have. One way to do this is to refer back to plates from your multiples rounds of plaque assays for purification. It is very likley that one of your dilutions yielded a webbed plate. You can easily calculate the number of plaques that were on that webbed plate because you also performed serial dilutions and obtained plates in that same experiment that yielded a plate with a countable number of plaques. (An example of how to perform this calculation is provided later in <a class = 'chap-linking' tabindex = '0'    data-link-to = '7-2' >protocol 7.1, Making Webbed Plates from Lysates of Known Titer</a>.) If your previous plaque assays never generated a plate that resembles a webbed plate, you could still use information from plates that yielded a high density of plaques to estimate the number of plaques needed to generate a webbed plate. For example, doubling the number of plaques on a plate may make a webbed plate. Alternatively, you could calculate the number of plaques needed to cover a plate based on plaque diameter and plate surface area.</p><p>Once you know how many plaques are needed for a webbed plate, you know how many phage particles to use in a plaque assay to generate a webbed plate. Remember, each plaque on a plate originates from a single plaque. Using the titer of your lysate, calculate the volume of your lysate that will contain the desired amount of phage particles.</p><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-7-1-1.jpg' alt = 'A flowchart for amplification: making webbed plates leads to collecting plate lysates leads to spot titer leads to full plate titer and then the question: is your titer high enough? If no, then return to making webbed plates. If yes, then proceed to extraction, microscopy and archiving phage samples.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 7.0-3. Flow chart of experiments in phage amplification.</div></div>"},"7-2":{content:"<h2>Objective:</h2><p>To create a plate with a very high density of plaques from a lysate of known titer</p><hr><br><h2>Rationale:</h2><p>Even if you have a small volume of titered lysate, it can be used to make additional lysate for experiments like DNA extraction and transmission electron microscopy (TEM) and for long-term storage. Lysates of high titer can be collected by flooding plates with as many plaques as possible. These plates are called “webbed” plates because they have densely packed confluent plaques with only wisps, or a “web” of bacteria, left between them.</p><p>There are many approaches to determining how many phages (i.e., how much phage lysate) need to be added to a plate to obtain a web. The method presented below relies on information from the plaque assay you used to create your initial lysate. With this method you can calculate the volume of  lysate needed to generate a webbed plate. Typically, a single webbed plate allows you to harvest ~ 4 ml of a highly concentrated phage lysate. It is recommended that you harvest 8–10 ml of lysate; therefore, you will need two or three webbed plates.</p><hr><br><h2>Supplies:</h2><ul><li>Lysate of known titer</li><li>Agar plates</li><li>Host bacterial cultures</li><li>Phage buffer</li><li>Top agar, molten (between 55 - 60 ˚C)</li><li>Microcentrifuge tubes</li><li>5 ml serological pipettes</li></ul><hr><br><h2>Procedure:</h2><ol style = 'list-style-type:upper-alpha;'> <li>Prepare your bench for aseptic work and assemble your supplies.</li> <li>Obtain the titered lysate you prepared from your last round of phage purification using the<a class = 'chap-linking' tabindex = '0'    data-link-to = '6-5' > Collecting Plate Lysates (6.3) </a>protocol and consult the protocols in chapter 6 for calculating titer. </li> <li>Estimate the number of plaques you need and calculate the volume of lysate necessary to generate a webbed plate. <ol> <li>Using data from a previous plaque assay, for example from your most recent full-plate titer of your lysate, estimate how many plaques would result in a webbed plate.</li> <li>Calculate how many phage were on the webbed plate used to make the lysate. (See Helpful Tips for sample calculations.)<br> OR</li> <li>Choose a plate with a high density of plaques and count how many plaques are in a quadrant. Use a sharpie to mark each plaque on the bottom of the plate as you count it so you don’t count it twice.</li> <li>Estimate how many more plaques you need to make a webbed plate.</li> <li>Calculate the volume of lysate needed to generate a webbed plate.<br> <br> <span ><strong>Example</strong></span>: Vince has medium-sized plaques and estimates that he needs approximately 3,000 plaques to create a webbed plate. His lysate has a titer of  5 x 10<sup>9</sup> pfu/ml. To find the volume of lysate needed to create a webbed plate, divide the number of plaques  needed by the titer of the lysate and convert to microliters.<br> <br> Volume needed (ml)<sub>lysate</sub><br>  =  3.0 x 10<sup>3</sup> pfu/ 5 x 10<sup>9</sup> pfu/ml<br>  =  6.0 x 10<sup>-7 </sup>ml lysate<br>  =  (6.0 x 10<sup>-7 </sup>ml) x (1000 &micro;l/ml)<br>  =  6.0 x 10<sup>-4</sup> &micro;l<br> <em>Although you cannot pipette 6.0 x 10<sup>-4</sup> ul of lysate, you can pipette 6.0 &micro;l of a 10<sup>-4 </sup>dilution, or 60 &micro;l of a 10<sup>-5 </sup>dilution.</em></li> </ol> </li> <li>Perform the dilutions according to the protocols for <a class = 'chap-linking' tabindex = '0'    data-link-to = '6-4' >Serial Dilutions(6.2)</a>. <ol> <li>Plaque size often decreases the closer the plaques are on a plate. To ensure that you obtain proper webbed plates plates, it is a good idea to “bracket” the number of phage particles you put on the plate. This means you should also prepare plates with fewer phage and with more phage than the optimum number estimated. For larger plaques you may want to double and halve the number of phage. For smaller plaques you may want to use 10 times more and 10 times less. This is phage-dependent.<br> <br> <span><strong>Example</strong></span>: After Vince looked at his plates, he decided that for his medium-sized plaques his brackets should be 5 times fewer and 5 times greater than his estimate of 3,000 (i.e., 600 and 15,000) plaques. Therefore, in addition to plating 60 &micro;l of a 10<sup>-5 </sup>dilution, he will also plate 12 &micro;l of a 10<sup>-5</sup> dilution and 30 &micro;l of a 10<sup>-4</sup> dilution.<br>  </li> </ol> </li> <li>Plate your dilutions to create webbed plates. <ol> <li>Retrieve the proper number of agar plates and host bacterial cultures needed to plate your dilutions, including your bracket dilutions and replicates.</li> <li>Infect the bacterial host cells with the appropriate volume and dilutions of your lysate and and incubate the mixture for 5–10 minutes for phage attachment. Then add 3 ml of molten top agar to the mixture and plate. (If you need additional direction, refer to the <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-5' >Plaque Assay(5.3)</a> protocol.)</li> <li>Remember that each webbed plate will yield ~ 3–5 ml of lysate and that you will need a minimum of 8–10 ml for subsequent experiments and long-term storage. Therefore, you may need to plate each dilution two or three times to make additional webbed plates. Check with your instructor.<br> </li> </ol> </li> <li>Incubate (without inverting) for at least 24 hours at the proper temperature. <ol> <li>Choose the best webbed plate and collect a lysate according to the <a class = 'chap-linking' tabindex = '0'    data-link-to = '6-5' >Collecting Plate Lysates (6.3)</a> protocol.</li> </ol> </li> </ol></ul><hr><br><h3>Helpful Tips:</h3><ul><li>Making webbed plates that will yield lysates with titers above 5 x 10<sup>9</sup> is highly phage-dependent. Therefore, you need to understand <em>your</em> phage, including its plaque morphology and behavior to be successful.<br> <br> <span><strong>Example</strong></span>: Priscilla collected her initial lysate for phage “Axelito” from the 10<sup>-2</sup> dilution plate generated during the last round of purification of phage “Axelito.” Her notebook says that the 10<sup>-4</sup> dilution plate had 40 plaques on it and was made by infecting the host culture with 10 &micro;l of the 10<sup>-4 </sup>dilution of a picked plaque.<br> <br> She can directly determine the number of plaques on the webbed plate from the number of plaques on the 10<sup>-4</sup> plate because both plates were generated as part of a serial dilution series. Since 10<sup>-2</sup> is 100-fold more concentrated than 10<sup>-4</sup> she can multiply the number of plaques (40) on the 10<sup>-4</sup> plate by 100 to for a value of 4.0 x 10<sup>3</sup> plaques.<br> <br> When she made her lysate she noted that the 10<sup>-2</sup> plate was not quite a perfect webbed plate. From the picture of the plate in her notebook she estimates that a perfect webbed plate would have twice as many plaques on it, or about  8.0 x 10<sup>3 </sup>pfu.<br> <br> Next, Priscilla calculates how much of Axelito’s initial lysate she will need to create new webbed plates, so that she can collect enough lysate for DNA isolation and TEM. She calculated Axelito’s <span >lysate titer to be 5.0 x 10<sup>10 </sup>pfu/ml. If she needs 8.0 x 10<sup>3 </sup>pfu per plate:</span><br> <br> (8.0 x 10<sup>3</sup> pfu/ml) / ( 5.0 x10<sup>10</sup> pfu/ml)  =  1.6 x 10<sup>-7</sup> ml<br> <br> Priscilla converts 1.6 x 10<sup>-7</sup> ml to microliters by multiplying by 10<sup>3</sup> &micro;l/ml, so she knows that she needs 1.6 x 10<sup>-4</sup> &micro;l of Axelito’s lysate per plate to create a good webbed plate. She can’t pipette that small a volume, but she can pipette 1.6 ul of a 10<sup>-4</sup> dilution, or 16 ul of a 10<sup>-5</sup> dilution.<br> <br> Because Pricilla needs 8 ml of lysate, she decides to make two webbed plates by infecting two host cell aliquots with 16 &micro;l of a 10<sup>-5</sup> dilution. To make sure she has a good webbed plate tomorrow when she comes to the lab, and because she had tiny plaques, she decides to make plates with 10 times fewer and 10 times more phage on them as well. Therefore, she makes four more plates, two with 16 &micro;l of the 10<sup>-4</sup> dilution, and two with 16 &micro;l of the 10<sup>-6</sup> dilution.</li></ul>"},"7-3":{content:"<h2>Objective:</h2><p>To include your phage in the Actinobacteriophage Database to make it available for other scientists</p><hr><br><h2>Rationale:</h2><p>The Actinobacteriophage Database is a permanent online record of actinobacteriophage isolates and their characteristics. It provides a means of quickly identifying and sorting through the thousands of phages in the SEA-PHAGES archives as a first step toward further and more extensive analyses. Once a new record for a phage isolate is started, information about the phage can be contributed by both the isolator and by the SEA-PHAGES team. The more information you can provide, the better!</p><p>You can begin creating an entry for a phage as soon as you have a positive spot test, though it is recommended that you wait until you have a high titer lysate for that phage. At the latest, you must add your phage to the database to receive its genome sequence. As the isolator, you are responsible for collecting all the information for the phage you will ''add'' to the database, including your phage’s name and conditions of the collection site—especially GPS coordinates, temperature of the isolation, host strain, naming notes, and whether your phage is from direct or enriched isolation. As you progress through the characterization process, you can add additional information, such as plaque pictures, genomic restriction digest pictures, and an electron micrograph.</p><hr><br><h2>Procedure:</h2><ol style = 'list-style-type:upper-alpha;'> <li>Name your phage according to the rules found in the “Phages” dropdown menu and “Naming Rules” on <a href = 'https://phagesdb.org/' target = '_blank'>phagesdb.org</a>.<br> </li> <li>Log in to <a a href = 'https://phagesdb.org/' target = '_blank'>phagesdb.org</a> or create an active PhagesDB account by clicking on the “Sign in or Register” button at the bottom of the left column on the page. </li> <li>Enter your phage on PhagesDB. <ol> <li>Once you have logged in to PhagesDB, add a phage by selecting the “Data” dropdown menu and clicking on 'Add Phage.'</li> <li>Fill in as many fields as possible, paying special attention to the following: <ol style = 'list-style-type:lower-alpha;'> <li>GPS coordinates: If your GPS fields were not prepopulated by the SEA-PHAGES App, you can find the GPS coordinates by using a web mapping service application. One such service can be found via this link:<a href = ' http://www.heywhatsthat.com/profiler.html ' target = '_blank'> http://www.heywhatsthat.com/profiler.html </a> OR http://bit.ly/yqKB . Simply drag the map to the location where the environmental sample was found, zoom in as much as possible, and click on the map to obtain the GPS coordinates of that location. The output should be to the right of the map and in the N/W Decimal Degree format.</li> <li>Program</li> <li>Institution</li> <li>Bacterial host</li> <li>Name  <ul style = 'list-style-type:disc' > <li>Convert the GPS coordinates to the correct format if necessary. For example, the coordinates <em>40° 26’ 46” N, 79° 57’ 11” W </em>from the iPhone’s “Compass” app will be converted into <em>40.446111 N, 79.953056 W</em>.<br> <span ><strong>Important</strong></span>: Phages must have GPS coordinates to be submitted to GenBank after sequencing and annotation.</li> </ul></li>  <li>If you found your phage using an enriched culture, click “Yes” to “From Enriched Soil” or “No” if you used a direct isolation.</li> <li >Fill in the Discovery Notes to explain where the sample was collected and the soil conditions, etc.</li> <li>Complete the Naming Notes to explain where your phage name came from and why you chose the name.</li> <li>Enter the details about your phage’s plaque morphology, including size, turbidity, margins, etc.</li> <li >The “Morphotype” of your phage is determined from the TEM data. Details about the different morphotypes can be found in <a class = 'chap-linking' tabindex = '0'    data-link-to = '8' >Chapter 8, Viewing Phage Particles by Transmission Electron Microscopy</a>.</li> <li >The Cluster, Subcluster, Sequencing Facility and Method, and GenBank Accession Number can be filled in only after a phage’s genome has been sequenced. Therefore, you should not fill in these fields.</li> <li >If you know the titer of the sample to be archived at the University of Pittsburgh, enter the titer here (or return later and enter the data).</li> <li >Upload unaltered images of your phage where indicated. If you do not have a particular image at this time, you will need to return and add it at a later date.</li> </ol>         <li>Click “Submit” to submit your request to PhagesDB. Your submission will be reviewed and approved by a scientist at the University of Pittsburgh; therefore, your submission will not be visible immediately. Please submit your phage only ONCE! <ol style = 'list-style-type:lower-alpha;'> <li> Once approved, information is publically available for everyone to see and use.</li> </ol> </li> <li >Once your submission is accepted, you need to create thumbnails of the images that will be displayed with your phage’s profile.     <ol     > <li>From the main PhageDB page select the “Data” dropdown menu and click on “Thumbnails.”</li><li>Type in your phage’s name.</li><li>Read the instructions for “Making Thumbnails for Plaque and EM Pictures for PhagesDB.”</li><li>Upload images that fulfill the requirements in the instructions.</li></ol> <li >If you need to modify or add data to your phage’s PhagesDB profile, you can do so at any time by clicking on the “Data” dropdown menu and selecting “Modify Phage.”<br>  </li>   </ol><li>Once your phage is sequenced, the SEA-PHAGES team will add data, such as the phage cluster, the genome sequence, and the type of ends. Finally, when your phage has been annotated, submitted to GenBank, and published, the SEA-PHAGES team will add the GenBank accession number and publication data. Phages MUST have GPS coordinates for GenBank submission. Only phages that have been, or will be, archived at the University of Pittsburgh should be entered into the database.</li>"},"7-4":{content:"<h2>Objective:</h2><p>To prepare a high-titer lysate for long-term storage</p><hr><br><h2>Rationale:</h2><p>Your phage samples are a valuable resource for current and future in-depth research. There are two objectives to archiving your phage. First, when prepared and stored properly, your phage samples can remain viable for decades. Second, you need to verify the data (including names and titers of your samples) entered at PhagesDB is complete and accurate for easy referencing. You will create three samples for archiving; two will be stored at the University of Pittsburgh, in the lab of Dr. Graham Hatfull, and another at your institution.</p><p>Important: This protocol is provided as a classroom supplement to the complete SEA-PHAGES Lysate Archiving Protocol on phagesdb.org. This protocol is updated periodically so please check that you are using the most up-to-date version of this protocol.</p><hr><br><h2>Supplies:</h2><ul><li>Barcoded tubes provided by the Hatfull lab</li><li>Sample tubes provided by your institution</li><li>DMSO mixture</li><li>Lysate (at least 5 x 10<sup>9</sup>pfu/ml)</li></ul><hr><br><h2>Procedure:</h2><ol style = 'list-style-type:upper-alpha;'> <li>Prepare your bench for aseptic work and assemble your supplies.<br>  </li> <li>Certify phage samples on Phagesdb.org. <ol> <li>Make certain that the details of your phage samples have been recorded on the phages database at phagesdb.org according to the protocol <a class = 'chap-linking' tabindex = '0'    data-link-to = '7-3' >Entering Phages into the Actinobacteriophage Database (7.2)</a>.</li> <li>Double-check the spelling of the phage name and the recorded titer of the lysate. Alert your instructor if you find any discrepancies.<br>  </li> </ol> </li> <li>Prepare archiving tubes. <ol> <li>Retrieve two barcoded tubes for storage at the University of Pittsburgh and a third tube for storage at your institution. This third tube does not need to be barcoded.</li> <li>Properly label each tube with your phage’s name according to Figure 7.3-1C, making sure it is legible.</li> </ol> </li> <li>Prepare DMSO/lysate mixture.<br> <span ><strong>   Important</strong></span>: Turn off your flame, as DMSO and its vapors are flammable! <ol> <li>Dispense 4.0 ml of your high-titer lysate into a sterile 15 ml conical tube.</li> <li>Add 280 &micro;l DMSO to the tube, cap the tube, and then vortex the mixture.</li> </ol> </li><p><strong>Important:</strong> Dimethyl sulfoxide (DMSO) is a cryoprotectant that prevents ice crystals from forming. (Ice crystals can cause damage to phage particles during long-term storage at -80 °C.)</p> <li>Aliquot the mixture into the storage tubes<ol><li>Pipette the lysate/DMSO mixture into each of the labeled bar-coded tubes so that the cap threads are not covered, approximately 1.4 ml.  the remainder can go into the archive tube (not bar-coded) stored at your institution.</li><li>Avoid overfilling and underfilling the tubes. Examples are shown in Figure 7.3-1G–J. (Ignore that the tubes have beads.)</li><li>Take care to close the tubes properly.</li></ol><li>Prepare the tubes for storage<ol><li>Place the barcoded tubes in the storage box provided in the same order as shown on the archive list. (Your instructor will request a complete archive list once all information is entered in <a href = 'https://phagesdb.org/' target = '_blank'>PhagesDB</a>.)</li><li>Store samples for shipment to University of Pittsburgh at 4 °C.</li><li>Samples stored at your institution can be frozen immediately.</li></ol></li></li></ol><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-7-4-1.jpg' alt = 'A picture of 11 tubes labeled A through K illustrating proper procedures and poor examples. Tubes C through F are proper procedure and tubes G through J are poor examples.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 7.3-1. Examples of proper and improper filling of archiving tubes. The pink shows the liquid level in the tubes. (Note:  Beads are no longer being used.)</div></div><hr><br><h3>Helpful Tips:</h3><ul><li>A video of this protocol can be viewed at:<a href = ' https://www.youtube.com/watch?v = 5GjAjky81D4 ' target = '_blank'> https://www.youtube.com/watch?v = 5GjAjky81D4 </a>.  (Again, ignore the beads.)</li></ul>"},"7-5":{content:"<h2>Making Webbed Plates</h2> <ul><li>When making webbed plates, you will want to standardize as many variables between preparing the initial lysate and the webbed plates. Using the same batch of cells (culture) and plates (so that they have the same water concentration), as well as adsorption and incubation time will ensure consistent results.</li>  <li>The moisture content of the plates has a huge impact on the quality of the resulting webbed plates. It is best to use fresh plates that have been cured (left out on the bench, lids on and facing down, to dry) for approximately 24 hours.</li>  <li>Once a webbed plate has been observed after at least 24 hours of incubation, you can allow this plate to incubate for an additional 24 hours to allow for further phage replication. These additional rounds of phage replication would ideally fully clear the remaining bacteria on the plate, which will increase the titer of the resulting lysate.</li>  <li>Once the desired amount of lysate is created, it must be titered according to the spot titer protocol or by doing a full plate titer.</li>  <li>When pooling lysates take care not to dilute a high titer lysate with a lysate made from a marginal plate with a low titer. It is recommended not to pool lysates from different rounds of infection.</li></ul></p>"},"8-1":{content:"<p>Phages, like most microorganisms, are too small to be seen with the naked eye. Traditional compound light microscopes, like those found in most biology labs, use a series of lenses to magnify a specimen and a light source to illuminate it. A simple light microscope can magnify an object 40x to 1,000x. Even the most sophisticated light microscopes cannot magnify an object beyond ~ 1,500x due to the properties of light and its ability to resolve, or distinguish between, two objects that are less than 0.2 &micro;m apart. Although light microscopy allows us to see individual microorganisms such as bacteria, it is not sufficient for viewing phages. To overcome this limitation, scientists employ microscopes that use a beam of electrons instead of light, which allows for magnification up to 10,000,000x.   </p><p>   Transmission electron microscopy (TEM) is based on the same principles as light microscopy, but it uses a beam of electrons rather than light to “illuminate” the specimen. Electron microscopes are usually very large (Figure 8.0-1) and use a hot wire, called a filament, to produce a stream of electrons that are focused into a beam by magnets. Much the same way as light-microscopes rely on glass slides that are transparent to light, you must use slides that are transparent to electrons. Electron microscope slides, which are called grids, are typically made of thin copper or nickel mesh and can be coated with carbon and heat-resistant plastics such as formvar, depending on the sample to be examined. Biological samples and the carbon/plastic coating are transparent to electrons, yet they are extremely fragile, so the metal mesh provides stability and support. For liquid phage samples, such as lysates, 400-mesh copper grids coated with carbon and formvar are appropriate (“400-mesh” means that there are 400 holes in the copper foil of the grid).</p><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-8-1-1.jpg' alt = 'A picture of a desk with computer monitors and a transmission electron miscroscopy on top.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 8.0-1. Transmission electron microscopy (TEM).</div></div><p>An ideal TEM sample has a concentration of 1010 pfu/ml and should be relatively free of cellular membranes and debris. You can concentrate your sample and remove debris by pelleting your phage in a microcentrifuge and resuspending the pellet in fresh phage buffer. When pelleting a phage sample the densest biomolecule is DNA, followed by protein, with lipid membranes being the lightest (even lighter than water). Because phage particles are composed of both DNA and protein, they sediment to the bottom of a centrifuge tube faster than either lipid membranes or host proteins. By pelleting only the phage particles in your lysate and then resuspending the pellet in clean phage buffer, you will have a clean, concentrated sample for electron microscopy.     </p><p>  When a phage sample is applied to a grid, the phage particles adhere to the carbon coating by means of electrostatic interactions. The longer the lysate sits on the grid, the more phage particles stick to the coating. Using distilled water, you can rinse away extraneous salts and cell debris from the grid, and then stain the grid with a heavy metal salt, such as uranyl acetate (UA). The best images are obtained when the stain is carefully dried into a thin film across the entire grid, pooling around the phage and barely coating their surfaces. When the grid is placed in the electron microscope, electrons within the beam penetrate biological components, such as phage and the carbon/plastic grid coating, but are scattered by the heavy metal stain surrounding the phage and  by the copper support mesh. The electrons that pass through the sample strike a phosphorescent screen and create an image called a micrograph. A good micrograph should provide a clear outline and some appreciable detail of the phage.  </p><p>     A lot can be learned about a phage based on its morphology. For example, you can tell if it is a siphoviridae with a long flexible tail (Figure 8.0-2A), a myoviridae with a contractile tail (Figure 8.0-2B), or a podoviridae (Figure 8.0-2C), with a stumpy noncontractile tail. Two phages with similar plaque morphologies may be distinguishable by calculating their respective head and tail sizes from a micrograph. The morphology may also provide information about a phage’s genome. For instance, phages that have large capsids typically contain more DNA (that is, a longer genome) than phage with smaller capsids. Phages with long tails have a long tape-measure gene, which codes for a protein that determines the length of its tail.<div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-8-1-2.jpg' alt = 'Illustrations of three types of phages and the electron micrograph pictures.  Mycobateriophage scarlette is labeled A and is a multifaceted geometric shape with a long tail with a cone at the end. Mycobateriophage gabriel is a multifaceted geometric shape with a shorter tail and six prongs at the end. Cyanobacteriophage Syn5 is a multifaceted geometric shape with almost no tail and spikes at the end.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 8.0-2. Phage morphology. Cartoons and transmission electron micrographs show three different phage morphologies. (A) Mycobacteriophage Scarlett is a siphoviridae with a long flexible tail. (B) Mycobacteriophage Gabriel is a myoviridae with a short contractile tail. (C) Cyanobacteriophage Syn5 is a podoviridae with a stumpy noncontractile tail. Size bars  =  50 nm.</div></div> "},"8-2":{content:"<h2> Objective: </h2><p>To prepare your phage sample for viewing with a transmission electron microscope </p><hr><br><h2>Rationale:</h2><p> Individual phage particles are too small to be seen with light microscopes, necessitating the use of transmission electron microscopy (TEM). To observe your phage using TEM, a sample of your lysate should first be concentrated and cleared of protein and bacterial membrane debris. This is accomplished by pelleting your phage in a microcentrifuge and resuspending the phage pellet in clean phage buffer.</br> The concentrated sample is then placed on an electron microscopy grid. Once on the grid, salts and remaining bacterial debris are rinsed away and the sample is stained with uranyl acetate. When the stained grid is placed in the electron microscope, the beam of electrons penetrates the phage particles, but not the stain surrounding your phage, making the phage visible on the dark background.</br> <strong>Important:</strong> You will use uranyl acetate (UA) as your stain. UA is a hazardous substance requiring special safely precautions. Please follow your instructor’s directions for handling and disposing of UA-exposed materials. </p><hr><br><h2>Supplies:</h2><ul> <li>High-titer lysate</li><li>Phage buffer</li><li>EM reverse forceps</li><li>200–400 mesh carbon–formvar-coated copper grids</li><li>Wedges of Whatman filter paper for wicking</li><li>Sterile, <i> filtered</i> water</li><li> 1 % uranyl acetate (filtered immediately before use) </li></ul><hr><br><h2>Procedure:</h2><ol style = 'list-style-type:upper-alpha;'> <li>Prepare your phage samples. <ol > <li> Aseptically transfer 100 &micro;l of your high-titer lysate into a sterile microcentrifuge tube. The ideal concentration of your resuspended sample is 10 <sub>10</sub> pfu/ml, so you should titer your lysate and then choose an appropriate volume, centrifugation time, and resuspension volume according to Table 8.1a-1. </li><table> <caption> <p><strong><span style = 'font-size:14px'>Table 8.1a-1. Centrifuge times to pellet phage samples in a microcentrifuge.</span></strong></p></caption> <tbody> <tr> <th scope = 'col'> <p>Initial concentration (pfu/ml)</p></th> <th scope = 'col'> <p>Initial volume</p></th> <th scope = 'col'> <p>Time</p></th> <th scope = 'col'> <p>Final resuspension volume</p></th> </tr><tr> <td> <p>10<sup>10</sup></p></td><td> <p>100 &micro;l</p></td><td> <p>22 min</p></td><td> <p>100 &micro;l</p></td></tr><tr> <td> <p>5x10<sup>9</sup></p></td><td> <p>100 &micro;l</p></td><td> <p>22 min</p></td><td> <p>50 &micro;l</p></td></tr><tr> <td> <p>1x10<sup>9</sup></p></td><td> <p>500 &micro;l</p></td><td> <p>30 min</p></td><td> <p>50 &micro;l</p></td></tr><tr> <td> <p>5x10<sup>8</sup></p></td><td> <p>500 &micro;l</p></td><td> <p>30 min</p></td><td> <p>10 &micro;l</p></td></tr><tr> <td> <p>1x10<sup>8</sup></p></td><td> <p>1000 &micro;l</p></td><td> <p>40 min</p></td><td> <p>10 &micro;l</p></td></tr></tbody> </table> <li> Balance the tube(s) and centrifuge according to Table 8.1a-1 at 4 °C at top speed in a microcentrifuge to pellet the phage particles to the bottom of the tube. </li><li> Using a micropipettor, carefully remove as much of the supernatant as possible without disrupting the pellet. You should remove the majority of the supernatant, leaving as little liquid as possible without dislodging the pellet. (If you can’t see the pellet, don’t worry, it is still there! You just have a really clean sample.) </li><li> Resuspend the pellet in the proper volume of phage buffer (Table 8.1a-1) and mix gently using the pipette tip. Note that vigorous pipetting can damage the phage particles. </li><li> Proceed with the rest of the protocol immediately to avoid damaging the phage heads. </li></ol></li> <li>Prepare your work area. (This may have been done before class by your instructor.)<ol> <li>Put on a fresh pair of gloves.</li><li>Cover the designated work area with bench paper or a large Kimwipe to create a clean work surface.</li><li>Using EM forceps, remove a fresh grid from a box of unused grids, touching only the very edge of the grid. If you are not using reverse forceps, slide the silicon clamp down the forceps so that they stay closed around the grid when you let go of the forceps.</li><li>Place the forceps on the paper so that the grid’s dark-and-shiny side is facing UP.</li></ol> </li> <li>Mount and stain your phage. <ol> <li>Using a micropipettor, place 5 &micro;l of lysate onto the grid without touching the tip to the grid itself.</li><li>Allow the lysate to sit on the grid for at least 2 to 10 minutes, but do not let the sample evaporate completely! During this time the phage will settle and adsorb onto the grid.</li><li>Rinse the grid once using the following method: <ol> <li>Using the forceps, turn the grid so it is at a 45° angle.</li><li>Carefully pipette 60 &micro;l of sterile water across the dark-and-shiny face of the grid, allowing it to drip off the other side into a Petri dish. It will take about six big drops.</li><li>Rotate the grid back so that the dark-and-shiny side is facing up again. If necessary, wick away any excess water by placing a fresh wedge of filter paper against one edge of the grid. </br> <strong> Important:</strong> Work quickly and carefully! Do NOT allow the grid to dry out or the phage capsids can collapse.</li></ol> </li><li>Add 5 &micro;l of 1 % uranyl acetate to the grid.<br> <strong> Important: </strong>Uranyl acetate is a very toxic compound. You should wear gloves throughout this procedure and when working in any area where this material has been used. </li><li>Immediately begin to wick off excess stain by using a wedge of filter paper. UA staining occurs by leaving a very thin layer of stain dried across the entire grid. You should continue to wick away the stain until the surface of the grid looks like a rainbow oil slick. Then allow the grid to air dry before putting it safely back into the grid box.<br> </li></ol> </li><li>Observe your phage. <ol> <li>Place your grid in the designated grid box for storage. Be sure to accurately record the location of your grid in the box.</li><li>Transport your samples to your EM facility for imaging.<br> </li></ol> </li><li>Calculate the capsid diameter and tail length relative to the size bar. <ol> <li>Using a ruler, measure the widest point (edge-to-edge, not vertex-to-vertex) of the capsid and the length of the tail (excluding the capsid and any visible tail fibers). If possible, measure multiple phage heads and tails and average their respective values.</li><li>Measure the length of the size bar with the ruler.</li><li>Using the known and relative lengths of the size bar, calculate the length of the capsid and tail. <p> <strong>Example</strong>: <p>The phage in the electron micrograph below has an average capsid diameter of 1.6 cm (16 mm) and a tail length of 4.8 cm (48 mm). The 100 nm size bar is measured to be 2.4 cm long.</p><div class = 'img-wrap'> <div class = 'img-container'> <image src = './assets/images/figure-8-2-1.png' alt = 'Electron micrograph picture of a phage with a scale marked at the bottom of 100 nanometers.'/> </image> </div></div><p>To find the scaled (actual) size of the tail set up a simple ratio and solve for the unknown scaled size:<br>(scaled size bar) / (measured size bar) = (unknown scaled sized) / (measured size)<br>(100 nm) / (24 mm) = (unknown scaled tail size) / 48 mm<br>unknown scaled tail size = (100 nm)(48 nm) / (24 mm)<br>unknown scaled tail size = 200 nm</p></li><li>Compare your capsid and tail lengths with those of your classmates.</li><li>Record your findings for entry in Phagesdb according to the protocol <a class = 'chap-linking' tabindex = '0'    data-link-to = '7-3'> <strong>Entering Phage into the Actinobacteriophage Database </strong> (7.2)</a>.</li> </ol></ul><hr><br><h3>Helpful Tips:</h3><ul style = 'list-style-type:disc'> <li>The EM forceps are very delicate. Never set them down without first replacing the protective cover.</li><li>It is extremely important that the grids do not dry out, especially between the last water wash and the addition of the stain. The capsid proteins destabilize in water (because there are no salts available) and the capsid will collapse or burst open if the grid dries out before stain is added. The stain acts as a fixative and will help stabilize the capsids when they dry.</li><li>Positively stained capsids (the phage heads look black because the stain has permeated the capsid) should not be used when describing the size or the morphology of the phage. Positive staining and capsid collapse can be caused by allowing the grid to dry out.</li><li>If a white precipitate is visible on the grids after drying, a new grid needs to be made. The precipitates are leftover salts from the lysate that were not wicked off completely and can damage the TEM.</li><li>Best results are obtained when FRESH lysates are used for the EM staining experiments.</li></ul>"},"8-3":{content:"<h2> Objective:</h2><p> To prepare your phage sample for viewing with a transmission electron microscope </p><hr><br><h2>Rationale:</h2><p> Individual phage particles are too small to be seen with light microscopes, necessitating the use of transmission electron microscopy (TEM). To observe your phages using TEM, a sample of your lysate should first be concentrated and cleared of protein and bacterial membrane debris. This is accomplished by pelleting your phage in a microcentrifuge and resuspending the phage pellet in clean phage buffer. The concentrated sample is then placed on an electron microscopy grid. Once on the grid, salts and remaining bacterial debris are rinsed away and the sample is stained with uranyl acetate. When the stained grid is placed in the electron microscope, the beam of electrons penetrates the phage particles but not the stain surrounds the phage, making it visible on the dark background. </br> <strong> Important: </strong> You will use uranyl acetate (UA) as your stain. UA is a hazardous substance requiring special safely precautions. Please follow your instructor’s directions for handling and disposal of UA-exposed materials. </p><hr><br><h2>Supplies:</h2><ol> <li>High-titer lysate</li><li>EM forceps</li><li>Pelco tabs</li><li>200–400 mesh carbon–formvar-coated copper grids</li><li>Wedges of Watman filter paper for wicking</li><li>Sterile <i>filtered</i> water</li><li>Phage buffer</li><li> 1 % uranyl acetate  </li></ol><hr><br><h2>Procedure:</h2><ol> <li>Prepare phage samples.<ol> <li>Aseptically transfer 1 ml of your high-titer lysate into a sterile microcentrifuge tube.</li><li>Balance the tube(s) and centrifuge for 1 hour at 4 °C at top speed to concentrate the phage particles at the bottom of the tube.</li><li>Using a micropipettor, carefully remove as much supernatant as possible without disrupting the concentrated phage at the bottom of the tube.</li><li>Add 100 &micro;l of phage buffer and let resuspend at 4 °C for 30 minutes to one hour.</li><li>Proceed with the rest of the protocol immediately to avoid damaging the phage heads.</li></ol> </li><li>Prepare your work area. (This may have been done before class by your instructor.) <ol> <li>Put on a fresh pair of gloves.</li><li> Cover the designated work area with bench paper or a Kimwipe to create a clean work surface.</li><li>Remove the cover from a 5 x 5 cm piece of parafilm, and place the parafilm into the lid of Petri dish. </li><li>Place a PELCO Tab or small piece of double-sided tape onto the parafilm in the lid of the Petri dish. Expose the adhesive or the tab.</li><li>Using EM forceps, remove a fresh grid from a box of unused grids, touching only the very edge of the grid.</li><li>Place the grid dark-and-shiny side UP, on the edge of the tab or double-sided tape so that only the very edge of the grid (no more than 0.5 mm) is touching the adhesive.</li></ol> </li><li>Mount and stain your phage. <ol> <li>Using a micropipettor, place 10 &micro;l of lysate onto the grid without touching the tip to the grid itself. </li><li>Allow the phage settle and attach onto the grid for at least 2 minutes, or according to the times in Table 8.1b-1.<br>  <table> <caption> <p><span><strong>Table 8.1b-1. Approximate times that it should take phage to settle onto the EM grid based on titer.</strong></span></p></caption> <tbody> <tr> <th scope = 'col'> <p><strong>Lysate titer</strong></p></th> <th scope = 'col'> <p><strong>Approximate time</strong></p></th> </tr><tr> <td> <p>10<sup>6</sup> -10<sup>7</sup></p></td><td> <p>5-7 minutes</p></td></tr><tr> <td> <p>10<sup>8</sup></p></td><td> <p>5 minutes</p></td></tr><tr> <td> <p>≥ 10<sup>9</sup></p></td><td> <p>2 minutes</p></td></tr></tbody> </table> <span>.</span></li><li>Using a small wedge of filter paper, wick off the excess fluid.</li><li>Rinse the grid two times by the following method: <ol> <li>Carefully pipette 10 &micro;l of sterile water onto the grid. Allow it to sit for 2 minutes.</li><li>Wick off the water using a fresh wedge of filter paper.<br><span><strong>Important</strong></span>: Work quickly and carefully! Do NOT allow the grid to dry out!</li></ol> </li><li>Add 10 &micro;l of 1 % uranyl acetate to the grid.<br><span><strong>Important</strong></span>: Uranyl acetate is a very toxic compound. You should wear gloves throughout this procedure and when working in any area where this material has been used.</li><li>Let it sit for 2 minutes.</li><li>Wick off excess stain by using a wedge of filter paper. UA staining occurs by leaving a very thin layer of stain dried across the entire grid. You should continue to wick away the stain until the surface of the grid looks like a rainbow oil slick. Then allow the grid to air dry before putting it safely back into the grid box.</li></ol> </li><li>Observe your phage. <ol> <li>Place your grid in the designated grid box for storage. Be sure to accurately record the location of your grid in the box.</li><li>Transport your samples to your EM facility for imaging.<br> </li></ol> </li><li>Calculate the capsid diameter and tail length relative to the size bar.<ol> <li>Using a ruler, measure the widest point (edge-to-edge, not vertex-to-vertex) of the capsid and the length of the tail (excluding the capsid and the tail tip). If possible, measure multiple phage heads and tails and average their respective values.</li><li>Measure the length of the size bar with the ruler.</li><li>Using the known and relative lengths of the size bar, calculate the length of the capsid and tail.</li><p>Example: The phage in the electron micrograph below has an average capsid diameter of 1.6 cm (16 mm) and a tail length of 4.8 cm (48 mm). The 100 nm size bar is measured to be 2.4 cm long.</p><div class = 'img-wrap'> <div class = 'img-container'> <image src = './assets/images/figure-8-2-1.png' alt = 'Electron micrograph picture of a phage with a scale marked at the bottom of 100 nanometers.'/> </image> </div><div class = 'title-container'></div></div><p>To find the scaled (actual) size of the tail set up a simple ratio and solve for the unknown scaled size:(scaled size bar) / (measured size bar) = (unknown scaled sized) / (measured size)(100 nm) / (24 mm) = (unknown scaled tail size) / 48 mmunknown scaled tail size = (100 nm)(48 nm) / (24 mm)unknown scaled tail size = 200 nm</p></ol> </li><li>Compare your capsid and tail lengths with those of your classmates.</li><li>Record your findings for entry in Phagesdb according to the protocol <a class = 'chap-linking' tabindex = '0' data-link-to = '7-3'><strong>Entering Phage into the Actinobacteriophage Database </strong> (7.2)</a>.</li></ol><hr><br><h3>Helpful Tips:</h3><ul> <li> The EM forceps are very delicate. Never set them down without first replacing the protective cover. </li><li> It is extremely important that the grids do not dry out, especially between the last water wash and the addition of the stain. The capsid proteins destabilize in water (because there are no salts available) and the capsid will collapse or burst open if the grid dries out before stain is added. The stain acts as a fixative and will help stabilize the capsids when they dry. </li><li> Positively stained capsids (the phage heads look black because the stain has permeated the capsid) should not be used when describing the size of the morphology of the phage. Positive staining and capsid collapse can be caused by allowing the grid to dry out. </li><li> If a white precipitate is visible on the grids after drying a new grid needs to be made. The precipitates are leftover salts from the lysate that were not wicked off completely and can damage the TEM. </li><li> Best results are obtained when FRESH lysates are used for the EM staining experiments. </li></ul>"},"8-4":{content:"<h2> Objective: </h2><p>To prepare your phage sample for viewing with a transmission electron microscope </p><hr><br><h2>Rationale:</h2><p> Individual phage particles are too small to be seen with light microscopes, necessitating the use of transmission electron microscopy (TEM). To observe your phage using TEM, a sample of your lysate should be of high titer (> = 109 pfu/ml). The phage in the sample are placed on an electron microscopy grid. Once on the grid, salts and remaining bacterial debris are rinsed away and the sample is stained with an appropriate stain (e.g. uranyl acetate, ammonium molybdate or lanthanide salts). When the stained grid is placed in the electron microscope, the beam of electrons penetrates the phage particles, but not the stain surrounding your phage, making the phage visible on the dark background. </br> <strong>Important:</strong> You will use lanthanide salts as your stain. Please follow your instructor’s directions for handling and disposing of these materials. </p><hr><br><h2>Supplies:</h2><ul> <li>High-titer lysate</li><li>Phage buffer</li><li>EM reverse forceps</li><li>200–400 mesh carbon–formvar-coated copper grids</li><li>Wedges of Whatman filter paper for wicking</li><li>Sterile, filtered water</li><li> Appropriate stain (e.g. 2 % uranyl acetate, ammonium molybdate or lanthanide salts) Important: Heavy metals should be handled with caution. You should wear gloves throughout this procedure and when working in any area where this material has been used. </li></ul><hr><br><h2>Procedure:</h2><ol style = 'list-style-type:upper-alpha;'> <li>Prepare your phage samples. <ol> <li> Aseptically transfer 100 &micro;l of your high-titer lysate into a sterile microcentrifuge tube. The ideal concentration of your resuspended sample is 1010 pfu/ml, so you should titer your lysate and then choose an appropriate volume, centrifugation time, and resuspension volume according to Table 8.1a-1. </li><table> <caption> <p><strong><span style = 'font-size:14px'>Table 8.1a-1. Centrifuge times to pellet phage samples in a microcentrifuge.</span></strong></p></caption> <tbody> <tr> <th scope = 'col'> <p>Initial concentration (pfu/ml)</p></th> <th scope = 'col'> <p>Initial volume</p></th> <th scope = 'col'> <p>Time</p></th> <th scope = 'col'> <p>Final resuspension volume</p></th> </tr><tr> <td> <p>10<sup>10</sup></p></td><td> <p>100 &micro;l</p></td><td> <p>22 min</p></td><td> <p>100 &micro;l</p></td></tr><tr> <td> <p>5x10<sup>9</sup></p></td><td> <p>100 &micro;l</p></td><td> <p>22 min</p></td><td> <p>50 &micro;l</p></td></tr><tr> <td> <p>1x10<sup>9</sup></p></td><td> <p>500 &micro;l</p></td><td> <p>30 min</p></td><td> <p>50 &micro;l</p></td></tr><tr> <td> <p>5x10<sup>8</sup></p></td><td> <p>500 &micro;l</p></td><td> <p>30 min</p></td><td> <p>10 &micro;l</p></td></tr><tr> <td> <p>1x10<sup>8</sup></p></td><td> <p>1000 &micro;l</p></td><td> <p>40 min</p></td><td> <p>10 &micro;l</p></td></tr></tbody> </table> <li> Balance the tube(s) and centrifuge according to Table 8.1a-1 at 4 °C at top speed in a microcentrifuge to pellet the phage particles to the bottom of the tube. </li><li> Using a micropipettor, carefully remove as much of the supernatant as possible without disrupting the pellet. You should remove the majority of the supernatant, leaving as little liquid as possible without dislodging the pellet. (If you can’t see the pellet, don’t worry, it is still there! You just have a really clean sample.) </li><li> Resuspend the pellet in the proper volume of phage buffer (Table 8.1a-1) and mix gently using the pipette tip. Note that vigorous pipetting can damage the phage particles. </li><li> Proceed with the rest of the protocol immediately to avoid damaging the phage heads. </li></ol> </li>      <li>Prepare your work area. (This may have been done before class by your instructor.)<ol> <li>Put on a fresh pair of gloves.</li><li>Cover the designated work area with bench paper or a large Kimwipe to create a clean work surface.</li><li>Cut a 5x5 cm piece of parafilm and place into the lid of a Petri dish with the waxy side facing up.</li></ol> </li><li>Prepare lysate, water wash, and stain. <ol> <li>Using a micropipettor, transfer 10-15 &micro;l of high titer lysate to the surface of the parafilm. It will make a droplet.</li><li>In a separate area of the parafilm, transfer 15 &micro;l of sterile, filtered water to make a separate droplet.</li><li>In a third area of the parafilm, transfer 15 &micro;l of stain to make a separate droplet.<br> </li></ol> </li><li>Mount and stain your phage. <ol> <li>Using EM forceps, remove a fresh grid from a box of unused grids, touching only the very edge of the grid.</li><li>Place the forceps on the high titer lysate droplet so that the grid’s dark-and-shiny side is facing down toward the liquid. Carefully release the grid. The grid will float on the droplet.</li><li>Allow the grid to sit on the lysate for at least 2 to 10 minutes (use the longer times if lysate titer is not higher than 10<sup>9</sup> pfu/ml). During this time the phage will adsorb onto the grid.</li><li>Rinse the grid once using the following method: <ol style = 'list-style-type:lower-alpha;'> <li>Carefully grasp the edge of the grid with the forceps. While holding, use a small (2-to-3-cm) wedge of Whatman filter paper to wick off the excess fluid.</li><li>Place the grid, dark-and-shiny side down, onto the droplet of water. Leave for 30 seconds.</li><li>Remove the grid from the drop with forceps. Wick excess liquid immediately using an unused corner of the filter paper wedge.<br><span>Important</span>: Work quickly and carefully! Do NOT allow the grid to dry out or the phage capsids can collapse.</li></ol> </li><li>Place grid on droplet of staining liquid with dark-and-shiny side down. Allow to stain for 2 minutes.<br> </li><li>Carefully grasp edge of grid with the forceps. Immediately begin to wick off excess stain by using a wedge of filter paper. Lanthanide staining occurs by leaving a very thin layer of stain dried across the entire grid. You should continue to wick away the stain until the liquid no longer enters the filter paper. Then allow the grid to air dry before putting it safely back into the grid box.<br> </li></ol> </li><li>Observe your phage. <ol> <li>Place your grid in the designated grid box for storage. Be sure to accurately record the location of your grid in the box.</li><li>Transport your samples to your EM facility for imaging.<br> </li></ol> </li><li>Calculate the capsid diameter and tail length relative to the size bar. <ol> <li>Using a ruler, measure the widest point (edge-to-edge, not vertex-to-vertex) of the capsid and the length of the tail (excluding the capsid and any visible tail fibers). If possible, measure multiple phage heads and tails and average their respective values.</li><li>Measure the length of the size bar with the ruler.</li><li>Using the known and relative lengths of the size bar, calculate the length of the capsid and tail. <p> <span><strong>Example</strong></span>: <p>The phage in the electron micrograph below has an average capsid diameter of 1.6 cm (16 mm) and a tail length of 4.8 cm (48 mm). The 100 nm size bar is measured to be 2.4 cm long.</p><div class = 'img-wrap'> <div class = 'img-container'> <image src = './assets/images/figure-8-2-1.png' alt = 'Electron micrograph picture of a phage with a scale marked at the bottom of 100 nanometers.'/> </image> </div> <div class = 'img-hr'></div> <div class = 'title-container'> </div></div><p>To find the scaled (actual) size of the tail set up a simple ratio and solve for the unknown scaled size:<br>(scaled size bar) / (measured size bar) = (unknown scaled sized) / (measured size)<br>(100 nm) / (24 mm) = (unknown scaled tail size) / 48 mm<br>unknown scaled tail size = (100 nm)(48 nm) / (24 mm)<br>unknown scaled tail size = 200 nm</p></li><li>Compare your capsid and tail lengths with those of your classmates.</li><li>Record your findings for entry in Phagesdb according to the protocol <a class = 'chap-linking' tabindex = '0' data-link-to = '7-3'><strong> Entering Phage into the Actinobacteriophage Database </strong>(7.2)</a>.</li></ol> </li></ul><hr><br><h3>Helpful Tips:</h3><ul style = 'list-style-type:disc'> <li>The EM forceps are very delicate. Never set them down without first replacing the protective cover.</li><li>It is extremely important that the grids do not dry out, especially between the last water wash and the addition of the stain. The capsid proteins destabilize in water (because there are no salts available) and the capsid will collapse or burst open if the grid dries out before stain is added. The stain acts as a fixative and will help stabilize the capsids when they dry.</li><li>Positively stained capsids (the phage heads look black because the stain has permeated the capsid) should not be used when describing the size or the morphology of the phage. Positive staining and capsid collapse can be caused by allowing the grid to dry out.</li><li>If a white precipitate is visible on the grids after drying, a new grid needs to be made. The precipitates are leftover salts from the lysate that were not wicked off completely and can damage the TEM.</li><li>Best results are obtained when FRESH lysates are used for the EM staining experiments.</li></ul>"},"8-5":{content:"<h2>Working with the delicate EM grids.</h2>  <ul><li  >For transmission electron microscopy, phage samples are deposited on 200-400 mesh carbon-stabilized, formvar coated copper TEM grids. It is best if these grids are “fresh” (<6 month) and not saved from year to year since the formvar gets brittle with age. When storing unused grids, place in a dry environment.</li> <li  >TEM forceps come in two varieties. Traditional forceps are held closed with your thumb and index finger, while reverse forceps remain closed at rest, and are opened by exerting pressure with your thumb and index finger. Which protocol you use to stain your grids will depend on which forceps you want to use. For protocol 8.1a you need reverse forceps, but protocol 8.1b is better suited to regular forceps. Either forcep will work well for protocol 8.1c. A summary of the protocols is provided below: <ul type = 'disc'><li  > <strong> Protocol 8.1a</strong> uses reverse forceps to hold the grid while the sample and stain are added to the grid surface. The grid is rinsed by holding the forceps and the grid at an angle and dropping sterile filtered water across the grid surface. Using the reverse forceps reduces damage to the delicate grid since it is only manipulated when it is removed from the grid box, and again when it is returned to the grid box. The rest of the time it is safely held in the forceps.</li> <li  > <strong>Protocol 8.1b </strong> uses sticky Pelco tabs in the bottom of a Petri dish to hold the grid in place while the sample, rinses, and stain are deposited on the grid and wicked off using filter paper. It can be difficult to move the grids from the grid box onto the Pelco tab, and off again, without damaging the grid. A video protocol from Gonzaga University is available<a href = 'https://www.youtube.com/watch?v = F0ZNUykXovk' target = '_blank'> here</a>.</li>  <li  > <strong>Protocol 8.1c </strong> is called the parafilm drop method, in which a droplet of phage sample, water, and stain are placed on a piece of parafilm. The grid is then “floated” on these liquid droplets, sequentially.</li></ul> </li> <li  > When removed from the grid box you will see that the girds have a “dark and shiny” side and a “light and matte” side. You will want to put the sample on the “dark and shiny” side. This dark side is the side that formvar has been layered on and covered with the carbon. The light side actually has some depth to it, and the sample and stain can actually pool in the boxes made by the mesh rather than spreading evenly across the carbon surface. Such pooling will negatively impact your ability to visualize your phage by TEM.</li><li  > These grids are very delicate and should not be crumpled or bent prior to viewing on the TEM. To avoid damaging the grids, handle them carefully and try to touch only the outer edge with the forceps when moving them between grid boxes and the preparation area. However, even if the grids do get crumpled or bent, it is usually possible to unfold them and use them to visualize phage anyway. </li> <li  > Most commercially purchased formvar coated grids have been glow-discharged for you. If you are preparing your own grids you will need to treat them in a glow-discharge unit. Carbon and plastic are hydrophobic, while phage lysates are hydrophilic. Imagine trying to evenly spread a thin film of water across a piece of saran wrap! The water beads and pools. The same thing would happen to phage samples if the grids were not glow-discharged prior to application of the sample. In a glow-discharge unit, grids are placed under vacuum on a metal surface, and high-voltage electricity is added to the system at a slight distance. In a glass glow discharge unit, it is possible to see a purple glow evenly surrounding the high voltage source through the vacuum and the metal plate that hold the grids. By glow-discharging grids for up to a minute, it is possible to artificially charge the surface of the carbon/plastic, thus allowing hydrophilic samples to evenly spread across it.</li> </ul> <h2> Staining</h2> <ul> <li  > The method presented for staining your EM grids utilizes uranyl acetate, a  toxic compound for which you should read the MSDS information carefully. When dealing with uranyl acetate, you should wear gloves and exposure should be minimized. You should set up a designated staining station where all work using uranyl acetate will be done and waste collected. The uranyl acetate waste will need to be saved separately and disposed of according to your institution’s hazardous waste policies.</li><li  > Set up a “staining area” on a piece of absorbent bench paper. This will help you keep the uranyl acetate in one area as well as provide a clean white surface in case a grid is accidentally dropped. </li> <li  > It is important that the 1-2 % uranyl acetate be filtered after it is made, before use. The 1- 2 % uranyl acetate can be made ahead of time and stored for long periods of time (years!) if it is protected from light with aluminum foil.</li> <li  > A staining alternative to uranyl acetate is non-toxic lanthanide salts (e.g. UranyLess). You can substitute lanthanide salts for uranyl acetate and simply follow the rest of the protocol. The contrast is not usually as good in the TEM images, and so we recommend uranyl acetate. </li> </ul>   <h2> Other Tips </h2> <ul> <li  > It is important to have a scale bar on each micrograph so that the size of the capsid and tail can be measured. Comparing the measurements of phage particles can be important for determining if a phage is unique or may be the same as another phage. For example, multiple phages can produce a common plaque morphology but may have very different capsid and tail dimensions. The scale bar is also required if the micrograph is to be included in research presentations and publications. Please remind your microscopist to add a scale bar to each image.      </li> <ul>  "},"9-1":{content:"<p>Once you have obtained a high-titer lysate of your clonal phage population, you can further characterize your phage by analyzing its genome. A genome is defined as an organism’s complete set of genetic material, comprising all of its genes and regulatory sequences. While virus genomes are unique in that they can be made of RNA or DNA, most phages isolated in this course use DNA as their genetic material. To characterize your phage’s DNA, you first need to extract it from your phage lysate. Once isolated, you can study the phage DNA by using experiments like restriction enzyme digestion and DNA sequencing.  </p><p>     Your lysate contains your clonal phage population as well as the remnants of lysed bacterial cells, including bacterial nucleic acids (DNA and RNA), proteins, and lipids. Because the presence of bacterial DNA and RNA will interfere with future experiments designed to study your phage genome, the initial step in DNA extraction is the destruction of bacterial nucleic acids by using enzymes called nucleases. Your phage genomic DNA is surrounded by the protein shell of the phage capsid and therefore protected from the nucleases. After a brief incubation period, the nucleases are inactivated with the addition of denaturants like guanidine hydrochloride. Denaturants unfold all proteins, including the nucleases and phage capsids, thereby simultaneously inactivating the nucleases and releasing phage genomic DNA into your lysate. </p><p>      Once the phage genomic DNA has been released, you can purify it from the rest of the lysate. First, you add a resin containing microscopic polymer beads that bind to the long DNA molecules of your phage genome. This resin/DNA mix is then loaded onto a column that contains a very fine filter that prevents the beads and bound DNA from escaping the column. This allows you to “wash” away unwanted contaminating molecules such as proteins, lipids, nucleotides, denaturant, and salts with isopropanol. The DNA is then released, or eluted, from the resin by adding hot water to the column and collecting the liquid that contains your phage genomic DNA. The concentration of DNA in your sample is determined by using spectrophotometry, fluorimetry, or by viewing it on an agarose gel. </p><p>      The protocol you will use takes advantage of a commercially available kit. Kits like these are available from many different vendors and are designed to make common molecular biology protocols, like DNA extractions, easier, faster, and generally more accessible. Not too long ago, kits were not readily available and scientists spent much of their time mixing solutions and spending hours performing protocols that now only take a few minutes. These kits are not magic, but instead are based on the inherent chemical properties of the molecules involved, like proteins and DNA. Understanding what each step accomplishes, and the chemistry behind it, is especially important when it comes to troubleshooting and understanding why an experiment didn’t go as planned. It is also important to keep in mind that not all protocols or kits will work with all organisms, so understanding the principles behind what you are doing can be crucial to your success.</p> <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-9-1-1.jpg' alt = 'A flowchart for extraction: extracting phage D-N-A leads to the question: do you have enough D-N-A? If no, then return to extracting phage D-N-A. If yes, then proceed to characterization.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 9.0-1. Flow chart of phage DNA extraction experiments..</div></div> "},"9-2":{content:"  <h2>Objective:</h2>  <p > To isolate genomic DNA from phage</p> <hr><br><h2>Rationale:</h2> <p>Once you have a prepared a phage lysate, your phage DNA can be isolated and studied. The first step in extracting phage DNA is to remove bacterial DNA and RNA from your lysate using nuclease enzymes. Next, the phage DNA is released from the phage capsid, bound to a solid matrix according to its inherent chemical properties, rinsed, and eluted, yielding a sample of pure phage DNA. </p>  <hr><br><h2>Supplies:</h2> <ul> <li   >1 ml phage lysate (titer ≥ 5 x 109 pfu/ml)</li> <li   >Nuclease mix</li> <li   >EDTA (0.5 M) (optional)</li> <li   >Proteinase K (20 mg/ml) and SDS (10 %) (optional)</li> <li   >2 ml DNA clean-up resin (Promega Wizard DNA Clean-Up Kit)</li> <li   >2 DNA clean-up columns (Promega Wizard DNA Clean-Up Kit)</li> <li   >3 ml syringes</li> <li   >6 ml 80 % isopropanol, freshly prepared</li> <li   >ddH2O pre-warmed (95 °C)</li>  </li> </ul> <hr><br><h2>Procedure:</h2>   <ol style = 'list-style-type:upper-alpha;'> <li  >Prepare your bench and assemble your supplies.    </li> <li  >Degrade bacterial DNA/RNA in high-titer phage lysate. <ol> <li  >Aseptically transfer 1 ml of phage lysate into a microcentrifuge tube.</li> <li  >Wearing gloves and working in the designated area, add 5 &micro;l nuclease mix to the lysate.   <span style = ' ;'><br><strong>Important</strong></span>: The enzymes (RNase in particular) are very stable and can persist and contaminate equipment and supplies throughout the laboratory. Take precautions to keep and use them in the designated area. </li> <li   value = '3'>Mix gently but thoroughly by repeated inversions—do not vortex!</li> <li   value = '4'>Incubate at 37 °C for 10 minutes or room temperature for 30 minutes.</li> <li   value = '5'>Remove and discard your gloves before returning to your bench.</li> <li   value = '6'>Optional: Add 15 &micro;l EDTA to the nuclease-treated lysate and mix gently. <ol style = 'list-style-type:lower-alpha;'> <li  >EDTA will inactivate the nucleases by chelating, or binding, divalent cations required by the nucleases for activity.</li> </ol> </li> <li  >Optional: Add 0.5 &micro;l Proteinase K and 50 &micro;l SDS to the nuclease-treated lysate and mix gently. Incubate at 37 °C for 10 minutes. <ol start = '1' style = 'list-style-type: lower-alpha;'> <li  >Proteinase K is added to degrade the nucleases added in Step 2. SDS stimulates the activity of Proteinase K.</li> </ol> </li> </ol> </li>  <li   value = '3'>Denature the protein capsid to release phage DNA. <ol> <li  >Put on a fresh pair of gloves.</li> <li  >Add 2 ml of DNA clean-up resin to a 15 ml conical tube. <ol style = 'list-style-type:lower-alpha;'> <li  >The DNA resin is a slurry solution containing microscopic polymer beads. Make sure that the bottle of resin is well mixed, the precipitate dissolved by heating to 37 °C, and the beads resuspended before aliquoting your 2 ml. (Your instructor may have done this for you.)   <span style = ' ;'><br><strong>Important</strong></span>: The resin contains guanidinium thiocyanate, a chemical that denatures proteins. Do not get it on your skin!</li> </ol> </li> <li   value = '3'>Transfer your nuclease-treated phage lysate from the microcentrifuge tube to the 15 ml conical tube containing resin.</li> <li   value = '4'>Mix the solution by gently inverting the tube repeatedly for 2 minutes.    </li> </ol> </li> <li   value = '4'>Isolate the phage genomic DNA. <ol> <li  >Label two Wizard Kit columns with your initials.</li> <li  >Remove the plungers from two 3 ml syringes and attach a column to each syringe barrel.</li> <li  >Follow the steps below for each column at the same time: <ol style = 'list-style-type:lower-alpha;'> <li  >Set the column and syringe barrel on a new microcentrifuge tube.</li> <li  >Transfer 1.5 ml of phage DNA/resin solution to the column using a pipette.</li> <li  >Do not discard the empty 15 ml conical tube.</li> <li  >Insert a plunger into the syringe and carefully push all the liquid through, collecting the flow-through in the used 15 ml conical tube from above.   <span style = ' ;'><br><strong>Important</strong></span>: The DNA is bound to the polymer beads that pack into the column as the liquid is pushed through. It is VERY important to maintain a firm, gentle, unrelenting, and even pressure on the syringe. Do not let the plunger pop out of the syringe barrel because releasing the vacuum will ruin the column.</li> <li   value = '5'>Once the liquid is expelled, maintain pressure on the plunger as you dry residual liquid by touching the tip of the column to a paper towel.</li> <li   value = '6'>Unscrew the column from the syringe barrel before releasing the plunger and set the column into a clean microcentrifuge tube.</li> <li   value = '7'>Remove the plunger from the syringe barrel, and then reattach the syringe barrel to the column.</li> </ol> </li> <li  >Wash the salts from the DNA (now in the column) with the following steps for each column: <ol style = 'list-style-type:lower-alpha;'> <li  >Add 2 ml 80 % isopropanol to each syringe barrel/column and push the liquid through the column, repeating steps 3(d)–3(f).</li> <li  >Repeat twice, for a total of three isopropanol washes.</li> </ol> </li> <li  >Remove residual isopropanol. <ol style = 'list-style-type:lower-alpha;'> <li  >With each column in a fresh 1.5 ml microcentrifuge tube, spin at 10,000 × g for 5 minutes. <ol style = 'list-style-type:lower-roman;'> <li  >The column will prevent the microfuge tube lids from closing. Arrange the open tubes in the centrifuge so that the lids point toward the center of the rotor.</li> </ol> </li> <li  >Transfer columns to new 1.5 ml microcentrifuge tubes. Spin at 10,000 × g for 1 additional minute to remove any residual isopropanol.</li> <li  >Evaporate the last traces of isopropanol by removing your columns from the microcentrifuge tubes and placing them directly in a 90 °C heating block for 60 seconds.   <span style = ' ;'><br><strong>Important</strong></span>: Leaving the columns in the heat block for more than 1 minute can lead to DNA damage.</li> </ol> </li> <li   value = '6'>Elute the phage DNA from the columns. <ol style = 'list-style-type:lower-alpha;'> <li  >Place each column in a clean microcentrifuge tube and apply 50 &micro;l of 90 °C sterile ddH<sub>2</sub>O directly to each column.   <span style = ' ;'><br><strong>Important</strong></span>: Keep the ddH<sub>2</sub>O in the heating block so that it remains at 90 °C.</li> <li   value = '2'>Incubate columns for 1 minute at room temperature.</li> <li   value = '3'>Spin at 10,000 × g for 1 minute in a microcentrifuge.</li> <li   value = '4'>Combine the products from both microcentrifuge tubes into one tube; this is your eluted phage DNA.    </li> </ol> </li> </ol> </li> <li   value = '5'>Determine the concentration of your phage DNA. <ol> <li  >Using a spectrophotometer (fluorimeter, or Nanodrop) and a protocol from your instructor, quantify your phage DNA.</li> <li  >Place at 4 °C for short-term storage (1–2 weeks) or at -20 °C for long-term storage.</li> </ol> </li> </ol> </div> <div  > <hr><br><h3>Helpful Tips:</h3>  <ul> <li  >Make sure you have a titer of 5 x 10<sup>9</sup> or higher before proceeding with the protocol, or you may not isolate enough DNA.</li>  <li  >If your titer is less than 5 x 10<sup>9 </sup>you can increase the concentration by pelleting your phage.</li>  <li  >Spinning above 10,000 x g can lead to the column getting stuck in the microcentrifuge tube, the microcentrifuge tube breaking, and both getting stuck in the microcentrifuge.</li>  <li  >Insufficient rinsing with isopropanol will lead to residual guanidinium that will affect your ability to estimate DNA concentration via spectrophotometry.</li>  <li  >Failure to remove all of the isopropanol can cause problems when loading agarose gels later.</li> <li  >A brief video demonstrating this protocol is available <a href = 'https://seaphages.org/video/46/' target = '_blank'>here</a>.</li> </ul>"},"9-3":{content:"  <h2>Objective :</h2><p> To concentrate liquid phage samples prior to extracting DNA</p>    <hr><br><h2>Rationale:</h2> <p> Once you have a prepared a phage lysate, your phage DNA can be isolated and studied. If you have a lysate with a titer lower than 5 x 109 the phage can be concentrated prior to DNA isolation to increase yield. The first step in extracting phage DNA is to remove bacterial DNA and RNA from your lysate using nuclease enzymes. After precipitating you phage using phage precipitant solution containing polyethylene glycol (PEG), and pelleting in a high speed centrifuge, the phage DNA is released from the phage capsid, bound to a solid matrix according to its inherent chemical properties, rinsed, and eluted, yielding a sample of pure phage DNA. </p> <hr><br><h2>Supplies:</h2> <ul> <li  >4 - 10 ml phage lysate</li> <li  > Phage precipitant solution (0.4 ml per 1 ml phage lysate) </li> <li  >Nuclease mix</li> <li  >15 ml Oak Ridge Tube (optional)</li> <li  >15 ml conical tubes</li> <li  >Sterile water (0.5 ml)</li> <li  > 2 ml DNA clean-up resin (Promega Wizard DNA Clean-Up Kit) </li> <li  > 2 DNA clean-up columns (Promega Wizard DNA Clean-Up Kit) </li> <li  >3 ml syringes</li> <li  >6 ml 80 % isopropanol, freshly prepared</li> <li  >ddH2O pre-warmed (95 °C)</li> </ul> </div> <hr><br><h2>Procedure:</h2>   <ol style = 'list-style-type: upper-alpha'> <li  > Prepare your bench and assemble your supplies.<br />   </li>   <li  > Degrade bacterial DNA/RNA in phage lysate. <ol> <li  > Aseptically transfer 4 - 10 ml of phage lysate into a conical tube.<br /> Note: This step can be done in an Oak Ridge tube if an appropriate centrifuge for these tubes are available. </li> <li  > Wearing gloves and working in the designated area, add 5 &micro;l nuclease mix (per 1 ml phage lysate) to the lysate.<br /> <strong>Important:</strong> The enzymes (RNase in particular) are very stable and can persist and contaminate equipment and supplies throughout the laboratory. Take precautions to keep and use them in the designated area. </li> <li   value = '3'> Mix gently but thoroughly by repeated inversions, and incubate at 37 °C for 10 minutes or at room temperature for 30 minutes. </li> <li   value = '4'> Optional: Add 15 &micro;l EDTA (per 1 ml phage lysate) to the nuclease-treated lysate and mix gently. <ol style = 'list-style-type: lower-alpha'> <li  > EDTA will inactivate the nucleases by chelating, or binding, divalent cations required by the nucleases for activity. </li> </ol> </li> <li  > Optional: Add 0.5 &micro;l Proteinase K and 50 &micro;l SDS (per 1 ml phage lysate) to the nuclease-treated lysate and mix gently. Incubate at 37 °C for 10 minutes. <ol style = 'list-style-type: lower-alpha'> <li  > Proteinase K is added to degrade the nucleases added in Step 2. SDS stimulates the activity of Proteinase K.<br />   </li> </ol> </li> </ol> </li> <li   value = '3'> Precipitate phage particles. <ol> <li  > Add 0.4 ml of phage precipitant solution to every 1.0 ml of the nuclease-treated lysate, and cap the tube. </li> <li  >Mix gently but thoroughly by inversion.</li> <li  > Prepare 1 ml aliquots of the mixture in microcentrifuge tubes. </li> <li  > Incubate 30 minutes on ice or, for maximum yield, overnight at 4 °C. </li> <li  > Place the tube in a high-speed centrifuge, balance the tubes, and spin at 10,000 × g for 20 minutes. Centrifugation should be done at 4 °C. </li> <li  > Decant the supernatant being careful not to disturb the pellet. Drain excess liquid from the pellet by inverting for 2 to 3 minutes on a paper towel. Discard the paper towel and your gloves.<br /> <strong>Important:</strong> Remove as much phage precipitant solution as possible with out disturbing the phage pellet. Failure to do so will interfere with or clog the column and significantly reduce your DNA yield.<br />   </li> </ol> </li> <li   value = '4'> Re-suspend the phage pellet. <ol> <li  > Add 0.5 ml of sterile ddH<sub>2</sub>0 to the pellet in one microcentrifuge tube. </li> <li  >Gently re-suspend the pellet by pipetting up and down.</li> <li  > Transfer the resuspension to the pellet in the next centrifuge tube, and gently re-suspend the pellet by pipetting up and down.<br /> Note: Skip step 3 and 4 if using one Oak Ridge tube per lysate. </li> <li  > Continue to do so until all the phage pellets have been resuspended in the 0.5 ml of ddH<sub>2</sub>0. </li> </ol> </li> </ol> <ol style = 'list-style-type: upper-alpha'> <li   value = '1'> Denature the protein capsid to release phage DNA. <ol> <li  >Put on a fresh pair of gloves.</li> <li  > Add 2 ml of DNA clean-up resin. <ol style = 'list-style-type: lower-alpha'> <li  > The DNA resin is a slurry solution containing microscopic polymer beads. Make sure that the bottle of resin is well mixed, the precipitate dissolved by heating to 37 °C, and the beads resuspended before aliquoting your 2 ml. (Your instructor may have done this for you.)<br /> <strong>Important:</strong> The resin contains guanidinium thiocyanate, a chemical that denatures proteins. Do not get it on your skin! </li> </ol> </li> <li   value = '3'> Mix the solution by gently inverting the tube repeatedly for 2 minutes.<br />   </li> </ol> </li> <li   value = '2'> Isolate the phage genomic DNA. <ol> <li  >Label two Wizard Kit columns with your initials.</li> <li  > Remove the plungers from two 3 ml syringes and attach a column to each syringe barrel. </li> <li  > Follow the steps below for each column at the same time: <ol style = 'list-style-type: lower-alpha'> <li  > Set the column and syringe barrel on a new microcentrifuge tube. </li> <li  > Transfer 1.25 ml of phage DNA/resin solution to the column using a pipette. </li> <li  >Do not discard the empty 15 ml conical tube.</li> <li  > Insert a plunger into the syringe and carefully push all the liquid through, collecting the flow-through in the used 15 ml conical tube from above.<br /> <strong>Important:</strong> The DNA is bound to the polymer beads that pack into the column as the liquid is pushed through. It is VERY important to maintain a firm, gentle, unrelenting, and even pressure on the syringe. Do not let the plunger pop out of the syringe barrel because releasing the vacuum will ruin the column. </li> <li   value = '5'> Once the liquid is expelled, maintain pressure on the plunger as you dry residual liquid by touching the tip of the column to a paper towel. </li> <li   value = '6'> Unscrew the column from the syringe barrel before releasing the plunger and set the column into a clean microcentrifuge tube. </li> <li   value = '7'> Remove the plunger from the syringe barrel, and then reattach the syringe barrel to the column. </li> </ol> </li> <li  > Wash the salts from the DNA (now in the column) with the following steps for each column: <ol style = 'list-style-type: lower-alpha'> <li  > Add 2 ml 80 % isopropanol to each syringe barrel/column and push the liquid through the column, repeating steps 3(d)–3(f). </li> <li  > Repeat twice, for a total of three isopropanol washes. </li> </ol> </li> <li  > Remove residual isopropanol. <ol style = 'list-style-type: lower-alpha'> <li  > With each column in a fresh 1.5 ml microcentrifuge tube, spin at 10,000 × g for 5 minutes. <ol style = 'list-style-type: lower-roman'> <li  > The column will prevent the microfuge tube lids from closing. Arrange the open tubes in the centrifuge so that the lids point toward the center of the rotor. </li> </ol> </li> <li  > Transfer columns to new 1.5 ml microcentrifuge tubes. Spin at 10,000 × g for 1 additional minute to remove any residual isopropanol. </li> <li  > Evaporate the last traces of isopropanol by removing your columns from the microcentrifuge tubes and placing them directly in a 90 °C heating block for 60 seconds.<br /><strong> Important: </strong>Leaving the columns in the heat block for more than 1 minute can lead to DNA damage. </li> </ol> </li> <li   value = '6'> Elute the phage DNA from the columns. <ol style = 'list-style-type: lower-alpha'> <li  > Place each column in a clean microcentrifuge tube and apply 50 &micro;l of 90 °C sterile ddH<sub>2</sub>O directly to each column.<br> <strong>Important:</strong> Keep the ddH<sub>2</sub>O in the heating block so that it remains at 90 °C. </li> <li   value = '2'> Incubate columns for 1 minute at room temperature. </li> <li   value = '3'> Spin at 10,000 × g for 1 minute in a microcentrifuge. </li> <li   value = '4'> Combine the products from both microcentrifuge tubes into one tube; this is your eluted phage DNA.<br />   </li> </ol> </li> </ol> </li> <li   value = '3'> Determine the concentration of your phage DNA. <ol> <li  > Using a spectrophotometer (fluorimeter, or Nanodrop) and a protocol from your instructor, quantify your phage DNA. </li> <li  > Place at 4 °C for short-term storage (1–2 weeks) or at -20 °C for long-term storage. </li> </ol> </li> </ol> <hr><br><h3>Helpful Tips:</h3> <ul> <li  > Spinning above 10,000 x g in the microcentrifuge can lead to the column getting stuck in the microcentrifuge tube, the microcentrifuge tube breaking, and both getting stuck in the microcentrifuge. </li> <li  > Insufficient rinsing with isopropanol will lead to residual guanidinium that will affect your ability to estimate DNA concentration via spectrophotometry. </li> <li  > Failure to remove all of the isopropanol can cause problems when loading agarose gels later. </li> </ul> "},"9-4":{content:"    <h2>Objective :</h2><p> To concentrate liquid phage samples prior to extracting DNA </p>   <hr><br><h2>Rationale:</h2><p> Once you have a prepared a phage lysate, your phage DNA can be isolated and studied. If you have a lysate with a titer lower than 5 x 109 the phage can be concentrated prior to DNA isolation to increase yield. The first step in extracting phage DNA is to remove bacterial DNA and RNA from your lysate using nuclease enzymes. After precipitating you phage using zinc chloride, and pelleting in a high speed centrifuge, the phage DNA is released from the phage capsid, bound to a solid matrix according to its inherent chemical properties, rinsed, and eluted, yielding a sample of pure phage DNA. </p>  <hr><br><h2>Supplies:</h2> <ul> <li   >phage lysate (4 - 10 ml)</li> <li   >15 ml conical tube</li> <li   >2 M ZnCl<sub>2</sub> (25 &micro;l per 1 ml of phage lysate), freshly prepared and filter sterilized</li> <li   >Nuclease mix</li> <li   >0.1 M EDTA (0.5 ml)</li> <li   >Proteinase K (20 mg/ml) and SDS (10 %) (optional)</li> <li   >2 ml DNA clean-up resin (Promega Wizard DNA Clean-Up Kit)</li> <li   >2 DNA clean-up columns (Promega Wizard DNA Clean-Up Kit)</li> <li   >3 or 5 ml syringes</li> <li   >6 ml 80 % isopropanol, freshly prepared</li> <li   >ddH<sub>2</sub>O pre-warmed (95 °C)    </li> </ul> <hr><br><h2>Procedure: </h2>   <ol style = 'list-style-type:upper-alpha'> <li   >Prepare your bench and assemble your supplies.    </li> <li   >Degrade bacterial DNA/RNA in ml phage lysate. <ol> <li   >Aseptically transfer 4 - 10 ml of phage lysate into a 10 ml conical tube.</li> <li   >Wearing gloves and working in the designated area, add 5 &micro;l nuclease mix (per 1 ml phage lysate) to the lysate. <br>  <strong>Important:</strong> The enzymes (RNase in particular) are very stable and can persist and contaminate equipment and supplies throughout the laboratory. Take precautions to keep and use them in the designated area.</li> <li   value = '3'>Mix gently but thoroughly by repeated inversions, and incubate at 37 °C for 10 minutes or at room temperature for 30 minutes.    </li> </ol> </li> <li   value = '3'>Precipitate phage particles. <ol> <li   >Using a 5 ml pipette, portion out the nuclease-treated lysate as 1 ml aliquots in clean microcentrifuge tubes.</li> <li   >To each 1 ml of nucelase-treated lysate, add 25 &micro;l ZnCl<sub>2</sub>.</li> <li   >Mix gently but thoroughly by inversion.</li> <li   >Incubate at 37 °C for 10 minutes<sub>.</sub></li> <li   >Place the tube in a high-speed centrifuge, balance the tubes, and spin at 10,000 × rpm for 1 minute.</li> <li   >Remove and discard the supernatant from each microcentrifuge tube using a micropipettor, being careful not to disturb the pellets. This pellet contains your phage particles. Proceed immediately to Step D.    </li> </ol> </li> <li   value = '4'>Re-suspend the phage pellet in 1.0 ml of EDTA. <ol> <li   >Distribute a total of 1.0 ml of 0.1M EDTA equally between all the microcentrifuge tubes containing a pellet from Step C6.   For example, if you have 5 microcentrifuge tubes from Step C6, add 200 &micro;l of EDTA to each microcentrifuge tube.</li> <li   >Working quickly but gently, re-suspend each pellet by pipetting up and down.</li> <li   >Once all the pellets have been resuspended, combine all the resuspensions into one microcentrifuge tube.</li> <li   >Optional: Add 0.5 &micro;l Proteinase K and 50 &micro;l SDS to the nuclease-treated lysate and mix gently. Incubate at 37 °C for 10 minutes. <ol style = 'list-style-type:lower-alpha'> <li   >Proteinase K is added to degrade the nucleases added in Step 2. SDS stimulates the activity of Proteinase K.</li> </ol> </li> </ol> </li> </ol> <ol style = 'list-style-type:upper-alpha'> <li   value = '1'>Denature the protein capsid to release phage DNA. <ol> <li   >Put on a fresh pair of gloves.</li> <li   >Transfer the 1.0 ml of phage pellet resuspension to a clean 15 ml conical tube.</li> <li   >Add 2 ml of DNA clean-up resin to the phage pellet resuspension. <ol style = 'list-style-type:lower-alpha'> <li   >The DNA resin is a slurry solution containing microscopic polymer beads. Make sure that the bottle of resin is well mixed, the precipitate dissolved by heating to 37 °C, and the beads resuspended before aliquoting your 2 ml. (Your instructor may have done this for you.) <br>  <strong >Important:</strong> The resin contains guanidinium thiocyanate, a chemical that denatures proteins. Do not get it on your skin!</li> </ol> </li> <li   value = '4'>Mix the solution by gently inverting the tube repeatedly for 2 minutes.</li> </ol> </li> </ol> <ol style = 'list-style-type:upper-alpha'> <li   value = '1'>Isolate the phage genomic DNA. <ol> <li   >Label two Wizard Kit columns with your initials.</li> <li   >Remove the plungers from two syringes and attach a column to each syringe barrel.</li> <li   >Follow the steps below for each column at the same time: <ol style = 'list-style-type:lower-alpha'> <li   >Set the column and syringe barrel on a new microcentrifuge tube.</li> <li   >Transfer 1.5 ml of phage DNA/resin solution to the column using a pipette.</li> <li   >Do not discard the empty 15 ml conical tube.</li> <li   >Insert a plunger into the syringe and carefully push all the liquid through, collecting the flow-through in the used 15 ml conical tube from above.  <br> <strong>Important:</strong> The DNA is bound to the polymer beads that pack into the column as the liquid is pushed through. It is VERY important to maintain a firm, gentle, unrelenting, and even pressure on the syringe. Do not let the plunger pop out of the syringe barrel because releasing the vacuum will ruin the column.</li> <li   value = '5'>Once the liquid is expelled, maintain pressure on the plunger as you dry residual liquid by touching the tip of the column to a paper towel.</li> <li   value = '6'>Unscrew the column from the syringe barrel before releasing the plunger and set the column into a clean microcentrifuge tube.</li> <li   value = '7'>Remove the plunger from the syringe barrel, and then reattach the syringe barrel to the column.</li> </ol> </li> <li   >Wash the salts from the DNA (now in the column) with the following steps for each column: <ol style = 'list-style-type:lower-alpha'> <li   >Add 2 ml 80 % isopropanol to each syringe barrel/column and push the liquid through the column, repeating steps 3(d)–3(f).</li> <li   >Repeat twice, for a total of three isopropanol washes.</li> </ol> </li> <li   >Remove residual isopropanol. <ol style = 'list-style-type:lower-alpha'> <li   >With each column in a fresh 1.5 ml microcentrifuge tube, spin at 10,000 × g for 5 minutes. <ol style = 'list-style-type:lower-roman'> <li   >The column will prevent the microfuge tube lids from closing. Arrange the open tubes in the centrifuge so that the lids point toward the center of the rotor.</li> </ol> </li> <li   >Transfer columns to new 1.5 ml microcentrifuge tubes. Spin at 10,000 × g for 1 additional minute to remove any residual isopropanol.</li> <li   >Evaporate the last traces of isopropanol by removing your columns from the microcentrifuge tubes and placing them directly in a 90 °C heating block for 60 seconds.  <br><strong> Important:</strong> Leaving the columns in the heat block for more than 1 minute can lead to DNA damage.</li> </ol> </li> <li   value = '6'>Elute the phage DNA from the columns. <ol style = 'list-style-type:lower-alpha'> <li   >Place each column in a clean microcentrifuge tube and apply 50 &micro;l of 90 °C sterile ddH<sub>2</sub>O directly to each column.   <br><strong>Important:</strong> Keep the ddH<sub>2</sub>O in the heating block so that it remains at 90 °C.</li> <li   value = '2'>Incubate columns for 1 minute at room temperature.</li> <li   value = '3'>Spin at 10,000 × g for 1 minute in a microcentrifuge.</li> <li   value = '4'>Combine the products from both microcentrifuge tubes into one tube; this is your eluted phage DNA.    </li> </ol> </li> </ol> </li> <li   value = '2'>Determine the concentration of your phage DNA. <ol> <li   >Using a spectrophotometer (fluorimeter, or Nanodrop) and a protocol from your instructor, quantify your phage DNA.</li> <li   >Place at 4 °C for short-term storage (1–2 weeks) or at -20 °C for long-term storage.</li> </ol> </li> </ol> <hr><br><h3>Helpful Tips:</h3> <ul> <li  > Spinning above 10,000 x g in the microcentrifuge can lead to the column getting stuck in the microcentrifuge tube, the microcentrifuge tube breaking, and both getting stuck in the microcentrifuge. </li> <li  > Insufficient rinsing with isopropanol will lead to residual guanidinium that will affect your ability to estimate DNA concentration via spectrophotometry. </li> <li  > Failure to remove all of the isopropanol can cause problems when loading agarose gels later. </li> </ul> "},"9-5":{content:"<h2>Enough DNA?</h2>  <ul><li  >DNA yield per phage lysate processed should ideally be >8 &micro;g of purified phage DNA (at a concentration ~100 ng/&micro;l), for restriction enzyme digests and DNA sequencing.<br> <strong>Important: </strong>The minimum amount of DNA required for sequencing is 4 &micro;g, at 40 ng/&micro;l.</li> <li  >There are many little things that can help with the success of the DNA extraction protocol but the most important factor is starting with a phage titer of 109 pfu/ml or greater. In our experience, titers below this value do not yield adequate amounts of DNA. However, if a titer greater than 109 pfu/ml cannot be obtained, you can try concentrating the sample using several strategies presented below. Regardless of the method you choose to concentrate a phage lysate, it is important to use freshly prepared lysates. This is because phage particles in older lysates are often less stable than in fresh lysates, and the process of concentrating phage can physically damage some fraction of phage particles, often resulting in the release of DNA from the phage particles. Presented here are 3 strategies for concentrating phage: <ol><li  > <strong> Protocol 9.2a: DNA Extraction following Phage Precipitation with PEG.</strong> This protocol involves spinning the lysate at 10,000 x g after precipitation using ZnCl2. It uses ZnCl2 to precipitate phage prior to DNA extraction, and is as reliable for concentrating phage as using PEG. This protocol is faster and does not require centrifugation at 4 °C.</li> <li  > <strong>Protocol 9.2b: DNA Extraction following Phage Precipitation with ZnCl2 </strong> uses sticky Pelco tabs in the bottom of a Petri dish to hold the grid in place while the sample, rinses, and stain are deposited on the grid and wicked off using filter paper. It can be difficult to move the grids from the grid box onto the Pelco tab, and off again, without damaging the grid. A video protocol from Gonzaga University is available here.</li> </ol> </ul><h2> Treatment with nucleases:</h2> <ul> <li  >If your phage is particularly unstable, it may be worth incubating the sample at 30 °C or room temperature, instead of 37 °C.</li><li  > For some phages, the may be nucleases that are co-purified with your DNA. We have observed this for phages isolated using <i>Microbacterium foliorum</i>. In these instances, the addition of cations during the restriction enzyme digest reaction leads to rapid degradation of phage DNA by those nucleases. To avoid co-purification of nucleases, ProteinaseK can be added during the DNA extraction process to degrade the nucleases. The addition of ProteinaseK in the DNA Extraction protocol is presented as an optional step.    </li> <li  >  It is a good idea to set up a separate area where to work with the nucleases, and use a dedicated set of pipettes, to avoid the dispersal of DNase and RNase around the lab.    </li> </ul>  <h2> Other important considerations when extracting phage DNA:</h2> <ul>  <li  > It is important to use <strong>freshly made 80 % isopropanol</strong> since the alcohol will evaporate and change in concentration overtime. Too low of an isopropanol concentration in the wash will cause the DNA to elute off the column during the wash step.</li> <li  > Prior to elution, the column is placed directly in the 90 °C heat block (not inside the 1.5 ml tube, but directly into the block). This step removes any residual isopropanol remaining after the two spins. Again, this is a critical step to assure accurate DNA concentrations, as well as preventing the restriction enzyme reactions from floating out of the wells prior to gel electrophoresis. If RE enzyme samples float when the gels are loaded, you can use DNA loading dye at a final concentration of 2x instead of 1x. Alternatively, you can supplement your loading dye with glycerol to 50 % (v/w). Commercially purchased 6 x loading dye typically has glycerol at 30 % (v/w).  <br><strong> Important:</strong> The DNA is eluted using water heated to 90 °C in the heat block. The kit instructs that TE (Tris EDTA) be used for elution, but the sequencing protocol at the University of Pittsburgh requires the samples be in water.</li> <li  > Using high quality microcentrifuge tubes with a reinforced lip will prevent the tube from collapsing during centrifugation with the DNA isolation columns. These tubes are available from Eppendorf as Polypropylene, Snap-cap, Flex-tubes (Fisher Scientific Catalog Number 05 402).  <br><strong> Important:</strong> Spinning DNA isolation columns in non-reinforced microcentrifuge tubes beyond 10,000 rpm should be avoided, as the columns can break into the centrofuge tubes and become stuck in the centrifuge rotor. </li> <li  > All wash steps in the DNA extraction protocol (9.1) are performed on the DNA clean-up columns by forcing isopropanol through the column with a syringe. An alternative protocol (9.1b) has been included in this Instructor’s Guide, in which all but the last wash step are performed in a microcentrifuge tube and by centrifugation. The advantage of this alternative protocol is that the tedious task of forcing isopropanol over the column is minimized. Though DNA yields reported using this protocol are similar to that when using protocol 9.1, it should be noted that this protocol has not been extensively tested.  <br><strong> Important:</strong> If using the alternate protocol (9.1b), it will be important to ensure that the resin is not remove along with the supernatant during washes.  </li> <ul>  "},"10-1":{content:" <p> Another step in characterizing your phage is to analyze its genome. Ideally, we would sequence the genome of every new bacteriophage isolated during the SEA-PHAGES course. Unfortunately, sequencing technology has not yet progressed to the point where this is feasible in the available time frame. Luckily, you can still get a sense of the character of your phage’s genome by comparing the genetic “fingerprints” generated by restriction endonuclease enzymes. </p> <h3> Restriction Enzymes </h3> <p> A restriction enzyme recognizes and cuts a specific 4–6 base pair sequence of DNA, called a restriction site. The incubation of a piece of DNA with a restriction enzyme is called a restriction digestion. An example of a restriction site is GGATCC, recognized by the restriction enzyme BamHI. The number of times a particular enzyme like BamHI cuts your phage DNA depends on the number of times the restriction site occurs in the genome. For example, if GGATCC occurs once in your phage genome and the genome is mixed with BamHI in a restriction digestion, the genome will be cut into two fragments of DNA. Whereas if GGATCC occurs twice in the phage genome of a classmate and the genome is mixed with BamHI in a restriction digestion, your classmate’s phage genome will be cut twice yielding three fragments of DNA. The number and sizes of fragments that are generated are part of the genetic fingerprint of your phage. By performing multiple restriction digestions with different enzymes, you can create a robust genetic fingerprint of your phage, which can then be compared to those of other phages. </p> <h3> Gel Electrophoresis </h3> <p> You will use gel electrophoresis to separate the DNA fragments in your restriction digestions and visualize the genetic fingerprint of your phage. Gel electrophoresis is a common way to separate and visualize macromolecules like DNA, RNA, and proteins. Even though the exact protocols differ for different macromolecules, the rationale remains the same: charged macromolecules migrate through a molecular sieve (a gel) when an electric current is applied. DNA is negatively charged, and therefore, when placed in a gel and exposed to current, it migrates toward the positive electrode. As the charged macromolecules move through the porous matrix of the gel, the smaller molecules encounter less resistance and move faster than the larger molecules. As a result, the DNA fragments become separated by size. </p> <br /> <p> To perform DNA gel electrophoresis, first the gel is cast by melting agarose (a substance extracted from seaweed) in buffer and then it is poured into the mold of an electrophoresis apparatus where it solidifies after cooling. The gel mold includes a comb that creates depressions called “wells” in the gel into which you pipette your phage DNA sample. Since DNA is invisible to the naked eye, a fluorescent DNA-binding dye is added to the gel prior to pouring. When bound to DNA, the dye fluoresces (glows) when exposed to ultra violet light. As a result, you are able to visualize the location and abundance of the DNA in the gel. It is important to note that these dyes work by binding to DNA—meaning they can also bind to your DNA! Therefore, always be cautious and wear gloves when working with these mutagenic dyes. Traditionally, ethidium bromide (EtBr) has been the most popular dye, but now less toxic alternatives are available and should be considered. After the gel has cooled and set, it is placed in the electrophoresis apparatus, submerged in buffer, and then it is ready to be loaded with your phage DNA sample. Before loading your restriction digest samples into the gel, you must mix them with a loading buffer. The loading buffer typically contains glycerol—which helps your sample sink to the bottom of the well and not diffuse into the buffer—and one or more dyes such as bromophenol blue. Bromophenol blue is a dye that migrates through the gel at a similar rate as a 300 bp DNA fragment when exposed to electricity. Its use allows you to see your sample as you pipette it into the wells of the gel and to track the movement of the DNA through the gel during electrophoresis. </p> <br /> <p> Once your samples are loaded into the wells on the gel, the electrophoresis apparatus is connected to a power source and an electric current flows through the gel, causing the DNA fragments to begin migrating. Because the different-sized DNA fragments migrate at different speeds, each fragment will end up as a band on the gel that is visible when the gel is exposed to the proper wavelength of light. The number and sizes of these bands are the genetic fingerprint of your phage! </p> <br /> <p> A photograph of your phage’s genetic fingerprint (Figure 10.0-1) contains valuable information about your phage that can be used for characterization and comparison purposes. To determine the size of each fragment, or band, as it appears on the gel, a ladder containing DNA fragments of known size is also run on the gel. This ladder can be used to estimate the size of each band from your sample, or, when used to generate a standard curve, to determine the exact size of each band. Once you have determined the number and sizes of the bands from your restriction-digested phage sample, you can interpret them to see how they compare with other characterized phages. </p> <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-10-1-1.jpg' alt = 'A photograph of a gel with D-N-A bands from the elecrophoresis apparatus. There are seven lanes marked ladder, uncut D-N-A, and 5 different enzymes. Measurements of D-N-A length in base-pairs along the side of the picture range from 10,000 to 500.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 10.0-1. Restriction enzyme gel. A 0.8 % agarose gel loaded with a 10 kb ladder and phage DNA. Each lane is labeled according to its contents.</div></div>"},"10-2":{content:" <ul> <li  > If uncut DNA from Protocol 10.1 appears to be degraded when run on an agarose gel (i.e. a smear without the discrete band typical of uncut phage genomic DNA), there may be nucleases persisting in your DNA prep. Nuclease in your DNA prep, if present, will be inactive because of the lack of cofactors required for activity. However, upon the addition of restriction enzyme buffer (i.e. the uncut control in Protocol 10.1), the necessary cofactors for nuclease activity are introduced. To test for the presence of nucleases in your DNA prep, include an additional control in Protocol 10.1 in which neither restriction enzyme nor buffer are included. </li> <li  > The percentage of agarose can be adjusted to resolve different sized bands: 0.8 % will resolve larger bands (greater than 10 kb), 2.5 % may help resolve the size of very small bands (less than 500 bp). </li> <li  > The voltage of the gel can be adjusted to run the gels faster or slower. If you run a gel above 100 Volts, the bands may smear. </li> <li  > Stains other than EtBr can be used for visualization of DNA bands and may be less toxic, as well as eliminate the use of a UV light box. These dyes are generally more expensive but the benefits may outweigh the cost. </li> </ul>"},"10-3":{content:"<h2>Objective:</h2><p> To cut your phage genome into multiple fragments based on its DNA sequence </p><hr><br><h2>Rationale:</h2><p>Restriction endonuclease enzymes recognize and cleave DNA at specific 4–6 bp sequences called restriction sites. Digestion of a phage genome with a given restriction enzyme results in the genome being cut into fragments of different sizes, depending on the number and locations of the restriction sites. Since the genome sequences of different phages are highly variable, the frequency and location(s) of a particular restriction site are not constant across all phages’ genomes. As a result, digestion of different genomes with the same enzyme may yield different numbers and sizes of DNA fragments. These fragments constitute a genetic “fingerprint” that is unique to each phage.</p><hr><br><h2>Supplies:</h2><ul> <li>Phage DNA</li><li>37 °C water bath</li><li>65 °C heat block</li><li>Microcentrifuge tubes</li><li>Restriction enzymes* with buffers</li></ul><p>* Specific sets of restriction enzymes are best suited for studying phages isolated on a given bacterial host. Refers to the specific host information page for the recomemnded enzymes to use.</p><hr><br><h2>Procedure:</h2><ol style = 'list-style-type:upper-alpha;'> <li>Prepare genomic DNA. <ol> <li>Gently mix your DNA sample by either flicking the closed tube with your finger or vortexing it on low. </li><li>Incubate the tube at 65 °C for 10 minutes, and then quickly place it on ice. Quick spin the tube in a microcentrifuge for less than 1 minute to move all of the liquid to the bottom of the tube.</li><li>Using the concentration of your DNA sample, calculate the volume of DNA sample needed to obtain 0.5 &micro;g of DNA.<br><br><span><strong>Example</strong></span>: You want to digest 0.5 &micro;g of your phage DNA. If your DNA sample has a concentration of 125 &micro;g/ml, calculate how many &micro;l of DNA are needed to obtain 0.5 &micro;g:<br>&micro;l DNA = 0.5 &micro;g (ml/125 &micro;g)(1000 &micro;l/1 ml) = 4 &micro;l</li></ol> </li><li>Set up restriction enzyme digest reactions. <ol> <li>Set up a reaction in a microcentrifuge tube for each enzyme according to Table 10.1-1. Include a negative control in which no restriction enzyme is added.<br>  <table> <caption style = 'text-align: left;'> <span style = 'font-size:14px; '>Table 10.1-1. Components of restriction enzyme digest reactions.</span> </caption> <tbody> <tr> <th scope = 'col'> Solution </th> <th scope = 'col'> Volume </th> </tr><tr> <td> Sterile ddH<sub>2</sub>O </td><td> To final volume of 25 &micro;l </td></tr><tr> <td> 10X Reaction Buffer </td><td> 2.5 &micro;l </td></tr><tr> <td> Restriction Enzyme </td><td> 0.5 &micro;l </td></tr><tr> <td> Phage Genomic DNA </td><td> Equivalent to 0.5 &micro;g </td></tr></tbody> </table> <div style = 'clear:both; '><br><strong>Important</strong></span>: Add your phage DNA last to prevent contamination of the enzyme stocks!<br> </div></li><li>Mix the contents of each tube gently and quick spin the tube in a microcentrifuge for less than 1 minute to move all of the liquid to the bottom of the tube.</li><li>Incubate at 37 °C for up to 1 hour. (Note: Some enzymes may require only 15 minutes, so follow your instructor’s directions.)<br> </li></ol> </li><li>Quick spin the tube in a microcentrifuge for less than 1 minute to move all of the liquid to the bottom of the tube. Store at -20 °C until ready to use.</li><li>To visualize your digested phage DNA, follow the protocols for <a class = 'chap-linking' tabindex = '0' data-link-to = '10-4'> <strong>Casting Agarose Gels </strong> (10.2) </a> and <a class = 'chap-linking' tabindex = '0' data-link-to = '10-6'> <strong> Analyzing Restriction Enzyme Gels </strong>(10.4)</a>.</li></ol> <hr><br><h3>Helpful Tips:</h3></p><ul> <li>Do not vortex genomic DNA on high because it can cause it to shear.</li> <li>Restriction enzymes must be kept cold at all times. Keep them in an insulated ice block or ice bucket unless they are being used. When handling the tubes, hold them by the top to avoid heating the enzyme with your fingers.</li> <li>Be attentive when pipetting small volumes. It is best to pipette small volumes into liquid already in the tube because they can adhere to the pipette tip via cohesion.</li> <li>Highly concentrated phage genomic DNA tends to aggregate in solution. Heating for 15 minutes at 55 °C before pipetting can help ensure consistent concentration.</li> <li>Remember that 1 ng/&micro;l = 1 &micro;g/ml.</li></ul>"},"10-4":{content:"<h2>Objective:</h2><p> To prepare an agarose gel for electrophoresis </p><hr><br><h2>Rationale:</h2><p>Gel electrophoresis is a common method of separating DNA fragments according to size. The gel is prepared using the polymer agarose that is purchased as a powder, suspended in liquid buffer, and melted before use. The concentration of agarose needed in your gel depends on the size of DNA fragments you want to separate. Typically, agarose gels are prepared at concentrations of 0.7 %–2 %.</p><hr><br><h2>Supplies:</h2><ul> <li>Agarose</li> <li>1X TBE running buffer (TBE  =  0.089 M Tris-Boric Acid, 0.002 M-EDTA)</li> <li>DNA dye (ethidium bromide or alternative)</li> <li>Erlenmeyer flask</li> <li>Electrophoresis apparatus and power supply<br>  </li> </ul><hr><br><h2>Procedure:</h2><ol style =  'list-style-type:upper-alpha; '> <li>Pour a 0.8 % (w/v) agarose gel. <ol> <li>Set up gel apparatus according to your instructor’s directions.</li> <li>Prepare enough 0.8 % agarose gel to cover the tips of the gel combs by ~2–3 mm.<br> <br> <strong>Example</strong>: You want to prepare 30 ml of 0.9 % agarose. How much powdered agarose will you add to 30 ml of buffer?<br> <br> concentration of solute (w/v, %)  =  [ (mass agarose, <em>g</em>) / (volume agarose, <em>ml</em>) ] x 100<br> 0.9 %  =  [ (X, <em>g</em>) / (30 <em>ml</em>) ] x 100<br> [(0.9 %) (30 <em>ml</em>)] / 100  =  (X, <em>g</em>)<br> X  =  0.27 <em>g</em><br>   <ol start =  '1 ' style =  'list-style-type: lower-alpha; '> <li>Weigh out the appropriate mass of agarose powder, and then transfer the powder to an Erlenmeyer flask.</li> <li>Add the appropriate volume of 1X TBE buffer to the agarose powder. Swirl gently to mix. Take note of the volume of liquid in the flask.</li> <li>Heat the mixture in the microwave just until it boils (1–2 minutes). As soon as it boils, stop heating.</li> <li>Using a heat-resistant glove or mitt, <em>carefully</em> remove the flask from the microwave, remembering to face the open mouth of the flask away from yourself and others.</li> <li>Very gently, swirl to mix and then examine the solution for small transparent clumps. If clumps remain, return the flask to the microwave and continue heating until the mixture boils again. Repeat this process until no clumps are visible.<br> <span style =  ' ; '><strong>Important</strong></span>: Take care not to splash the hot liquid or touch the hot flask.</li> <li>Check the volume of the solution. If it has decreased, bring it back to the original volume using ddH<sub>2</sub>O. Swirl to mix.</li> <li>Allow the solution to cool to between 50 °C and 60 °C (very warm to the touch but not hot enough to burn). This typically takes 10–15 minutes.</li> </ol> </li> <li>Once the solution is sufficiently cool, put on latex gloves and add the gel dye of your choice. <ol start =  '1 ' style =  'list-style-type: lower-alpha; '> <li>If using ethidium bromide (EtBr) as a DNA dye, add enough to achieve a final concentration of 0.5 &micro;g/ml in the agarose solution. Swirl to mix.<br> <strong><span style =  ' ; '>Important</span></strong>: Use extreme caution when handling EtBr and other DNA dyes. They are mutagens that intercalate (insert) between nucleotide bases.<br> <br><strong>Example</strong></span>: You want 100-ml agarose with 0.5 &micro;g/ml EtBr and therefore need to use an unknown volume of 10 mg/ml EtBr stock. (Remember that 10 mg/ml  =  10 &micro;g/&micro;l). Using the C<sub>1</sub>V<sub>1 </sub> =  C<sub>2</sub>V<sub>2</sub> equation:<br> <br> (100 ml) (0.5 &micro;g/ml EtBr)  =  ( X ) (10 mg/ml)<br> (100)(0.5 &micro;g)  =  X (10 &micro;g/&micro;l)<br> X  =  5 &micro;l</li> </ol> </li> <li>Pour the agarose/EtBr mixture into the prepared gel apparatus, being careful not to introduce any bubbles. Insert the comb to cast the wells.</li> <li>Allow the gel to cool for 20–30 minutes.</li> <li>Once the gel has solidified, very carefully remove the comb by pulling it slowly straight up. Once the comb is removed, gently lift the gel platform out of the casting tray.</li> <li>Place the platform with the solidified gel into the gel box. The wells should be at the (-) cathode end of the box, where the black lead is connected.</li> <li>Pour 1X TBE buffer into the gel apparatus until your gel is submerged by ~ 1/4 inch of buffer.<br>  </li> </ol> </li> <li value =  '2 '>Load your samples and run the gel according to your instructor’s directions and the <a class = 'chap-linking' tabindex = '0' data-link-to = '10-5'><strong>Gel Electrophoresis </strong>(10.3) </a> protocol.</li> </ol><hr><br><h3>Helpful Tips:</h3><ul><li>If you want to achieve better resolution and separation of smaller fragments you can run up to a 2% agarose gel.</li><li>TAE running buffer can be substituted for TBE.</li><li>When microwaving agarose, use an Erlenmeyer flask or bottle that can accommodate at least 10 times the volume being heated to prevent it from boiling over.</li></ul>"},"10-5":{content:"<h2>Objective:</h2><p>  To separate DNA fragments via agarose gel electrophoresis</p><hr><br><h2>Rationale:</h2><p> DNA fragments can be separated by size using an agarose gel. When exposed to an electric current, the DNA molecules migrate through the sieve-like agarose gel because of their inherent negative charge. Smaller DNA fragments move more easily, and therefore more quickly, than larger DNA molecules through the meshwork of agarose polymers. After electrophoresis, the gel is illuminated using specific wavelengths of light to visualize the stained DNA and photographed. The sizes of DNA fragments, called bands, are calculated by comparing them to a DNA ladder containing DNA fragments of known size.</p><hr><br><h2>Supplies:</h2><ul> <li>Pre-poured agarose gel</li> <li>DNA loading dye</li> <li>Electrophoresis apparatus and power supply</li> <li>DNA cut with restriction enzymes</li> <li>DNA ladder<br>  </li> </ul> <hr><br><h2>Procedure:</h2><ol style = 'list-style-type:upper-alpha;'> <li>Set up the gel electrophoresis equipment with a gel prepared according to the <a class = 'chap-linking' tabindex = '0' data-link-to = '10-4'>Casting Agarose Gels(10.2)</a> protocol. <ol> <li>Wearing gloves, orient the gel in such a way that the wells are closest to the cathode (black electrode).<br>  </li> </ol> </li> <li>Prepare your restriction enzyme digest samples for electrophoresis. <ol> <li>Add 5 &micro;l of concentrated 6x loading dye to each 25 &micro;l restriction enzyme sample.</li> <li>Place samples at 65 °C, either in a heat block or a hot water bath, for 5 minutes. Immediately place the samples on ice to cool, and then spin them in a microcentrifuge for ~ 15 seconds at 10,000 rpm. <ol > <li>This step prevents annealing of the cohesive ends of phage DNA.<br>  </li> </ol> </li> </ol> </li> <li>Load the gel in the following order:<strong>    <br> Ladder     Uncut DNA     Enzyme-1     Enzyme-2      Enzyme-3      Enzyme-n       </strong> <ol> <li>Carefully load the gel with the proper volume of DNA ladder. Each manufacturer is different, so follow the manufacturer’s instructions.</li> <li>Using a fresh tip on your micropipettor for each sample, pipette 20 &micro;l  (or as much will fit in the well) of each RE reaction into the wells in the order suggested above. <ol> <li>Holding the pipette in both hands, place your elbows on either side of the gel apparatus.</li> <li>Situate your eyes directly above the wells to make the wells easier to see.</li> <li>Place the pipette tip directly above the well, just below the surface of the buffer. Do not try to get the pipette tip into the well, or you might puncture the bottom of the gel.</li> <li>Slowly depress the pipette plunger, allowing the solution to slowly sink into the well.</li> <li>Remove the pipette from the gel before releasing the plunger.</li> </ol> </li> <li >Draw a picture of the gel in your lab notebook, making note of where your samples are relative to your classmates’ samples. (Once the gels are loaded, everyone’s samples look alike!)</li> <li >Plug the electrodes into the appropriate locations on the power supply. Turn on the power supply and set the voltage to 100 V.<br> <span ><strong>Important</strong></span>: Remember, DNA runs toward the RED electrode!</li> <li >Run the gel until the blue dye front has migrated at least 3.5 inches from the well. This will take approximately 1 hour.</li> <li >Turn off the power supply.<br>  </li> </ol> </li> <li >Photograph the gel. <ol> <li>Using gloves, carefully remove the gel from the electrophoresis chamber.</li> <li>Photograph your gel and obtain a copy of the gel photograph, following your instructor’s directions.</li> <li>Label your copy of the gel photograph to include the names of each enzyme used above its corresponding lane. Below is an example of a properly labelled gel photograph (Figure 10.3-1).</li> <li>Include this labelled copy of the gel photograph in your laboratory notebook and upload a copy to phagesDB.  <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-10-5-1.png' alt = 'A photograph of a gel with D-N-A bands from the elecrophoresis apparatus. There are seven lanes marked 1-k-b ladder, uncut D-N-A, and 5 different enzymes. '/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 10.3-1. Example of a well-labelled restriction enzyme gel photograph, which includes information about the various enzymes used.</div></div> </li> </ol> </li> <li >Clean up your work area. <ol> <li>If another gel isn’t going to be run, empty the buffer into the sink, rinse out the gel apparatus, and set it aside to dry. (The buffer can be reused, so check to see if anyone else will need to run a gel in the next few days.)</li> <li>Dispose of your gel as directed by your instructor.</li> </ol> </li> </ol><hr><br><h3>Helpful Tips:</h3> <ul><li>Loading dyes are prepared at a higher concentration than used in the gel. If your loading dye is labeled as 6X, this means it is six times as concentrated as its working concentration (1X). </li> <li>If your samples float when you try to load the gel, they contain residual isopropanol. Consult with your instructor, who may suggest you add additional DNA loading dye. The excess glycerol should cause the samples to sink to the bottom of the well. </li> </ul>"},"10-6":{content:"<h2>Objective:</h2><p> To examine and interpret the results of your restriction enzyme digest</p><hr><br><h2>Rationale:</h2><p>DNA fragments of a phage genome cleaved by restriction enzymes can be separated by size using gel electrophoresis. The result is a series of DNA bands on the gel that reflect the sizes of each DNA fragment. The number of DNA fragments and the size of each fragment is a genetic “fingerprint” for your phage and can be used as a diagnostic tool for comparing phages.</p><hr><br><h2>Supplies:</h2><ul> <li>Photo of gel</li><li>Ruler</li><li>DNA ladder with band sizes</li></ul><hr><br><h2>Procedure:</h2><ol style = 'list-style-type:upper-alpha; '> <li>Perform a qualitative gel assessment. <ol> <li>Obtain a printout of your gel photo or access a digital image. <ol style = 'list-style-type:lower-alpha; '> <li>Label each lane according to its contents.</li><li>Using the “map” of the DNA ladder, label each ladder band with its size in base pairs.</li><li>Looking at each lane, and the gel as a whole, answer these questions: <ul  style = 'list-style-type: disc'> <li>Is there DNA in each lane? How do you know?</li><li>Did each restriction enzyme cut the phage DNA? How do you know?</li><li>Which restriction enzyme cut the most? Which cut the least?</li><li>If there is a problem with your gel, can you identify the cause?</li></ul> </li><li>Upload a labeled digital image of your gel to the profile for your phage that you created on phagesDB.<br> </li></ol> </li></ol> </li><li>Perform a quantitative gel assessment—calculate band size. <ol> <li>Using a ruler and a pen, draw a line at the top of the gel photo, above the wells or below the wells. The line will act as a reference for measuring the distance migrated by the DNA (Figure 10.4-1). <div class = 'img-wrap'> <div class = 'img-container'> <image src = './assets/images/figure-10-1-1.jpg' alt = 'A photograph of a gel with D-N-A bands from the elecrophoresis apparatus. There are seven lanes marked ladder, uncut D-N-A, and 5 different enzymes. Measurements of D-N-A length in base-pairs along the side of the picture range from 10,000 to 500. There is a horizintal line drawn across the top of the gel labelled line drawn through wells. There is a vertical two sided arrow from the horizontal line to the top of a D-N-A band in lane 6 labelled meaure from line to band.'/> </image> </div>  <div class = 'img-hr'></div> <div class = 'title-container'>Figure 10.4-1. Sample restriction enzyme gel. Agarose gel with lanes and ladder labeled. </div></div>' </li><li>Make a standard curve of the DNA ladder. <ol start = '1 ' style = 'list-style-type: lower-alpha; '> <li>Using Table 10.4-1 as a guide, make a table in your notebook or an Excel spreadsheet.</li><li>Fill in the values for known DNA fragments sizes of the ladder. <ol start = '1 ' style = 'list-style-type: lower-roman; '> <li>DNA ladders (also known as standards or markers) contain DNA fragments of known sizes. The fragment sizes are manufacturer-specific; therefore, it is important to use the proper map when assigning band sizes to your ladder.</li></ol> </li><li>Measure the distance migrated, in centimeters, of each band in the ladder. Start at the line you drew at the top of the gel and stop at the top of each band.</li><li>Record these data in the table or spreadsheet.</li><li>Create a semi-log plot of these data points to create a standard curve. The graph can be made by hand or by using Excel.<p></p> <caption> <p style = 'text-align: center; '><span ><strong>Table 10.4-1. Sample data table for creating DNA standard curve for agarose gel shown in Figure.</strong></span></p></caption><table align = 'center'> <thead><tr><th>Distance Traveled (mm)</th><th>DNA Ladder Fragment Size (bp)</th></tr></thead> <tbody> <tr> <td> <p>15</p></td><td> <p>10,000</p></td></tr><tr> <td> <p>17</p></td><td> <p>6,000</p></td></tr><tr> <td> <p>20</p></td><td> <p>3,000</p></td></tr><tr> <td> <p>23</p></td><td> <p>2,000</p></td></tr><tr> <td> <p>27</p></td><td> <p>1,500</p></td></tr><tr> <td> <p>31</p></td><td> <p>1,000</p></td></tr><tr> <td> <p>50</p></td><td> <p>500</p></td></tr></tbody></table><p  >.</span><br><strong>To create a graph by hand on semi-log graph paper:</strong></u></p><ol start = '1 ' style = 'list-style-type: lower-roman;  '> <li>Label the axes according to the example in Figure 10.4-2</li><ul> <li style = 'list-style-type: disc;  '> The y-axis is divided into three “cycles,” each with 10 sections representing logarithmic distances. Using a semi-log plot lets you visualize exponential data without calculating the logarithm.</li></ul></ol><ol start = '2 ' style = 'list-style-type: lower-roman;'> <li>Plot the data accordingly and create a “best fit” line.</li></ol><p > <strong> To create a graph using a spreadsheet in Excel:</strong></u></p><ol start = '1 ' style = 'list-style-type: lower-roman; '> <li>Select the data entered in your spreadsheet in the format shown in Table 10.4-1.</li><li>Insert a marked scatter plot.</li><li>Format the y-axis to use a logarithmic scale, add major and minor gridlines, and label the axes.</li><li>Right-click a data point to add a trend line, making sure it is exponential. Have the line equation displayed on the chart.</li></ol> </li>  </ol>    <li  >Estimate restriction fragment lengths by using the standard curve. For each restriction enzyme: <ol start = '1 ' style = 'list-style-type: lower-alpha; '> <li>Measure the distance (cm) migrated, from the line drawn through the wells to the top of each band.</li><li>Find this distance on the x-axis of your graph and move straight up until you intersect your standard curve. From that point, move left until you intersect the y-axis. This value is the size (bp) of your restriction fragment.</li><li>Make a table of these data in your notebook. <ol start = '1 ' style = 'list-style-type: lower-roman; '> <li>You can double-check your estimates by substituting the distance migrated (x) into the equation of the best-fit line and calculating the size in bp (y).</li></ol> </li></ol> </li><li >            Compare the results from your gel with those for other bacteriophage</li></ol>    </ol> </div> <div class = 'img-wrap'> <div class = 'img-container'> <image src = './assets/images/figure-10-6-1.jpg' alt = 'Example of a semi-log graph paper for D-N-A standard curve. Distance is on the x-axis measured in millimeters ranging from zero to 50 in 10 millimeter increments. D-N-A ladder fragment size in b-p is on the y-axis with values ranging from 100 to 100,000.'/> </image> </div>  <div class = 'img-hr'></div> <div class = 'title-container'>Figure 10.4-2. Example of semi-log graph paper for DNA standard curve. The x-axis is logarithmic, allowing exponential data to fit on the graph and be linearized without taking the logarithm. The y-axis is the distance migrated by the bands in the DNA ladder.</div></div></ol>         "},"11-1":{content:"<p>The microbial world is a dynamic place, constantly changing and evolving. Bacteria must compete for resources, such as food, oxygen, and light, and aggressively defend themselves against other microbes to be successful. Bacteria have many ways of rendering themselves insensitive to phage infection; consequently, bacteriophages must constantly evolve to remain competitive. This continual warfare drives the gene flow between bacteria and bacteriophages. Whereas some of the mechanisms by which bacteria can become sensitive or insensitive to phage infection are well understood (and are described below), many more remain to be discovered.</p> <h2>Mechanisms of Bacterial Insensitivity</h2> <p>Bacterial insensitivity to phage infection can be categorized, according to mechanism, as immunity, exclusion, or resistance. Bacteria can acquire insensitivity to infection if phage proteins from a previous phage infection are expressed. This insensitivity to infection is due to either <strong>immunity</strong> or <strong>exclusion</strong>, depending on the phage proteins expressed and how they act to prevent further infection.</p> <p>Bacteria can also acquire insensitivity to phage infection because of DNA mutations that cause changes in the bacterial receptor that is recognized by a phage for infection. In addition, bacteria may possess phage defense systems such as restriction endonucleases or CRISPR arrays (discussed in Chapter 1) that protect them from phage infection. Bacterial insensitivity resulting from these kinds of defense systems, including mutation, is called resistance. Immunity, exclusion, and resistance are described in more detail below.</p> <h2>Lysogeny</h2> <p>A basic characteristic of a phage is its life cycle. The phages isolated in the SEA-PHAGES program are all capable of forming a plaque, which means they can undergo the lytic cycle. However, some phages that form plaques have an alternate life cycle option; that is, after infection this type of phage represses the lytic functions of its genes and has its genome stably maintained within the bacterial cell throughout multiple rounds of cell division. This latent form of phage is called a prophage. <strong>Lysogeny</strong> occurs when the phage genome is integrated into the bacterial genome through homologous recombination or through the circularization of the phage genome and its maintenance as a large plasmid in the bacterial host. Bacterial cells that carry the phage genome are called <strong>lysogens</strong> and phages that form them are said to be <strong>temperate</strong>.</p> <h2>Repressors</h2> <p>During lysogen growth the expression of genes involved in the lytic cycle is repressed. In many actinobacteriophages this occurs through the constant expression of a protein known as an immunity repressor. This repressor protein binds to specific sequences on the prophage genome at sites called <strong>operators</strong> to block transcription initiation or at sites called <strong>stoperators</strong> to prevent protein elongation. This means that the lysogen contains excess immunity repressor proteins that diffuse freely through the bacterium during its normal life cycle. Should another phage infect the lysogen (a process called <strong>superinfection</strong>), the immunity repressor proteins can bind to the new invading phage DNA if that genome contains the same specific operator and stoperator sequences. In this way, immunity repressor proteins offer the <strong>lysogen</strong> immunity to superinfection from phages identical or closely related to the prophage.</p> <p>Immunity repressors are highly specific for their operator and stoperator sequences. Even a single base change in the operator/stoperator sequence can alter or prevent binding to the target DNA. Therefore, phages that cannot infect a lysogen, because of actions of the immunity repressor, share at least some genomic similarity to the prophage’s operator/stoperator sequences. Frequently, the inability of two different actinobacteriophages to infect each other’s lysogens, called <strong>homoimmunity</strong>, is a property shared by members of the same subcluster. It is important to note that lysogens are only considered “immune” to superinfection if the immunity repressor is the mechanism by which infection is prevented.</p> <h2>Exclusion</h2> <p>In addition to the immunity repressor protein, some phages express proteins that can alter the surface of the bacterial cell. When these proteins or carbohydrates appear on the bacterial cell surface, the prophage can prevent other phages from binding to the cell and injecting their DNA. This ability is called surface <strong>exclusion</strong>. Unlike immunity, which is generally restricted to genomically related phages, exclusion depends on interactions between the tail fiber(s) of the infecting phage, which is the primary means of attachment to the cell, and the receptor on the surface of the lysogen. Tail fibers are highly mosaic and modular, and they may or may not be shared among phages that are otherwise very similar at the genomic level.</p>  <h2>Other Factors</h2> <p>Finally, it is important to note that not that every phage infection of a bacterial population will yield intrinsically insensitive host cells. There have been cases of transient insensitivity owing to the stage of a bacterial cell cycle as well as permanent resistance caused by mutation. A liquid culture of bacteria in stationary phase likely has 108–109 cells per milliliter! A mutant present in the culture at only a few tenths of a percent will rapidly become the dominant strain when the bulk of the cells in the culture are destroyed by phage infection.</p> <p>So how do you tell the difference between a true lysogen and a naturally resistant bacterial cell in the lab? Lysogens will be insensitive to superinfection from all subsequent attempts at infection with the initial phage—no matter how many times the lysogen is streak-purified or regrown from a liquid stock. Also, liquid cultures of lysogens will release phage into the supernatant of the culture. This is because some of the cells in the population are undergoing spontaneous induction of the lytic cycle at all times. And finally, the gold standard for lysogen confirmation is to demonstrate that the prophage is in the cell. This can be done using a technique called polymerase chain reaction (PCR) and amplifying the new junction between the bacterial and phage genomes. Or confirmation can be done by carefully repeating rounds of purification and then sequencing of all of the DNA in the cell to look for the presence of the prophage genome.</p> <p>It is possible to isolate lysogens by carefully purifying them away from exogenous infecting phage. Once isolated, lysogens can be used to create bacterial lawns to test the ability of other phages to infect the new lysogen. This immunity data can be helpful in characterizing a whole collection of sequenced or unsequenced phages. It is likely that temperate phages carry a variety of genes that are expressed in lysogens and influence phage sensitivity. Therefore, exploring these patterns can provide valuable insights into phage mechanisms of bacterial sensitivity and insensitivity to bacteriophage.</p> "},"11-2":{content:"<h2>Objective:</h2> <p>To create a lysogen with your phage</p> <hr><br><h2>Rationale:</h2> <p> The phages isolated in the SEA-PHAGES program are all capable of forming plaques, which means they can undergo the lytic cycle. However, some phages that can form plaques are temperate phage; that is, after infecting a host bacterium this type of phage can repress the lytic functions of its genes, with the result that its genome is stably maintained within the host cell throughout multiple rounds of cell division. The host bacterium that contains a stable copy of a phage genome is called a lysogen. Lysogens are isolated by carefully purifying them away from exogenous infecting phage. Once isolated, lysogens can be used as host bacterial lawns to test, using a sensitivity assay, the ability of other phages to infect the new lysogen.</p> <hr><br><h2>Supplies:</h2> <ul><li>Host bacterial culture (250 &micro;l)</li><li>Agar plate</li><li>Top agar</li><li>Phage buffer</li><li>Microcentrifuge tubes</li></ul> <hr><br><h2>Procedure:</h2> <h3>Day 1</h3> <ol style = 'list-style-type:upper-alpha;'><li>Prepare your bench for aseptic work and assemble your supplies.</li><li>Create a spot plate of your lysate.<ol><li>Serially dilute your lysate to 10<sub>-8</sub> according to the <a class = 'chap-linking' tabindex = '0' data-link-to = '6-4'> Serial Dilutions (6.2) </a> protocol.</li><li>Make a spot plate according the <a class = 'chap-linking' tabindex = '0' data-link-to = '6-6'>Spot Titer (6.4)</a> protocol.</li></ol></li></ol> <h3>Day 2</h3><ol style = 'list-style-type:upper-alpha;'><li>Check plates each day for a “mesa.”<ol><li>A mesa is an overgrowth of bacterial cells surrounded by the zone of clearing caused by phage infection and lysis. It will look like a cloudy center in of one of your spots. A mesa is caused by bacterial cell growth in the presence of phage and therefore is an excellent source of lysogens.</li><ol  style = 'list-style-type:lower-alpha;'><li>A mesa will not likely appear on day one. A true mesa is generally as well grown as the unspotted lawn by day two or three (Figure 11.1-1).</li></ol></li></ol>  <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-11-1-1.png' alt = 'A picture of four rows labeled phages A through D. Phage A shows five large dark spots. Phage B an C show three large dark spots and one very small one. And phage D shows two large dark spots and a smaller one.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 11.1-1. This picture depicts four phages spotted at 10-fold dilutions (100 -> 10-4). A mesa is recognizable for phages A, B, and C. However, phage D may also have produced lysogens. Continue with the protocol regardless of whether there is visible bacterial growth in the spot. Note that phage A has mesas to at least the 10-3 spot, while phages B and C have visible mesas to 10-2. Warning: do not plate multiple phages for lysogen isolation on a single plate as depicted in the photo; you could select double-lysogens in the process!</div></div> <li>  Streak cells from the mesa onto a new plate.<ol><li>Label the bottom of an agar plate.</li><li>Using a sterile stick, pipette tip, or sterile loop, touch the mesa growing in the spot.<ol style = 'list-style-type:lower-alpha;' ><li>If you are using a wire loop, sterilize it by heating it in a flame until red hot and then allowing it to cool before picking up the bacteria.</li></ol></li><li>Touch the stick or loop to the agar and streak back and forth across the top one-third of the agar. Discard the stick or reflame the loop.<ol style = 'list-style-type:lower-alpha;'><li>The objective of this step is to spread the bacteria out as much as possible.</li></ol></li><li>With a new wooden stick (or sterile loop), drag through the first streak once and continue to streak back and forth in an adjacent one-third of the agar, making sure not to overlap the first streak again.<ol style = 'list-style-type:lower-alpha;'><li>This second streak allows you to spread just a fraction of the bacteria from the first streak, thereby diluting it.</li></ol></li><li>Repeat step 4 to create a third streak on the remaining one-third portion of the plate.<ol style = 'list-style-type:lower-alpha;'><li>This third streak further dilutes the bacteria sample. Hopefully, you will have individual bacterial cells in this third streak that will grow into isolated colonies.</li></ol></li></li><li>Incubate at the appropriate temperature for 2–4 days.</li></ol></ol>    </li>       <h3>Day 3</h3> <ol style = 'list-style-type:upper-alpha;'><li>Test and confirm putative lysogens.<ol><li>Choose 6–10 colonies to test for lysogeny. Not all of them will be lysogens. Test for lysogeny via <a class = 'chap-linking' tabindex = '0' data-link-to = '11-3'> Patch Assay (11.2)</a> and <a class = 'chap-linking' tabindex = '0' data-link-to = '11-4'>Liquid Phage Release (11.3)</a>, or PCR amplify the attachment sites in the lysogen.</li></ol></li></ol> <hr><br><h3>Helpful Tips:</h3><ul><li>When making a lysogen, you may need three rounds of streak purification to remove exogenous phage from your initial pick from the mesa. Each round requires approximately 4 days of incubation.</li><li>Remember, you will be streaking cells, not phage, as is discussed in the purification protocol. A plate streaked with phage and nothing else will look completely blank no matter how long you incubate it for.</li><li>Never plate for mesas with more than one phage per plate during isolation. You run the risk of creating double lysogens if phage spots are side by side.</li><li>Try to pick up cells from the top of the colony. Do your best to avoid scraping the agar surface around your colony to avoid picking up additional exogenous phage.</li></ul>    "},"11-3":{content:"<h2>Objective:</h2> <p>To identify lysogens by patching putative lysogens onto a layer of host cells and incubating to observe clearing in the lawn caused by lysis</p> <hr><br><h2>Rationale:</h2> <p>All lysogen cultures release phages into the environment as the individual cells spontaneously undergo the lytic cycle. Thus, it is possible to differentiate true lysogens from insensitive bacterial cells by testing for the presence of phage in the cells. This method tests for phage release by patching putative lysogens onto a host-seeded plate, incubating them until cell growth is visible, and observing signs of cell lysis.</p> <hr><br><h2>Supplies:</h2><ul><li>Plates with bacterial colonies that are potential lysogens</li><li>Agar plates</li><li>Top agar</li><li>Host bacterial culture (250 ml)</li><li>Sterile sticks or wire loops for bacterial streaking</li></ul><hr><br><h2>Procedure:</h2> <h3>Day 1</h3><ol  style = 'list-style-type:upper-alpha;'><li>Prepare your bench for aseptic work and assemble your supplies.</li><li>Prepare two plates for this procedure.<ol><li>Draw identical grids on the bottom of two agar plates and number the squares 1–10 (or however many putative lysogen colonies you are testing).</li><li>Label both plates. One plate is your experimental plate on which you will observe lysis around lysogens on a host cell lawn after incubation; the other plate is your lysogen-only plate, which will contain your patched lysogens.<ol style = 'list-style-type:lower-alpha;'><li>You will pick confirmed lysogens from the lysogen-only plate for future experiments.</li></ol></li><li>Obtain a 250 &micro;l culture of host bacteria.</li><li>For this part of the experiment you will need 3 ml of molten top agar. Your instructor may provide this for you or you may need to make it according to protocol found in the <a class = 'chap-linking' tabindex = '0' data-link-to = '12'>Toolbox </a>.</li><li>Using a sterile 5 ml pipette, transfer 3 ml of molten top agar to a culture tube containing host bacteria and then immediately draw the solution back into the same pipette.<ol style = 'list-style-type:lower-alpha;'><li>Dispense the top agar-bacteria mixture onto the experimental plate ONLY.</li><li>Gently, but quickly, tilt the plate in multiple directions until the top agar mixture evenly coats the agar plate.</li><li>Do not prepare a lawn on the lysogen-only plate.</li></ol></li><li>Let the top agar set and dry completely. This should take approximately 15 minutes.</li></ol></li><li>Make your patch plates.  <ol><li>Pick a colony from the lysogen streak plate, made according to the protocol <a class = 'chap-linking' tabindex = '0' data-link-to = '11-2'>Creating and Testing Lysogens , Streak Method (11.1)</a>, by lightly touching a sterile loop or sterile toothpick to the center of an isolated colony.<ol style = 'list-style-type:lower-alpha;'><li>If you are using a wire loop, sterilize it by heating it in a flame until red hot and then allowing it to cool before picking up the bacteria.</li></ol></li><li>Streak a small quantity of candidate lysogen cells to each plate by dragging the loop or toothpick gently across the surface of the lysogen-only plate first, followed by the experimental plate (the plate with the prepared bacterial lawn), as shown in Figure  11.2-1.<ol style = 'list-style-type:lower-alpha;'><li>Do not dig into the agar surface of the places. The surface of the lysogen-only plate is far more firm that that of the experimental plate with the prepared host cell lawn.</li><li>You may get better results on the prepared lawn by lightly touching the top agar with the loop or toothpick in a series of 6–8 adjacent “taps” instead of dragging it through the top agar. </li></ol></li></ol></li>    </ol> <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-11-3-1.jpg' alt = 'A photograph of two plates. One plate is transparent and labeled lysogen only  with 5 white lines of varying thickness labeled A through E. The other is an opaque plate labeled experimental plate and has 5 transparent lines of varying thickness labeled A through E.'/></image></div> <div class = 'img-hr'></div> <div role = 'figure' class = 'title-container'>Figure 11.2-1. Patch plates to test putative lysogens. The lysogen-only plate is a plain agar plate with no host bacterial lawn and patched with five putative lysogen colonies. The experimental plate has the same bacterial colonies patched onto a wild-type host bacterial lawn. Patches A, B, and E test positive as lysogens because of clearing from phage release. Patches C and D do not show clearing caused by phage release, but only removal of the lawn from the patching process.</div></div> <p color:red><strong>Important: </strong>Use the same loop or toothpick for both plates, without getting a new one, because you want the same colony to grow on both plates.</p> <h3>Day 2</h3><ol   style = 'list-style-type:upper-alpha;'><li>Identify lysogens.<ol><li>Look for clearings in the lawn caused by cell lysis around the patches on the experimental plate (Figure 11.2-1). These are candidate lysogens.</li></ol></li><li>Streak each positive lysogen candidate from the lysogen-only plate onto an agar plate to isolate single colonies.<ol><li>Label the bottom of an agar plate. <ol style = 'list-style-type:lower-alpha;'><li>Indicate if it is your first, second, or third round of lysogen purification.</li></ol></li><li>Using a sterile stick, pipette tip, or sterile loop, touch the center of the lysogen colony you are purifying from the lysogen-only plate.<ol><li>If you are using a wire loop, sterilize it by heating it in a flame until red hot and then allowing it to cool before picking up the bacteria.</li></ol></li><li>Touch the stick or loop to the agar and streak back and forth across the top one-third of the agar. Discard the stick or reflame the loop.<ol      style = 'list-style-type:lower-alpha;'   ><li>The objective of this step is to spread the bacteria out as much as possible.</li></ol></li><li>With a new wooden stick (or sterile loop), drag through the first streak once and continue to streak back and forth in an adjacent one-third of the agar, making sure not to overlap the first streak again.<ol style = 'list-style-type:lower-alpha;'><li>This second streak allows you to spread just a fraction of the bacteria from the first streak, thereby diluting it.</li></ol></li><li>Repeat step 4 to create a third streak on the remaining one-third portion of the plate.<ol style = 'list-style-type:lower-alpha;'><li>This third streak further dilutes the bacteria sample. Hopefully, you will have individual bacterial cells in this third streak that will grow into isolated colonies.</li></ol></li><li>Incubate at the appropriate temperature for 2–4 days (until single colonies are visible).</li></ol>  </li><li>Repeat the process of picking isolated colonies from the streak plates and streaking to isolation three consecutive times to successfully purify a lysogen.</li><li>Perform the patch test again on your new lysogen candidates (taken from single colonies from your third round of purification) to confirm that they are indeed lysogens.<ol style = 'list-style-type:lower-alpha;'><li>Unlike the first time you performed the patch test, these cells will have no exogenous phage from the initial high-titer mesa spots associated with the lysogens. </li></ol></li><li>You are ready to inoculate a liquid culture for testing using the protocols for <a class = 'chap-linking' tabindex = '0' data-link-to = '11-4'>Liquid Phage Release (11.3) </a> and <a class = 'chap-linking' tabindex = '0' data-link-to = '11-5'>Sensitivity Assay (11.4)</a>.<ol style = 'list-style-type:lower-alpha;'><li>An aliquot(s) of any and all lysogens that are confirmed should be frozen for future use.</li></ol></li></ol> <hr><br><h3>Helpful Tips:</h3> <ul><li>It is very important that potential lysogens are purified away from exogenous phage that may have stuck to the cells during the initial mesa creation. Otherwise, you will never be able to differentiate between a lysogen releasing new phage via spontaneous induction and a mutant resistant cell line that has been coated in a large amount of the initial phage.</li><li>Make sure that the top agar is completely set before you make your patches.</li></ul>"},"11-4":{content:"<h2>Objective:</h2> <p>To determine if a putative lysogen is releasing phage into a liquid culture</p> <hr><br><h2>Rationale:</h2> <p>In a lysogen culture phages are constantly being released because of spontaneous induction of the lytic cycle. A liquid culture of a lysogen will have a fairly high titer of phage in the supernatant that can be detected via spot titering on a wild-type host lawn.</p> <hr><br><h2>Supplies:</h2> <ul><li>Purified streak plate of lysogen candidate</li><li>Liquid bacterial growth media</li><li>Culture tubes</li><li>Top agar</li><li>Agar plate</li><li>Microcentrifuge tubes</li></ul> <hr><br><h2>Procedure:</h2> <p><h3>Day 1</h3></p> <ol style = 'list-style-type:upper-alpha;'><li>Prepare your bench for aseptic work and assemble your supplies.</li><li>Make a liquid culture of a potential lysogen. <ol><li>Using a sterile stick, pipette tip, or sterile loop, pick a single colony of your potential lysogen from a streak purification plate prepared according to the protocol for <a class = 'chap-linking' tabindex = '0' data-link-to = '11-3'>Testing and Verification of Potential Lysogens: Patch Assay (11.2).</a></li><li>Inoculate 3 ml of appropriate medium (in a culture tube) with the picked colony by swishing the stick, tip, or loop in the media.<ol style = 'list-style-type:lower-alpha;'><li>Incubate the tube by shaking it at 220 rpm for 2–4 days.</li><li>This culture can also be used to prepare sensitivity assays and to create a freezer stock of your lysogen.</li></ol></li></li>   </ol></ol> <h3>Day 2</h3><ol style = 'list-style-type:upper-alpha;'><li>Prepare a lawn with wild-type host cells and top agar according to the <a class = 'chap-linking' tabindex = '0' data-link-to = '5-8'>Spot Test (5.6)</a> or <a class = 'chap-linking' tabindex = '0' data-link-to = '6-6'>Spot Titer (6.4)</a> protocols.</li><li>Prepare the supernatant of the lysogen liquid culture.  <ol><li>Pipette 500 &micro;l of your potential liquid lysogen culture into a 1.5 ml microcentrifuge tube.</li><li>Spin at top speed in a microcentrifuge for 1 minute to pellet the bacterial cells.</li><li>Remove 100 &micro;l of the supernatant, and transfer it to a new 1.5 ml microcentrifuge tube.</li><li>Make 10-fold serial dilutions of the supernatant to 10-8 in phage buffer according to the <a class = 'chap-linking' tabindex = '0' data-link-to = '6-4'>Serial Dilutions (6.2) </a> protocol.</li><li>Pipette 3 &micro;l spots of each dilution onto the prepared lawn and let them sit until they are absorbed.</li><li>Incubate at the appropriate growth temperature for 24–48 hours.</li></ol></li>   </ol> <h3>Day 3</h3> <ol style = 'list-style-type:upper-alpha;'><li>Observe the spot plates for plaques.<ol><li>If individual plaques appear in the higher dilution spots (i.e., most dilute samples) you may have a stable lysogen in your initial liquid culture.</li><li>If no plaques appear, this is most likely NOT a lysogen.</li> </ol></li><li>You can continue to investigate your lysogen by using it to perform a <a class = 'chap-linking' tabindex = '0' data-link-to = '11-5'>Sensitivity Assay (11.4)</a> with other phages.</li></ol>  <hr><br><h3>Helpful Tips:</h3>  <ul><li>Liquid cultures started from single colonies may experience clumping as seen with wild-type M. smegmatis mc2155. To avoid this you may want to use the detergent Tween in your starting liquid culture media. If Tween is used, you must passage this first culture into a Tween-free media before testing for phage sensitivity because the presence of Tween in the culture can inhibit phage attachment to the host.</li><li>When you create a freezer stock of your lysogen from a liquid culture, remember to test and save the same liquid culture. Do not assume that different colonies from the same plate are identical.</li><li>Be careful to not confuse “killing from without” with phage infection. Phage infection will yield plaques at low lysate concentrations. “Killing from without” will produce a zone of clearing that dilutes out and will not produce plaques. This is because phages that can kill a bacterial cell from without are not undergoing a normal lytic cycle. Killing from without occurs when cells are exposed to a high concentration of phage and die without the phage propagating normally.</li></ul>    "},"11-5":{content:"<h2>Objective:</h2><p>To test the ability of a phage or panel of phages to infect a confirmed lysogen</p> <hr><br><h2>Rationale:</h2><p>Sensitivity assays are spot titers performed on a confirmed lysogen lawn to test those cells’ sensitivity to phage infection. Immunity to superinfection is defined as the ability of a lysogen to resist infection from a bacteriophage via its immunity repressor. Although the results will not tell us definitively the mechanism behind cellular insensitivity to phage infection, it is a good first line of inquiry.</p> <hr><br><h2>Supplies:</h2> <ul> <li>Liquid cultures of potential lysogens</li> <li>Liquid culture of wild-type host cells</li> <li>Lysates of phages to test on the lysogens</li> <li>Agar plates</li> <li>Phage buffer</li> <li>Top agar</li> </ul> <hr><br><h2>Procedure:</h2> <h3>Day 1</h3> <ol style = 'list-style-type:upper-alpha;'> <li>Prepare your bench for aseptic work and assemble your supplies. <ol> <li>You will need a liquid lysogen culture prepared according to the <a class = 'chap-linking' tabindex = '0' data-link-to = '11-4'>Testing and Verification of Potential Lysogens: Liquid Phage Release (11.3) </a> protocol.</li> <li>You will also need a liquid wild-type bacterial host culture.</li> </ol></li> <li>Prepare bacterial lawns of each bacteria to be tested, including all potential lysogens and wild-type host cells, according to the <a class = 'chap-linking' tabindex = '0' data-link-to = '5-8'>Spot Test (5.6)</a> or <a class = 'chap-linking' tabindex = '0' data-link-to = '6-6'>Spot Titer (6.4)</a> protocols.<ol> <li>Label the plates, including the cell type (wild type, lysogen) and name(s) of the phage(s) being tested.</li> <li>Let plates solidify.</li> </ol> </li> <li>Perform the 10-fold <a class = 'chap-linking' tabindex = '0' data-link-to = '6-4'>Serial Dilutions (6.2) </a>protocol on the phage lysates you wish to assay for their ability to superinfect the lysogen. <ol> <li>Lysates should be diluted to 10-8 in phage buffer.</li> <li>Record the titers of the undiluted lysates in your notebook.</li> </ol></li> <li>Spot dilutions and controls on each prepared bacterial lawn.  <ol> <li>One at a time, aseptically transfer 3 &micro;l of each lysate dilution onto the prepared bacterial lawns. <ol style = 'list-style-type:lower-alpha;'> <li> Dilutions of up to four lysates can be spotted on a single plate if you pipette carefully.</li> <li>Always use new tips for each spot. You do not want to contaminate the lysate dilutions with the various cells onto which you are plating.</li> </ol> <li>Use 3 &micro;l of sterile phage buffer as the negative control.</li> <li>Allow the liquid from the spots to dry completely.</li> <li>Carefully invert the plates and incubate at the proper temperature for 24-48 hours.<ol style = 'list-style-type:lower-alpha;'> <li>To prevent the spots from running, you can postpone inversion until after the first 24 hours have passed.</li> </ol></li>             </li> </li>        </ol> </ol> <h3>Day 2</h3> <ol style = 'list-style-type:upper-alpha;'> <li>Check for evidence of insensitivity. <ol> <li>Check for spots and individual plaques after 24 and 48 hours. <ol style = 'list-style-type:lower-alpha;'> <li>Do you have individual plaques in your higher dilutions (i.e., most dilute samples)? See Figure 11.4-1 for an example.<ol style = 'list-style-type:lower-roman;'> <li>Calculate the titers of phage in the lysate able to infect the lysogen and the wild-type host according to the <a class = 'chap-linking' tabindex = '0' data-link-to = '6-6'> Spot Titer (6.4) </a>protocol.</li> </ol></li>  <li>If you have spots only on samples of lower dilutions (i.e., the most concentrated samples), you are probably witnessing “killing from without.” This is a phenomenon by which a high-titer lysate can cause cell lysis when it is spotted on a lawn. However, spots of more dilute samples do not produce individual plaques. Killing from without is not indicative of immunity.</li> </ol></li>       <li>Photograph your results.</li> </ol></li> <li>Interpret your immunity assay results. <ol> <li>Calculate plating efficiency.  <ol style = 'list-style-type:lower-alpha;'> <li>Calculate the efficiency according to the formula: Titer of phage on lysogen / titer pf phage on wild type host  =  plating efficiency</li> <li>The simplest plating efficiencies to interpret are either 0, because the phage did not infect the lysogen, or 1, because the phage infected both strains of bacteria equally. <ol style = 'list-style-type:lower-roman;'> <li>Some plating efficiencies are more difficult to interpret. For example, some phage lysates will appear to have a substantially lower titer when plated on your lysogen than when plated wild-type cells. This is called reduced plating efficiency. This can be due to a variety of reasons, including surface exclusion or restriction.</li> </ol></li> </li>            </ol> <li>What can you determine about your phage from your results? <ol style = 'list-style-type:lower-alpha;'> <li>Do you see equivalent infection on your control and experimental plates for your initial phage, or is your phage unable to infect its own lysogen, as in Figure 11.4-1? </li> </ol> <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-11-5-1.jpg' alt = 'A picture showing portions of two different plates next to each other.  Eight phages are labeled by row along the side of the photograph from top to bottom: baka, courthouse, little E, omega, optimus, thibault, jasper, and rosebush. The plate on the left is labeled M. smegmatis wild type and has varying numbers of dark spots in each of the phage rows. The plate on the right is labeled M. smegmatis courthouse lysogen and has several dark spots only in the phage rows thibault and rosebush.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 11.4-1. Sensitivity assay example. Portions of two plates are shown. The left plate has a lawn of wild-type M. smegmatis mc2155. The right plate has a lawn of the M. smegmatis mc2155 lysogen of the Cluster J phage Courthouse. Eight phages were serially diluted and plated on both lawns, and their names are listed on the left. The top six phages are members of Cluster J (Baka through Thibault), Jasper is a member of cluster A1, and Rosebush is a member of Cluster B2. The Courthouse lysogen is insensitive to infection from the Cluster J phages, most likely through immunity and the action of the Courthouse immunity repressor expressed within the lysogen. Some “killing from without” is visible in the Thibault row. Phage Jasper is also unable to infect the lysogen, most likely due to the presence of the A1 immunity repressor gene that Courthouse also carries in its genome (in addition to its regular Cluster J one). Finally, Rosebush is also unable to infect the lysogen, most likely due to exclusion or restriction, as Rosebush does not appear to have any obvious integration machinery or immunity repressor.</div></div></li>  <li>What can you learn from patterns of infection with other phages? <ol style = 'list-style-type:lower-alpha;'> <li> Examining the genomes of sequenced phages may lead you to some interesting genome features. However, do not limit yourself to testing only sequenced phages because you may learn something about the character of unsequenced phages, based on their ability to infect a panel of known lysogens. </li> </ol></li> </li>     </ol> </ol> <hr><br><h3>Helpful Tips:</h3> <ul> <li>Be careful to not confuse “killing from without” with phage infection. Phage infection will yield plaques at low lysate concentrations. “Killing from without” will produce a zone of clearing that dilutes out and will not produce plaques, as phages that can kill bacteria from without are not undergoing a normal lytic cycle. Killing from without occurs when cells are exposed to a high concentration of phage and die without the phage propagating normally. A good analogy of cell death from phage “killing from without” is dropping a water balloon on a box of needles. You will not make any more needles, but you will definitely break your water balloon.</li> <li>A lysogen will not be sensitive to infection by the same phage that created it.</li> <li>No immunity assay is valid unless a wild-type control is included in the assay. Why?</li> <li>If an attP/attB site is identified, design PCR primers to amplify across the attL or attR sites of the integrated prophage. Design control primers to amplify only across the attB site. The attB primers should work on a non-lysogen, but will be too far apart to work on an intact lysogen.</li> </ul>"},"11-6":{content:"<h2>Objective:</h2> <p>To test the ability of a phage to infect an alternative host from which it was isolated</p> <hr><br><h2>Rationale:</h2> <p>Host range assays are spot titers performed to test the ability and efficiency of a given phage to infect bacteria different from the isolation host.  Results are reported as a ratio that relates how well the phage infects the test bacteria compared to the isolation host, called Efficiency of Plating (EOP).</p> <hr><br><h2>Supplies:</h2> <ul><li>Liquid cultures of test bacteria</li><li>Liquid culture of isolation host</li><li>Phage lysates with a titer of > 5 x 109 pfu/ml</li><li>Agar plates</li><li>Phage buffer</li></ul> <hr><br><h2>Procedure:</h2> <h3>Day 1</h3> <ol style = 'list-style-type:upper-alpha;'><li>Prepare your bench for aseptic work and assemble your supplies.<ol><li> You will need a liquid culture for each bacterial species that you plan to test for host range of your phage. You will also need a liquid culture of the bacterial host on which you initially isolated your phage.</li><li>You will need an aliquot of phage lysate for each phage you wish to test, with a minimum titer of >5 x 109 pfu/ml.</li></ol></li><li>Prepare a bacterial lawn for each bacteria to be tested, including the original isolation host, according to the <a class = 'chap-linking' tabindex = '0' data-link-to = '6-6'>Spot Titer (6.4) </a>protocol.<ol><li>Label the bottom of each agar plate your name, the date, and the bacterial species. The configuration of how you will spot the dilutions of you phage(s) should be determined now, labeling accordingly.  See Figures 6.4-1 and 11.4-1 for examples.</li><li>Once your bacterial lawn has been prepared, allow plates dry for at least 20 minutes.</li></ol></li><li>Perform serial dilutions on the phage lysates you wish to assay for their ability to infect alternate hosts.<ol><li>Prepare ten-fold dilutions of phage lysate(s) according to <a class = 'chap-linking' tabindex = '0' data-link-to = '6-4'><strong>Serial Dilution</strong> (6.2)</a> protocol.</li> </ol></li><li>Spot dilutions on each prepared bacterial lawn.<ol><li>One at a time, aseptically transfer 3 &micro;l of each sample dilution onto the prepared bacterial lawns.<ol style = 'list-style-type:lower-alpha;'><li>Always use new tips for each spot to avoid contamination.</li></ol></li><li>Use 3 &micro;l of sterile phage buffer as the negative control.</li><li>Allow the liquid from the spots to dry completely.</li><li>Carefully invert the plates and incubate at the proper temperature for 24-48 hours.</li></ol></li></ol> <h3>Day 2</h3><ol style = 'list-style-type:upper-alpha;'><li>Check for evidence of phage infection. <ol><li>After 24 and 48 hours, check each spot on the agar plates. <ol style = 'list-style-type:lower-alpha;'><li>Do you have individual plaques at some spots? The ability to observe individual plaques will allow you to calculate a titer.</li><li>*If you have spots of lysis but no observable individual plaques for any dilutions, this will require further investigation and involve repeating the experiment using full plates (i.e. <a class = 'chap-linking' tabindex = '0' data-link-to = '6-3'>Plaque Assay (6.1)</a> instead of spot tests).</li></ol></li> <li>Calculate the titer, in pfu/ml, of phages able to infect each host tested, including the isolation host, according to the <a class = 'chap-linking' tabindex = '0' data-link-to = '6-6'> Spot Titer (6.4) </a>protocol.</li><li>Photograph your results.  Carefully record your observations.</li></ol></li>      <li>Interpret your host range assay results.<ol><li>Do you see equivalent infection on your isolation host and the test bacteria for your phage? Is it reduced or is your phage unable to infect the experimental host bacteria?</li><li>Calculate Efficiency of Plating (EOP).<ol style = 'list-style-type:lower-alpha;'><li>For each host and phage tested, calculate the EOP according to the formula:(Titer of phage on test species) / (Titer of phage on isolation host)  =  EOP </li><li>The simplest interpretation of results are that the phage infected both types of bacteria equally, reported as 1, or the phage did not infect the test host, reported as <[1/number of phage particles plated]**.</li><li>Record EOPs and interpretations in your notebook.</li></ol></li></li></ol></ol> <hr><br><h3>Helpful Tips:</h3> <ul><li>*Be careful to not confuse “killing from without” with phage infection. Phage infection will yield plaques at low lysate concentrations. “Killing from without” will produce a zone of clearing at high lysate concentrations with no observable plaques as the lysate is diluted. Killing from without occurs when cells are exposed to a high concentration of phage and die without the phage propagating normally.</li><li>**Do not report EOP as “0”, instead report the EOP as <[1/highest number of phage particles plated]. If you calculated your phage lysate titer as 5 x109 pfu/ml from plaques observed on the isolation host and you spotted 10 &micro;l of the undiluted sample, the highest number of phage particles you spotted is 5 x 107 particles. You observed less than 1 plaque on the test bacterial species, and should record your results as < 1/(5 x 107).</li><li>No host range assay is valid unless the isolation host is included in the assay. It is also not valid if the titer of the phage is low.</li><li>It is not informative to report the results as +/-. </li><li>If you are testing new hosts where no phages have been found, it may be difficult to optimize the conditions for this experiment.  Consider phage-hunting on those hosts!</li></ul>"},"12-1":{content:"<p> This chapter contains a collection of protocols that apply to a broad range of experiments. Some, like Plaque Streak Plates are alternative protocols that you may not use, while others, like Making Top Agar, will be used often. </p>"},"12-2":{content:"<h2>Objective:</h2> <p> To make molten agar/media mixture for growing bacterial lawns on agar plates</p> <hr><br><h2>Rationale:</h2> <p>Mixing a small amount of bacteria with molten top agar and spreading it over the surface of an agar plate results in a smooth lawn perfect for viewing bacteriophage plaques. When working with M. smegmatis, top agar is made by mixing 2X Middlebrook Top Agar with an equal volume of 7H9 liquid media.</p> <hr><br><h2>Supplies:</h2> <ul> <li>100mM CaCl2 (sterile) stock (1 ml)</li><li>7H9 liquid media neat (50 ml aliquot)</li><li>2X top agar (50 ml aliquot)</li></ul> <hr><br><h2>Procedure:</h2> <ol style = 'list-style-type:upper-alpha;'> <li>Prepare your bench for aseptic work and assemble your supplies.</li> <li>Prepare 7H9 media. <ol><li>Using aseptic technique add 1 ml of CaCl2 to a 50 ml aliquot of 7H9 neat media.</li><li>Place in 55 °C bath.</li></ol></li> <li>Melt the 2X top agar in a microwave. <ol><li>Make sure the cap is loose!</li><li>The 2X top agar should come to a boil but not boil over.</li><li>The agar must be completely melted, so carefully swirl the bottle and check to make sure there are no clumps.</li><li>Once the agar is completely melted, place the 2X top agar in a 55 °C bath.</li></ol></li> <li>Aseptically combine the hot 2X top agar and  7H9/CaCl2 to make 1X top agar. <ol> <li>Keep at 55 °C until use.</li><li>The CaCl2 will precipitate out of solution and form a hazy layer on the bottom of the bottle. It will go back into solution when swirled.</li></ol></ol></li>"},"12-3":{content:"<h2>Objective:</h2> <p> To generate well-isolated plaques containing only one kind of phage</p> <hr><br><h2>Rationale:</h2> <p> Before you begin characterizing a phage, you need to ensure that you have a pure phage population. This can be accomplished by repeatedly isolating plaques. Plaque streak plates are a quick and economical method for doing this. The process relies on streaking a phage sample on agar to dilute the sample. Once bacteria are added to the plate, the diluted phage sample should give rise to well-isolated plaques (Figure 12.2-1). By repeating this process of isolating plaques, you will obtain a clonal phage population. Note that for some phages it is more difficult to obtain well-isolated plaques with this method. It is therefore best to also perform this in combination with at least one round of serial dilution/plaque assay to ensure good purification.</p> <hr><br><h2>Supplies:</h2> <ul> <li>Plate with plaques to purify or liquid sample from picking a plaque </li><li>Agar plate</li><li>Top agar</li><li>Bacterial host culture</li><li>Sterile sticks (If you do not have sterile sticks, sterile pipette tips can be used.)</li></ul> <hr><br><h2>Procedure:</h2> <ol style = 'list-style-type:upper-alpha;'><li>Prepare your bench for aseptic work and assemble your supplies.</li><li>Prepare your negative control plate.</li><ol><li>Label the bottom of the plate with your name, date, streak plate, and negative control.</li><li>Using aseptic technique, remove a sterile stick from its packaging.</li><li>Gently streak back and forth across the top one-third of the plate without lifting the stick from the agar. This is your first streak (Figure 12.2-1).<p><strong>Important:</strong> Try to skim across the top of the agar without gouging it with the stick.</p></li><li>With a new wooden stick, drag through the area streaked in the previous step, overlapping ONLY on the first two to three strokes. Continue to streak back and forth in an adjacent one-third of the agar, making sure to not to overlap the first streak again.</li><li>Repeat Step 4 to create a third streak on the remaining one-third of the agar.</li></ol><div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-13-3-3.png' alt = 'Image has two parts: on the right, labeled A, is an illustration of 4 blue circles. The first has white spots of various sizes with a stick pointing to one labeled: touch the center of a well isolated plaque using a sterile wooden stick. Three blue circles underneath show lines across the plate, labeled first streak, second streak and third streak with arrows between each circle labeled new wooden stick.  There is a red x on the side of the circle marked third streak. On the left is a picture of a opaque plate labeled plaque streak and lightly transparent circles along streaks in areas on the plate. There is an x marked on the lower part of the plate.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 12.2-1. (A) Streaking for isolation of a phage. The center of a well-isolated plaque is touched with a sterile wooden stick. On the first streak, this stick is used to streak an area of roughly one-third of a fresh agar plate. A new stick is then used to streak an adjacent un-streaked area (one-third) of the plate for the second streak, making sure to overlap the original streaked area with only the first few strokes. Again, a new stick is used for the third streak to make a few overlapping strokes of the second streak. The area where the third streak was made is the most dilute and is designated by the red X. (B) A sample streak plate showing good plaque isolation.</div></div><li>Gather your phage samples.</li><ol><li>Your phage sample may be a plaque, either from a plaque assay or another streak plate. It may also be a liquid sample prepared according to the <a class = 'chap-linking' tabindex = '0' data-link-to = '5-6'> Picking a Plaque (5.4)</a> protocol.</li><li>If you are picking a plaque directly from a plate, whether from a plaque assay or another streak plate, draw a circle around the plaque on the bottom of the plate and label it. Record the details in your lab notebook.</li></ol><li>Prepare your agar plates.</li><ol><li>Obtain as many agar plates as you have phage samples to purify.</li><li>Label each plate according to the phage sample, as described in step B(1).</li></ol><li>Make a streak plate.</li><ol><li>Remove a sterile wooden stick from its packaging.</li><li>Prepare for your first streak.</li><ol style = 'list-style-type:lower-alpha;'><li>If you are working directly from a plaque on a plate, gently touch the center of the plaque once with the edge of the stick. </li><li>If you are working from a liquid phage sample, swirl the stick in the sample.</li></ol><li>Touch the stick to the agar on the appropriately labeled plate and streak back and forth across the top one-third of the agar. Discard the stick.</li><li>With a new wooden stick, drag through the first streak once and continue to streak back and forth in an adjacent one-third of the agar, making sure to not to overlap the first streak again.</li><ol style = 'list-style-type:lower-alpha;'><li>This second streak allows you to spread just a fraction of the phage from the first streak, thereby diluting it.</li></ol><li>Repeat Step 4 to create a third streak on the remaining one-third portion of the plate.</li><ol style = 'list-style-type:lower-alpha;'><li>This third streak further dilutes the phage sample. Hopefully, you will have individual phage particles in this third streak.</li></ol><li>Mark the area where your third streak ends with an “X” on the bottom of the plate. This is the region that has the least number of phage particles on your streak plate.</li><li>Repeat Steps 1–6 with a new plate for each phage sample.</li></ol><li>Add bacteria to your streak plates. For this part of the experiment you will need 3 ml of molten top agar per streak plate. Your instructor may provide this for you or you may need to make it according to the protocol found in the <a class = 'chap-linking' tabindex = '0' data-link-to = '12'> <strong>Toolbox</strong></a>.</li><ol><li>For each sample, including your negative control:</li><ol style = 'list-style-type:lower-alpha;'><li>Use a sterile 5 ml pipette to aseptically transfer 3 ml of molten top agar to a culture tube containing 250 &micro;l of host bacteria.</li><li>Immediately aspirate (suck-up) the mixture back up into the pipette.</li><ol style = 'list-style-type:lower-roman;'><li>Try to avoid generating or withdrawing bubbles, as they can look like plaques on plates.</li></ol></ol><li>Dispense the top agar/bacteria mixture slowly onto the streak plate, depositing the mixture on the area marked “X.”</li><ol style = 'list-style-type:lower-alpha;'><li>Do not swirl the plate!</li></ol><li>Allow the mixture to spread across the plate from the most dilute point to the more concentrated areas. If necessary, tilt the plate very slightly.</li><li>Repeat this process for each of your samples.</li><li>Let the plates sit, undisturbed, for ~20 minutes to allow the top agar to solidify.</li></ol><li>Incubate plates at the proper temperature.</li><ol><li>After the top agar has solidified (~20 minutes), invert the plates rapidly (but gently). Inverting the plates prevents condensation from dripping onto the top agar in the incubator.</li><li>Place inverted plates in the proper incubator.</li></ol></ol> <hr><br><h3>Helpful Tips:</h3><ul><li>It is a good idea to streak the phage plaques in duplicate until you have mastered the technique. This will help ensure that you get an isolated single plaque every time.</li><li>Pick your plaques as soon as possible. The longer the phage is on the plate, the farther it diffuses toward and into adjacent plaques.</li><li>It is best to only pick plaques for purification that are well isolated from adjacent plaques. A well-isolated plaque is at least more than 0.5 cm away from adjacent plaques.</li></ul>"},"12-4":{content:"<h2>Objective:</h2> <p>To keep a visual record of phage plates and plaque morphology</p><hr><br><h2>Rationale:</h2><p> Properly documenting progress and results is the cornerstone of good scientific technique. For this reason, it is imperative that you take pictures documenting plaque morphology throughout the purification process to track your progress and report findings.<br> Different laboratory equipment exist to take pictures of plaques, but when unavailable, plaque pictures can also be taken with a digital camera or a smart phone. Detailed instructions about how to use a GelDoc or other common imaging instruments should be available at your institution, so only details regarding how to take a good picture with a digital camera are outlined in this protocol.</p> <hr><br><h2>Supplies:</h2> <ul><li>Plates to be photographed</li><li>Digital camera</li></ul><hr><br><h2>Procedure:</h2> <ol style = 'list-style-type:upper-alpha;'><li>Assemble your supplies.</li><li>The ideal plate for a picture contains about 20–40 plaques, depending on the size of the plaques. At this concentration, individual plaques are clearly visible and features (such as formation of halos) are clear.</li><li>Remove the lid. The agar side should be facing the lens of the camera.</li><li>Place the plate on top of a dark surface. Slide a black sheet of paper or black binder underneath the plate if necessary.</li><li>Make sure the field of view captures the entire surface area of the plate, with little to no space around it.</li><li>Make sure the lens is exactly parallel to the surface of the plate. Plates photographed at an angle will appear misshapen.</li><li>Make sure the picture is in focus. Edges of plaques should not be blurry.</li><li>Do not use a flash. Take the photo under good lighting conditions to avoid glare.</li><li>Look at the examples in Figure 12.3-1 below. Is your photo acceptable?</li><li>Include a copy of the photo in the proper location in your laboratory notebook. Include a detailed figure legend.</li><ol>  <div class = 'img-wrap'> <div class = 'img-container'><image src = './assets/images/figure-13-4-1.jpg' alt = 'A photograph with 9 tiles of pictures showing incorrect ways of photographing plates.'/></image></div> <div class = 'img-hr'></div> <div class = 'title-container'>Figure 12.3-1. Examples of poor plate pictures compared with an acceptable plate picture suitable for uploading to the Actinobacteriophage Database.</div></div>"},"14-1":{content:""},"14-2":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><strong>AD Supplement (0.5 L)</strong></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p><em>M. smegmatis </em>mc<sup>2</sup> 155 only</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>NaCl</p><p>&nbsp;</p><p>Albumin (Fraction V)</p></br><p>ddH<sub>2</sub>O</p></br><p>Dextrose</p></td><td colspan = '2'><p>4.25 g</p><p>&nbsp;</p><p>25 g</p></br><p>To 0.5 L</p></br><p>10 g</p></td><td><p>145 mM</p></br><p>5.0 %</p><p>2.0 %</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Add 400 mL of ddH<sub>2</sub>O to a 1-L Erlenmeyer flask.</li><li>Weigh out NaCl and albumin, and slowly, while stirring, add to the ddH<sub>2</sub>O in the Erlenmeyer flask.</li><li>Stir until dissolved.</li><li>Once dissolved, continue stirring and slowly add the dextrose.</li><li>Once the dextrose is dissolved, transfer the solution to a 1-L &nbsp;OR 0.5 L graduated cylinder, and bring the volume to 0.5 L with ddH<sub>2</sub>O.</li><li>Sterilize.</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Filter-sterilize; <span>do not autoclave.</span></p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At 4 ˚C, indefinitely if not contaminated</p></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Recipe:</p><ul><li>7H9 liquid medium complete</li></ul></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>Adding the dextrose last helps keep it from recrystallizing on the bottom of the flask.</li><li>Make sure the albumin is completely in solution before the filter-sterilization step. To dissolve the albumin, use a large stir bar spinning at a moderate to high speed. Very vigorous stirring will cause the albumin to denature and collect at foam/bubbles at the surface of the solution.</li><li>Standard medium for the growth of mycobacteria (Middlebrook) generally contains ADC (for albumin, dextrose and catalase) or OADC (ADC plus oleic acid). This guide uses the recipe for ADC but without catalase, hence the term “AD supplement”.</li><li>0.5 L of AD supplement is require to prepare 5 L of 7H9 liquid media .</li><li>Approximately 2.5 L of 7H9 liquid media is used per semester for a class of 20 students.</li></ol></td></tr></tbody></table>"},"14-3":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '6'><p><strong>1 M CaCl<sub>2</sub> (100 mL)</strong></p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '5' rowspan = '1'>General</td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td colspan = '4'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>CaCl<sub>2</sub></p><p>&nbsp;</p><p>ddH<sub>2</sub>O</p></td><td colspan = '4'><p>14.7 g of CaCl<sub>2</sub>.2H<sub>2</sub>0 <strong>or</strong></p><p>11.1 g of CaCl<sub>2</sub> (anhydrous)</p><p>see instructions below</p></td><td><p>1 M</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '5'><p>Note: You will need stock solutions of CaCl<sub>2</sub> at 1 M and 100 mM.</p><p>&nbsp;</p><p>For 1 M CaCl<sub>2</sub>:</p><ol><li>Add 90 mL of ddH<sub>2</sub>O to a large beaker.</li><li>Weigh out CaCl<sub>2</sub> and slowly, while stirring, add to the ddH<sub>2</sub>O until the CaCl<sub>2 </sub>is dissolved.</li><li>Transfer the solution to a graduated cylinder, and add ddH<sub>2</sub>O to 100 mL.</li><li>Sterilize.</li></ol><p>&nbsp;</p><p>For 100 mM CaCl<sub>2</sub>:</p><ol><li>Add 50 mL of 1 M CaCl<sub>2</sub> to a large beaker.</li><li>Add ddH<sub>2</sub>O to 500 mL</li><li>Sterilize*</li></ol><p>*if steps 1 &amp; 2 are performed using sterile ddH<sub>2</sub>O and sterile 1M calcium chloride, as well as using aseptic technique, there is no need to sterilize the solution.</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '5'><p>Filter-sterilize; <span>do not autoclave.</span></p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '5'><p>At room temperature, indefinitely. <span>Do not refrigerate</span></p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '5' rowspan = '1'><p>In Recipe:</p><ul><li>&nbsp;7H9 liquid medium complete and top agar</li><li>PYCa liquid medium, agar, and top agar.</li><li>Phage buffer</li></ul></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '5'><ol><li>CaCl<sub>2 </sub>should not be autoclaved as it can precipitate out of solution at high temperatures.</li><li>Due to the high concentration of salts in Middlebrook 7H9 media, CaCl<sub>2 </sub>has a tendency to precipitate out of solution. As such, CaCl<sub>2</sub> will ideally be added to 7H9 media immediately before the solution is used.</li><li>100 mL of both 1 M CaCl<sub>2 </sub>and 100 mM CaCl<sub>2</sub> should be sufficient for preparing all reagents requiring CaCl<sub>2</sub> for ~ 20 students per semester.</li></ol></td></tr></tbody></table>"},"14-4":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>0.8 M Ca(NO<sub>3</sub>)<sub>2</sub> (100 mL)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p><em>Streptomyces spp. </em>only</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p><strong>Ca(NO<sub>3</sub>)<sub>2</sub></strong></p><p><strong>ddH<sub>2</sub>O</strong></p></td><td colspan = '2'><p>18.89 g of Ca(NO<sub>3</sub>)<sub>2</sub>.4H<sub>2</sub>0</p><p>To 100 mL</p></td><td><p>0.8 M</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Add 80 mL of ddH<sub>2</sub>O to a large beaker.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</li></ol><ol><li value = '2'>Weigh out Ca(NO<sub>3</sub>)<sub>2</sub> and slowly, while stirring, add to the ddH<sub>2</sub>O until the Ca(NO<sub>3</sub>)<sub>2 </sub>is dissolved</li></ol><ol><li value = '3'>Transfer the solution to a graduated cylinder, and add ddH<sub>2</sub>O to 100 mL.</li></ol><ol><li value = '4'>Sterilize.</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Autoclave.</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, indefinitely. <span>Do not refrigerate</span></p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Recipe:</p><ul><li>Media Supplement for <em>Streptomyces</em> phage</li></ul></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><p>&nbsp;</p></td></tr></tbody></table>"},"14-5":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '6'><p><strong>Carbenicillin (CB) at 1000X</strong></p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '5' rowspan = '1'><p><em>M. smegmatis </em>mc<sup>2</sup>155 only</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td colspan = '3'><p><strong>Amount</strong></p></td><td colspan = '2'><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>Carbenicillin (powder)</p></td><td colspan = '3'><p>50 mg/ml</p></td><td colspan = '2'><p>stock = 1000X</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '5'><ol><li>Weigh out 100 to 600 mg of carbenicillin powder and place it in a 10-15 mL tube.</li></ol><ol><li value = '2'>Divide the number of milligrams of carbenicillin by 50 to determine the volume (in milliliters) of ddH<sub>2</sub>O to add (e.g., for 477 mg of CB, add 9.5 mL ddH<sub>2</sub>O).</li></ol><ol><li value = '3'>Stir or shake until dissolved.&nbsp;</li><li>Filter-sterilize (0.22&micro;m pore size) into a fresh sterile tube.</li><li>Prepare 1 mL aliquots and freeze at -80 ˚C</li></ol></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '5'><p>Filter-sterilize (0.22&micro;m pore size) into a fresh sterile tube.</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '5'><p>At -80 ˚C for 3 years.</p><p>At 4 ⁰C for ≤ 60 days.</p></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '5' rowspan = '1'><p>In Recipe: 7H9 Liquid Medium</p></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>Label Stock Solution </strong></p></td><td colspan = '5'><p>CB, 1000X, date, initials</p></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare Final Solution </strong></p></td><td colspan = '5'><ol><li>Remove a frozen aliquot, allow to thaw, and use as follows. Do not return thawed aliquot to -80 ˚C. Instead, store thawed aliquots at 4 ˚C</li><li>When using, add stock solution at 1:1000. For example, for 1 L of medium, add 1 mL of stock.</li></ol></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '5'><ol><li>If your host bacteria is resistant to CB, it can be added to prevent growth of other bacteria.</li></ol><ol><li value = '2'>Mechanism of action: CB is a semi-synthetic penicillin that interferes with bacterial cell-wall synthesis.</li></ol><ol><li value = '3'>50 mL of carbenicillin should be sufficient per semester for a class of ~ 20 students.</li></ol></td></tr></tbody></table>"},"14-6":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '5'><p><strong>Cycloheximide (CHX)</strong></p></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '4' rowspan = '1'><p>General</p></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td colspan = '3'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>Cycloheximide (powder)</p></td><td colspan = '3'><p>10 mg/ml</p></td><td><p>stock = 1000X</p></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '4'><ol><li>Weigh out 50 to 100 mg of cycloheximide powder and place it in a 10-15 mL tube.</li></ol><ol><li value = '2'>Divide the number of milligrams of cycloheximide by 10 to determine the volume (in milliliters) of ddH<sub>2</sub>O to add (e.g., for 51 mg of CHX, add 5.1 mL ddH<sub>2</sub>O).</li></ol><ol><li value = '3'>Stir or shake until dissolved.&nbsp;</li><li>Filter-sterilize (0.22&micro;m pore size) into a fresh sterile tube.</li><li>Prepare 1 mL aliquots and freeze at -80 ˚C.</li></ol></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '4'><p>Filter-sterilize (0.22&micro;m pore size) into a fresh sterile tube.</p></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '4'><p>At -80 ˚C for 1 year.</p><p>At 4 ⁰C for ≤ 60 days.</p></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '4' rowspan = '1'><p>In Recipe:</p><ul><li>7H9 Liquid Medium</li><li>PYCa Liquid Medium</li><li>Luria Agar</li><li>PYCa Agar</li></ul></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '4'><p>&nbsp;</p></td></tr><tr><td><p><strong>Label Stock Solution </strong></p></td><td colspan = '4'><p>CHX, 1000X, date, initials</p></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare Final Solution </strong></p></td><td colspan = '4'><ol><li>Remove a frozen aliquot, allow to thaw, and use as follows. Do not return thawed aliquot to -80 ˚C. Instead, store thawed aliquots at 4 ˚C.</li></ol><ol><li value = '2'>When using, add stock solution at 1:1000. For example, for 1 L of medium, add 1 mL of stock.</li></ol></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '4'><ol><li>CHX is added to media to prevent growth of fungi and yeast.</li></ol><ol><li value = '2'>Mechanism of action: CHX inhibits protein biosynthesis in eukaryotic organisms. It interferes with the peptidyl transferase activity of the 60S ribosome, thus blocking translational elongation.</li></ol><ol><li value = '3'>50 mL of cycloheximide should be sufficient per semester for a class of ~ 20 students.</li></ol></td></tr></tbody></table>"},"14-7":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>40 % Dextrose (200 mL)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>Dextrose anhydrous</p></br><p>ddH<sub>2</sub>O</p></td><td colspan = '2'><p>80 g</p></br><p>To 200 mL</p></td><td><p>40 %</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><p>1. Add 100 mL of ddH<sub>2</sub>O to a beaker, and then add the dextrose while stirring.</p><p>2. Top up with ddH<sub>2</sub>O to 200 mL, and continue stirring until dextrose has dissolved. If the dextrose does not fully dissolve, gently warm the solution on the stir plate (or in a microwave).</p><p>3. Filter-sterilize the solution. Do not autoclave.</p></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Filter-sterilize. <span>Do not autoclave.</span></p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, indefinitely if not contaminated.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Recipe:&nbsp;</p><ul><li>PYCa media</li><li>PYCa agar plates&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</li><li>PYCa top agar</li><li>Media Supplement for <em>Streptomyces</em> phage</li></ul></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>100 mL of 40 % dextrose should be sufficient for preparing all reagents and media requiring dextrose, per semester for a class of ~ 20 students.</li></ol></td></tr></tbody></table>"},"14-8":{content:"<table border = '1' cellpadding = '0' cellspacing = '0' width = '431'><tbody><tr><td colspan = '4'><p><strong>Enrichment Broth (100 mL)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p><em>M. smegmatis </em>mc<sup>2</sup>155 only</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td><p><strong>Amount</strong></p></td><td colspan = '2'><p><strong>Final Concentration</strong></p></td></tr><tr><td><p><strong>7H9 liquid media neat</strong></p></br><p><strong>100 mM CaCl<sub>2 </sub>stock</strong></p></td><td><p>99 mL</p></br><p>1 mL</p></td><td colspan = '2'><p>1X</p></br><p>1 mM</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'>All ingredients must be added to a sterile bottle or flask using aseptic technique. The final medium can be filter-sterilized.</td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>All ingredients are sterile. The final medium can be filter-sterilized.</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, for up to 1 year. Discard if solution is discolored, contaminated or has precipitation.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3'><p>In Protocol: 5.2, 5.5</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>If antibiotics or antimicrobials are added, the medium must be stored at 4 ⁰C.</li><li>A good rule of thumb is that CaCl<sub>2</sub> should be added to all solutions that come in contact with phage, and ideally, this should happen immediately before the solution is used. This is because CaCl<sub>2</sub> will precipitate out when cooling after autoclaving or when refrigerated.</li><li>30 ml of Enrichment broth is required per student per soil sample tested. &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</li></ol></td></tr></tbody></table>"},"14-9":{content:"<table border = '1' cellpadding = '0' cellspacing = '0' width = '431'><tbody><tr><td colspan = '4'><p><strong>EDTA 0.5 M (100 mL)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td><p><strong>Amount</strong></p></td><td colspan = '2'><p><strong>Final Concentration</strong></p></td></tr><tr><td><p><strong>EDTA disodium salt dihydrate</strong></p><p><strong>ddH<sub>2</sub>O</strong></p></td><td><p>18.61 g</p><p>&nbsp;</p><p>To 100 mL</p></td><td colspan = '2'><p>0.5 M</p><p>&nbsp;</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Weigh out EDTA and place in a large beaker.</li><li>Add approximately 80 mL of ddH<sub>2</sub>O and stir until dissolved.</li><li>Adjust the pH to 8.0 with NaOH.</li><li>Transfer the solution to a graduated cylinder and add ddH<sub>2</sub>O to 100 mL.</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Filter-sterilize or autoclave</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, indefinitely if not contaminated.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3'><p>In Protocol: 9.1a, 9.1b, 9.2a, 9.2b</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>For protocol 9.2b, prepare a 0.1 M EDTA solution by diluting the 0.5 M EDTA solution in water.&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</li></ol></td></tr></tbody></table>"},"14-10":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>40 % Glycerol (500 mL)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>ddH<sub>2</sub>O</p></br><p>Glycerol (also known as glycerin or glycerine)</p></td><td colspan = '2'><p>To 500 mL</p></br><p>200 ml</p></td><td><p>&nbsp;</p><p>40 %</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Measure 250 mL ddH<sub>2</sub>O and place in a large beaker.</li><li>Slowly add 200 mL of glycerol while stirring.</li><li>When the glycerol has completely mixed with the water, transfer the solution to a graduated cylinder.</li><li>Bring the volume to 500 mL with ddH<sub>2</sub>O. Mix.</li><li>Sterilize</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Filter-sterilize or autoclave.</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, indefinitely if not contaminated.</p></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Recipe:</p><ul><li><p>7H9 liquid medium<br>&nbsp;</p></li></ul><p>In Protocol:</p><ul><li><p>Making glycerol stocks</p></li></ul></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><p>100 mL of 40 % glycerol should be sufficient for all reagents and media requiring glycerol per semester.</p></td></tr></tbody></table>"},"14-11":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>80 % Isopropanol (50 ml)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>99% Isopropanol</p><p>ddH<sub>2</sub>O</p></td><td colspan = '2'><p>40 ml</p><p>To 50 ml</p></td><td><p>80 %</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Measure 40 ml isopropanol and place in a sterile conical tub&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</li><li>Slowly add ddH<sub>2</sub>O to 50 ml.</li><li>Mix using a vortex or by inversion.</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Not required</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p><span>Important: </span>Do Not Store.<br>80 % isopropanol should be prepared and used within the same day.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Protocol: 9.1, 9.2</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>Prepare 80 % isopropanol only as needed. Isopropanol will evaporate with time, resulting in a solution with reduced isopropanol concentration. Isopropanol solutions below 80 % will result in premature elution of DNA during phage DNA extraction.</li></ol><ol><li value = '2'>6 mL of isopropanol is used per attempt at DNA extraction from phage lysates.</li></ol></td></tr></tbody></table>"},"14-12":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>1 M MgCl<sub>2</sub> (100&nbsp; mL)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p><em>Streptomyces spp. </em>only</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p><strong>MgCl<sub>2</sub></strong> hexahydrate*</p><p><strong>ddH<sub>2</sub>O</strong></p><p>&nbsp;</p><p><span><em><span>*<strong>MgCl<sub>2</sub></strong> hexahydrate can be substituted with 9.5 g of&nbsp; <strong>MgCl<sub>2</sub></strong> anhydrous in this recipe</span></em></span></p></td><td colspan = '2'><p>20.33 g</p><p>To 100 mL</p></td><td><p>1 M</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Weigh out MgCl<sub>2</sub> and place in a large beaker containing approximately 75 mL of ddH<sub>2</sub>O, and stir until MgCl<sub>2 </sub>is dissolved.</li><li>Transfer the solution to a graduated cylinder, and add ddH<sub>2</sub>O to 100 mL.</li><li>Sterilize</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Autoclave or filter-sterilize</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, for a year.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Recipe:</p><ul><li>Media Supplement for <em>Streptomyces</em> phage</li></ul></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><p>&nbsp;</p></td></tr></tbody></table>"},"14-13":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '6'><p><strong>1 M MgSO<sub>4</sub> (100 mL)</strong></p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td colspan = '2'><p><strong>For bacterial host</strong></p></td><td colspan = '4' rowspan = '1'><p>General</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td colspan = '3'><p><strong>Amount</strong></p></td><td colspan = '2'><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>MgSO<sub>4 </sub>(anhydrous)</p></br><p>ddH<sub>2</sub>O</p></td><td colspan = '3'><p>12.04 g</p></br></br><p>To 100 mL</p></td><td colspan = '2'><p>1 M</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '5'><ol><li>Weigh out MgSO<sub>4</sub> and place in a large beaker containing approximately 95 mL of ddH<sub>2</sub>O, and stir until MgSO<sub>4 </sub>is dissolved.</li><li>Transfer the solution to a graduated cylinder, and add ddH<sub>2</sub>O to&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 100 mL.</li><li>Sterilize.</li></ol></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '5'><p>Autoclave or filter-sterilize</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '5'><p>At room temperature, for a year.</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '5' rowspan = '1'><p>In Recipe:</p><ul><li>Phage buffer</li></ul></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '5'><ol><li>10 mL of MgSO<sub>4 </sub>is required to prepare 1 L of phage buffer.</li><li>100 mL of MgSO<sub>4</sub> is sufficient for 1 – 5 semesters for a class of ~ 20 students.</li></ol></td></tr></tbody></table>"},"14-14":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>Nuclease Mix (5 mL)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>NaCl</p><p>ddH<sub>2</sub>O</p><p>DNase 1</p><p>RNase A</p><p>Glycerol</p></td><td colspan = '2'><p>0.09 g</p><p>4.25 mL</p><p>250 &micro;L of 5-mg/mL stock</p><p>125 &micro;L of 10-mg/mL stock</p><p>2.5 mL</p></td><td><p>150 mM</p><p>&nbsp;</p><p>0.25 mg/mL or 500 KU/ml</p><p>0.25 mg/mL or 12.5 U/mg</p><p>50 %</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><p><strong>Note</strong>: Begin by preparing the RNase A and DNase 1 stock solutions according to the manufacturer’s protocol. RNAse A stock solutions are typically prepared in Tris-Cl (10 mM; pH 7.5) and NaCl (15 mM), and includes a 5-min heating step at 100 ˚C.</p><ol><li>Dissolve the NaCl in 4.25 mL of ddH<sub>2</sub>O in a sterile 15-mL conical tube.</li><li>Transfer 2.1 mL of the NaCl solution into a new 15-mL conical tube.<br><span><span>Note: the remaining 2.1 mL can be stored indefinitely and used at a later date to prepare more nuclease mix.</span></span></li><li>Add the glycerol to the NaCl solution, and mix with gentle inversion.</li><li>Add the RNase and DNase stock solutions, and mix with gentle inversion.</li><li>Add ddH<sub>2</sub>O to a final volume of 5 mL, and mix with gentle inversion until the solution is homogeneous.</li><li>Aliquot into microcentrifuge tubes.</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Not required</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At -20 ⁰C, for up to 2 years.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Protocol: 9.1</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>Be extra careful not to contaminate pipettes, benches, or other reagents with this nuclease mixture, because it will degrade any DNA samples.</li><li>Prepare DNase I and RNase A stock solutions according to the instructions provided by the manufacturer.</li><li>5 &micro;L of nuclease mix is used per attempt at DNA extraction from phage lysates.</li><li>Estimate 1-2 attempts at DNA extraction per student.</li></ol></td></tr></tbody></table>"},"14-15":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '7'><p><strong>Phage Buffer (PB; 1 L)</strong></p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '6' rowspan = '1'><p>General</p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td colspan = '3'><p><strong>Amount</strong></p></td><td colspan = '3'><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>1 M Tris stock (pH 7.5)</p><p>1 M MgSO<sub>4</sub>&nbsp;stock</p><p>NaCl</p><p>ddH2O</p><p>100 mM CaCl<sub>2 </sub>stock</p><p>Glycerol (100 %), if using*</p></td><td colspan = '3'><p>10 mL</p><p>10 mL</p><p>4 g</p><p>up to 1 L</p><p>10 mL</p><p>100 mL</p></td><td colspan = '3'><p>10 mM</p><p>10 mM</p><p>68 mM</p><p>&nbsp;</p><p>1 mM</p><p>10 %</p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '6'><ol><li>Add 800 mL of&nbsp; ddH<sub>2</sub>O to a large flask or beaker.</li></ol><ol><li value = '2'>Add all ingredients to the water, while stirring. Glycerol should be added at this stage, if using (see notes*).</li></ol><ol><li value = '3'>Stir until all components have completely dissolved.</li></ol><ol><li value = '4'>Add ddH<sub>2</sub>O to 1 L.</li></ol><ol><li value = '5'>Filter-sterilize.</li></ol><ol><li value = '6'>Aliquot into sterile bottles, flasks, or tubes in 50-mL or 100-mL (or other) portions as needed.</li></ol></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '6'><p>Filter-sterilize</p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '6'><p>At room temperature, for up to 3 years. Discard if solution is discolored, contaminated or has precipitation.</p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '6' rowspan = '1'><p>In Protocol:</p><p>5.3, 5.4, 5.6, 6.1, 6.2, 6.3, 6.4, 6.5, 7.1, 8.1a, 8.1b, 11.1, 11.4</p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '6'><ol><li>*If your phage is particularly unstable in phage buffer, you may consider supplementing your phage buffer with glycerol to 10 %.</li><li>1 L of phage buffer is sufficient per semester for a class of ~ 20 students.</li></ol></td></tr></tbody></table>"},"14-16":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>Phage Precipitation Solution (100 mL)</strong></p><p><span><strong>[If using, prepare as necessary]</strong></span></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>ddH<sub>2</sub>O</p><p>Polyethylene glycol 8000</p><p>(PEG 8000)</p><p>NaCl</p></td><td colspan = '2'><p>To 100 mL</p><p>30 g</p><p>&nbsp;</p><p>19.3 g</p></td><td><p>&nbsp;</p><p>30 %</p><p>&nbsp;</p><p>3.3 M</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Add 60 mL of ddH<sub>2</sub>O, followed by 19.3 g NaCl, to a glass bottle with a screw cap, and swirl until the NaCl is completely dissolved.</li></ol><ol><li value = '2'>Slowly add 30 g of PEG 8000. Add only a few grams at a time, alternately swirling and gently heating in a microwave (e.g. by heating only for 10 sec at a time) until all the PEG is dissolved.</li></ol><ol><li value = '3'>Add sterile water to 100 mL.</li></ol><ol><li value = '4'>Add a clean magnetic stir bar and stir until homogeneous.</li></ol><ol><li value = '5'>Filter-sterilize.</li></ol><ol><li value = '6'>Dispense into sterile bottles or tubes.</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Filter-sterilize.</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, indefinitely. Discard if solution is deteriorating, discolored, contaminated or has precipitation.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Protocol: 9.1 of Instructors Guide</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>The NaCl and the PEG 8000 are difficult to dissolve. Several iterations of heating and stirring may be required. If there are precipitates in the solution, do not use it! Rather, attempt to dissolve particulates by alternately heating the solution in a microwave and swirling. Allow the solution to cool before using. Use this solution only at room temperature.</li></ol><ol><li value = '2'>Up to 4 mL of phage precipitation buffer is used per attempt at DNA extraction using the PEG precipitation protocol.</li></ol></td></tr></tbody></table>"},"14-17":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '5' rowspan = '1'><p>General</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td colspan = '3'><p><strong>Amount</strong></p></td><td colspan = '2'><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>SDS powder</p><p>ddH<sub>2</sub>O</p></td><td colspan = '3'><p>1 g</p><p>up to 10 mL</p></td><td colspan = '2'><p>10 %</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '5'><p><span>Important:</span> SDS may cause respiratory irritation if inhaled. Is using loose powder of SDS (instead of micropellets), a mask should be used when weighing out the powder.<br>&nbsp;</p><ol><li>Weigh out 1g of SDS powder and place it in a 50 mL tube.</li></ol><ol><li value = '2'>Add ddH<sub>2</sub>O to 10 mL.</li></ol><ol><li value = '3'>Invert the tube gently and repeated until the SDS is fully dissolved. Vigorous shaking will lead to excessive foaming.</li><li>Filter-sterilize (0.22&micro;m pore size) into a fresh sterile tube.</li><li>Prepare 1 mL aliquots.</li></ol></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '5'><p>Filter-sterilize (0.22&micro;m pore size) into a fresh sterile tube.</p></td></tr><tr><td colspan = '6'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '5'><p>At room temperature, indefinitely if not contaminated.</p></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '5' rowspan = '1'><p>In Protocol: 9.1a, 9.1b, 9.2a, 9.2b</p></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>Label Stock Solution </strong></p></td><td colspan = '5'><p>10 % SDS</p></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '5'><ol><li>2 mL of 10 % SDS should be sufficient per semester for a class of ~ 20 students.</li></ol></td></tr></tbody></table>"},"14-18":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>1X TBE</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>10X TBE</p><p>ddH<sub>2</sub>O</p></td><td colspan = '2'><p>100 mL</p><p>900 mL</p></td><td><p>1X (89-mM Tris, 89-mM boric acid, and 2-mM EDTA)</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><p>Dilute the 10X TBE to 1X with the volumes above (i.e., 100 and 900 mL). There is no need to check or adjust the pH of this solution, and it does not have to be sterilized before use.</p></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Not required</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, indefinitely. Discard is discolored, contaminated or has precipitation.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Protocol: 10.2</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><p>Volume of buffer will depend on size of gel and gel tank</p></td></tr></tbody></table>"},"14-19":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>1 M Tris (base), pH 7.5 (100 mL)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>Trizma base</p><p>ddH<sub>2</sub>O</p></td><td colspan = '2'><p>12.11 g</p><p>To 100 mL</p></td><td><p>1 M</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Add approximately 95 mL of ddH<sub>2</sub>O to a large beaker.</li></ol><ol><li value = '2'>Add Trizma base to the breaker while stirring.</li></ol><ol><li value = '3'>When the Trizma base is dissolved, bring the pH to 7.5 with HCl.</li></ol><ol><li value = '4'>Transfer the solution to a graduated cylinder, and add ddH<sub>2</sub>O to 100 mL</li><li value = '5'>&nbsp;Sterilize</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Autoclave or filter-sterilize</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, for up to a year.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Recipe:</p><ul><li>Phage buffer</li></ul></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>10 mL of 1M Tris is required to prepare 1 L of phage buffer.</li></ol><ol><li value = '2'>100 mL of 1M Tris is sufficient for 1 – 5 semesters for a class of ~ 20 students.</li></ol></td></tr></tbody></table>"},"14-20":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>20 % Tween80 (100 ml)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>ddH<sub>2</sub>O</p><p>Tween80</p></td><td colspan = '2'><p>To 100 ml</p><p>20 ml</p></td><td><p>20 %</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Measure 60 mL of ddH<sub>2</sub>O and place in a medium-size beaker.</li></ol><ol><li value = '2'>Slowly add 20 mL of Tween80 while stirring.</li></ol><ol><li value = '3'>Once the Tween80 has completely dissolved, transfer the solution to a graduated cylinder and bring to 100 mL with ddH<sub>2</sub>O.</li></ol><ol><li value = '4'>Sterilize</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Filter-sterilize. Do not autoclave</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, indefinitely if not contaminated</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Recipe:</p><ul><li>7H9 liquid medium</li></ul></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>It may take a while for Tween80 to dissolve. Add Tween 80 slowly and alternate between stirring and warming the solution. For example, alternate the beaker between the stir plate and at 55 ⁰C water bath to dissolve the Tween80.</li><li>100 mL of 20 % Tween80 is sufficient for 1 – 10 semesters for a class of ~ 20 students.</li></ol></td></tr></tbody></table>"},"14-21":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>Uranyl Acetate Stain</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>Uranyl acetate</p><p>ddH<sub>2</sub>O</p></td><td colspan = '2'><p>0.1 g</p><p>10 mL</p></td><td><p>1 % w/v</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Weigh out the uranyl acetate and place in a conical containing approximately 9 mL of ddH<sub>2</sub>O, and stir</li><li>When the uranyl acetate is dissolved, bring the final volume up to 10 mL with ddH<sub>2</sub>O.</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Prepare a 0.22-&micro;m filter by passing 3 mL of ddH<sub>2</sub>O through the filter. Discard the filtered ddH<sub>2</sub>O. Filter-sterilize the uranyl acetate solution and dispense in 1-mL aliquots.</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>Cover each aliquot in foil to protect from light. Store at room temperature, for up to 3 years. Discard appropriately if precipitation occurs.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Protocol:</p><p>8.1a, 8.1b</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>This is a highly toxic compound. Always wear gloves when preparing or working with it.</li></ol><ol><li value = '2'>5 &micro;L of uranyl acetate stain is used per sample to be imaged.</li></ol></td></tr></tbody></table>"},"14-22":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>Water, sterile (200 mL)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>ddH<sub>2</sub>O</p><p>&nbsp;</p></td><td colspan = '2'><p>200 mL</p><p>&nbsp;</p></td><td><p>&nbsp;</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Measure 200 mL ddH<sub>2</sub>O (ideally deionized water) and place bottle with a cap.</li><li>Sterilize</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Filter-sterilize or autoclave.</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, indefinitely if not contaminated.</p></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Recipe:</p><ul><li><p>80 % Isopropanol<br>&nbsp;</p></li></ul><p>In Protocol:</p><ul><li><p>DNA Extraction</p></li><li><p>Restriction Enzymes Digests</p></li></ul></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><p>500 mL of sterilized water should be sufficient for all reagents and experiments per semester.</p></td></tr></tbody></table>"},"14-23":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p>Zinc Chloride (2 M)</p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p>General</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>ZnCl<sub>2</sub></p><p>ddH<sub>2</sub>O</p></td><td colspan = '2'><p>10.9 g</p><p>To 40 mL</p></td><td><p>2 M</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Weigh out ZnCl<sub>2</sub> and place in a conical tube containing approximately 35 mL of ddH<sub>2</sub>O, and vortex until ZnCl<sub>2 </sub>is dissolved.</li><li value = '2'>Add ddH<sub>2</sub>O to 40 mL.</li><li>Sterilize.</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Filter-sterilize.</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p><span>Important:</span> Do Not Store.</p><p>*Prepare immediately before use.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Protocol:</p><p>9.2b</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><p>* &nbsp;When prepared at this high concentration, ZnCl<sub>2 </sub>will precipitate after ~ 1 hour at room temperature.</p></td></tr></tbody></table>"},"14-24":{content:""},"14-25":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'> <tbody> <tr> <td colspan = '4'> <p><strong>Middlebrook 7H9 Liquid Medium: Neat (900 mL)</strong></p></td></tr><tr> <td colspan = '4'> <p>&nbsp;</p></td></tr><tr> <td> <p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'> <p><em>M. smegmatis </em>mc<sup>2</sup>155 only</p></td></tr><tr> <td colspan = '4'> <p>&nbsp;</p></td></tr><tr> <td> <p><strong>Ingredients</strong></p></td><td colspan = '2'> <p><strong>Amount</strong></p></td><td> <p><strong>Final Concentration</strong></p></td></tr><tr> <td> <p>7H9 broth base</p><p>40% glycerol stock&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p><p>ddH<sub>2</sub>O</p></td><td colspan = '2'> <p>4.7 g</p><p>5 mL</p><p>To 900 ml</p></td><td> <p>&nbsp;</p></td></tr><tr> <td colspan = '4'> <p>&nbsp;</p></td></tr><tr> <td> <p><strong>To Prepare</strong></p></td><td colspan = '3'> <ol> <li>Add broth base to 850 mL of ddH2O in an Erlenmeyer flask.</li><li>While stirring, add the glycerol. Stir until the broth base powder is completely dissolved.</li><li>Bring up to 900 mL with ddH2O in a graduated cylinder.</li><li>Aliquot into 90-mL, 450-mL, or 900-mL portions as needed.</li><li>Sterilize</li></ol> </td></tr><tr> <td colspan = '4'> <p>&nbsp;</p></td></tr><tr> <td> <p><strong>To Sterilize</strong></p></td><td colspan = '3'> <p>Autoclave.</p></td></tr><tr> <td colspan = '4'> <p>&nbsp;</p></td></tr><tr> <td> <p><strong>To Store</strong></p></td><td colspan = '3'> <p>At room temperature, indefinitely if not contaminated or discolored.</p></td></tr><tr> <td colspan = '4'> <p>&nbsp;</p></td></tr><tr> <td> <p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'> <p>In Recipe:</p><ul> <li>Growing <em>Mycobacterium smegmatis</em> mc<sup>2</sup>155</li><li>1X 7H9 top agar</li></ul> </td></tr><tr> <td colspan = '4'> <p>&nbsp;</p></td></tr><tr> <td> <p><strong>Notes</strong></p></td><td colspan = '3'> <ol> <li>This medium will be diluted by the addition of 10 % (v/v) AD supplement (see Middlebrook 7H9 Liquid Medium: Complete). This is why medium is dispensed in 9/10 volumes (e.g., 450 mL).</li></ol> <ol> <li value = '2'>5 L of 7H9 liquid media should be sufficient per semester for a class of 20 students. </li></ol> </td></tr></tbody></table>"},"14-26":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>Middlebrook 7H9 Liquid Medium: Complete (100 mL)</strong></p><p><strong><span>(For bacterial growth without Tween 80)&nbsp;</span>&nbsp;&nbsp;&nbsp;&nbsp;</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p><em>M. smegmatis </em>mc<sup>2</sup>155 only</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>7H9 liquid medium: neat</p><p>AD supplement</p><p>CB stock</p><p>CHX stock</p><p>100 mM CaCl<sub>2</sub>&nbsp;stock</p></td><td colspan = '2'><p>89 ml</p><p>10 ml</p><p>100 &micro;l</p><p>100 &micro;l</p><p>1 ml</p></td><td><p>1X</p><p>10 %</p><p>50 &micro;g/ml</p><p>10 &micro;g/ml</p><p>1 mM</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><p>All ingredients must be added to a sterile bottle or flask using aseptic technique. The final medium cannot be autoclaved or filter-sterilized.</p></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>All ingredients are already sterile</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At 4 ⁰C, for up to 30 days. Discard if media is discolored or contaminated.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Recipe:</p><ul><li>Growing <em>Mycobacterium smegmatis</em> mc2155</li></ul></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>Make complete 7H9 medium as needed; refrigeration may cause the CaCl<sub>2</sub> to precipitate. If antimicrobials and AD supplement have been added, the medium must be stored at 4 °C.</li><li>A good rule of thumb is that CaCl<sub>2</sub> should be added to all solutions that come in contact with phage, and ideally, this should happen immediately before the solution is used. This is because CaCl<sub>2</sub> will precipitate out when cooling after autoclaving or when refrigerated.</li></ol></td></tr></tbody></table>"},"14-27":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>Middlebrook 7H9 Liquid Medium: Complete with Tween80</strong></p><p><strong><span>(For bacterial growth with Tween 80)&nbsp;&nbsp;&nbsp;</span>&nbsp;&nbsp;</strong>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p><em>M. smegmatis </em>mc<sup>2</sup>155 only</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>7H9 liquid medium</p><p>AD supplement</p><p>CB stock</p><p>CHX stock</p><p>100 mM CaCl<sub>2</sub>&nbsp;stock</p><p>20 % Tween80 stock</p></td><td colspan = '2'><p>90 ml</p><p>10 ml</p><p>100 &micro;L</p><p>100 &micro;L</p><p>1 ml</p><p>250 &micro;L</p></td><td><p>1X</p><p>10 %</p><p>50 &micro;g/ml</p><p>10 &micro;g/ml</p><p>1 mM</p><p>0.05 %</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><p>All ingredients must be added to a sterile bottle or flask using aseptic technique. The final medium cannot be autoclaved or filter-sterilized.</p></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>All ingredients are already sterile</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At 4 ⁰C, for up to 30 days. Discard if media is discolored or contaminated.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Recipe:</p><ul><li>Growing <em>Mycobacterium smegmatis</em> mc2155</li></ul></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>In liquid culture, <em>Mycobacterium smegmatis</em> tends to clump. When liquid cultures are grown from bacterial colonies, Tween80 should be added to 0.05% to prevent clumping.</li></ol><ol><li value = '2'>This culture is then sub-cultured into medium without Tween80 for use in phage infections.</li></ol><ol><li value = '3'>Make complete 7H9 medium as needed; refrigeration will cause the calcium to precipitate. If antimicrobials and AD supplement have been added, the medium must be stored at 4 °C.</li></ol><ol><li value = '4'>A good rule of thumb is that CaCl<sub>2</sub> should be added to all solutions that come in contact with phage, and ideally, this should happen immediately before the solution is used. This is because CaCl<sub>2</sub> will precipitate out when cooling after autoclaving or when refrigerated.</li></ol></td></tr></tbody></table>"},"14-28":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '8'><p><strong>1X Middlebrook Top Agar (TA; 100mL)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong></p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td colspan = '2'><p><strong>For bacterial host&nbsp;&nbsp; </strong></p></td><td colspan = '6' rowspan = '1'><p><em>M. smegmatis </em>mc<sup>2</sup>155 only</p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td colspan = '4'><p><strong>Amount</strong></p></td><td colspan = '3'><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>100 mM CaCl<sub>2 </sub>stock</p><p>7H9 liquid media neat</p><p>2X TA</p></td><td colspan = '4'><p>1 mL</p><p>50 mL</p><p>50 mL</p></td><td colspan = '3'><p>1 mM</p><p>&nbsp;</p><p>1 X</p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '7'><ol><li>Using aseptic technique, add 1 mL of CaCl<sub>2</sub> stock to the 50 mL of 7H9 medium.</li><li>Place into a 55 °C water bath.</li><li>Melt the 2X TA in a microwave. Make sure the cap is loose! The agar must be completely melted, so carefully swirl the bottle and check to make sure there are no clumps. The 2X TA should come to a boil but not boil over.</li><li>Once the agar is completely melted, place the 2X TA into a 55 °C water bath. When both solutions are at 55 °C, aseptically add the medium-calcium mixture to the 2X TA and swirl.</li></ol></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '7'><p>All ingredients must be added using aseptic technique. The final medium cannot be autoclaved or filter-sterilized.</p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '7'><p>In the 55 °C water bath ≤ 7days. Cooling to room temperature will cause the CaCl<sub>2</sub> to precipitate out of solution.</p><p>Discard if media is discolored or contaminated.</p><p>Repeated melting and solidifying of top agar will cause the agar to deteriorate and not solidify properly on plates.</p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '7' rowspan = '1'><p>In Protocol:</p><p>5.3, 5.6, 6.1, 6.4, 6.5, 7.1, 11.1, 11.2, 11.3, 11.4, 12.1,12.2</p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '7'><ol><li>The final agar concentration will be 0.4%</li></ol><ol><li value = '2'>A good rule of thumb is that CaCl<sub>2</sub> should be added to all solutions that come in contact with phage, and ideally, this should happen immediately before the solution is used. This is because CaCl<sub>2</sub> will precipitate out when cooling after autoclaving or when refrigerated.</li></ol><p>3. 3 – 5 mL of top agar is used per plating.</p></td></tr></tbody></table>"},"14-29":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '5'><p ><strong>2X Middlebrook Top Agar (2X TA; 1 L)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong></p></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td colspan = '2'><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p><em>M. smegmatis </em>mc<sup>2</sup>155 only</p></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td colspan = '2'><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td colspan = '2'><p><strong>7H9 broth base</strong></p><p><strong>Agar</strong></p><p><strong>ddH<sub>2</sub>O</strong></p></td><td colspan = '2'><p>4.7 g</p><p>8 g</p><p>1 L</p></td><td><p>&nbsp;</p><p>0.8 %</p></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '4'><ol><li>Place ddH<sub>2</sub>O and a stir bar, then broth base and agar, into an Erlenmeyer flask; put onto a stir plate.</li><li>Stir until the broth base powder is completely dissolved (the agar will not dissolve until it is heated).</li><li>Autoclave the flask, and cool to 55 °C in a 55 °C water bath.</li><li>Gently swirl the solution to evenly distribute the agar.</li><li>In a hood, using aseptic technique, dispense into sterile bottles using only half the available volume (e.g., for a 100 mL bottle, add 50 mL of 2X TA; this will allow you to add 50 mL of media directly to the molten 2X TA to prepare 1X TA). Keep any lids loose until the agar has solidified.</li><li>Tighten lids for storage.</li><li>To melt stored top agar for use, heat in a microwave using 30 second intervals until agar has completely melted.</li><li>Repeated melting and solidifying of top agar will cause the agar to deteriorate and not solidify properly on plates.</li></ol></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td colspan = '2'><p><strong>To Sterilize</strong></p></td><td colspan = '3' ><p>Autoclave</p></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td colspan = '2'><p><strong>To Store</strong></p></td><td colspan = '3' ><p>As a solid, store at room temperature, indefinitely. Discard if media is discolored or contaminated.</p></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td colspan = '2'><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Recipe:</p><ul><li>1X Middlebrook Top Agar<br>&nbsp;</li></ul><p>In Protocol:</p><ul><li>12.1</li></ul></td></tr><tr><td colspan = '5'><p>&nbsp;</p></td></tr><tr><td colspan = '2'><p><strong>Notes</strong></p></td><td colspan = '3' ><p>2.5 L of 2X top agar is sufficient for per semester for ~ 20 students</p></td></tr></tbody></table>"},"14-30":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '8'><p><strong>Luria Agar (L-Agar) Plates (1 L) &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</strong></p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td colspan = '2'><p><strong>For bacterial host</strong></p></td><td colspan = '6' rowspan = '1'><p><em>M. smegmatis </em>mc<sup>2</sup>155 only</p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td colspan = '4'><p><strong>Amount</strong></p></td><td colspan = '3'><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>Luria broth base</p><p>Agar</p><p>ddH<sub>2</sub>O</p><p>CB stock (if using)</p><p>CHX stock</p></td><td colspan = '4'><p>15.5 g</p><p>15 g</p><p>To 1 L</p><p>1.0 mL</p><p>1.0 mL</p></td><td colspan = '3'><p>&nbsp;</p><p>&nbsp;</p><p>&nbsp;</p><p>50 &micro;g/mL</p><p>10 &micro;g/mL</p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '7'><ol><li>To 1 L of ddH<sub>2</sub>O in a large flask, add 15.5 g of Luria broth base and 15 g agar. Mix well with a magnetic stir bar.</li></ol><ol><li value = '2'>Autoclave the flask.</li></ol><ol><li value = '3'>Cool to 55 °C in a 55 °C water bath.</li></ol><ol><li value = '4'>Once the agar has cooled to 50-55 °C, aseptically add antimicrobials as necessary.</li></ol><ol><li value = '5'>Mix well by swirling, and avoid bubbles.</li></ol><ol><li value = '6'>Using aseptic technique, pour agar into Petri dishes.</li><li value = '7'>Allow the agar to set overnight.*</li></ol></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '7'><p>Autoclave (see above)</p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '7'><p>At 4 °C, for up to 3 months. Plates can still be used beyond 3 months, although the antimicrobials may have been partially degraded. Discard if plate is discolored or contaminated.</p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '7' rowspan = '1'><p>In Protocol:</p><p>5.3, 5.4, 5.6, 6.1, 6.4, 6.5, 7.1, 11.1, 11.2, 11.3, 11.4, 12.2</p></td></tr><tr><td colspan = '8'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '7'><ol><li>Prior to adding antibiotics or antimicrobials, the cooled medium can be aliquoted into sterile bottles, allowed to solidify, and stored at room temperature. The solidified agar can be melted by autoclaving for 5 mins, or in a microwave. Antimicrobials should only be added once the freshly melted agar has cooled to 55 ˚C.</li><li>Prior to use, agar plates stored at 4 ˚C should be allowed to warm to room temperature.</li><li>1 L of agar is sufficient to prepare 50 agar plates.</li><li>* Though the agar will set within 1 – 2 hours, allowing the plates to sit at room temperature overnight helps remove some water content within the agar. Drying plates in this manner helps prevent top agar, when used, from slipping on the surface of the agar.</li><li>Approximately 1,500 agar plates are used per semester per class of 20 students. Consider preparing plates in batches of 500 plates.</li></ol></td></tr></tbody></table>"},"14-31":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>Media Supplement for <em>Streptomyces</em> phage (32.5 mL)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p><em>Streptomyces </em>spp. only</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td><p><strong>Amount</strong></p></td><td colspan = '2'><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>1 M MgCl<sub>2</sub></p><p>0.8 M Ca(NO<sub>3</sub>)<sub>2</sub></p><p>40% Dextrose</p></td><td><p>10 mL</p><p>10 mL</p><p>12.5 mL</p></td><td colspan = '2'><p>10 mM (after addition to culture media)</p><p>&nbsp; 8 mM (after addition to culture media)</p><p>0.5% (after addition to culture media)</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Aseptically add each component to a sterile container to mix or add each directly to cooled, sterile culture media (per 1 L).</li><li>Mix by gently swirling.</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Autoclave each component separately.</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, for up to 1 year. Discard if solution is discolored, contaminated or has precipitation.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3'><p>Nutrient Broth with Media Supplement for <em>Streptomyces</em> phage, Nutrient Agar with Media Supplement for <em>Streptomyces</em> phage</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><p>If using a different species of <em>Streptomyces</em> for phage isolation, a different base media may be used with the same supplement.</p></td></tr></tbody></table>"},"14-32":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '3'><p><strong>Nutrient Broth (1 L)&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </strong></p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '2' rowspan = '1'><p><em>Streptomyces </em>spp. only</p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p><strong>Nutrient broth powder</strong></p><p><strong>ddH<sub>2</sub>O </strong></p></td><td><p>8 g</p><p>&nbsp;</p><p>To 1 L</p><p>&nbsp;</p></td><td><p>1X</p><p>&nbsp;</p><p>&nbsp;</p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '2'><ol><li>Mix 8 g of Nutrient broth powder with 1 L of ddH<sub>2</sub>O in a large flask and mix well with a magnetic stir bar.</li><li>Aliquot as needed.</li><li>Sterilize</li></ol></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '2'><p>Autoclave</p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '2'><p>At room temperature or 4 °C.</p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '2'><p><em>Streptomyces</em> liquid culture growth for starter cultures.</p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '2'><ol><li>Because <em>Streptomyces</em> cultures are prone to clumping when grown in liquid, it is highly recommended that all cultures contain 5% w/v PEG 8000, which serves as a physical dispersant, and that cultures be shaken vigorously in a baffled flask.&nbsp; PEG can be added to mixture prior to autoclaving.</li></ol><ol><li value = '2'>5 L of 7H9 liquid media should be sufficient per semester for a class of 20 students.</li></ol></td></tr></tbody></table>"},"14-33":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '3'><p><strong>Nutrient Broth with Media Supplement for <em>Streptomyces</em> phage (1 L)</strong></p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '2' rowspan = '1'><p><em>Streptomyces </em>spp. only</p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p><strong>Nutrient broth powder</strong></p><p><strong>ddH<sub>2</sub>O </strong></p><p><strong>Media Supplement for <em>Streptomyces</em> phage</strong></p></td><td><p>8 g</p><p>&nbsp;</p><p>To 1 L</p><p>&nbsp;</p><p>&nbsp;</p><p>32.5 mL</p></td><td><p>1X</p><p>&nbsp;</p><p>&nbsp;</p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '2'><ol><li>Mix 8 g of Nutrient broth powder with 1 L of ddH<sub>2</sub>O in a large flask and mix well with a magnetic stir bar.</li><li>Aliquot as needed.</li><li>Autoclave.</li><li>Add Media Supplement for <em>Streptomyces</em> phage (32.5 mL/1L) after media has cooled.</li></ol></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '2'><p>Autoclave</p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '2'><p>At room temperature or 4 °C.</p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '2'><p><em>Streptomyces</em> liquid culture growth in which phage infection will occur (including enrichment cultures)</p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '2'><p>Because <em>Streptomyces</em> cultures are prone to clumping when grown in liquid, it is highly recommended that all cultures contain 5% w/v PEG 8000, which serves as a physical dispersant, and that cultures be shaken vigorously in a baffled flask.&nbsp; PEG can be added to mixture prior to autoclaving.</p></td></tr></tbody></table>"},"14-34":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>Nutrient Agar with <em>Streptomyces </em>phage supplement (NA+) Plates (1 L)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3' rowspan = '1'><p><em>Streptomyces </em>spp. only</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td><p><strong>Amount</strong></p></td><td colspan = '2'><p><strong>Final Concentration</strong></p></td></tr><tr><td><p><strong>Nutrient agar powder</strong></p><p><strong>ddH<sub>2</sub>O</strong></p><p><strong>Media Supplement for <em>Streptomyces</em> phage</strong></p></td><td><p>23 g</p><p>To 967.5 L</p><p>&nbsp;</p><p>&nbsp;</p><p>32.5 mL</p></td><td colspan = '2'><p>1X</p><p>&nbsp;</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Mix 23 g of Nutrient agar powder with ddH<sub>2</sub>O to 967.5 mL total volume in a large flask and mix well with a magnetic stir bar.</li><li>Autoclave the flask.</li><li>Cool to 55 °C in a 55 °C water bath.</li><li>Once the agar has cooled to 50-55 °C, aseptically add 32.5 mL of sterile Media Supplement for <em>Streptomyces</em> phage.</li><li>Mix well by swirling, and avoid bubbles.</li><li>Using aseptic technique, pour agar into Petri dishes.</li><li>Allow the agar to set overnight.*</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Autoclave (see above)</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature or 4 °C, for up to 3 months. Discard if plate is discolored or contaminated.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3'><p>In <em>Streptomyces</em> plate growth protocols.</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>Prior to adding media supplements, the cooled medium can be aliquoted into sterile bottles, allowed to solidify, and stored at room temperature. The solidified agar can be melted by autoclaving for 5 mins, or in a microwave. Supplements should only be added once the freshly melted agar has cooled to 55 ˚C.</li><li>Prior to use, agar plates stored at 4 ˚C should be allowed to warm to room temperature.</li><li>* Though the agar will set within 1 – 2 hours, allowing the plates to sit at room temperature overnight helps remove some water content within the agar. Drying plates in this manner helps prevent top agar, when used, from slipping on the surface of the agar.</li><li>1 L of agar is sufficient to prepare 50 agar plates.</li><li>Approximately 1,500 agar plates are used per semester per class of 20 students. Consider preparing plates in batches of 500 plates.</li></ol><p>&nbsp;</p></td></tr></tbody></table>"},"14-35":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '7'><p>&nbsp;</p><p><strong>PYCa Agar Plates (1 L)</strong></p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '6' rowspan = '1'><p>General (excluding <em>M. smegmatis</em> and <em>Streptomyces </em>spp.)</p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td colspan = '4'><p><strong>Amount</strong></p></td><td colspan = '2'><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>Agar</p><p>Yeast extract</p><p>Peptone</p><p>1 M Cacl<sub>2</sub></p><p>Dextrose (40 %)</p><p>ddH<sub>2</sub>O</p><p>CHX stock</p></td><td colspan = '4'><p>15.0 g</p><p>1 g</p><p>15 g</p><p>4.5 mL</p><p>2.5 mL</p><p>990 mL</p><p>1.0 mL</p></td><td colspan = '2'><p>1.5 %</p><p>1 g/L</p><p>15 g/L</p><p>4.5 mM</p><p>0.1 %</p><p>&nbsp;</p><p>10 &micro;g/mL</p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '6'><p><strong>Note</strong>: A video protocol for this recipe card is available <a href = 'https://seaphages.org/video/55' target = '_blank'><span><strong>here</strong></span></a>.<br>&nbsp;</p><ol><li>Place water and a stir bar, followed agar, peptone and yeast extract, into an Erlenmeyer flask. Put onto a stir plate.</li></ol><ol><li value = '2'>Stir until the yeast extract and peptone are completely dissolved (the agar will not dissolve until it is heated).</li></ol><ol><li value = '3'>Autoclave the flask, and cool to 60 °C in a 60 °C water bath.</li></ol><ol><li value = '4'>In a hood, aseptically add dextrose and CaCl<sub>2</sub>.</li></ol><ol><li value = '5'>Add antibiotics/antimicrobials stock solutions, if using.</li></ol><ol><li value = '6'>Mix well by swirling, and avoid bubbles.</li></ol><ol><li value = '7'>Using aseptic technique, pour agar into Petri dishes.</li><li value = '8'>Allow the agar to set overnight.*</li></ol></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '6'><p>Autoclave</p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '6'><p>At 4 °C, for up to 3 months. Plates can still be used beyond 3 months, although the antimicrobials may have been partially degraded. Discard if plate is discolored or contaminated.</p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '6' rowspan = '1'><p>In Protocol: 5.3, 5.4, 5.6, 6.1, 6.4, 6.5, 7.1, 11.1, 11.2, 11.3, 11.4, 12.2</p></td></tr><tr><td colspan = '7'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '6'><ol><li>Prior to adding antibiotics or antimicrobials, the cooled medium can be aliquoted into sterile bottles, allowed to solidify, and stored at room temperature. To melt many batches of solidified agar,&nbsp; autoclave for ~ 10 -15 mins. Alternatively, melt the agar in small bathes using a microwave. Antimicrobials should only be added once the freshly melted agar has cooled to 55 ˚C.</li><li>Prior to use, agar plates stored at 4 ˚C should be allowed to warm to room temperature.</li><li>* Though the agar will set within 1 – 2 hours, allowing the plates to sit at room temperature overnight helps remove some water content within the agar. Drying plates in this manner helps prevent top agar, when used, from slipping on the surface of the agar.</li><li>1 L of agar is sufficient to prepare 50 agar plates.</li><li>Approximately 1,500 agar plates are used per semester per class of 20 students.<br>- Consider preparing plates in batches of 500 plates.<br>- Often, a limiting factor to preparing plates in large quantities is adequate space at 4 ˚C to store those plates.</li></ol></td></tr></tbody></table>"},"14-36":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>PYCa Liquid Medium (1 L)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3'><p>General (excluding <em>M. smegmatis</em> and <em>Streptomyces </em>spp.)</p></td></tr>   <tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>Yeast extract</p><p>Peptone</p><p>ddH<sub>2</sub>O</p><p>1 M CaCl<sub>2</sub>&nbsp;stock</p><p>Dextrose (40 %)</p><p>CHX stock</p></td><td colspan = '2'><p>1.0 g</p><p>15.0 g</p><p>990 ml</p><p>4.5 ml</p><p>2.5 ml</p><p>1 ml</p></td><td><p>1 g/L</p><p>15 g/L</p><p>&nbsp;</p><p>4.5 mM</p><p>0.1 %</p><p>10 &micro;g/ml</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><ol><li>Add 990 mL of ddH<sub>2</sub>O into a large flask. Then weigh out and add peptone and yeast extract to the flask.</li></ol><ol><li value = '2'>Stir until completely dissolved.</li></ol><ol><li value = '3'>Autoclave.</li></ol><ol><li value = '4'>Once cool, aseptically add dextrose and CaCl<sub>2</sub>.</li></ol><ol><li value = '5'>Aliquot as 100-mL, 500-mL, or 1000-mL portions, in sterile vials, &nbsp;as needed.*</li></ol><ol><li value = '6'>If using CHX, add prior to use.</li></ol><p>* It may be more convenient to aliquot the media (lacking dextrose and calcium chloride) into vials prior to sterilization by autoclaving. In this case, the vials do not need to be sterile, but dextrose and calcium chloride stock solutions should still only be added after sterilization. You will also need to adjust the volume of dextrose and calcium chloride stock solutions added to each vial.</p></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Autoclave.</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><p>At room temperature, indefinitely, or at 4 °C for up to 30 days if antimicrobials have been added.</p><p>Discard if media is discolored or contaminated.</p></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Protocol: 5.2, 5.5, &amp; 12.1</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><ol><li>If antimicrobials have been added, the medium must be stored at 4 °C.</li></ol><ol><li value = '2'>5 L of PYCa liquid media should be sufficient per semester for a class of 20 students.</li></ol></td></tr></tbody></table>"},"14-37":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '4'><p><strong>PYCa Top Agar (PYCa TA at 1X concentration; 1 L)</strong></p></td></tr>   <tr><td><p><strong>For bacterial host</strong></p></td><td colspan = '3'><p>General (excluding <em>M. smegmatis</em> and <em>Streptomyces </em>spp.)</p></td></tr><tr><td><p>&nbsp;</p></td><td colspan = '2'><p>&nbsp;</p></td><td><p>&nbsp;</p></td></tr><tr><td><p><strong>Ingredients</strong></p></td><td colspan = '2'><p><strong>Amount</strong></p></td><td><p><strong>Final Concentration</strong></p></td></tr><tr><td><p>Agar</p><p>Yeast extract</p><p>Peptone</p><p>1 M Cacl<sub>2</sub></p><p>Dextrose (40 %)</p><p>ddH<sub>2</sub>O</p></td><td colspan = '2'><p>4.0 g</p><p>1 g</p><p>15 g</p><p>4.5 mL</p><p>2.5 mL</p><p>990 mL</p></td><td><p>0.4 %</p><p>1 g/L</p><p>15 g/L</p><p>4.5 mM</p><p>0.1 %</p></td></tr>   <tr><td><p><strong>To Prepare</strong></p></td><td colspan = '3'><p><strong>Note</strong>: You may choose to prepare a large volume of media and then transferring aliquots into smaller vials after sterilization (as described below), or preparing the media in smaller vials before sterilization. If choosing the latter, adjust the weight and volumes of each media component accordingly.<br>&nbsp;</p><ol><li>Add water and a stir bar, followed by agar, peptone and yeast extract, into an Erlenmeyer flask; put onto a stir plate.</li><li value = '2'>Stir until the yeast extract and peptone is completely dissolved (the agar will not dissolve until it is heated).</li></ol><ol><li value = '3'>Autoclave the flask.</li></ol><ol><li value = '4'>Once autoclaved and cooled slightly (not below 60 ˚C), add dextrose and CaCl<sub>2</sub>, using aseptic technique. Gently swirl the flask to mix the solution.</li></ol><ol><li value = '5'>In a hood, using aseptic technique, dispense into 50 mL or 100 mL sterile bottles.</li></ol><ol><li value = '6'>If using immediately, maintain top agar in the molten form by placing it at 55 ˚C. If not, keep bottle lids slightly loose until the agar has solidified. Once solidified, tighten lids for storage.</li></ol><ol start = '7'><li>To melt stored top agar for use, make sure the cap is loose, and heat in microwave using 30 second intervals. The top agar should come to a boil but not boil over. As the agar must be completely melted, <span><strong>carefully</strong></span> and gently swirl the bottle with the loose cap facing away from you and others. Check to make sure there are no clumps.</li></ol><ol start = '8'><li>Once the agar is completely melted, place the top agar into a 55 °C water bath to cool before use.</li></ol><ol start = '9'><li>Repeated melting and solidifying of top agar will cause the agar to deteriorate and not solidify properly on plates.</li></ol></td></tr>   <tr><td><p><strong>To Sterilize</strong></p></td><td colspan = '3'><p>Autoclave</p></td></tr>   <tr><td><p><strong>To Store</strong></p></td><td colspan = '3'><ul><li>As a solid, store at room temperature, indefinitely. Discard if media is discolored or contaminated.</li><li>As molten agar, store for up to 10 days. Agar stored in its molten form deteriorates over time and may not solidify properly on plates.</li></ul></td></tr>   <tr><td><p><strong>Usage</strong></p></td><td colspan = '3' rowspan = '1'><p>In Protocol:<br>5.3, 5.6, 6.1, 6.4, 6.5, 7.1, 11.1, 11.2, 11.3, 11.4, 12.1, 12.2</p></td></tr>   <tr><td><p><strong>Notes</strong></p></td><td colspan = '3'><p>5 L of PYCa top agar is sufficient for per semester for ~ 20 students .</p></td></tr></tbody></table>"},"14-38":{content:""},"14-39":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '2'><p><strong>Growing bacteria from a frozen stock (Streak Plate)</strong></p></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td><p><span>Note</span>: A video demonstrating this protocol is available at <a href = 'https://seaphages.org/video/54/' target = '_blank'><strong>here</strong></a>.<br>&nbsp;</p><ol><li>Refer to Host Basics in the Phage Discvery Guide for specific guidelines when working with a particular bacterial host. In particular, take note of the appropriate media (agar) and incubation temperature.</li></ol><ol><li value = '2'>Label an appropriate agar plate.</li></ol><ol><li value = '3'>Once you are ready to streak a plate, obtain your frozen stock from the freezer and complete Step 4 quickly&nbsp; to minimize the amount of thawing of your frozen stock.</li><li>Using an inoculation loop or sterile wooden stick, scrape the surface of a frozen stock, and immediately return the stock to the freezer.</li><li>Aseptically spread the bacteria at the end of the loop or wooden stick onto the agar plate by streaking for single colonies.</li><li>Incubate for 2 – 3 days at the appropriate temperature.</li><li>Monitor growth over time to ensure bacterial growth parameters are as expected (e.g. time for colony formation and color/texture of colonies.</li><li>Once single colonies are visible and large enough for picking (see image of plate in “Host Basics” of the Phage Discovery Guide), plates should be stored either at room temperature or at 4 ˚C. To prevent over drying, the plates should be wrapped in parafilm.</li></ol></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td><p>At room temperature or at 4°C.</p></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td><p>In Protocol: Streak plate</p></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td><ol><li>This is your P0FF (passage 0 from frozen) stock. Depending on the host bacteria, this stock culture can be stored at room temperature or at 4°C for several weeks. We recommend storage for no more than a couple of weeks.</li><li>We suggest that you start P0FF stock cultures every week or every other week.</li></ol></td></tr></tbody></table>"},"14-40":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '3'><p><strong>Growing liquid cultures of bacteria from a single colony</strong></p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td colspan = '2'><p><span>Note</span>: A video demonstrating this protocol is available at <a href = 'https://seaphages.org/video/56/' target = '_blank'><strong>here</strong></a>.</p><p>&nbsp;</p><ol><li>Refer to Host Basics in the Phage Discovery Guide for specific guidelines when working with a particular bacterial host. In particular, take note of the appropriate media (liquid) and incubation temperature.</li></ol><ol><li value = '2'>Add 50 mL of the appropriate liquid medium to a 250-mL baffled flask. If using, add antibiotics or antimicrobials.</li></ol><ol><li value = '3'>Using an inoculation loop or sterile wooden stick, pick a small piece from the middle of a single colony of bacteria on an agar plate. Avoid using colonies from plates older than 2 weeks.</li></ol><ol><li value = '4'>Aseptically transfer the bacteria from the loop or wooden stick into the medium in the flask.</li></ol><ol><li value = '5'><strong>Tween80 should be added to 0.05% when culturing bacteria that tend to clump (e.g. <em>M. smegmatis </em>mc<sup>2</sup>155). </strong>Cultures prepared with Tween80 must be subcultured to remove Tween80. See <a class = 'chap-linking' tabindex = '0'    data-link-to = '14-41' >instructions for subculturing from cultures prepared with Tween80</a>.</li></ol><ol><li value = '6'>Incubate, with shaking (250 rpm), at the recommended temperature.</li></ol><ol><li value = '7'>Monitor growth over time to ensure culture growth parameters are as expected (e.g. duration before culture is expected to become saturated)</li></ol><ol><li value = '8'>Once the culture is saturated, store at room temperature.</li></ol><p><span>Important</span>: Whenever a liquid culture of bacteria is prepared, the culture should be tested as follows:</p><ul><li>Prepare a streak plate from the culture, and monitor for the expected growth rate and colony morphology of the bacterial strain. If the growth rate or colony morphology is not as expected, the culture may be contaminated.</li><li>Prepare a top agar lawn using an aliquot of the bacterial culture, and test the ability of a host-specific phage to infect the top agar lawn using a spot test <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-8' >Protocol 5.6</a>. If the host-specific phage is unable to infect the bacterial lawn, the bacterial culture may be contaminated.</li></ul></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td colspan = '2'><p>At room temperature or at 4°C, for 1 – 2 weeks.</p></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td colspan = '2' rowspan = '1'><p>In Protocol:</p><ul><li>Growing liquid cultures for use in phage experiments</li></ul></td></tr><tr><td colspan = '3'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td colspan = '2'><ol><li>This is your P1FF (passage 1 from frozen) stock. This stock culture can be stored at room temperature, the duration for which will depend on the bacterium. Check Host Basics in this Guide for information about a specific bacterium.</li></ol><ol><li value = '2'>It can be aseptically aliquoted into smaller volumes for storage (this minimizes the potential for contamination).</li></ol><ol><li value = '3'>We recommend that you start P1FF stock cultures weekly, depending on use and the size of a given class. Regardless, because of the possibility of contamination, a backup P1FF culture should be available at all times.</li></ol><ol><li value = '4'>Single colonies from an agar plate can be used to inoculate multiple P1FF cultures for as long as the plate remains uncontaminated.</li></ol><ol><li value = '5'>If your PIFF does<strong> <span>NOT </span></strong>contain Tween80, this culture can be used for experiments with phage.</li></ol><ol><li value = '6'>Tween80 is added to cultures for some bacteria to minimize clumping during bacterial growth. <strong>The presence of Tween80, however, will likely inhibit phage infection.<br>For this reason:</strong><br>To grow liquid cultures for use in phage experiments, sub-inoculate from a P1FF culture into liquid media without Tween80.&nbsp; For instructions, see the recipe card “Growing liquid cultures of bacteria for use in phage experiments.”</li></ol></td></tr></tbody></table>"},"14-41":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '2'><p><strong>Preparing a culture without Tween80 from one with Tween80</strong></p></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td><p>Note: This protocol assumes a saturated liquid culture has been prepared, and contains Tween80.<br>&nbsp;</p><ol start = '1'><li>Add 50 mL of the appropriate liquid media (i.e., without Tween80) to a 250-mL baffled flask. For larger volumes, adjust accordingly (see notes). If using, add antibiotics or antimicrobials.</li><li >Aseptically, add a 1:1000 volume of the saturated culture containing Tween80. For example, for a 50-mL culture, add 50-&micro;L of your culture that contains Tween80.</li><li >Incubate, with shaking (250 rpm), at the recommended temperature.</li></ol></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td><p>At room temperature or at 4 °C.</p></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>Usage</strong></p></td><td><p>In Protocol: 5.2 - 7.2</p></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p><p>&nbsp;</p></td><td><p>Note: A culture prepared from a single colony is known as a P1FF. A new culture prepared from a PIFF is known as a P2FF.<br>&nbsp;</p><ol start = '1'><li>The addition of small (≤1:1000) volumes of the PIFF culture (containing Tween80) when preparing a sub-culture without Tween80 reduces the likelihood of bacterial clumping in subculture. If clumping does occur, continue incubating the subculture up to an additional 48 hours.</li><li>This culture can be stored at room temperature, the duration for which will depend on the bacterium. Check Host Basics in this guide for information about a specific bacterium.</li><li>To prepare additional cultures without Tween80, it is best to go back to the initial P1FF stock culture, so long as the P1FF culture is not contaminated, is not clumpy, or has not been stored for beyond the recommended time (see Host Basics in this guide)</li><li>The volume of a culture should be approximately one-fifth the size of the flask. For example, for 200 mL of bacterial culture, use a 1-L baffled flask. This maximizes aeration when the bacteria are grown on a shaker.</li><li>Tween80 will inhibit phage infection. It is advisable to double-check that any medium that will be used in phage experiments does not include Tween80.</li></ol></td></tr></tbody></table>"},"14-42":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '2'><p><strong>Preparing frozen stocks of bacteria</strong></p></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td><p><span>Note</span>: A video demonstrating this protocol is available at <a href = 'https://seaphages.org/video/57/' target = '_blank'><strong>here</strong></a>.</p><p>&nbsp;</p><ol><li>Label the appropriate number of cryo-tubes with the name of the bacteria, the date of preparation, and initials of the person preparing the stock.</li></ol><ol><li value = '2'>Add 300 &micro;L of 40 % glycerol to each tube.</li></ol><ol><li value = '3'>Add 700 &micro;L of bacterial culture to each tube.</li></ol><ol><li value = '4'>Mix by inversion.</li><li>Store at -80 °C.</li></ol></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td><p>at -80 °C, indefinitely.</p></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td><ol><li>To use frozen bacterial stocks, refer to the recipe card “Growing bacterial cultures from frozen a stock.”</li><li>It is important that only one person access any given&nbsp; glycerol stock. When a glycerol stock is first used, it should be labelled with the name of the person who is using it.</li></ol></td></tr></tbody></table>"},"14-43":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '2'><p><strong>Preparing Additional Phage Lysate &amp; Long-term Storage</strong></p></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Prepare</strong></p></td><td><ol><li>First, determine the titer of your phage lysate by performing a full plate titer (<a class = 'chap-linking' tabindex = '0'    data-link-to = '6-7' >Protocol 6.5</a>).</li><li>Based on the calculated titer, prepare webbed plates (<a class = 'chap-linking' tabindex = '0'    data-link-to = '7-2' >Protocol 7.1</a>).<br><em>Note: You can flood those plates from Step.1 that are webbed or have very high densities of plaques. This lysate should also be titered as part of Step.4.</em></li><li>Prepare additional lysate by flooding the webbed plates (<a class = 'chap-linking' tabindex = '0'    data-link-to = '6-5' >Protocol 6.3</a>). Proceed to Step 4, ideally as soon as Protocol 6.3 is complete.</li></ol><ol start = '4'><li>Determine the titer of the freshly prepared phage lysate by performing a full plate titer (<a class = 'chap-linking' tabindex = '0'    data-link-to = '6-7' >Protocol 6.5</a>). Ideally, the titer of the lysate will be at 10<sup>9 </sup>pfu/ml or greater. If the titer is below 10<sup>9</sup> pfu/ml, return to Step 2 and attempt the following modification: When webbed plates have been identified, incubate those webbed plates for an additional 24 - 48 hrs such that the bacteria remaining within the 'webbing' have also been lysed by phage before proceeding to Step 3.</li><li style  =  'text-align :start;'>Next:</li></ol><ol style  =  'list-style-type : lower-roman '><li><strong>To prepare freezer stocks</strong>, add 1 ml of the high-titer phage lysate to a labeled cryotube, then add 70 &micro;L of DMSO. Mix by inversion and store at -80 °C. Ideally, 3 - 4 freezer stocks will be prepared.</li><li><strong>To maintain a working stock,</strong> store the lysate at 4 °C.<br><span><strong>Important</strong></span>: The stability of phage particles in lysate stored at 4 °C will vary from phage to phage. Generally, phage stability parallels the titer, with phages more stable at higher titers. Regardless, the titer of phage lysates can drop from one day to another, or remain stable for weeks. It is therefore important to recalculate the titer of your lysate periodically.</li><li><strong>To archive phage lysates,</strong> refer to the archiving protocol (<a class = 'chap-linking' tabindex = '0'    data-link-to = '7-4' >Protocol 7.3</a>).<br><strong><span>Note</span>:</strong> You will receive bar-coded cryo-tubes and DMSO from the program in the fall of Year 1.&nbsp; All samples for archiving at the University of Pittsburgh should be prepared as described in the Phage Discovery Guide.</li></ol></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>To Store</strong></p></td><td><p>Freezer stocks -80 °C, indefinitely. Working stocks at 4 °C, with the titer checked periodically (e.g. biweekly)</p></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>Notes</strong></p></td><td><ol><li>To use frozen phage lysates, scrape a few microliters from the surface of the frozen lysate using a sterile micropipette tip, and immediately return the stock to the freezer to prevent thawing of the frozen stock. Repeated freeze-thaw cycles can damage phage particles.</li></ol><ol><li value = '2'>Transfer the scraped lysate to a microcentrifuge tube pre-filled with 100 &micro;L of phage buffer.</li><li value = '3'>The resuspension can then be used to prepare a working stock of phage lysate.</li><li value = '4'>Avoid thawing the frozen stock.</li></ol></td></tr></tbody></table>"},"14-44":{content:"<table border = '1' cellpadding = '0' cellspacing = '0'><tbody><tr><td colspan = '2'><p><strong>Received Biologicals from the Program? Follow these Next Steps.</strong></p></td></tr><tr><td colspan = '2'><p>&nbsp;</p></td></tr><tr><td><p><strong>Bacteria</strong></p></td><td><p>Bacterial strains are typically shipped as glycerol/freezer stocks to program members, and are intended for use in preparing additional freezer stocks for long-term storage.</p><p>Upon receiving these glycerol/freezer stocks, store these stocks at -80 °C.</p><p>&nbsp;</p><p>To prepare additional glycerol/freezer stocks:</p><ol><li>Streak out bacteria to obtain single colonies. Follow the instructions provided in ''<a class = 'chap-linking' tabindex = '0'    data-link-to = '14-39' >Growing bacteria from a frozen stock</a>''.</li><li>Within days of obtaining single colonies, prepare a liquid culture of the bacterial strain. Follow the instructions provided in ''<a class = 'chap-linking' tabindex = '0'    data-link-to = '14-40' >Growling liquid stock of cultures from a single colony</a>''.</li><li>Once the liquid culture is saturated, prepare additional frozen stocks following the instructions provided in ''<a class = 'chap-linking' tabindex = '0'    data-link-to = '14-42' >Preparing frozen stocks of bacteria</a>''.</li></ol><p>&nbsp;</p><p><span>Important</span>: Whenever a liquid culture of bacteria is prepared, the culture should be tested as follows:</p><ul><li>Prepare a streak plate from the culture, and monitor for the expected growth rate and colony morphology of the bacterial strain. If the growth rate or colony morphology is not as expected, the culture may be contaminated.</li><li>Prepare a top agar lawn using an aliquot of the bacterial culture, and test the ability of a host-specific phage to infect the top agar lawn using a spot test <a class = 'chap-linking' tabindex = '0'    data-link-to = '5-8' >Protocol 5.6</a>. If the host-specific phage is unable to infect the bacterial lawn, the bacterial culture may be contaminated.</li></ul></td></tr><tr><td colspan = '2'>&nbsp;</td></tr><tr><td><p><strong>Phage</strong></p></td><td><p>Phage are typically shipped as working stocks (i.e. the stock lacks a cryo-protectant and should not be placed in a freezer). Upon receiving phage lysate from the program, store the lysate at 4 °C. Within weeks of receiving the lysate, prepare additional lysate for long-term storage and for use in research following instruction provided in ''<a class = 'chap-linking' tabindex = '0'    data-link-to = '14-43' >Preparing Additional Phage Lysate &amp; Long-term Storage</a>''.</p></td></tr></tbody></table>"},"14-45":{content:""},"14-46":{content:"<p><span><h2>Notes About Recipes:</h2></span></p><p><h3>About ddH2O </h3></p><p>Water should be double-distilled at a minimum. Filtered water (e.g., Millipore) is ideal. Regardless of water availability, consistency of preparation is essential (i.e., always use the same water).<br>&nbsp;</p><p><h3>About Labeling</h3></p><p>Clearly label all media and solutions, and date the additions of supplements. Sterile solutions should also be clearly labeled as such, and any autoclave tape (indicating sterility) should be left on the bottles.<br>&nbsp;</p><p><h3>About Adjusting the pH of Solutions</h3></p><p>Be sure to check each recipe for the proper acid or base solution that should be used to adjust the pH (if needed) of each solution or reagent.</p>"},"14-47":{content:"<p>&nbsp;</p><p><span>When using an autoclave, selecting the correct settings is crucial.</span></p><p><span><h2>Cycle Type:</h2></span></p><p><span>Most autoclaves are equipped with 3 settings: Liquid, Wrapped (or Vacuum), andUnwrapped (or Gravity). </span></p><ul><li><span><strong>Liquid Cycle</strong>: When sterilizing liquids by autoclave, the LiquidCycle should be selected. The Liquid Cycle is programmed to release the chamber pressure slowly at the endof the cycle, which prevents your super-heated liquids from rapidly bubbling over.</span></li><li><span><strong>Wrapped (or Vacuum) Cycle</strong>: Most consumables and glassware used,as well as biohazard waste generated, should be sterilized in an autoclave using the Wrapped (or Vacuum)Cycle. This cycle mechanically replaces the air in the chamber with steam, which allows the steam to accessareas where air may otherwise be trapped (e.g. pipette tip boxes). Note that this cycle is notpre-programmed to release chamber pressure slowly, and should not be used for liquids.</span></li><li><span><span><strong>Unwrapped (or Gravity) Cycle</strong>:Unwrapped utensils can be sterilized using this cycle. Unlike the Wrapped Cycle, the Unwrapped cycleuses gravity to displace the air in the chamber. As such, air trapped in containers (e.g. pipette tipboxes) may not be replaced with steam, limiting sterilization.</span></span></li></ul><p>&nbsp;</p><p><span><h2>Cycle temperature and duration:</h2></span></p><ul><li><strong>Liquids:</strong> Liquids should be autoclaved at 121 ˚C (250 ˚F). The cycle duration varies dependingon the amount of liquid being autoclaved. As a rule, 500 ml of liquid should be autoclaved for 40 mins. An extra5 minutes should be added to the overall duration for each additional 500 ml of liquid.</li><li><strong>Non-liquids:</strong> Most consumables and glassware used, as well as biohazard waste generated, shouldbe sterilized in an autoclave set at 121 ˚C (250 ˚F) for 20 mins.</li></ul>"}}}};